<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>Business case</title>
    <meta charset="utf-8" />
    <meta name="author" content="Alejandro Cáceres" />
    <script src="BusinessCase_files/header-attrs/header-attrs.js"></script>
    <link href="BusinessCase_files/remark-css/default.css" rel="stylesheet" />
    <link href="BusinessCase_files/remark-css/default-fonts.css" rel="stylesheet" />
    <link rel="stylesheet" href="my.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">
class: center, middle, inverse, title-slide

# Business case
## Almirall
### <a href="#alejandro-caceres">Alejandro Cáceres</a>
### 2021/17/12

---




class: middle, center

## Who am I?

---

class: middle

##Alejandro Cáceres

- &lt;b&gt;Senior statistician&lt;/b&gt; in the Bioinformatics group at Barcelona Institute of Global Health &lt;https://www.isglobal.org/&gt;
  
- Adjunct &lt;b&gt;lecturer&lt;/b&gt; in statistics at the Universitat Politectica de Catalunya &lt;https://eebe.upc.edu/es&gt;
  
- Lead research on methods for &lt;b&gt;personalized medicine&lt;/b&gt;

- High dimensional data including exposures (exposomic), biomarkers (genomic, transcriptomic).

- First author of articles in high impact journals introducing new biomarkers for cancer, Alzheimer's disease, Asthma, Obesity, and more recently &lt;b&gt;psoriasis&lt;/b&gt; and COVID19. 

- Recent work is directly applicable to one of Almirall's interests relating the use of &lt;b&gt;Tildrakizumab&lt;/b&gt;.

---

class: middle, center

##Tildrakizumab

---

class: middle, center

##Tildrakizumab

### How to characterize psoriasis patients that benefit more from a 200mg dose of Tildrakizumab than a 100mg dose?



---

class: middle, center

##Tildrakizumab

- Humanized monoclonal antibody of IL-23 that inhibits proinflamatory processes of psoriasis. 

- Almirall obtained a license from Sun Pharma for its development and commercialization in Europe.


---

class: middle, center

##Biologics of psoriasis

&lt;img src="img/treatments.JPG" style="width:100%"  align="center"&gt;



---

class: middle

##Tildrakizumab PASI75: 

-  week 16: 100mg `\(66.3\% (n = 59)\)`, 200mg `\(74.4\% (n = 64)\)` (Papp et al. 2015)

- week 12: 100mg `\(64\% (n=197)\)`, 200mg `\(62\% (n=192)\)`  (reSURFACE1, Reich et al. 2017)  

- week 12: 100mg `\(61\% (n=188)\)`, 200mg `\(66\% (n=206)\)`  (reSURFACE2, Reich et al. 2017)  



---

class: middle, center


&lt;img src="img/study1.JPG" style="width:75%"  align="center"&gt;


---

class: middle, center


&lt;img src="img/study2.JPG" style="width:60%"  align="center"&gt;


---

class: middle

## Facts

- Tildrakizumab is an &lt;b&gt;efficacious&lt;/b&gt; biologic aiming at IL-23 p19

- IL-23 p19 treatments may be &lt;b&gt;more&lt;/b&gt; efficacious than TNF and IL-12/IL-23 p40 treatments but &lt;b&gt;less&lt;/b&gt; than IL-17 treatments.

- There is &lt;b&gt;substantial room&lt;/b&gt; for increasing efficacy in all treatments, efficacy substantially drops for PASI90 and PASI100 (although they remain stable over long periods of treatment).



---

class: middle, center

##Challenges: 

- Can we increase Tildrakizumab efficacy by detecting a group of subjects that respond better to a given dose?

- Can we increase Tildrakizumab efficacy by detecting a group of subjects that respond better to Tildrakizumab than other biologics?


---

class: middle, center

## Can we increase Tildrakizumab efficacy by detecting a group of subjects that respond better to a given dose?

I **propose** to answer this question from the most simple to more complex scenarios, which depend on the type of available data.  


---

class: middle

## Main effects


- Is there a consistent and significant increase in PASI improvement for 200mg in relation to 100mg?

- Is it on average better to treat with 200mg than 100mg?

---

class: middle

##Main effects, logistic regression

Are the PASI75 response trajectories at 200mg consistently higher than those at 100mg?

- Visual inspection of reSURFACE1 and reSURFACE2 suggest that this is the case (short and long term).

- Statistical analysis (logistic regressions) can be used to determine the differences in response between doses at each treatment week (i.e. `\(w12\)`, `\(w148\)`). 

`$$z_i = \beta_0 + \beta_1 t_i + \epsilon_i$$`
$$   
r_i= 
`\begin{cases}
    1,&amp; \text{if } z_i&gt;0 \\
    0,&amp; \text{if } z_i&lt;0 \\
\end{cases}`
$$
where `\(r_i\)` and `\(t_i\)` are response and treatment of `\(i\)` at a given week. 

inR: &lt;code&gt;glm(r ~t, family="binomial")&lt;/code&gt;   



---

class: middle, center

## Main effects, long term

&lt;img src="img/study2.JPG" style="width:60%"  align="center"&gt;


---

class: middle, center

## Main effects, long term: individual data?

&lt;img src="img/random.png" style="width:60%"  align="center"&gt;


---

class: middle

## Main effects, long term mixed effects 


Are the PASI75 response trajectories at 200mg consistently higher than those at 100mg?

Taking into account the longitudinal and nested structure of the data: 

- We can model the &lt;b&gt;long term efficacy&lt;/b&gt; by a linear model for
a fixed effect on dose and a random effect on patients. 

`$$R_{i}= \beta_0 + \beta_1 t_i + \alpha _{i} + \epsilon _i$$`
`\(R_i=(r_{w28}, r_{w29}, ..., r_{w148})^t_i\)`

`\(\alpha _{i} \sim N(0, d^2)\)`

`\(\epsilon _{i} \sim N(0, \sigma^2)\)`

In R: &lt;code&gt; lme(R ∼ t, random = ∼1 | patient)&lt;/code&gt;   




---


class: middle, center

## Main effects, short term

&lt;img src="img/study1.JPG" style="width:60%"  align="center"&gt;


---

class: middle

## Main effects, short term: individual data? 


The objective is to achieve a model that allows comparisons of the maximum PASI achieved between doses. 

&lt;img src="img/mx.png" style="width:70%"  align="center"&gt;



---

class: middle

## Main effects, short term mixed effect modelling 


- We can model the &lt;b&gt;maximum short term efficacy&lt;/b&gt; using a logistic model of three parameters, with a
a fixed effect on the dose and a random effect on patients. We can model the improvement of each patient by 

$$
PASI(w_i) = \frac{d_i e^{b_i(\log(w_i)+w_i)}}{1+e^{b_i(\log(w_i)+e_i)}}
$$

where `\(w\)` is the week, `\(d_i\)` is the effect for the maximum PASI improvement obtained for subject `\(i\)`, and `\(e_i\)` the median of `\(w\)` and `\(b_i\)` the rate of the effect. The parameters are taken as random on the patient and the treatment as fixed.

In R: &lt;code&gt;medrm(r ~ t, random=b + d + e ~ 1|patient, fct=LL.3())&lt;/code&gt;




---

class: middle

## Expectation 


- If a consistent and significant increase in PASI is found for 200mg &gt; 100mg then the recommendation, depending on size effect and other factors, would be to prescribe 200mg instead of 100mg for **all patients**. 

- We can already see that the size of the effect is **small**

- However, it is important to accurately model the main effects to gain a proper **understanding** of data.

- Higher gains in efficacy may be obtained from finding a **group** of patients that responds better to 200mg treatment. 



---

class: middle

## Modulators of main effects 

Known risk factors for psoriasis include:

- age
- sex
- BMI, diabetes, metabolic syndrome
- mental stress
- genetic
- smoking, alcohol, air pollution

Do any of these factors **modulate** the association between response and Tildrakizumab dose?


---

class: middle

## Interaction analysis

&lt;img src="img/age.JPG" style="width:80%"  align="center"&gt;

For instance: age of onset 

---

class: middle

## Interaction analysis


At a given week ( `\(w28\)`, `\(w148\)` ) one can test the interaction in the logistic model: 

`\(z_i = \beta_0 + \beta_1 t_i +  \beta_2 m_i + \beta_3 t_im_i + \epsilon_i\)`, { `\(r_i= 1\)` if `\(z_i&gt;0\)`, `\(r_i=0\)` if `\(z_i&gt;0\)`}

where `\(m_i\)` is the modulator (age) and we are interested in testing `\(H_0:\beta_3= 0\)`

R: &lt;code&gt;glm(r ~t*m+t+m, family="binomial")&lt;/code&gt;   


---

class: middle

## Interaction analysis: all modulators 

- we can run **massive univariate** interaction analyses across all available modulator data and select those whose interaction is significant (adjusted by multiple testing). 

- for long term longitudinal data we can test the mixed model with interaction

`$$R_{i}= \beta_0 + \beta_1 t_i + \beta_2 m_i + \beta_3 t_im_i+ \alpha _{i} + \epsilon _i$$`

- Those with higher interactions, depending of the effect, can be used to stratify the population into higher responders to 200mg (or to 100mg).

- reSURFACE1 can be used as discovery study and reSURFACE2 as validation study. 


---

class: middle

## Modulators of heterogeneity 


&lt;img src="img/sex.JPG" style="width:100%"  align="center"&gt;

---

class: middle

## Intraclass correlation differences

The long term longitudinal data suggest the PASI assessments across weeks can be treated as repeated measures on patients. 

Higher ICC= `\(d^2/(d^2+\sigma^2)\)` (mixed model) means higher heterogeneity in response across individuals. 

(U Ettinger, SCR Williams, D Patel, TM Michel, A Nwaigwe, &lt;b&gt;A Caceres&lt;/b&gt;, et al. Neuroimage 45 (2), 549-561)



---

class: middle

## Intraclass correlation differences: all modulators

- we can run **massive univariate** associations for differences in ICC analyses across all modulators available and select those whose difference between groups is significant (adjusted by multiple testing). 

- Those with higher differences, depending of the effect, can be used to stratify the population into consisten less variable responders to 200mg (or to 100mg).

- reSURFACE1 can be used as discovery study and reSURFACE2 as validation study. 


---

class: middle

## Biomarkers of reponse derived from high dimensional data

The acquisition (availability) of high dimensional data that fully specifies each subject can be used to determine &lt;b&gt;composite biomarkers&lt;/b&gt; that modulate response to tildrakizumab dose. 

- Molecular data (genomics, transriptomics, proteomics)
- Epidemiological data (demographics, clinical histories)



---

class: middle

## Genomics and transcriptomics

Composite biological markers:

- Genome-wide association studies have been identified a set of genetic variants that is highly associated with age of onset. The variant are involved in Th17 pathways (Nat Tsoi, et al. Nat Commun. 2017; 8: 15382).

  - Is the genetic risk score associated with Tildrakizumab dose respose? (interaction analysis:  &lt;code&gt;glm(r ~t*risk_score, family="binomial")&lt;/code&gt;)


- Psoriasis response to different treatments can be assessed with the activity of few genes (transcriptomic profiling: Wang,G. et al. JAMA Dermatology. 2020; 156, 1057–1065.):
  
    - Is the transcription profile associated with Tildrakizumab dose respose? (interaction analysis:  &lt;code&gt;glm(r ~t*trascription_profile, family="binomial")&lt;/code&gt;)



---

class: middle

## Genomics and transcriptomics

Other composite biological biomarkers:

- Defective immune response is associated with genetic mosaicism and cancer (Zekavat et al. Nature Medicine. 2021 27, 1012–1024, Perez-Jurado, &lt;b&gt;A Caceres&lt;/b&gt; et al. https://doi.org/10.1101/2020.04.19.20071357)

 - Is genetic mosiacism in skin a modulator of  response to Tildrakizumab dose (intereaction analysis).
 
- A transcription profile associated with differences on T cell abundance in blood between sexes (&lt;b&gt;A Caceres&lt;/b&gt;, et al. JNCI. 2020; 112, 913)   

 - Is mosiacism in skin a modulator of  response to Tildrakizumab dose (intereaction analysis).


---

class: middle

## Biomarkers of reponse derived from machine learning on high dimensional data

- Supervised &lt;b&gt;machine learning classifiers&lt;/b&gt; (linear discriminants, neural networks, support vector machines) can be used to predict responders from no responders at 200mg.

 - Is the classification a modulator of response to Tildrakizumab dose (interaction analysis)?

Further interpretation and validation of the classifier may be required for clinical use.  


---

class: middle

## Modulators that define groups of high benefit from 200mg with high dimensional data

- Individuals fully characterized by their values in &lt;b&gt;big number&lt;/b&gt; of features.

- Can we identify the features and their combinations of those individuals that benefit most from a treatment rather than another? (&lt;b&gt;A Caceres&lt;/b&gt; et al. 2021. Bioinformatics, under submission)


---

class: middle

## Which individuals benefit more from 200mg than 100mg?

Proposal: 

- &lt;b&gt;Causal inference&lt;/b&gt; in high dimensional (transcriptomic) data. 
- Causal random forest.
- Derive a profile of high responders (reSURFACE1). 
- Target individuals in new studies (reSURFACE2)
- Predict modulation of treatment benefit by the profile. 
- Application to epidemiological data.



---

class: middle
   
## Profiling a group of high benefit to 200mg 
      
  
 1) Select informative features `\(m_i\)` from interaction analyses ( `\(i=1...n\)`, `\(n\)`: number of features) .

&lt;code&gt;glm(r ~ t*m_i, family="binomial")&lt;/code&gt; in reSURFACE1.



&lt;img src="img/vol.JPG" style="width:60%"  align="center"&gt;

  
---

class: middle

## Profiling a group of high benefit to 200mg 

 2) Apply causal random forest  on a set of fetures `\(p\)` at *baseline* (Athey et al. 2019 Ann Stat; 47, 1179, 1203).
    
      
&lt;img src="img/rcf.JPG" style="width:75%"  align="center"&gt;
    
  
---

class: middle

## Profiling a group of high benefit to 200mg 

 3) Create profile of high responders.
    
      
&lt;img src="img/target.JPG" style="width:75%"  align="center"&gt;
 
 
---

class: middle


## Profiling a group of high benefit to 200mg 

 4) Target high responders in reSURFACE2.
    
      
&lt;img src="img/mat.JPG" style="width:75%"  align="center"&gt;
 


---

class: middle


## Profiling a group of high benefit to 200mg 

5) Confirm that the profiles modulate the response for different doses 

&lt;img src="img/resp.JPG" style="width:75%"  align="center"&gt;

Conclusion: Individuals who belong to the profile should be targeted with 200mg. 

---

class: middle
   
## Advantages of the method

- A clear characterization of the high responder profile is obtained.

- I developed and implemented  a software [package](https://github.com/teff-package/teff)

- It has been proven in the context of psoriasis for [brodalumab treatment](https://alejandro-isglobal.github.io/teff/teff.html)

- A transcriptomic profile was identified of individuals who would benefit most from brodalumab rather than [placebo](https://github.com/alejandro-isglobal/alejandro-isglobal.github.io/blob/master/BIOINF-2021-2268_Proof_hi.pdf).

- The profile involves 12 genes, involves in immune cell response (top gene NR4A2 has been shown to stop full Th17 differentiation in vitro).

- Individuals that did not respond to brodalumab and had low T cell counts in blood are the highest potential responders to [brodalumab](https://github.com/alejandro-isglobal/alejandro-isglobal.github.io/blob/master/supplementary.pdf).


---

class: middle
   
## Advantages of the method: Brodalumab Vs Etarnecept


&lt;img src="img/tcellad.jpg" style="width:70%"  align="center"&gt;

---

class: middle


## Advantages of the method

- I applied the method in a different context for high dimensional epidemiological data (high differences in obesity between sexes). I won the exposome data challenge 2021 with this application.

&lt;img src="img/datch.JPG" style="width:75%"  align="center"&gt;


---

class: middle
   
## Other machine learning methods

Methods to find subgroups with positive treatment effects. 

R-packages:

- [personalized](https://github.com/cran/personalized) (LASSO)
- [FindIt](https://imai.fas.harvard.edu/software/FindIt.html) (Uplift trees)
- [upLift](https://cran.r-project.org/web/packages/uplift/index.html) (k-nearest neighboors)
- [bart](https://cran.r-project.org/web/packages/BART/index.html) (Bayesian additive regression trees)


---

class: middle
   
## Data

- What type of data is already available?

- What type of data could be obtained?

- Prove of concept analysis can be done with available data at public repositories (dbGAP, GEO, 23andMe, UK biobank).

## Support

- In-house statisticians for modelling data.

- Keep collaborations with UPC and ISGlobal for access to data and expertise on machine learnig methods.

- In-house clinical researchers for the interpretation and assessment of the value of results and for developing analysis strategies. 

---

class: middle


## Conclusion

- Important gains in the efficacy of tildrakizumab may be obtained by characterizing patients that would benefit the most for a given dose.

- Other important gains may also be obtained by finding those who will benefit most in relation to other biologics. 

- Relevant data are needed. 

- Time-frame depends on available data. 



---

class: middle

##You can find me at 
  
- [linkedin](https://es.linkedin.com/in/alejandro-caceres-dominguez-7449aa176)

- [google scholar](https://scholar.google.es/citations?user=s1D-6WAAAAAJ&amp;hl=es)

- [gitHub](https://github.com/alejandro-isglobal)

- [my blog](https://alejandro-isglobal.github.io/)



---
class: middle
name: Alejandro-Caceres

.left-col-50[
&lt;img src="img/caceres.jfif" width = "200px"/&gt;
### Alejandro Cáceres
&lt;i class="fas fa-envelope"&gt;&lt;/i&gt;&amp;nbsp;alejandrocaceresdominguez@gmail.com&lt;br /&gt;
]


## Thank you!


    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
