% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
%
\documentclass[
]{book}
\usepackage{amsmath,amssymb}
\usepackage{iftex}
\ifPDFTeX
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math} % this also loads fontspec
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
\usepackage{lmodern}
\ifPDFTeX\else
  % xetex/luatex font selection
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\usepackage{longtable,booktabs,array}
\usepackage{calc} % for calculating minipage widths
% Correct order of tables after \paragraph or \subparagraph
\usepackage{etoolbox}
\makeatletter
\patchcmd\longtable{\par}{\if@noskipsec\mbox{}\fi\par}{}{}
\makeatother
% Allow footnotes in longtable head/foot
\IfFileExists{footnotehyper.sty}{\usepackage{footnotehyper}}{\usepackage{footnote}}
\makesavenoteenv{longtable}
\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
% Set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{5}
\usepackage{booktabs}
\ifLuaTeX
  \usepackage{selnolig}  % disable illegal ligatures
\fi
\usepackage[]{natbib}
\bibliographystyle{plainnat}
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\urlstyle{same}
\hypersetup{
  pdftitle={Statistical Data Analysis for Experimental Sciences},
  pdfauthor={Alejandro Cáceres},
  hidelinks,
  pdfcreator={LaTeX via pandoc}}

\title{Statistical Data Analysis
for
Experimental Sciences}
\author{Alejandro Cáceres}
\date{Barcelona, 2025-09-25}

\begin{document}
\maketitle

{
\setcounter{tocdepth}{1}
\tableofcontents
}
\hypertarget{introduction}{%
\chapter{Introduction}\label{introduction}}

This book originated from course materials (2019/2024) for graduate students, enrolled in the Master of Multidisciplinary Research in Experimental Sciences program, at the Barcelona Institute of Science and Technology (BIST), and undergraduate engineering students at the Universitat Politècnica de Catalunya (UPC).

The primary objective of this book is to provide readers from experimental sciences with a comprehensive and unified understanding of statistical thinking for scientific inquiry. Emphasis is placed on grasping both the concepts and practical know-how for asking meaningful questions with relevant data.

Experimental scientists face several obstacles when trying to understand statistics, which must be addressed early to set their learning process on the right track. One often-overlooked challenge is the need to clearly distinguish between \emph{observation} and \emph{abstraction}---for example, between data and outcome, or between the average and the mean. A useful simile is to consider \emph{observation} as the territory to be explored and \emph{abstraction} as the map representing that territory. Mistaking the map for the territory not only confuses students but also limits the early understanding of scientific practice, as it is easily forgotten that abstractions ultimately rest on refined human experience \citep{Frank2024}.

In addition, concepts as deduction and inference often remain at an intuitive level rather than being made explicit. For example, why do I know that the probability of rolling a 1 on a die is \emph{exactly} 1/6, but the probability of experiencing a hurricane in the Gulf of Mexico in a given day can only be \emph{approximately} 0.05?.

One of the greatest challenges teaching statistics arises when framing it as a method for understanding a population. Experimental scientists are primarily focused on gaining knowledge through measurements in experiments that, when repeated, yield varying results. The concept of a `population of experiments' may seem unnecessary or even awkward, as this population often must be considered infinite in order to account for the likelihood of future outcomes from unperformed experiments, uncreated specimens, or yet-to-be-born individuals.

A key insight is that the probabilities used in statistics are statements about future experiments, informed by past experiences. In this sense, statistics serves as a method to systematize our past experiences, allowing us to infer underlying properties of experiments and make informed predictions about future outcomes.

By centering statistics as a method to analyze the outcomes of random experiments---characterized by knowable parameters and modifiable conditions---a unified perspective emerges, one that is more easily relatable to the empiricist. The parameters, here, appear as abstract properties of the random experiment in question that can be either known by design or inferred by the actual realization of the experiment a number of times. Think of the elasticity constant of a spring or the reproduction rate of a bacteria. When the experimenter willingly changes a condition of the experiment (e.g.~temperature), she is expecting to influence the property described by the parameter and, therefore, the value of its inference. Since the outcomes of the experiment vary due to error, she needs arguments to demonstrate that increasing the temperature produces oscillation amplitudes, or higher bacterial yield, that cannot be achieved by the natural variation of the experiment but rather to her intervention. This is the explanatory approach that it is consistently taken throughout the book.

Emphasis is placed on the analysis of real data. Concepts are developed from the need to explore and understand insightful experiments. Sometimes a formal definition precedes the example; other times, the example illustrates the need to develop a concept. Plots and formulas are used. This is statistical thinking as it is applied in experimental science research. From this perspective, the book focuses on the use of analytic tools to develop a rigorous thinking about the experiment, rather than the rigorous development of the tools. Great mathematical statistics books cover that field \citep{davison2003statistical, MillerFreund2014, casella2002statistical}.

Historical data and classical experiments are widely used to explain statistical concepts and to highlight their critical role in shaping scientific conclusions. Analyzing data from iconic experiments not only demonstrates the central importance of statistics in scientific discovery but also allows students to learn through meaningful, relevant examples from the greatest scientists. Statistics textbooks are often filled with unmemorable or artificial cases, whereas Mendel's genetic data vividly illustrate the necessity of probability laws; Rutherford and Geiger elegantly show that alpha particles must have independent sources of radioactive emission (atoms) using a Poisson distribution; and the debate over the extinction of the dinosaurs by a meteorite can be explored through a simple boxplot of the ages of meteorite debris. Paradigm-shifting examples are discussed throughout the book with the original published data. We discuss scientific discoveries from diverse areas such as physics (e.g., the electron e/m ratio, Maxwell's gas viscosity, Hubble's law), genetics (Mendel's laws, lactase gene selection, knockout experiments), epidemiology (John Snow's cholera study, the Framingham cohort), and ecology and earth sciences (dinosaur extinction, Fisher's diversity index, Keeling's CO2 emissions).

Applying modern analytical methods to hystorical datasets serves to demonstrate what would now be considered preliminary analysis. Our goal is not to reinterpret the original experiments---their conclusions stand as part of the historical record---but rather to use them as instructive tools. Any scientific inaccuracies should be understood in the context of pedagogical intent.

Publicly accessible data are also downloaded to illustrate various hypothesis tests within a research inquiry, such as data on lithium batteries, electrocardiograms, and earthquake frequency. Data from a misophonia study, kindly lent by Dr.~Antonia Ferrer, are used throughout the book for hands on exercises of basic concepts to encourage the reader to learn how to gain scientific insight form a real empirical study. When data for some examples are created, emphasis is placed on the plausibility of similar experiments with real-world parameters, such as in the designs of a conductivity resistor, a copper wire of high purity and a pacemaker.

Readers are expected to have prior exposure to basic mathematical concepts at the undergraduate level. Statistics is expressed in mathematical language and implemented using programming languages. Put simply, we need a map and a compass. While both will be utilized, this is not a book about mathematics or computer programming. The primary goal is to understand the application of statistical thinking and its solutions in scientific inquiry.

With that clarified, the book aims to help readers develop a proficient use of both mathematical concepts and programming functions, without focusing on theorem proofs or code compilation. We will primarily use the R and Python programming languages. Although prior knowledge of R and Python is not a prerequisite, readers are encouraged to consult an introductory reference \citep{matloff2011art, mckinney2022python}.

Our focus lies in using statistics to guide the exploration of natural phenomena through experimental data. As an analytical tool, statistics not only sharpens scientific understanding but also strengthens the integrity of our conclusions by helping us ask the right questions, strive for precise thinking, and use accurate language. In an age of data-driven science, statistical literacy is a cornerstone of credible and collaborative discovery.

\hypertarget{how-to-read-the-book}{%
\section{How to read the book}\label{how-to-read-the-book}}

The book is meant to be read linearly, one chapter after another, as concepts build upon each other. Nonetheless, each chapter tackles a particular subject and is intended to be as self-contained as possible for future reference or study.

The first half of the book (up to Chapter 14 on hypothesis testing) serves as one semester of course material for second-year engineering students. The course material for master's students comprises all the chapters, with more emphasis on the second half. In this course, more technical chapters, such as those on discrete and continuous random variables (Chapters 4 and 5) and maximum likelihood (Chapter 12), can be quickly revised.

\hypertarget{acknowledgements}{%
\section{Acknowledgements}\label{acknowledgements}}

This book is the result of years of work in many different scientific research fields (physics, neuroimaging, genomics and epidemiology). My journey has taken me from abstract experiments (electrons orbiting black holes) to concrete ones (development of health biomarkers). Statistics and probability underpin them all. Along the way, I have had the privilege of meeting and discussing ideas with many great scientists---thank you all. I have also had the pleasure of learning from many great teachers from diverse disciplines. To my wife and sons, parents and sisters, I owe you all.

Special thanks to my colleagues at ISGlobal, UPC and BIST.

\hypertarget{data-description}{%
\chapter{Data description}\label{data-description}}

Data refers to the yield of observation through measurement. The experimental scientist's interaction with nature, achieved through the meticulous collection of data, forms the foundation of scientific knowledge. However, data is often collected with a specific purpose in mind. We aim to understand the natural processes that generate the data and the extent to which these processes depend on the particular experimental setup.

Observation and abstraction are thus intertwined in the scientific method, and statistics provides a systematic approach to addressing both. To explain statistical methods, we adopt the practical perspective that data are objects of observation, while models are objects of abstraction.

In this chapter, we will introduce tools for describing data. These tools include tables, figures, and descriptive statistics that summarize central tendency and dispersion. Before any explanation can be drawn from data, the data itself must first be observed. The goal is to develop an initial intuition by displaying the data in various forms, which will help us determine whether the questions we intend to ask are appropriate or whether we should explore new ones.

By exploring meaningful real data on Mendelian inheritance, misophonia and dinosaur extinction, we will also introduce key statistical concepts such as random experiments, observations, outcomes, and absolute and relative frequencies.

\hypertarget{scientific-method}{%
\section{Scientific method}\label{scientific-method}}

One of the goals of the scientific method is to provide a framework for solving problems that arise in the study of natural phenomena or in the design of new technologies.

Natural philosophers, and latter scientist, have developed a \textbf{method} over thousands of years to study natural phenomena. The method, which is continuously under development, can be associated with three different human activities:

\begin{itemize}
\tightlist
\item
  \emph{Observation} characterized by the acquisition of \textbf{data}
\item
  \emph{Reasoning} characterized by the development of mathematical \textbf{models}
\item
  \emph{Action} characterized by design of new \textbf{experiments} or technologies
\end{itemize}

Their complex interaction and results are the basis of the \emph{scientific activity}.

\includegraphics{./figures/stats.JPG}

To our understanding, \textbf{statistics} deals with the interaction between \emph{models} and \emph{data} (the bottom part of the figure).

Typical statistical questions are:

\begin{itemize}
\tightlist
\item
  What is the best model for my data (inference)?
\item
  What are the data that a certain model would produce (prediction)?
\end{itemize}

The substantive knowledge of scientific research---whether physical, chemical, biological, or otherwise---is derived from specific experiments conducted on phenomena of interest, most likely under a controlled lab environment. This represents the upper part of the triangle, where the research methods and technologies of various scientific disciplines can be situated.

\hypertarget{data}{%
\section{Data}\label{data}}

The data is presented in the form of observations.

An \textbf{observation} or \textbf{realization} is the acquisition of a number or characteristic from an experimental run. We may also think of the result of a measuring process.

For example, let us take the series of numbers produced by repeating an experiment, such as trying to start a newly assembled engine (1: success of starting an engine, 0: failure of starting it).

\[1, 0, 0, 1, 0, \mathbf{1}, 0, 1, 1, ...\]

The number one in bold is \textbf{an observation} from the sixth repeat of the experiment. That is, in that particular trial, we were able to start the engine.Remember that the observation is \textbf{concrete}, it is the number you get one day in the laboratory. An observation is the recording of a particular and concrete event, located in space and time. Observations can only be obtained by experiments.

By contrast, an \textbf{outcome} is a \textbf{possible} observation that could result from the experiment. No need to run the experiment to know the possibilities. Before we even run the first trial, we know that either the engine starts or not. In the example, the number \textbf{1} marks one of the possible observations (success), \textbf{0} is the other one (failure). The outcome of an experiment is \textbf{abstract}. We can obtain them by reasoning, no need to go to the lab. As such, the outcomes are general characteristics of the experiments we run. An outcome is a universal abstract entity, with no location in space and time.

\hypertarget{types-of-outcomes}{%
\section{Types of outcomes}\label{types-of-outcomes}}

In statistics we are mainly interested in two types of outcomes.

\begin{itemize}
\item
  \textbf{Categorical}: If the outcome of an experiment is a quality. They can be nominal (binary: yes, no; multiple: colors) or ordinal, when the qualities can be ranked (severity of a disease, emergency grades).
\item
  \textbf{Numeric}: If the outcome of an experiment is a number. The number can be discrete (number of messages received in an hour, number of leukocytes in the blood) or continuous (battery charge status, engine temperature).
\end{itemize}

\hypertarget{random-experiments}{%
\section{Random experiments}\label{random-experiments}}

We may boldly say that the subject matter of statistics is the gaining of knowledge from random experiments, the means by which we produce data.

\textbf{Definition:}

A \textbf{random experiment} is an experiment that gives a different result or outcome when repeated in the same way.

Random experiments are of different types, depending on how they are conducted:

\begin{itemize}
\tightlist
\item
  on the same object (person): temperature, sugar levels.
\item
  on different objects (animals): the weight of an animal.
\item
  on events (climate phenomena): the number of hurricanes per year.
\end{itemize}

\hypertarget{absolute-frequencies}{%
\section{Absolute frequencies}\label{absolute-frequencies}}

When we repeat a random experiment with \textbf{categorical} outcomes, we make a list of the results. We summarize the observations by counting how many times we saw a particular outcome.

For instance we compute the \textbf{absolute frequency}:

\[n_i\]

which is simply the number of times we observe the result \(i\).

\textbf{Example (mouse coating)}

Consider one version of the timeless random experiment of obtaining varying offspring from the same reproducing couple. Suppose we breed one male and one female mouse, both with grey coats, and observe the resulting coat color of their offspring.

A fine controlled experiment of a suitable couple was run by Crampe, which was latter reported by Allen in 1904 \citep{allen1904heredity}. Crampe repeated the experiment \(156\) times, that is, he produced \(156\) mouse off-spring from the same pair and recorded their color. The results, however, did not appear to strictly follow Mendel's rules of recessive inheritance for albinism, and at the time no detailed explanation of the full series was provided. Similar patterns of non-Mendelian inheritance were first explained in 1909 through Bateson's concept of epistasis (i.e., allele interaction) and, in the specific case of mouse coat colors, by Little in 1914 \citep{little1914possible}.

When we look at Crampe's mice, while most of the offspring have grey coats, several siblings are also black and albino. These are the results of the repetition of the random experiment

black, grey, black, albino, grey, grey, grey, grey, grey, black, grey, grey, grey, grey, grey, albino, grey, grey, grey, grey, grey, grey, black, grey, albino, grey, grey, albino, grey, black, black, grey, albino, albino, grey, albino, black, grey, grey, albino, grey, black, black, albino, albino, grey, albino, black, grey, black, black, black, grey, grey, grey, grey, black, black, albino, black, albino, black, black, albino, grey, grey, grey, grey, grey, albino, black, albino, grey, grey, grey, grey, grey, albino, albino, grey, grey, albino, albino, grey, grey, albino, black, black, grey, grey, black, albino, grey, albino, black, grey, grey, grey, grey, black, albino, grey, grey, grey, grey, black, grey, grey, grey, albino, black, albino, black, albino, grey, grey, grey, grey, grey, grey, grey, grey, albino, albino, grey, grey, grey, grey, grey, grey, black, grey, grey, grey, grey, albino, albino, black, albino, grey, albino, grey, albino, black, albino, grey, grey, grey, grey, albino, grey, grey, grey, grey, albino, black

Each experiment was the production of a specimen from the same grey couple and its color being recorded. The first \textbf{black} in the list is the observation of the first run of the random experiment (the first listed offspring of the couple). The last \textbf{black} is the observation number \(156\).

We can list the observations using a \textbf{frequency table} of the outcomes (categories), observing high variability in the color of the offspring

\[
\begin{array}{cc}
\mathbf{outcome} & \mathbf{n_i}  \\
\text{grey}  & 88     \\
\text{black}   & 31    \\ 
\text{albino}  & 37   \\ 
\end{array}
\]

The table summarizes the outcomes (first column) and the corresponding observations (second column). From the table, we can see that, for example, \(n_1=88\) represents the total number of grey mice observed. Additionally, we note that the total number of offspring is given by

\[\sum_{i=1}^m n_i= n = 156,\]
where \(m=3\) is the number of outcomes (the number of rows in the table), and \(n=156\) is the total number of observations (repetitions of the random experiment).

The frequency table in the example illustrates the well-known inheritance pattern of hybrid grey parents, which carry two versions of the C gene---grey (dominant) and black (recessive)---and two versions of another gene, A, associated with normal pigmentation (dominant) and albinism (recessive). This phenomenon, known as recessive epistasis, occurs when the offspring inherits the albinism mutation in gene A from both parents, thereby interfering with---or masking---the expression of grey or black coloration determined by gene C \citep{little1914possible}. Mechanistically, the protein encoded by the albino gene inhibits pigmentation driven by the color gene.

\hypertarget{relative-frequencies}{%
\section{Relative frequencies}\label{relative-frequencies}}

We can also summarize the observations by calculating the \textbf{proportion} of how many times we saw a particular result.

\[f_i = \frac{n_i}{n}\]

In our example, \(n_1=88\) grey mice were recorded, so we can ask about the proportion of grey from the total number of offspring observed: \(156\). We can copy these proportions \(f_i\) in the frequency table.

\[
\begin{array}{ccc}
\mathbf{outcome} & \mathbf{n_i} & \mathbf{f_i} \\
\text{grey}  & 88    & 0.56  \\
\text{black}   & 31     & 0.20  \\ 
\text{albino}  & 37     & 0.24 \\ \hline
\mathbf{sum} & 156 & 1 \\
\end{array}
\]

\(56\%\) of mice were grey. Relative frequencies are fundamental in statistics. They represent the proportion of one outcome relative to the other outcomes. Later, we will understand them as observations of quantities that provide empirical information about probabilities, which are abstract, unobserved but knowable properties of the random experiment.

In the mouse coat color example, the probabilities are derived from an inheritance model of recessive epistasis for albinos that states that the ratio of color outcomes is 9:3:4 for grey, black, and albino, respectively. This ratio have ideal frequencies \((0.5625, 0.1875, 0.25)\) for each outcome. No need to go into further genetics theory yet. However, while we will formally define probabilities in the following chapter, the point here is to note that probabilities are expected unobserved frequencies derived through reasoning, whereas relative frequencies are their observed empirical counterparts, obtained only through experimentation.

The numerical values of probabilities and relative frequencies often differ; expectation is not usually matched by observation. For instance, consider the relative frequencies obtained by Crampe: \((0.56, 0.20, 0.24)\). These are close to, but not identical with, those predicted by the model of recessive epistasis, where albinism and color are determined by different genes.

The available interpretation in Crampe's time was that of Mendelian recessive inheritance, in which two variants of a single gene lead to either albinism or black coloration, and grey color occurs when the two variants are inherited from one parent each. The expected Mendelian frequencies in this model are: \((0.5, 0.25, 0.25)\), also close to Crampe's data, though slightly less so than the epistatic model proposed by Little.

For absolute and relative frequencies, we have the important properties

\[\sum_{i=1}^m n_i = n\]

and

\[\sum_{i=1}^m f_i = 1\]

where \(m\) is the number of outcomes.

\hypertarget{bar-chart}{%
\section{Bar chart}\label{bar-chart}}

When we have a lot of outcomes and want to see which ones are most likely, we can use a bar chart. This is a plot of the frequencies \(n_i\) Vs the outcomes \(i\).

\includegraphics{_main_files/figure-latex/unnamed-chunk-5-1.pdf}

\hypertarget{pie-chart}{%
\section{Pie chart}\label{pie-chart}}

We can also visualize the relative frequencies \(f_i\) using a pie chart.

In the pie chart, the area of the circle represents the 100\% of the observations (proportion = 1) and the sections represent the relative frequencies of each outcome.

\includegraphics{_main_files/figure-latex/unnamed-chunk-7-1.pdf}

\hypertarget{ordinal-categorical-outcomes}{%
\section{Ordinal categorical outcomes}\label{ordinal-categorical-outcomes}}

The color of a mouse in the example above is a \textbf{categorical} nominal variable. Each observation belongs to a category (quality). The categories do not always have a certain order.

Sometimes \textbf{categorical} outcomes can be \textbf{sorted} when they have a natural ranking. This allows you to compute \textbf{cumulative frequencies}.

\textbf{Example (misophonia)}

A typical clinical study involves standardized measurements across multiple subjects or patients, often under controlled conditions. While individuals are clearly distinct, their observations are treated as statistically equivalent, meaning they are representative of the same random experiment that could be repeated for the same class of individuals.

This concept can be generalized to other scientific fields where experiments are conducted on specimens representing their respective classes. For example, consider measuring the hardness of an alloy. If the underlying experiment differs---either because the specimens belong to different classes (e.g., different alloys) or the measurement conditions (e.g., temperature) have changed---data interpretation and description must be adjusted accordingly.

In a clinical study run in Barcelona in \(2019\), \(123\) patients referred to a clinical psychologist for anxiety disorders were examined for their degree of misophonia. Misophonia is an involuntary experience of anxiety or anger triggered by specific sounds, and it may affect up to \(5\%\) of the population, often without their awareness. The study aimed to evaluate the risk factors and onset mechanisms associated with misophonia, a poorly recognized disorder at the time.

Each patient included in the study, was evaluated with the misophomia questionnaire AMISO, that scores patients into \(4\) different groups according to severity.

The results of the AMISO tests are

4, 2, 0, 3, 0, 0, 2, 3, 0, 3, 0, 2, 2, 0, 2, 0, 0, 3, 3, 0, 3, 3, 2, 0, 0, 0, 4, 2, 2, 0, 2, 0, 0, 0, 3, 0, 2, 3, 2, 2, 0, 2, 3, 0, 0, 2, 2, 3, 3, 0, 0, 4, 3, 3, 2, 0, 2, 0, 0, 0, 2, 2, 0, 0, 2, 3, 0, 1, 3, 2, 4, 3, 2, 3, 0, 2, 3, 2, 4, 1, 2, 0, 2, 0, 2, 0, 2, 2, 4, 3, 0, 3, 0, 0, 0, 2, 2, 1, 3, 0, 0, 3, 2, 1, 3, 0, 4, 4, 2, 3, 3, 3, 0, 3, 2, 1, 2, 3, 3, 4, 2, 3, 2

Each observation is, therefore, the result of the run of one particular random experiment: the measurement of the level of misophonia in a patient. This data series can be summarized using the frequency table

\[
\begin{array}{ccc}
\mathbf{outcome} & \mathbf{n_i} & \mathbf{f_i} \\
0   & 41    & 0.33333333  \\
1   & 5     & 0.04065041  \\
2   & 37    & 0.30081301  \\ 
3   & 31    & 0.25203252  \\
4   & 9     & 0.07317073  \\ \hline
\mathbf{sum} & 123 & 1
\end{array}
\]

We say that about \(25\%\) of the patients in the study had misophonia grade \(3\), or that the outcome of grade \(3\) was observed with \(25\%\) frequency. Note that the rows of table are now \textbf{ordered} by the severity of the disease.

\hypertarget{absolute-and-relative-cumulative-frequencies}{%
\section{Absolute and relative cumulative frequencies}\label{absolute-and-relative-cumulative-frequencies}}

Misophonia severity is \textbf{categorical} and \textbf{ordinal} because its outcomes can be meaningfully ordered by their degree.

When the outcomes of a random experiment can be ordered, it is useful to ask how many observations were obtained up to a given outcome. We call this number the \textbf{absolute cumulative frequency} up to the outcome \(i\), and compute it with sum the relative frequencies up to \(i\)
\[N_i =\sum_{k= 1}^i n_k\]
It is also useful to calculate the \textbf{proportion} of observations up to a given result.

\[F_i =\sum_{k= 1}^i f_k\]
This is called the \textbf{relative cumulative frequency}, or \textbf{frequency distribution} of the data.

We can add these frequencies to the \textbf{frequency table}

\[
\begin{array}{ccccc}
\mathbf{outcome} & \mathbf{n_i} & \mathbf{f_i} & \mathbf{N_i} & \mathbf{F_i}\\
0   & 41    & 0.33333333  & 41 & 0.3333333\\
1   & 5     & 0.04065041  & 46 & 0.3739837\\
2   & 37    & 0.30081301  & 83 & 0.6747967\\ 
3   & 31    & 0.25203252  & 114& 0.9268293\\
4   & 4     & 0.07317073  & \mathbf{123} & \mathbf{1}\\ \end{array}
\]

Therefore, \textbf{67\%} of patients had misophonia up to severity \textbf{2}, and \textbf{37\%} of patients had severity less than or equal to \textbf{1}.

\hypertarget{cumulative-frequency-graph}{%
\section{Cumulative frequency graph}\label{cumulative-frequency-graph}}

\(F_i\) is an important quantity because it allows us define the accumulation of frequencies up to \textbf{intermediate} levels of the outcomes.

The frequency up to an intermediate level \(x\) (\(i\leq x< i+1\)) is just the accumulation up to the lower possible outcome of the experiment \[F_x = F_i\]

\(F_x\) is therefore a function on a \textbf{continuous} range of values. We can draw it with respect to the outcomes; for instance, for the misophonia example

\includegraphics{_main_files/figure-latex/unnamed-chunk-12-1.pdf}

Therefore, we can say that \textbf{\(67\%\)} of the patients had misophonia up to severity \(2.3\), even though \(2.3\) is not an observed outcome.

\hypertarget{numerical-outcomes}{%
\section{Numerical outcomes}\label{numerical-outcomes}}

The result of a random experiment can produce a number. If the number is \textbf{discrete}, we can generate a frequency table, with absolute, relative, and cumulative frequencies, and illustrate them with bar, pie, and cumulative charts.

When the outcome is \textbf{continuous} the frequency table is not so useful, because it is unlikely to obtain a repetition of the same continuous number, to arbitrary small precision.

\textbf{Example (Dinosaur extinction)}

The Alvarez hypothesis states that the mass extinction in the Cretaceous-Paleocene boundary, \(\sim 66\) million of years ago (Ma), was due to the impact of a large meteorite in Yucatan's peninsula. Alvarez and son discovered a layer of extraterrestrial iridium, presumably from the meteorite, uniformly distributed around the globe at the depth close to the time of the extinction. A rival theory explained a gradual extinction by unprecedented volcanic activity, as the impact appeared to precede extinction in 300,000 years. After 30 years of dispute, Renne et al designed a method to precisely measure the age of collected geological samples \citep{Renne2013}, by assessing the quantity of radioactive potassium degradation into argon in impact melt droplets (tektites).

These were the ages in Ma of \(87\) tektites specimens for one of the experiments reported by Renne and colleagues:

65.664, 64.492, 66.239, 66.677, 66.137, 65.406, 65.980, 64.979, 66.302, 65.975, 66.252, 67.492, 65.792, 66.328, 65.975, 65.740, 66.462, 66.287, 67.281, 54.619, 40.915, 66.444, 65.710, 55.046, 65.463, 66.378, 65.774, 67.044, 65.756, 66.754, 65.809, 66.386, 51.835, 65.624, 63.565, 65.179, 65.896, 65.825, 66.257, 65.990, 65.421, 53.258, 66.884, 65.912, 65.693, 66.087, 65.857, 65.992, 65.916, 65.723, 66.275, 65.441, 66.630, 66.097, 69.700, 66.011, 66.218, 66.737, 60.071, 65.800, 62.429, 66.308, 67.036, 59.836, 66.149, 65.928, 65.951, 66.077, 70.980, 66.060, 66.207, 61.905, 66.055, 65.205, 66.433, 66.085, 66.001, 62.098, 66.064, 66.368, 66.348, 65.693, 66.035, 66.040, 66.27, 65.739, 66.023

If we look in detail these set of observations, we see that only two outcomes were repeated at \(65.693\) and \(65.975\), each. These are the result of measurement rounding or limited instrument resolution. While there was one single event that give rise to similar ages of the tektites, potassium decay into argon depends on so many factors that it is unlikely that two specimens have the same estimated aging, if we had sufficient resolution.

\hypertarget{transforming-continuous-data}{%
\section{Transforming continuous data}\label{transforming-continuous-data}}

Since observations of continuous outcomes cannot be counted in frequencies (at least by definition), we may transform them into ordered categorical outcomes. We do it in two steps:

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\tightlist
\item
  First, we cover the range of observations in regular intervals of the same size (bins)
\end{enumerate}

{[}40.9,43.9{]}, (43.9,46.9{]}, (46.9,49.9{]}, (49.9,52.9{]}, (52.9,55.9{]}, (55.9,59{]}, (59,62{]}, (62,65{]}, (65,68{]}, (68,71{]}

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{1}
\tightlist
\item
  Then, we map each observation to its interval: creating a categorical \textbf{ordered} outcomes; in this case, we have \(10\) possible outcomes
\end{enumerate}

(65,68{]}, (62,65{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (52.9,55.9{]}, {[}40.9,43.9{]}, (65,68{]}, (65,68{]}, (52.9,55.9{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (49.9,52.9{]}, (65,68{]}, (62,65{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (52.9,55.9{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (68,71{]}, (65,68{]}, (65,68{]}, (65,68{]}, (59,62{]}, (65,68{]}, (62,65{]}, (65,68{]}, (65,68{]}, (59,62{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (68,71{]}, (65,68{]}, (65,68{]}, (59,62{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (62,65{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}, (65,68{]}

Therefore, instead of saying that the first tektite had an age of \(65.664\) million of years, we say that its age is in the interval (or \textbf{bin}) between \((65,68]\).

\hypertarget{frequency-table-for-a-continuous-variable}{%
\section{Frequency table for a continuous variable}\label{frequency-table-for-a-continuous-variable}}

For a given regular partition of the range of the continuous outcomes into intervals, we can produce a frequency table as before

\[
\begin{array}{ccccc}
\mathbf{outcome} & \mathbf{n_i} & \mathbf{f_i} & \mathbf{N_i} & \mathbf{F_i}\\
\mathrm{[40.9,43.9]} & 1 & 0.01149425 &  1 & 0.01149425 \\ 
\mathrm{(43.9,46.9]} & 0 & 0.00000000 & 1 & 0.01149425 \\ 
\mathrm{(46.9,49.9]} & 0 & 0.00000000 &  1 & 0.01149425 \\ 
\mathrm{(49.9,52.9]} & 1 & 0.01149425 & 2 & 0.02298851 \\ 
\mathrm{(52.9,55.9]} & 3 & 0.03448276 &  5 & 0.05747126 \\ 
\mathrm{(55.9,59]} & 0 & 0.00000000 &  5 & 0.05747126 \\ 
\mathrm{(59,62]} & 3 & 0.03448276 &  8 & 0.09195402 \\ 
\mathrm{(62,65]} & 4 & 0.04597701 & 12 & 0.13793103 \\ 
\mathrm{(65,68]} & 73 & 0.83908046 & 85 & 0.97701149 \\ 
\mathrm{(68,71]} & 2 & 0.02298851 & \mathbf{87} & \mathbf{1} \\
\end{array}
\]

We see that \(83\%\) of the tektites had an age at this interval, already overlapping with the age of extinction; estimated at \(65.957\)Ma by fossil records. We still need some conceptual and analytic tools to be able to compare the true aging of two unobserved events, the extinction onset and meteorite impact, but tektite data give us the strong intuition that they were simultaneous in geological scale, and therefore that the meteorite was one of the causes of extinction.

\hypertarget{histogram}{%
\section{Histogram}\label{histogram}}

The histogram is the graph of the absolute (\(n_i\)) or relative \(f_i\) frequencies against the interval outcomes (bins). The histogram depends on the size of the bin.

\includegraphics{_main_files/figure-latex/unnamed-chunk-18-1.pdf}

The following is a histogram with \(50\) bins

\includegraphics{_main_files/figure-latex/unnamed-chunk-20-1.pdf}

We see, for instance, that the most frequent estimated impact age is close to \(66\)Ma, the age of extinction. The data already suggest that the meteorite impact and the dinosaur extinction are likely simultaneous events.

\hypertarget{cumulative-frequency-graph-1}{%
\section{Cumulative frequency graph}\label{cumulative-frequency-graph-1}}

We can also plot \(F_x\) against the outcomes. Since \(F_x\) has a continuous range, we can order the observations (\(x_1 <... x_j < x_{j+1} < x_N\)) and therefore

\[F_x = \frac{k}{N}\]

for \(x_{k} \leq x < x_{k+ 1}\) .

\(F_x\) is known as the \textbf{distribution} of the data. \(F_x\) does not depend on the size of the bin. However, its \textbf{resolution} depends on the amount of data.

\includegraphics{_main_files/figure-latex/unnamed-chunk-22-1.pdf}

We therefore see a steep increase in the number of observations between \(65\) and \(67\)Ma.

\hypertarget{summary-statistics}{%
\section{Summary Statistics}\label{summary-statistics}}

Summary statistics are numbers calculated from the data that tell us important characteristics of the numerical outcomes (discrete or continuous).

For example, we have statistics that describe extreme values:

\begin{itemize}
\tightlist
\item
  \textbf{minimum}: the minimum observation.
\item
  \textbf{maximum}: the maximum observation.
\end{itemize}

\hypertarget{average-sample-mean}{%
\section{Average (sample mean)}\label{average-sample-mean}}

An important statistic that describes the central value of the observations (where to expect most observations) is the \textbf{average}

\[\bar{x}=\frac{1}{N} \sum_{j=1}^N x_j\]

where \(x_j\) is the \textbf{observation} \(j\) from a total of \(N\) observations. We also call the average the \textbf{sample mean}. The sample is the set of observations, or the data obtained from the repetition of the random experiment.

\textbf{Example (Dinosaur extinction)}

The average estimated age for the deadly meteorite impact from tektites can be calculated directly from the \textbf{observations} in the usual way

\(\bar{x}= \frac{1}{ N}\sum_j x_j\)

\(= \frac{1}{87}(65.664 + 64.492 + 66.239... + 66.023) = 64.99741\)

The average is nearly a million years more recent than the start of the extinction. We will see that, under reasonable models and conditions, the average is an estimation that we can trust. We will say that it is the maximum likelihood value for the true aging. In this example, we can readily make some objections to this, as some few estimated ages appear to be excessively recent (\(40\)Ma) or excessively old (\(70\)Ma). As these observations do not gather with most of them, we have reasons to believe that they are not representative of the same potassium decay process that can age the impact, or that they are not suitable repetitions of the random experiment in question. Collection of those samples, processing of the material or potassium decay conditions may have substantially affected those specimens.

Now let us look as the average of \textbf{categorically ordered} outcomes. For these cases, it is insightful to show that we can \textbf{also} use the relative frequencies to calculate the average

\(\bar{x}=\frac{1}{ N}\sum_{i=1}^N x_j =\frac{1}{N}\sum_{i=1}^M x_i n_ {i}=\)
\[\sum_{i=1}^M x_i f_{i}\]

where we went from adding \(N\) \textbf{observations} to adding \(M\) \textbf{outcomes}.

The form \(\bar{x}= \sum_{i = 1}^M x_i f_i\) shows that the average is the \textbf{center of mass} of the data. As if each observation had a mass weight given by \(f_i\).

\textbf{Example (misophonia)}

The average \textbf{severity} of misophonia in the study can be calculated from the relative frequencies of the \textbf{outcomes}

\[
\begin{array}{ccc}
\mathbf{outcome} & \mathbf{n_i} & \mathbf{f_i} \\
0   & 41    & 0.33333333  \\
1   & 5     & 0.04065041  \\
2   & 37    & 0.30081301  \\ 
3   & 31    & 0.25203252  \\
4   & 9     & 0.07317073   
\end{array}
\]

\(\bar{x}=0\times f_ {0}+ 1\times f_{1}+2 \times f_{2}+3 \times f_{3}+4 \times f_{4}=1.691057\)

The average is also the center of mass for continuous outcomes; that is, the point where the relative frequencies of the bins balance. Clearly the average for a categorical ordered variables is not necessarily an outcome of the random experiment. There is no observable misophonia severity of \(1.691057\). Likewise, if the average birth rate in a country is \(1.5\) per family, there is no family with \(1.5\) children.

For continuous data, the average may be a possible observation and can still be considered the point where observations balance. We can imagine the histogram as a balancing rod, where we place sticks of different heights, and the mean represents the balancing point.

\includegraphics{_main_files/figure-latex/unnamed-chunk-25-1.pdf}

\hypertarget{median}{%
\section{Median}\label{median}}

Another measure of centrality is the median. The median \(x_m\), or \(q_{0.5}\), is the value below which we find half of the observations. When we order the observations \(x_1 \leq ... x_j \leq x_{j+1} \leq x_N\), we count them until we find half of them, then we report the value \(x_m\) that splits the observations in two equal parts.

If the number of observations is a pair number, we can be split then into two equal parts and \(x_m\) is the value between the observation \(x_{N/2}\) and \(x_{N/2+1}\). That is, if \(N\) is even

\[x_1 \leq... \leq x_{N/2} \leq x_{N/2+1} \leq x_N\]
then
\[x_1 \leq ... x_{N/2} \leq x_m \leq x_{N/2+1} \leq x_N\]

or \(x_m \in (x_{N/2}, x_{N/2+1})\). In general any value \(x_m\) in between these two numbers is a median, conventionally, if we need to report a single value we take the middle point between them. In the case that \(N\) is odd, the observation in the middle \(x_j\)

\[x_1 \leq ... x_{j-1} \leq x_j \leq x_{j+1} \leq x_N\]
for which \(j=(N+1)/2\), splits the data in half and it is the median
\(x_m=x_j\)

\textbf{Example (Dinosaur extinction)}

If we order the tektites estimated ages and cut them in half at position \(j=(87+1)/2=44\), we see that \(43\) observations (tektites) are below \(65.992\). The \textbf{median age} is therefore \(q_{0.5}= x_{44}=65.992\). Let us show the data explicitly. This is the fist half of the ordered data from the first to the \(43\)th position

40.915, 51.835, 53.258, 54.619, 55.046, 59.836, 60.071, 61.905, 62.098, 62.429, 63.565, 64.492, 64.979, 65.179, 65.205, 65.406, 65.421, 65.441, 65.463, 65.624, 65.664, 65.693, 65.693, 65.710, 65.723, 65.739, 65.740, 65.756, 65.774, 65.792, 65.800, 65.809, 65.825, 65.857, 65.896, 65.912, 65.916, 65.928, 65.951, 65.975, 65.975, 65.980, 65.99

The position \(44\)the position, or the median is

65.992

This is the second half of the data from the \(45\)th position to the last one.

66.001, 66.011, 66.023, 66.035, 66.040, 66.055, 66.06, 66.064, 66.077, 66.085, 66.087, 66.097, 66.137, 66.149, 66.207, 66.218, 66.239, 66.252, 66.257, 66.270, 66.275, 66.287, 66.302, 66.308, 66.328, 66.348, 66.368, 66.378, 66.386, 66.433, 66.444, 66.462, 66.630, 66.677, 66.737, 66.754, 66.884, 67.036, 67.044, 67.281, 67.492, 69.700, 70.98

In terms of frequencies, \(q_{0.5}\) makes the cumulative frequency \(F_x\) equal to \(0.5\)

\[\sum_{i = 1}^m f_i =F_{q_{0.5}}=0.5\]
that is

\[q_{ 0.5}= F^{-1}(0.5)\]

This last equation means that, in the distribution graph, the median \(q_{ 0.5}\) is the value of \(x\) at which we have climbed half of the total height of \(F_x\). We have accumulated half of the observations.

\includegraphics{_main_files/figure-latex/unnamed-chunk-30-1.pdf}

The average and median are not always the same. The point that divides half of the mass is not always the center of mass. In the tektite aging, it appears that the median is a better estimation of the meteorite impact age than the median because the median is \textbf{robust} to the presence of extreme observations or outliers. We see that the median is closer to the most frequent observations and to the expected time of extinction.

\includegraphics{_main_files/figure-latex/unnamed-chunk-32-1.pdf}

The median will be close to the average for histograms that look symmetrical about the average.

\hypertarget{dispersion}{%
\section{Dispersion}\label{dispersion}}

Other important summary statistics for observations are the statistics of \textbf{dispersion} or variability.

Many experiments may share their average, but differ in how spread the observations are.

\includegraphics{_main_files/figure-latex/unnamed-chunk-34-1.pdf}

The dispersion of the observations is a measure of the \textbf{noise} or randomness of the experiment. If observations do not vary, the random experiment gives always the same outcome and there is no need to learn statistics.

\hypertarget{sample-variance}{%
\section{Sample variance}\label{sample-variance}}

The dispersion about the average is measured by the sample variance

\[s^2=\frac{ 1}{ N-1} \sum_{j=1}^N ( x_j -\bar{x})^2\]

This number measures the average squared distance of the \textbf{observations} to the average. The reason for \(N-1\) will be explained when we talk about inference, when we study the spread of \(\bar{x}\), as well as the spread of the observations.

In terms of the frequencies of the outcomes that are \textbf{categorical and ordered}, we can \textbf{also} calculate the sample variance as

\[s^2=\frac{N}{N-1} \sum_{i=1}^M (x_i -\bar{x})^2 f_i\]

Take many observations and then make \(N/(N+1)\) close to \(1\), then \(s^2\) can be considered as the \textbf{moment of inertia} about the average of the observations.

The square root of the sample variance, \(s\), is called \textbf{standard deviation} of the sample.

\textbf{Example (Dinosaur extinction)}

The standard deviation of the tektite ages is

\(s= [\frac{ 1}{87-1}((64.99741-65.664)^2+ (64.99741-64.492)^2\)
\[+ ... + (64.99741-66.023)^ 2]^{1/2} = 3.939105\]

The tektites ages tend to differ from the age average in \(3.939105\)Ma. As this is a measure of how far the observations will be from the average then the summarized aging of the tektites can be written as

\[(\bar{x}, s)= (average=64.99741, sd= 3.939105)\]

\hypertarget{interquartile-range-iqr}{%
\section{Interquartile range (IQR)}\label{interquartile-range-iqr}}

The spread of the data can also be measured with respect to the median using the \textbf{interquartile range}, as follows:

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\tightlist
\item
  We define the \textbf{first} quartile as the value \(x\) that makes the cumulative frequency \(F_{q_{0.25}}\) equal to \(0.25\), or the value of \(x\) where we have accumulated a quarter of the observations. This is the value that splits the first quarter of the observations.
\end{enumerate}

\[q_{0.25}=F^{-1}(0.25)\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{1}
\tightlist
\item
  We define the \textbf{third} quartile as the value \(x\) that makes the cumulative frequency \(F_{q_{0.75}}\) equal to \(0.75\), or the value of \(x\) where we have accumulated three quarters of observations.
\end{enumerate}

\[q_{0.75}=F^{-1}(0.75)\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{2}
\tightlist
\item
  The \textbf{interquartile range} (IQR) is \[IQR=q_{0.75} - q_{0.25}\]
\end{enumerate}

This is the distance between the third and first quartiles and captures the central \(50\%\) of the observations

\textbf{Example (Dinosaur extinction)}

The first and third quartiles of the tektite data are

\[q_{0.25}=65.693\]
\[q_{0.75}=66.281\]
and therefore the IQR is \[IQR=66.281-65.693=0.588\].

\includegraphics{_main_files/figure-latex/unnamed-chunk-37-1.pdf}

A usual argument for reducing the dispersion of the observations, when extreme observations are obtained, is to discard those that are very far from the median. In particular, we can consider that observations with distances grater than \(1.5\times\)IQR are outliers, meaning that they are not representative of our random experiment. We may consider removing them form our data as a quality control measure. The outliers for the tektite data are:

40.915, 51.835, 53.258, 54.619, 55.046, 59.836, 60.071, 61.905, 62.098, 62.429, 63.565, 64.492, 64.979, 66.884, 67.036, 67.044, 67.281, 67.492, 69.700, 70.98

We thus see that the summarized aging for the tektites without those observations

\[(\bar{x}, s)= (average=66.01955, sd=0.339907)\]
gets closer to the median

\[(q_{0.5}, IQR)= (median=66.023, iqr=0.4675)\]
and substantially reduces the dispersion of the data. We also see that the median changes less than the average when we remove the outliers.

\hypertarget{boxplot}{%
\section{Boxplot}\label{boxplot}}

The interquartile range, median, and \(5\%\) and \(95\%\) of the data can be displayed in a \textbf{box plot}.

In the boxplot, the values of the outcomes are on the y-axis. The IQR is the box, the median is the middle line, and the whiskers mark the \(5\%\) and \(95\%\) of the data.

\includegraphics{_main_files/figure-latex/unnamed-chunk-41-1.pdf}

The estimated age of extinction, \(65.957\)Ma, is close to both the average (\(66.01955\)Ma) and the median (\(66.023\)Ma) ages of trusted tektite samples measured via potassium decay. This alignment suggests that the impact event and the onset of extinction could have been simultaneous on a geological timescale. Alternatively, if the impact occurred slightly later, it likely took place in an already ecologically stressed environment, amplifying its deadly consequences.

However, the use of the average (or median) derived from tektite chemistry to estimate the age of the impact raises an important question: how confident can we be in these estimates? If we were to analyze a different set of \(87\) tektites, we would likely obtain a different average. What if we analyzed \(200\) tektites? How much would the result vary? In other words, we aim to assess the level of certainty in each figure of our estimate. Intuitively, we are more confident in the first \(6\) of the \(60\) million years than in the last \(5\) of the \(50\) years in the figure \(66.01955\)Ma.

Could we achieve a precision small enough to rule out a temporal gap of \(300,000\) years between the impact and the extinction?

These are the critical questions we will address in the following chapters as we justify the use of the average (or any other statistic) and formulate its probable error. In their comprehensive study using multiple sources of evidence, Renne and colleagues achieved an unprecedented precision of \(11,000\) years for the impact age estimate, encompassing the onset of extinction. Their findings provide compelling evidence for the simultaneity of these events.

\hypertarget{questions}{%
\section{Questions}\label{questions}}

\textbf{1)} In the following boxplot, the first quartile and second quartile of the data are:

\textbf{\(\qquad\)a:} \((-1.00, 21.30)\); \textbf{\(\qquad\)b:} \((-1.00, 7.02)\); \textbf{\(\qquad\)c:} \((7.02, 7.96)\); \textbf{\(\qquad\)d:} \((7.02, 14.22)\)

\includegraphics{_main_files/figure-latex/unnamed-chunk-42-1.pdf}

\textbf{2)} The main disadvantage of a histogram is that:

\textbf{\(\qquad\) a :} Depends on the size of the bin ; \textbf{\(\qquad\)b :} Cannot be used for categorical outcome;
\textbf{\(\qquad\) c :} Cannot be used when the bin size is small;
\textbf{\(\qquad\) d :} Used only for relative frequencies;

\textbf{3)} If the relative cumulative frequencies of a random experiment with outcomes \(\{1,2,3,4\}\) are: \(F(1)=0.15, \qquad F(2)=0.60, \qquad F(3)=0.85, \qquad F(4)=1\).

Then the relative frequency for the outcome \(3\) is

\textbf{\(\qquad\)a:} \(0.15\); \textbf{\(\qquad\)b:} \(0.85\); \textbf{\(\qquad\)c:} \(0.45\); \textbf{\(\qquad\)d:} \(0.25\)

\textbf{4)} In a sample of size \(10\) from a random experiment we obtained the following data:

8, 3, 3, 7, 3, 6, 5, 10, 3, 8

The first quartile of the data is:

\textbf{\(\qquad\)a:} \(3.5\); \textbf{\(\qquad\)b:} \(4\); \textbf{\(\qquad\)c:} \(5\); \textbf{\(\qquad\)d:} \(3\)

\textbf{5)} Imagine that we collect data for two quantities that are not mutually exclusive, for example, the gender and nationality of passengers on a flight. If we want to make a single pie chart for the data, which of these statements is true?

\textbf{\(\qquad\)a :} We can \textbf{only} make a nationality pie chart because it has more than two possible outcomes;

\textbf{\(\qquad\)b :} We can make a pie graph for a new variable marking gender \textbf{and} nationality;

\textbf{\(\qquad\)c :} We can make a pie chart for the variable sex \textbf{or} the variable nationality;

\textbf{\(\qquad\)d :} We can only choose \textbf{whether} to make a pie chart for gender \textbf{or} a pie chart for nationality.

\hypertarget{exercises}{%
\section{Exercises}\label{exercises}}

\hypertarget{exercise-1}{%
\subsubsection{Exercise 1}\label{exercise-1}}

We have performed an experiment 8 times with the following results

3, 3, 10, 2, 6, 11, 5, 4

Answer the following questions:

\begin{itemize}
\tightlist
\item
  Calculate the relative frequencies of each result.
\item
  Calculate the cumulative frequencies of each result.
\item
  What is the average of the observations?
\item
  What is the median?
\item
  What is the third quartile?
\item
  What is the first quartile?
\end{itemize}

\hypertarget{exercise-2}{%
\subsubsection{Exercise 2}\label{exercise-2}}

We have performed an experiment 10 times with the following results

2.875775, 7.883051, 4.089769, 8.830174, 9.404673, 0.455565, 5.281055, 8.924190, 5.514350, 4.566147

Consider 10 bins of size 1: {[}0,1{]}, (1,2{]} \ldots( 9,10).

Answer the following questions:

\begin{itemize}
\item
  Calculate the relative frequencies of each result and draw the histogram
\item
  Calculate the cumulative frequencies of each result and draw the cumulative graph.
\item
  Draw a box plot .
\end{itemize}

\hypertarget{practice}{%
\section{Practice}\label{practice}}

In the misophonia study, the researchers wondered if the type of jaw given by cephalometric measures such as the convexity angle of the jaw would affect the severity of misophonia. The scientific hypothesis is that the angle of convexity of the jaw can influence hearing and its sensitivity, as the inner ear rests on top of the jaw. Summary of Python and R code can be found in the Apendix.

Load misophonia data from \url{https://alejandro-isglobal.github.io/SDA/data/Misophonia.txt}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Extract the misophonia severity variable
\end{enumerate}

\begin{itemize}
\tightlist
\item
  Do a bar plot and a pie chart
\end{itemize}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{1}
\tightlist
\item
  Extract the convexity angle variable
\end{enumerate}

\begin{itemize}
\tightlist
\item
  Calculate its sample mean (average), standard deviation and make a histogram
\item
  Calculate its median and inter-quartile range
\item
  Draw a boxplot for all the patients
\item
  Draw a boxplot for each misophonia severity
\end{itemize}

\href{https://colab.research.google.com/drive/1ncTaoBgskCJcBIb0-PdnbUFr_AKU0XRF?usp=sharing}{Solutions}

\hypertarget{probability}{%
\chapter{Probability}\label{probability}}

We may state that one of the primary objectives of experimental sciences is either to predict the outcomes of a random experiment or to gain control over the experiment to influence the frequencies of outcomes in a desired manner.

We hold a strong belief that advances in our knowledge and understanding of the experiment, framed within a coherent body of theories, will shed light on these objectives. Statistics plays a pivotal role in this endeavor, as it seeks to account for the randomness inherent in experiments and helps us extract meaningful signals from noise. Paradoxically, statistics involves the systematic study of aspects the experimentalist is not directly interested in: randomness and noise.

The central epistemic question behind statistics is this: How can we derive knowledge from a random experiment when its results vary each time it is conducted? There must be some invariant---a physical property of the experiment---that we can identify and, in certain cases, influence. A cornerstone of experimental sciences is the recognition that the propensities with which outcomes occur are intrinsic properties of the random experiment itself. When a random experiment is repeated, each outcome retains the same propensity of being observed. Sometimes a specific outcome will occur, and sometimes it will not, but as the experiment is repeated, the outcome will appear with a certain regularity: perhaps 2 out of 5 times, or 3 out of 7. While we can never predict with certainty for a finite number of repetitions, if the experiment were repeated an infinite number of times, the exact propensity of each outcome would be revealed.

In this chapter, we introduce the concept of probability as the value approached by relative frequencies when a random experiment is repeated infinitely. For experiments in which all outcomes have equal probabilities, this frequentist definition aligns with the classical definition of probability: the ratio of the number of favorable outcomes to the total number of possible outcomes. The frequentist definition also allows for a propensity interpretation, which views probabilities as abstract defining properties of the random experiment.

With the help of a dice, computer simulations and the inheritance of genetic traits, we will define and explain the fundamental rules of probability in alignment with relative frequency tables. Events will be introduced as the basic elements to which probabilities are assigned. Composite events will be constructed using the principles of set algebra.

The concept of the joint probability of two events will be derived from these axioms and will serve as the foundation for incorporating multiple simultaneous measurements in a random experiment. This will form the foundation of conditional probability and statistical independence, which will be explored in the next chapter.

\hypertarget{probability-mesurement}{%
\section{Probability mesurement}\label{probability-mesurement}}

We seek a measure of the likelihood that a particular outcome will occur in a future instance of a random experiment. These likelihoods will be referred to as probabilities of outcomes and will represent statements about their future occurrences.

We define the probability of an outcome as a measure of its propensity or likelihood of occurring, assigning it specific values:

\begin{itemize}
\tightlist
\item
  0, when the outcome has \textbf{no possibility} of occurring in the next run of the experiment.
\item
  1, when the outcome \textbf{is certain} to occur in the next run of the experiment.
\end{itemize}

Intermediate values can be easily conceived for a class of experiments with many outcomes, each equally likely.

\hypertarget{classical-probability}{%
\section{Classical probability}\label{classical-probability}}

As long as a random experiment has \(m\) possible outcomes that are all equally likely, the probability of each outcome \(i\) can be defined as:

\[P_i =\frac{1}{m}\].

Classical probability was explicitly defined by Laplace (1814).

Since every outcome is equally likely in this type of experiment, we may declare complete ignorance and, consequently, the best we can do is to equally distribute the same probability to each outcome. In other words, there are some random experiments for which we have no reason to assign more likelihood to one outcome than another.

Note that:

\begin{itemize}
\tightlist
\item
  \(P_i\) is an abstract defining property of the random experiment.
\item
  We do not observe \(P_i\).
\item
  We deduce \(P_i\) from the ratio above and have no need to carry out any experiment to know it.
\end{itemize}

\textbf{Example (dice):}

What is the probability that we will get \(2\) on the roll of a die?

\[P_2=1/6=0.166666\]

We reason that all other \(5\) numbers in the dice are equally likely to \(2\). The equal probability of the dice outcomes is a property of the random experiment of rolling it, which just states that the dice is fair and we should not expect one outcome rather than another.

\hypertarget{relative-frequencies-1}{%
\section{Relative frequencies}\label{relative-frequencies-1}}

What about random experiments whose possible outcomes are \textbf{not} equally likely?

How then can we define the probabilities of the outcomes?

\textbf{Example (mouse coating)}

Imagine that we obtain \(20\) mice from a couple, both of which are grey in color. We write down the colors of their progeny:

grey, grey, grey, grey, grey, albino, grey, black, albino, grey, black, grey, grey, grey, grey, black, grey, albino, black, grey

From this data, how sure are we of obtaining an albino offspring from a future breading of the couple?

The frequency table is

\[
\begin{array}{ccc}
\mathbf{outcome} & \mathbf{n_i} & \mathbf{f_i} \\
\text{grey}  & 13    & 0.65  \\
\text{black}   & 4     & 0.20  \\ 
\text{albino}  & 3     & 0.15  \\ \hline
\mathbf{sum} & 20 & 1
\end{array}
\]

and the bar plot

\includegraphics{_main_files/figure-latex/unnamed-chunk-48-1.pdf}

The \textbf{relative frequency} \(f_i =\frac{n_ i}{n}\) seems like a reasonable probability measure because

\begin{itemize}
\tightlist
\item
  it is a number between \(0\) and \(1\); and
\item
  it measures the proportion of the total number of observations that we obtained for a particular outcome.
\end{itemize}

Since \(f_{albino}=0.15\) then we would be about one sixth sure or, more precisely, three out of every \(20\) observations, of getting albino.

How good is \(f_i\) as a measure of the outcome's \(i\) likelihood?

Let us imagine, we repeat the previous experiment \(100,000\) more times. This is clearly impossible in practice but to achieve this, we will use the computer to simulate data based on a known inheritance model.

Here is the approach: Genetic theory for the epistatic model (Exercise 5), where albinism inhibits the grey and black coat colors, tell us that the relative ratios are 9:3:4 for grey, black and albino respectively. This means that out of \(16\) possible mice, \(4\) can be albino. To simulate this, we instruct the computer to create an urn containing \(9\) grey balls, \(3\) black balls, and \(4\) albino balls. A ball is drawn at random, its label is recorded, and then it is returned to the urn. This simulates running one single random experiment of epistatic inheritance.

We can repeat this process \(20\) times, as we did earlier, or \(100,000\) times, as we now show. The resulting frequency table is now as follows:

\[
\begin{array}{ccc}
\mathbf{outcome}   & \mathbf{n_i} & \mathbf{f_i} \\
\text{grey}  & 56281    & 0.56281  \\
\text{black}   & 18542    & 0.18542  \\ 
\text{albino}  & 25177    & 0.25177  \\
\hline
\mathbf{sum} & 100000 & 1
\end{array}
\]

and the bar plot is

\includegraphics{_main_files/figure-latex/unnamed-chunk-50-1.pdf}

We see that the frequency for \(f_{albino}\) is now \(0.25145\), which is closer to the likelihood of \(4\) in \(16\) (\(0.25\)) for producing an albino in the next offspring. Thus, we note that the probabilities measured by \(f_i\) vary with \(n\).

A crucial fact is that when we compute \(f_i\) with increasing values of \(n\), \(f_i\) \textbf{converges}.

In the following, graph each vertical section gives the relative frequency of each observation, for a given value of \(n\). We see that after \(n=1000\) (\(log10(n)=3\)) the sections' proportions hardly change with more \(n\), and the relative frequencies stabilize.

\includegraphics{_main_files/figure-latex/unnamed-chunk-51-1.pdf}

An important point needs to be made. The unequal frequencies \(f_1\), \(f_2\) and \(f_3\), at given \(n\) and shown in the figure, were obtained from an experiment with \(16\) equally likely outcomes. Each of the \(16\) balls in the urn has the same chance of being drawn. Four balls are labeled ``albino,'' giving them an accumulated probability of \(0.25\) of being drawn. In practice, after drawing a million balls, we observed a relative frequency of \(0.249898\) for albino. While this is not exactly \(0.25\), it is very close. As the number of draws increases, the relative frequency approaches the probability, becoming equal to it only at infinity.

By drawing an analogy between the urn and the reproduction of mice, we can imagine that the mice could reproduce to arbitrary numbers. While this is clearly impossible, it represents a step toward abstract thinking. If the mice could reproduce indefinitely, just as balls can be drawn from an urn indefinitely, then the relative frequency would converge to the probability. Specifically, the relative frequency \(f_i\) at infinity becomes an abstract quantity \(P_i\), the constant value to which \(f_i\) converges:

\[\lim_{n \to \infty} f_i = P_i\]

This quantity, \(P_i\), is what we call frequentist probability. It is important to note that \(f_i\) is derived from a finite number \(n\) of observations. In the lab, we can only perform a limited number of repetitions of a random experiment---always far from infinity. No amount of observations can definitively prove that the inheritance model is epistatic, especially with just \(20\) observations, or with real data from actual mice. However, as we will discuss in the following chapters, we can state with a certain level of confidence that the observed frequencies are consistent with an epistatic model.

\hypertarget{frequentist-probability}{%
\section{Frequentist probability}\label{frequentist-probability}}

We call \textbf{Probability} \(P_i\) the limit as \(n \rightarrow \infty\) of the \textbf{relative frequency} of observing the outcome \(i\) in a random experiment.

Defended by Venn (1876), the frequentist definition of probability is derived from (empirical) data/experience.

Note that:

\begin{itemize}
\tightlist
\item
  \(P_i\) is a property of an infinite repetition of the random experiment.
\item
  We do not observe \(P_i\), we observe \(f_i\) at a given value of \(n\).
\item
  \textbf{We estimate} \(P_i\) with \(f_i\) (usually when \(n\) is large), and write: \[\hat {P_ i}= f_i\]
\end{itemize}

Similar to the relationship between observation and outcome, there is a relationship between relative frequency and probability: a concrete value corresponding to an abstract quantity.

A suitable interpretation of probability in experimental sciences is as the propensity of a random experiment to produce a given outcome \citep{popper2002logic}. Probability is thus viewed as an abstract defining property of a random experiment, which can be either approximated as the experiment is repeated a large number of times, or known by experimental design.

\hypertarget{classical-and-frequentist-probabilities}{%
\section{Classical and frequentist probabilities}\label{classical-and-frequentist-probabilities}}

We have situations where classical probability can be used to find the limit of relative frequencies:

\begin{itemize}
\tightlist
\item
  If the results are \textbf{equally probable}, the classical probability gives us the limit:
\end{itemize}

\[P_i=lim_{n\rightarrow \infty} \frac{n_i}{n}=\frac{1}{m}\]

\begin{itemize}
\tightlist
\item
  If the results in which we are interested can be derived from other \textbf{equally probable} results, as in the mice coating example and the following one.
\end{itemize}

\textbf{Example (sum of two dice)}

Let us consider the sum of the outcomes when rolling two dice. We can determine the exact probabilities of the outcomes without actually rolling the dice.

This probability \textbf{follows} from the fact that the outcome of each die is \textbf{equally likely}. From this assumption, we can find that (Exercise 1)

\[
    P_i =
\begin{cases}
\frac{i-1}{36},& i \in \{2,3,4,5,6, 7\} \\
\frac{13-i}{36},& i \in \{8,9,10,11,12\} \\
\end{cases}
\]

Imagine, however, that we have a black box where we do not know that two dice are being thrown, but we can see the result of their roll displayed on a screen

\[5, 3, 11, 9, 5, 7\]

We are then able to closely approach the probabilities of the random experiment using the \textbf{relative frequencies} if we repeat the experiment a million times (as simulated by a computer)

\[
\begin{array}{cccc}
\mathbf{outcome}   & \mathbf{n_i} & \mathbf{f_i} & \mathbf{P_i} \\
2  & 28070  & 0.028070  & 0.02777778 \\
3  & 55617  & 0.055617  & 0.05555556 \\
4  & 82964  & 0.082964  & 0.08333333 \\
5  & 111113 & 0.111113  & 0.11111111 \\
6  & 138749 & 0.138749  & 0.13888889 \\
7  & 166717 & 0.166717  & 0.16666667 \\
8  & 139011 & 0.139011  & 0.13888889 \\
9  & 111193 & 0.111193  & 0.11111111 \\
10 & 83504  & 0.083504  & 0.08333333 \\
11 & 55397  & 0.055397  & 0.05555556 \\
12 & 27665  & 0.027665  & 0.02777778 \\ \hline
\mathbf{sum} & 1000000 & 1 & 1 \\ 
\end{array}
\]

Exact knowledge can be obtained from a model without the need to conduct an experiment, using the formula for \(P_i\). In contrast, precise knowledge about outcome likelihoods can be derived from large amounts of data, even without understanding the underlying mechanisms. Models are free to describe any experiment, real or not, while data can yield accurate predictions without necessarily deepening our understanding. Kant observed, `Thoughts without content are empty; intuitions without concepts are blind.' Thus, scientific content and insight are often achieved by contrasting a model's predictions with experimental results.

The motivation behind the frequentist definition of probability is empirical, while that of the classical definition is rational. Since both approaches are complementary, we often combine them with inference and deduction to determine the probabilities of our random experiment.

\hypertarget{sample-space}{%
\section{Sample space}\label{sample-space}}

Probabilities are numbers assigned to each possible outcome of a random experiment. However, probabilities are also applied to composite outcomes or \textbf{events}. For instance, we can ask about the probability of drawing a ball labeled ``albino'' when there are four such balls out of a total of 16. Each ``albino'' ball represents a different outcome of the drawing, but the event of drawing an ``albino'' ball is the same.

Therefore, before providing a formal definition of probabilities, we need a further characterization of the outcomes that a random experiment can produce.

The set of all possible outcomes of a random experiment is called the sample space and is denoted by \(S\).

The sample space can consist of categorical or numerical outcomes. It represents the range of outcomes that a random experiment can yield.

\emph{For example:}

\begin{itemize}
\tightlist
\item
  human temperature: \(S = (36, 42)\) degrees Celsius
\item
  sugar levels in humans: \(S =( 70,80) mg/ dL\)
\item
  the size of a production line screw: \(S =(70,72) mm\)
\item
  number of emails received in an hour: \(S = \{0, ...\infty \}\)
\item
  coat colors of mice: \(S= \{grey, black, albino\}\)
\item
  the throw of a dice: \(S= \{ 1, 2, 3, 4, 5, 6\}\)
\end{itemize}

\hypertarget{events}{%
\section{Events}\label{events}}

An \textbf{event} \(A\) is a \textbf{subset} of the sample space. It is a \textbf{collection} of possible outcomes.

\emph{Examples of events:}

\begin{itemize}
\tightlist
\item
  The event of a healthy temperature: \(A=(37,38)\) degrees Celsius
\item
  The event of producing a screw with a size range: \(A = (71.5mm, 71.6mm)\)
\item
  The event of receiving more than \(4\) emails in an hour: \(A= \{ 4, \infty \}\)
\item
  The event of obtaining an albino mouse \(A=\{albino\}\)
\item
  The event of obtaining a number less than or equal to 3 in the throw of a dice: \(A= \{ 1,2,3\}\)
\end{itemize}

An event refers to a possible set of primary \textbf{outcomes}.

\hypertarget{algebra-of-events}{%
\section{Algebra of events}\label{algebra-of-events}}

While it seems clear that the result of a random experiment can give us a specific outcome or a more general event, it is also expected that the result of the experiment can be a composite event.

For two events \(A\) and \(B\), we can construct the following \textbf{composite events} using the basic set operations:

\begin{itemize}
\tightlist
\item
  Complement \(A'\): the event of \textbf{not} \(A\)
\item
  Union \(A \cup B\): the event of \(A\) \textbf{or} \(B\)
\item
  Intersection \(A \cap B\): the event of \(A\) \textbf{and} \(B\)
\end{itemize}

\textbf{Example (dice)}

Let us imagine we intend to roll a die but first we want look at a range of interesting events (composite outcomes):

\begin{itemize}
\tightlist
\item
  a number less than or equal to three \(A:\{ 1,2,3\}\)
\item
  an even number \(B:\{ 2,4,6\}\)
\end{itemize}

Let us see how we can build new events with set operations:

\begin{itemize}
\tightlist
\item
  a number \textbf{not} less than three: \(A':\{4,5,6\}\)
\item
  a number less than or equal to three \textbf{or} even: \(A \cup B: \{ 1,2,3,4,6\}\)
\item
  a number less than or equal to three \textbf{and} even \(A \cap B: \{ 2\}\)
\end{itemize}

Random experiments produce a rich set of outcomes that can be gather together in events and transformed into other more complex events. Probabilities will be, consequently, defined on events.

\hypertarget{mutually-exclusive-events}{%
\section{Mutually exclusive events}\label{mutually-exclusive-events}}

Primary outcomes like rolling \(1\) and \(2\) on a die are events that cannot occur at the same run of the experiment, a single roll of the dice. We say that they are \textbf{mutually exclusive}.

In general, two events denoted as \(E_1\) and \(E_2\) are mutually exclusive when they have no element in common

\[E_1\cap E_2=\emptyset\]

\emph{Examples:}

The following events are mutually exclusive:

\begin{itemize}
\item
  The result that a patient has a misophonia severity \(1\) and severity \(4\). Only one severity is possible.
\item
  The results of obtaining \(12\) and \(5\) in the roll of two dice. If we get \(12\) we do not get \(5\).
\end{itemize}

However, the events of rolling a number ``less than or equal to three'' \textbf{and} ``even'' in a dice are \textbf{not} mutually exclusive. They share the outcome \(2\).

\hypertarget{definition-of-probability}{%
\section{Definition of probability}\label{definition-of-probability}}

The probability of an event, as an abstract quantity, is defined by a set of axioms. These axioms represent the minimum set of rules that relate to relative frequencies.

The probabilities of a random experiment's events are numbers satisfying the following properties or axioms:

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\item
  When the events \(E_1\) and \(E_2\) are mutually exclusive; that is, only one of them can occur, the probability of observing \(E_1\) \textbf{or} \(E_1\) is their sum:
  \[ P( E_1\cup E_2) = P(E_1) + P(E_2)\]
\item
  When \(S\) is the sample space, then its probability is \(1\) (at least something is observed): \[P(S)=1\]
\item
  The probability of any event is between \(0\) and \(1\) (a continuous and bounded measure) \[P(E) \in [0,1]\]
\end{enumerate}

Remarkably, the axioms were proposed by Kolmogorov in 1933 \citep{kolmogorov2013foundations}, after statistics had already been formally established as a distinct discipline and following the formulation of statistical and quantum mechanics---two prominent physics theories based on probability concepts. Advancement in these theories was possible because probabilities were treated as frequencies of infinite populations.

\hypertarget{probability-table}{%
\section{Probability table}\label{probability-table}}

Kolmogorov's axioms are the fundamental rules for defining a measure on the sample space and, as such, are independent of any specific interpretation. Probabilities, as mathematical objects, do not necessarily need to represent any random experiment. However, our interest lies in probabilities as measures of the likelihood of outcomes in real random experiments. Kolmogorov's axioms importantly support the close alignment of probabilities with relative frequencies, as they also form the foundational rules for constructing a probability table, similar to relative frequency tables.

Therefore, we can assert that frequencies tend to probabilities as the random experiment is repeated indefinitely, or that probabilities may be regarded as abstract defining properties of the experiment, representing the likelihood of its outcomes in the next repetition of the experiment.

\textbf{Example (dice)}

The probability table for the roll of a dice is

\[
\begin{array}{cc}
\mathbf{outcome}  & \mathbf{P_i} \\
1  & 1/6 \\
2  & 1/6 \\
3  & 1/6 \\
4  & 1/6 \\
5  & 1/6 \\
6  & 1/6 \\ \hline
S=\{1, 2, ... 6\} & 1 \\ 
\end{array}
\]

Let us verify the axioms:

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\item
  Where \(1 \cup 2\) is, for example, the \textbf{event} of rolling a \(1\) \textbf{or} a \(2\). So \[ P( 1 \cup 2)=P(1)+P(2)=2/6\]
\item
  Since \(S= \{ 1,2,3,4,5,6\}\) is made up of \textbf{mutually exclusive} outcomes, then
\end{enumerate}

\[P(S)=P(1\cup 2 ... \cup 6) = P(1)+P(2)+ ...+P(6)=1\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{2}
\tightlist
\item
  The probabilities of each outcome are between \(0\) and \(1\). This can be seen in the table.
\end{enumerate}

According to Kolmogorov's properties, only mutually exclusive outcomes can be organized into probability tables, similar to relative frequency tables.

\hypertarget{joint-probabilities}{%
\section{Joint probabilities}\label{joint-probabilities}}

Mutually exclusive events whose overall union is the sample space can be considered primary events. They constitute all the possible values of a type of measurement of the experiment that can be listed in a table.

However, we have seen that there are events that are not mutually exclusive. How are we to understand them and to list them on a probability table? Two different events with a common outcome may then constitute two different types of measurement, that sometimes coincide in that common outcome.

Let us consider the probability table for the dice and the events:

\begin{itemize}
\item
  a number less than or equal to three \(A:\{ 1,2,3\}\)
\item
  an even number \(B:\{ 2,4,6\}\)
\end{itemize}

which are not mutually exclusive coinciding in the outcome \(2\): \(A \cap B = \{2\}\).

In the rolling of a dice, we can observe there \emph{types of outcomes}, or three types of measurements: a number from 1 to 6 (outcome 1), the event of whether a number is \(\leq 3\) (outcome 2) or the event of whether a number is pair (outcome 3).

\[
\begin{array}{cccc}
\mathbf{outcome\, 1} & \mathbf{outcome\, 2} & \mathbf{outcome\, 3}  & \mathbf{P_i} \\
1 & A & B' & 1/6 \\
\mathbf{2} & \mathbf{A} & \mathbf{B} & \mathbf{1/6} \\
3 & A & B' & 1/6 \\
4 & A'& B & 1/6 \\
5 & A'& B' & 1/6 \\
6 & A'& B & 1/6 \\ \hline
S=\{1, 2, ... ,6\} & S=\{A \cup A'\} & S=\{B \cup B'\} & 1 \\ 
\end{array}
\]

Now, we can also observe the occurrence of the the crossed possibilities between outcomes 2 \textbf{and} 3. In particular, we can consider the joint event \(A\cap B\) as the event with two different qualities: ``being greater or equal to 3'' and ``pair'', namely, outcome 1 being \(2\).

For non mutually exclusive events, those that share common primary outcomes, we note that we can always decompose the sample space into \textbf{mutually exclusive} sets involving the intersections of the crossed possibilities between the events and their complements:

\[S=\{A\cap B, A \cap B', A'\cap B, A'\cap B'\}\]
If one of those events occur the others do not. We can now make a probability table of these events. The \textbf{joint probability} of \(A\) and \(B\) is the probability of \(A\) and \(B\). That is \[P( A \cap B)\] or \(P(A,B)\). The \textbf{probability table} for the joint probabilities will be as follows

\[
\begin{array}{ll}
\mathbf{outcome\, 4}  & \mathbf{joint\,probabilities} \\
 A\cap B=\{2\}    & P(A\cap B)=1/6 \\
 A\cap B'=\{1,3\} & P(A\cap B')=2/6 \\
 A'\cap B=\{4,6\} & P(A'\cap B)=2/6 \\
 A'\cap B'=\{5\}  & P(A'\cap B')=1/6 \\ \hline
S=\{A\cap B, ... A'\cap B'\} & P(S)=1 \\ 
\end{array}
\]

The outcomes of type 4 are, therefore, composed of 4 mutually exclusive events, namely a number in the dice that is: ``\(\leq 3\) and pair''; or ``\(\leq 3\) and not pair''; or ``not \(\leq 3\) and pair''; or ``not \(\leq 3\) and not pair''. The probabilities of the sum off all these composite events is \(1\).

The \textbf{marginals} of \(A\) and \(A'\) are the probabilities of each of those events. That is

\[
\begin{array}{ll}
\mathbf{outcome\, 2}  & \mathbf{marginal\,probabilities} \\
 A=\{1,2,3\}    & P(A)=3/6 \\
 A'=\{4,5,6\} & P(A')=3/6 \\ \hline
S=A\cup A' & P(S)=1 \\ 
\end{array}
\]

Note that the marginal \(P(A)\) is the addition of the joint probabilities where \(A\) occurs \[P(A)=P(A\cap B') + P(A \cap B)\] \[=2/6+1/6=3/6\] .

Similarly we have the marginals for \(B\) and \(B'\)

\[
\begin{array}{ll}
\mathbf{outcome\, 3}  & \mathbf{marginal\,probabilities} \\
 B=\{2,4,6\}    & P(B)=3/6 \\
 B'=\{1,3,5\} & P(B')=3/6 \\ \hline
S=A\cup A' & P(S)=1 \\ 
\end{array}
\]

And, the marginal \(P(B)\) is the addition of all the probabilities where event \(B\) occurs \[P(B)=P(A'\cap B) +P(A \cap B)\] \[=2/6+1/6=3/6\]

\hypertarget{contingency-table}{%
\section{Contingency table}\label{contingency-table}}

The joint and the marginal probability tables can be written in a single \textbf{contingency table}

\[
\begin{array}{ccc|c}
           & \mathbf{B}             & \mathbf{B'}             & \mathbf{marginals} \\ 
\mathbf{A}        & P(A \cap B)   & P(A \cap B')   & P(A)       \\ 
\mathbf{A'}       & P(A' \cap B)  & P(A' \cap B')  & P(A')      \\ \hline
\mathbf{marginals} & P(B)        & P(B')         & 1            \\ 
\end{array}
\]

Where the marginals are the sums in the margins of the table, for example:

\begin{itemize}
\tightlist
\item
  \(P(A)=P(A \cap B') + P(A \cap B)\)
\item
  \(P(B)=P(A' \cap B) + P(A \cap B)\)
\end{itemize}

In our example, the contingency table for the events \(A\) and \(B\) and their complements is

\[
\begin{array}{ccc|c}
           & \mathbf{B}             & \mathbf{B'}             & \mathbf{marginals} \\ 
\mathbf{A}        & 1/6   & 2/6   & 3/6       \\ 
\mathbf{A'}       & 2/6  & 1/6   & 3/6     \\ \hline
\mathbf{marginals} & 3/6        & 3/6         & 1            \\ 
\end{array}
\]

\hypertarget{the-addition-rule}{%
\section{The addition rule}\label{the-addition-rule}}

The addition rule allows us to calculate the probability of \(A\) \textbf{or} \(B\), \(P( A \cup B)\), in terms of the probability of \(A\) \textbf{and} \(B\), \(P(A \cap B )\). We can do this in three equivalent ways:

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\item
  Using only joint probabilities
  \[P( A \cup B)=P(A \cap B)+P(A\cap B')+P(A'\cap B)\]
\item
  Using the complement of the joint probability
  \[P(A \cup B)=1-P(A'\cap B')\]
\item
  Using the marginals and the joint probability
  \[P(A \cup B)=P(A) + P(B) - P(A\cap B)\]
\end{enumerate}

\textbf{Example (Mendel's first law)}

Every organism inherits two versions of a gene that we call alleles, one from each parent (\(X_m\),\(X_f\)). This is the joint event (\(X_m \cap X_f\)). The alleles can be identical or different, and sometimes can be linked to a characteristic of the organism. For the coat color of mice there is one allele for albino (\(A\)) and another for not-albino (\(A'\)), which allows any color.

Inheritance of the alleles is made at random with equal probabilities from the parental alleles. For one offspring, we define

\begin{itemize}
\item
  the event of inheriting albino allele from the mother \(A_m\)
\item
  the event of inheriting albino allele from the father \(A_f\)
\end{itemize}

If both parents are hybrids they both have alleles (\(A \cap A'\)) and their offspring can have four allele pairs

\[S =\{A_m \cap A_f,  A_m \cap A_f', A_m' \cap A_f, A_m' \cap A_f'\}\]

with equal probabilities of \(1/4\). The contingency table is

\[
\begin{array}{ccc|c}
                   & \mathbf{A_f}   & \mathbf{A_f'} & \mathbf{marginals} \\ 
\mathbf{A_m}       & 1/4   & \mathbf{1/4}  & 1/2                \\ 
\mathbf{A_m'}      & \mathbf{1/4}   & \mathbf{1/4}           & 1/2                 \\ \hline
\mathbf{marginals} & 1/2            & 1/2           & 1                   \\ 
\end{array}
\]

Having a color is dominant trait, which means that a mouse is colored if at least one the alleles is not albino. That is the event \((A'_m \cup A'_f)\).

Therefore, the probability of producing a color mouse is \(3/4\) that can be computed in three different ways:

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\item
  \(P( A'_m \cup A'_f)=P(A'_m \cap A'_f)+P(A_m\cap A_f')+P(A_m'\cap A_f)=1/4+1/4+1/4=3/4\)
\item
  \(P(A'_m \cup A'_f)=1-P(A_m\cap A_f)= 1-1/4=3/4\)
\item
  \(P( A'_m \cup A'_f)=P(A'_m) + P(A'_f) - P(A'_m\cap A'_f)=1/2+1/2-1/4=3/4\)
\end{enumerate}

In the contingency table, \(P( A'_m \cup A'_f)\) corresponds to the addition of the three cells in bold (method 1 above). The probability is also given by the addition of all cells but the \(1/4\) from the top left (method 2), or by adding the marginals and subtracting \(P(A'_m\cap A'_f)\) that has been added twice (method 3).

Color is a dominant trait because it masks albinism when present in only one allele. We can also say that albino is a recessive trait. Therefore, the probabilities for color and albino are given by the ratio of 3 to 1 (3:1), or probability \(3/4=0.75\), satisfying the Mendel's first law \citep{mendel1901experiments}.

We can consider more alleles at other genes that are not mutually exclusive with the inheritance of the albino alleles. For example, if we observe an additional trait (outcome type) with two alleles for the length of the ears (short or long), there will be \(16\) possible allele combinations, for all the possible crossings of paternal alleles. According to Mendel's second law, if the traits are independent, there will be four possible types of mice with different albino and tail length status, having ratios of 9:3:3:1 (see Exercise 4); with only one possibility in 16 for the albino and long ear mice.

William Bateson observed that some traits interfere with each other; for instance, being albino eliminates coat pigmentation. In such cases, inheritance deviates from Mendelian patterns and is considered epistatic, resulting in ratios like 9:3:4 (see Exercise 5), with only three observed colors.

After collecting data from several offspring, our scientific interest may lie in determining whether two traits are independent or if one trait suppresses the other, potentially to gain control over the first if harmful. While exact probabilities cannot be derived from relative frequencies, sufficient evidence may still be obtained to decide which scenario is more likely.

\hypertarget{questions-1}{%
\section{Questions}\label{questions-1}}

The following data is part of John Snow's study on the London 1854 cholera epidemic, which was foundational for modern epidemiology and early germ theory \citep{snow1855mode}. He famously showed that cholera was transmitted by water and that identified Southwark and Vauxall water supply was the likely source of the epidemic. The following data record the date of death and the water supply source used by 3920 deceased patients. While more deaths were recorded Southwark and Vauxall the number of houses supplied by each company needs to be taken into account for correct conclusions.\\
\[
\begin{array}{|l|r|r|r|r|r|r|}
\hline
\textbf{Week Ending} & \textbf{Total} & \textbf{S and V} & \textbf{Lambeth} & \textbf{Kent} & \textbf{Other} & \textbf{Not Ascertained} \\
\hline
\text{Sept 2, 1854}  & 670 & 399 & 45  & 38  & 72 & 116 \\
\text{Sept 9, 1854}  & 972 & 580 & 72  & 45  & 62 & 213 \\
\text{Sept 16, 1854} & 856 & 524 & 66  & 48  & 44 & 174 \\
\text{Sept 23, 1854} & 724 & 432 & 72  & 28  & 62 & 130 \\
\text{Sept 30, 1854} & 383 & 228 & 25  & 19  & 24 &  87 \\
\text{Oct 7, 1854}   & 200 & 121 & 14  & 10  & 9  &  46 \\
\text{Oct 14, 1854}  & 115 &  69 & 8   & 3   & 6  &  29 \\
\hline
\textbf{Total}       & \textbf{3920} & \textbf{2353} & \textbf{302} & \textbf{191} & \textbf{279} & \textbf{795} \\
\hline
\end{array}
\]

\textbf{1)} What is the estimated probability that a patient died on the 16th of September and likely drank from Southwark and Vauxhall's water supply?

\textbf{\(\qquad\)a:} \(524/3920\); \textbf{\(\qquad\)b:} \(524/2353\); \textbf{\(\qquad\)c:} \(524\); \textbf{\(\qquad\)d:} \(524/856\)

\textbf{2)} What is the estimated probability that the deceased patient was not from the borough of Lambeth?

\textbf{\(\qquad\)a:} \(302/3920\); \textbf{\(\qquad\)b:} \(3618/3920\); \textbf{\(\qquad\)c:} \(795/3920\); \textbf{\(\qquad\)d:} \(279/3920\)

\textbf{3)} What is the marginal probability of dying in October?

\textbf{\(\qquad\)a:} \(3605/100\); \textbf{\(\qquad\)b:} \(315/3920\); \textbf{\(\qquad\)c:} \(315/100\); \textbf{\(\qquad\)d:} \(3605/3920\)

\textbf{4)} What is the marginal probability that Kent water company supplied a deceased patient?

\textbf{\(\qquad\)a:} \(13/191\); \textbf{\(\qquad\)b:} \(191\); \textbf{\(\qquad\)c:} \(191/3920\); \textbf{\(\qquad\)d:} \(191/100\)

\textbf{5)} What is the probability of dying in September or from an unknown water source?
\textbf{\(\qquad\)a:} \(3680/3920\); \textbf{\(\qquad\)b:} \(3634/3920\); \textbf{\(\qquad\)c:} \(3620/3920\); \textbf{\(\qquad\)d:} \(3611/3920\)

\hypertarget{exercises-1}{%
\section{Exercises}\label{exercises-1}}

\hypertarget{classical-probability-exercise-1}{%
\subsubsection{Classical probability: Exercise 1}\label{classical-probability-exercise-1}}

\begin{itemize}
\item
  Write the table of \textbf{joint probability} for the \textbf{results} of rolling two dice; In the rows write the results of the first die and in the columns the results of the second die.
\item
  What is the probability of drawing \((3, 4)\) ? (A:1/36)
\item
  What is the probability of rolling \(3\) and \(4\) with any of the two dice? (A:2/36)
\item
  What is the probability of rolling \(3\) on the first die or \(4\) on the second? (To:11/36)
\item
  What is the probability of rolling \(3\) or \(4\) with any dice? (A:20/36)
\item
  Write the \textbf{probability table} for the result of the \textbf{add} of two dice. Assume that the outcome of each die is \textbf{equally likely}. Verify that it is:
\end{itemize}

\[
P_i=
\begin{cases}
\frac{i-1}{36},& i \in \{2,3,4,5,6, 7\} \\
\frac{13-i}{36},& i \in \{8,9,10,11,12\} \\
\end{cases}
\]

\hypertarget{frequentist-probability-exercise-2}{%
\subsubsection{Frequentist probability: Exercise 2}\label{frequentist-probability-exercise-2}}

The result of a randomized experiment is to measure the severity of misophonia \textbf{and} the state of depression of a patient.

Misophonia

\begin{itemize}
\tightlist
\item
  severity: \(S_M:\{M_ 0,M _1,M_2,M_3,M_4\}\)
\item
  Depression: \(S_ D:\{ D', D\}\))
\end{itemize}

Write the contingency table for the absolute frequencies (\(n_{ M,D }\)) for a study on a total of 123 patients in which it was observed

\begin{itemize}
\tightlist
\item
  100 individuals did not have depression.
\item
  No individual with misophonia 4 and without depression.
\item
  5 individuals with grade 1 misophonia and no depression.
\item
  The same number as the previous case for individuals with depression and without misophonia .
\item
  25 individuals without depression and grade 3 misophonia .
\item
  The number of misophonics without depression for grades 2 and 0 were distributed equally .
\item
  The number of individuals with depression and misophonia increased progressively
  in multiples of three, starting at 0 individuals for grade 1.
\end{itemize}

Answer the following questions:

\begin{itemize}
\tightlist
\item
  How many individuals had misophonia ? (A:83)
\item
  How many individuals had grade 3 misophonia ? (A:31)
\item
  How many individuals had grade 2 misophonia without depression? (A:35)
\end{itemize}

Write down the contingency table for relative frequencies \(f_{ M,D }\). Suppose \(N\) is large and the absolute frequencies \textbf{estimate} the probabilities \(f_{ M,D }=\hat {P}(M \cap D)\). Answer the following questions:

\begin{itemize}
\tightlist
\item
  What is the marginal probability of severity 2 misophonia ? (A: 0.3)
\item
  What is the probability of not being misophonic \textbf{and} not being depressed? (A:0.284)
\item
  What is the probability of being misophonic \textbf{or} depressed? (A: 0.715)
\item
  What is the probability of being misophonic \textbf{and} being depressed? (A: 0.146)
\item
  Describe in spoken language the results with probability 0.
\end{itemize}

\hypertarget{exercise-3}{%
\subsubsection{Exercise 3}\label{exercise-3}}

We have carried out a randomized experiment \(10\) times, which consists of recording the sex and vital status of patients with some type of cancer after 10 years of diagnosis. We got the following results

\[
\begin{array}{ccc}
\mathbf{Patient}   &\mathbf{Sex}   & \mathbf{Status}  \\
1 & male  & dead \\
2 &male  & dead \\
3 &male  & dead \\
4 &female& alive \\
5 &male  & dead \\
6 &female& alive \\
7 &female& dead \\
8 &female& alive \\
9 &male  & alive \\
10 &male  & alive \\
\end{array}
\]

\begin{itemize}
\tightlist
\item
  Create the contingency table for the number (\(n_{ i,j }\)) of observations of each result (\(A,B\))
\item
  Create the contingency table for the relative frequency (\(f_{ i,j }\)) of the results
\item
  What is the marginal frequency of being a man? (R/0.6)
\item
  What is the marginal frequency of being alive? (R/0.5)
\item
  What is the frequency of being alive \textbf{or} being a woman? (R/0.6)
\end{itemize}

\hypertarget{exercise-4}{%
\subsubsection{Exercise 4}\label{exercise-4}}

Consider an additional gene in mice that determines ear length, with short ears being the dominant trait. Each mouse inherits alleles (\(X_m\), \(X_f\), \(Y_m\), \(Y_f\)), where \(X\) represents the alleles for albinism and \(Y\) the alleles for ear length, inherited from the mother (subscript m) and the father (subscript f).

Suppose both parents are dihybrid---i.e., (\(A,A',B,B'\))---and therefore express the dominant phenotypes: non-albino with short ears. Show that if the traits for albinism and ear length assort independently, then there are four distinct phenotypic outcomes among the offspring that appear in a 9:3:3:1 ratio, illustrating Mendel's second law of independent assortment.

\hypertarget{exercise-5}{%
\subsubsection{Exercise 5}\label{exercise-5}}

Consider an additional gene in mice that determines black or grey color, with grey the dominant trait. Each mouse inherits alleles (\(X_m\), \(X_f\), \(Y_m\), \(Y_f\)), where \(X\) represents the alleles for albinism and \(Y\) the alleles for grey and black colors, inherited from the mother (subscript m) and the father (subscript f).

Suppose both parents are dihybrid---i.e., (\(A,A',B,B'\))---and therefore express the dominant phenotype: non-albino with grey color. Show that if the albinism removes pigmentation, then there are three distinct phenotypic outcomes among the offspring that appear in a 9:3:4 ratio, illustrating an apistatic model of inheritance.

\hypertarget{exercise-6}{%
\subsubsection{Exercise 6}\label{exercise-6}}

\begin{itemize}
\item
  From the second form of the addition rule, obtain the first and the third form.
\item
  What is the third form addition rule for the probability of three events \(P(A \cup B \cup C)\)?
\end{itemize}

\hypertarget{practice-1}{%
\section{Practice}\label{practice-1}}

Load misophonia data \url{https://alejandro-isglobal.github.io/SDA/data/Misophonia.txt}

\begin{itemize}
\item
  Compute the contingency table of absolute frequencies for misophonia diagnosis (Misophonia severity) and depression (Depression)
\item
  Compute the contingency table of relative frequencies for misophonia diagnosis and depression
\item
  Compare the differences with exercise 2.
\end{itemize}

\href{https://colab.research.google.com/drive/1ncTaoBgskCJcBIb0-PdnbUFr_AKU0XRF?usp=sharing}{Solutions}

\hypertarget{conditional-probability}{%
\chapter{Conditional probability}\label{conditional-probability}}

The final result of a random experiment can involve the observation of two or more types of outcomes. In a single realization of a random experiment, we may observe a list of observations of different types, each forming a column in an observation matrix. Repetitions of the random experiment add rows to the matrix. Random experiments thus increase in complexity.

Events that are not mutually exclusive naturally support random experiments where multiple types of outcomes can be measured. Joint probabilities of these events extend probability tables into contingency tables for two types of outcomes. This allows us to determine the likelihood of a specific pair of values occurring for the two outcome types. In some experiments, the measurement of one type outcome may provide information about the other, which may be, for instance, technically challenging to ascertain. For example, consider diagnostic tools. In other experiments, one outcome may be easier to control or may provide insights into the probabilities of the other outcome; take for instance the relationship between the temperature of an engine and the power it delivers.

In this chapter, we will introduce the concept of conditional probability. Conditional probability will be used to define statistical independence between two types of outcomes. Evaluating statistical independence is crucial for understanding and controlling random experiments. This topic will be explored further in subsequent chapters on hypothesis testing, where we will examine different ways of conditioning experiments and outcomes.

Additionally, we will discuss Bayes' theorem and one of its primary applications: assessing the predictive efficiency of diagnostic tools.

\hypertarget{joint-probability}{%
\section{Joint probability}\label{joint-probability}}

Recall that the joint probability of two events \(A\) \textbf{and} \(B\) is defined as the probability of their intersection

\[P( A ,B )=P(A \cap B)\]

Now imagine random experiments that measure simultaneously different types of outcomes:

\begin{itemize}
\item
  the throw of two dice: (\(n_1, n_2\))
\item
  height and weight of an individual: \((h, w)\)
\item
  position and speed of a molecule in a gas: \((x, v)\)
\item
  speed and distance of a galaxy: \((v, d)\)
\item
  in glycolysis (braking down of glucose), the time the process takes in the first two reactions (phosphorylation of glucose and isomerization of G6P): (\(t_1, t_2\))
\end{itemize}

In the first four cases both measurements may appear to be simultaneous characteristics of the experiment, and it is clear that we can report one and then the other, or the other way round. In this last case, while the gycolysis reactions always occur in the same order, when running the experiment, we may also report the times of reaction as (\(t_1 \cap t_2\)) or (\(t_ 2 \cap t_1\)). That is, the joint events are commutative, or in the observation matrix, the columns can be reordered.

We are now interested in whether the values of one type of outcome \textbf{conditions} the values of the other.

\hypertarget{statistical-independence-and-correlation}{%
\section{Statistical independence and correlation}\label{statistical-independence-and-correlation}}

In many cases, we are interested in whether certain values of one type of outcome are more likely to occur alongside specific values of another. Our goal is to distinguish between these two scenarios:

\begin{itemize}
\item
  \textbf{Independence} between events. For example, rolling a 1 on one die does not make it more likely to roll another 1 on a second die.
\item
  \textbf{Correlation} between events. For example, if a man is tall, he is probably heavy.
\end{itemize}

While independence between two events refers to the probability of one event being unaffected by the other event, dependence implies that the probability of the one event is higher when the other event occurs. Correlation thus refers to the likelihood of observing the two events together, or their joint event, or their constant conjunction. As joint events are commutative, correlation does not imply a temporal or causal relationship between the events. However, if there is a causal relationship then correlation is expected.

\textbf{Example (conductor)}

Imagine we conducted an experiment to find out if structural flaws in a material affects its electrical conductivity. We repeated the experiment in \(n\) specimens and measured both quantities.

The observation matrix would look like

\[
\begin{array}{ccc}
\mathbf{Conductor} & \mathbf{Structure} & \mathbf{Conductivity} \\ 
c_1 & \text{flaws} & \text{low} \\ 
c_2 & \text{no flaws} & \text{high} \\
c_3 & \text{flaws} & \text{low} \\
\vdots & \vdots & \vdots \\ 
c_i & \text{no flaws} & \text{low*} \\ 
\vdots & \vdots & \vdots \\ 
\vdots & \vdots & \vdots \\ 
c_n & \text{flaws} & \text{high*} \\ 
\end{array}
\]

We may expect that low conductivity occurs more often with flaws than without flaws if the flaws affect conductivity. That is a correlation between the flaws and low conductivity where there is more likelihood for the structure-conductivity pairs with no star. However, we need a tool to assess it.

Let us imagine that from the data we obtain the following \textbf{joint probability table}

\[
\begin{array}{ccc|c}
 & \mathbf{Flaws: F} & \mathbf{No\, flaws: F'} & \mathbf{sum} \\
\mathbf{Low\, conductivity: L} & 0.005 & 0.045 & 0.05 \\
\mathbf{High\, Conductivity: L'} & 0.095 & 0.855 & 0.95 \\ \hline
\mathbf{sum} & 0.1 & 0.9 & 1 \\
\end{array}
\]

where, for example, the joint probability that one conductor has low conductivity (\(L\)) and flaws (\(F\)) is

\begin{itemize}
\tightlist
\item
  \(P(L,F)=0.005\)
\end{itemize}

and the marginal probabilities are

\begin{itemize}
\tightlist
\item
  \(P(L)=P(L, F) + P(L, F')=0.05\)
\item
  \(P(F)=P(L, F) + P(L', F)= 0.1\).
\end{itemize}

\hypertarget{conditional-probability-1}{%
\section{Conditional probability}\label{conditional-probability-1}}

We will say that low conductivity is \textbf{independent} of having structural flaws if the probability of having low conductivity (\(L\)) is the same \textbf{whether} it has flaws (\(F\)) or not (\(F'\) ) .

Let us first consider only the materials that have flaws.

Among those materials that have flaws (\(F\)), what is the estimated probability that they have low conductivity?

Think of the observation matrix, and consider counting the number of specimens with low conductivity and flaws (\(n_{L,F}\)), and all the specimens with flaws \(n_{F}\). Then the fraction of materials with flaws that have low conductivity is

\(\frac{n_{L,F}}{n_{F}}=\frac{n_{L,F}/n}{n_{F}/n}= \frac{f_{L,F}}{f_{F}}\)
\[= \frac{\hat{P}( L,F )}{\hat{P}(F)}\]
where \(f_{L,F}\) are \(f_{F}\) the relative frequencies for \(n_{L,F}\) and \(n_{F}\); respectively, and \(n\) the total number of specimens tested. The last term is the estimated probability that an specimen has low conductivity \textbf{if} it has flaws. In the limit when \(n \rightarrow \infty\) is

\[P(L| F)= \frac{P(L,F)}{P(F)}=\frac{P(L\cap F)}{P(F)}\]

where the symbol \(|\) states that the random experiment of measuring conductivity is now run \textbf{only} on the specimens that \textbf{we know} they have flaws. Note that in \(P(L| F)\), \(F\) is not considered as an outcome but as a condition of the experiment. However, this new probability is computed from a random experiment where both \(L\) and \(F\) are outcomes \(P(L\cap F)\), by reducing the sample space to the specimens with the condition \(F\).

\textbf{Definition:}

The \textbf{conditional probability} of an event \(A\) given an event \(B\), denoted \(P(B| A)\) , is

\[P(A|B) = \frac{P(A\cap B)}{P(B)}.\]

We can prove that conditional probability satisfies the axioms of probability. The point here is that conditional probability can be understood as a probability where the conditional event \(B\) is fixed. That is, reconsidering the random experiment where the event \(B\) is a condition and part of its design. In our example, the new experiment includes only specimens with structural flaws.

\hypertarget{conditional-contingency-table}{%
\section{Conditional contingency table}\label{conditional-contingency-table}}

If we divide the columns of the joint probability table by their marginal probabilities \(P(F)\) and \(P(F')\), we obtain a \textbf{conditional contingency table}

\[
\begin{array}{cc|c}
    & \mathbf{Flaws: F} & \mathbf{No\, flaws: F'} \\ 
    \mathbf{Low\, conductivity: L} & P(L \mid F) & P(L \mid F') \\ 
    \mathbf{High\, conductivity: L} & P(L' \mid F) & P(L' \mid F') \\ 
    \mathbf{sum} & 1 & 1
\end{array}
\]

where the column probabilities sum to one. The first column shows the probabilities of low conductivity or not, only of the materials that have flaws (first condition: \(F\)). The second column shows the probabilities only for the materials that have no flaws (second condition: \(F'\)). This table refers to two different random experiments, one under the first condition and another under the second one.

Conditional probabilities are the probabilities of the joint events within each condition. We read them as:

\begin{itemize}
\tightlist
\item
  \(P(L| F)\): Probability of having low conductivity \textbf{if} it has flaws
\item
  \(P(L'| F)\): Probability of not having low conductivity \textbf{if} it has flaws
\item
  \(P(L|F ')\): Probability of having low conductivity \textbf{if} it has no flaws
\item
  \(P(L'|F ')\): Probability of not having low conductivity \textbf{if} it has no flaws
\end{itemize}

\hypertarget{statistical-independence}{%
\section{Statistical independence}\label{statistical-independence}}

In our example, the conditional contingency table is

\[
\begin{array}{cc|c}
    & \mathbf{Flaws: F} & \mathbf{No\, flaws: F'} \\ 
    \mathbf{Low\, conductivity:L} & 0.05 & 0.05 \\ 
    \mathbf{High\, conductivity: L'} & 0.95 & 0.95 \\ 
    \mathbf{sum} & 1 & 1
\end{array}
\]

We note that the conditional probabilities in this table are equal to the marginals \(P(L)\) and \(P(L')\) in the joint probability table (Section 4.2)

\begin{itemize}
\tightlist
\item
  \(P(L)=P(L| F)= P(L|F')\)
\item
  \(P(L')=P(L'| F)= P(L'|F')\)
\end{itemize}

This means that the probability of observing low conductivity is \textbf{not} dependent on whether a structural flaw is present.

Note that in the joint probability table, the most probable outcome was high conductivity with no flaws, \(P(L' \cap F') = 0.855\). At first glance, this might suggest a relationship between these two events. However, this apparent relationship arises because both low conductivity and flaws are relatively uncommon events (\(P(L) = 0.05\), \(P(F) = 0.1\)) compared to their complements. Conditioning on flaws eliminates the disparity between the marginal probabilities and reveals the actual independence of the events.

We conclude that, for the physical situation represented by this experiment, low conductivity in the material is not influenced by the presence of structural flaws. Equivalently, high conductivity is independent of the absence of flaws.

\textbf{Definition}

Two events \(A\) and \(B\) are statistically independent if either of the equivalent cases occurs:

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\tightlist
\item
  \(P(A| B)= P(A)\); \(A\) is independent of \(B\)
\item
  \(P(B| A)= P(B)\); \(B\) is independent of \(A\)
\end{enumerate}

and by the definition of conditional probability

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{2}
\tightlist
\item
  \(P(A\cap B)=P(A|B)P(B)=P(A)P(B)\)
\end{enumerate}

This third form is a statement about joint probabilities. It says that we can obtain joint probabilities by multiplying the marginal probabilities.

In our original joint probability table (Section 4.2),
we can confirm that all the entries of the matrix are indeed the product of the marginal probabilities. For example: \(P( L \cap F)=P(F)P(L)\) and \(P(L' \cap F')=P(L')P(F')\). Therefore, in our experiment, low conductivity is independent of having a structural flaw because their joint probability is the product of the marginals.

\textbf{Example (Mendel´s first law)}

We aim to confirm that the contingency table for Mendel's first law represents an independent inheritance model of parental alleles.

For mice, the contingency table for the albinism allele \(A\) of an offspring from parents, each with alleles (\(A, A'\)), is provided in Section 3.14.

\[
\begin{array}{ccc|c}
 & \mathbf{A_p} & \mathbf{A_p'} & \mathbf{sum} \\
\mathbf{A_m} & \mathbf{\frac{1}{4}} & \mathbf{\frac{1}{4}} & \frac{1}{2} \\
\mathbf{A_m'} & \mathbf{\frac{1}{4}} & \frac{1}{4} & \frac{1}{2} \\ \hline
\mathbf{sum} & \frac{1}{2} & \frac{1}{2} & 1 \\
\end{array}
\]

From this table, we see that the probability of getting the albinism allele \(A\) from the mother and from the father is the product of the marginals \(P(A_m, A_p)=P(A_m)P(A_p)=1/4\). Therefore, the inheritance of the maternal allele is independent from the paternal allele. If we build the conditional contingency table, we will see that inheriting a maternal allele is not conditioned by having inherited a paternal allele: \(P(A_m| A_p)=P(A_m)=1/2\).

\hypertarget{statistical-dependency}{%
\section{Statistical dependency}\label{statistical-dependency}}

An important example of statistical dependency or correlation is found in the performance of \textbf{diagnostic tools}, where we want to determine the state of a system with possible outcomes

\begin{itemize}
\tightlist
\item
  satisfactory (yes)
\item
  unsatisfactory (not)
\end{itemize}

using a test with results

\begin{itemize}
\tightlist
\item
  positive
\item
  negative
\end{itemize}

For example, we test a battery to see how long it can last. We perform a stress-strain test on a material to test its elasticity. We perform a PCR (polymerase chain reaction) to see if someone has an infection.

\hypertarget{diagnostic-test}{%
\section{Diagnostic test}\label{diagnostic-test}}

Let us consider diagnosing an infection with a new test. The infection status has two possible outcomes:

\begin{itemize}
\tightlist
\item
  yes (the patient is infected)
\item
  no (the patient is not infected)
\end{itemize}

The test has two possible outcomes:

\begin{itemize}
\tightlist
\item
  positive (the test detects the infection)
\item
  negative (the test does not detect the infection)
\end{itemize}

In the laboratory we run performance studies for the testing. In a control environment, where we know whether a patient has the infection, we run the random experiment of testing for the infection. If the testing has high performance, then its results will give mostly positive outcomes. However, we also expect some negatives as the test is not perfect. We also run second experiment where we test patients without the disease.

The conditional probability table describes the likelihoods of each testing outcome for each condition

\[
\begin{array}{cc|c}
    & \mathbf{Infection: yes} & \mathbf{Infection: no} \\ 
    \mathbf{Test: positive} & P(pos \mid yes) & P(pos \mid no) \\ 
    \mathbf{Test: negative} & P(neg \mid yes) & P(neg \mid no) \\ 
    \mathbf{sum} & 1 & 1
\end{array}
\]

The conditional table tells us that we are running an experiment where \textbf{we know} that the patient either has the disease or not. These are the \textbf{controlled conditions} of the experiment. Think for instance that you test the diagnostic tool in the hospital where you know the patients are infected. You also test the tool in healthy individuals who you know they are not infected.

Let us look at the table entries

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\item
  \(P(pos| yes)\) is called the \textbf{sensitivity} of the tool or the probability of a true positive: The probability of testing positive \textbf{if} a patient has the disease.
\item
  \(P(neg| no)\) is called the \textbf{specificity} of the tool or the probability of a true negative: The probability of testing negative \textbf{if} a patient does not have the disease.
\item
  \(P(pos| no)\) is called the probability of a false positive: the probability of testing positive \textbf{if} the patient does not have the disease.
\item
  \(P(neg| yes)\) is called the probability of a false negative: the probability of testing negative \textbf{if} the patient has the disease.
\end{enumerate}

High correlation (statistical dependence) between test and infection means high values for probabilities 1 and 2 in the diagonal (successes) \textbf{and} low values for probabilities 3 and 4 off the diagonal (errors).

\textbf{Example (COVID)}

Now, let us consider a real situation. In the early days of the sars-cov-2 pandemic (2019/2020), there was no measure of the effectiveness of PCRs in detecting the virus. One of the first published studies \citep{Woloshin2020FalseNegative} found that

\begin{itemize}
\tightlist
\item
  The PCR had a sensitivity of 70\%, in infection condition.
\item
  The PCR had a specificity of 94\%, in non-infected condition.
\end{itemize}

The conditional probability table for this study was

\[
\begin{array}{cc|c}
    & \mathbf{Infection: yes} & \mathbf{Infection: no} \\ 
    \mathbf{Test: positive} & 0.7 & 0.06 \\ 
    \mathbf{Test: negative} & 0.3 & 0.94 \\ 
    \mathbf{sum} & 1 & 1
\end{array}
\]

Therefore, the errors in the diagnostic tests had probabilities:

\begin{itemize}
\tightlist
\item
  \(P(pos| no)= 0.06\), for a false positive.
\item
  \(P(neg| yes)= 0.3\), for a false negative.
\end{itemize}

Can we say with this data that the test is useful to detect the infection?

\hypertarget{inverse-probabilities}{%
\section{Inverse probabilities}\label{inverse-probabilities}}

We are really interested in finding the probability of being infected if the test is positive \[P(yes| pos)\]

instead of the reported sensitivity \(P(pos|yes)\). In other words, you may want to run an experiment on a patient that you know is positive for the test and determine the probability that he is infected. Note that this experiment is not in a controlled environment of the lab, it is rather a surveying campaign on the population, where you collect all the positives and determine the fraction that were infected.

To compute these inverse probability, where the condition is now on the other event (positive), we follow the steps:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Recover the contingency table for \textbf{joint probabilities}, multiplying by the marginal \(P(yes)\) and \(P(no)\) that we need to know or be given.
\end{enumerate}

\[
\begin{array}{ccc|c}
 & \mathbf{Infection: yes} & \mathbf{Infection: no} & \mathbf{sum} \\
\mathbf{Test: positive} & P(pos \mid yes)P(yes) & P(pos \mid no)P(no) & P(pos) \\
\mathbf{Test: negative} & P(neg \mid yes)P(yes) & P(neg \mid no)P(no) & P(neg) \\ \hline
\mathbf{sum} & P(yes) & P(no) & 1 \\
\end{array}
\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{1}
\tightlist
\item
  Obtain the \textbf{conditional probability table} for rows.
\end{enumerate}

\[
\begin{array}{cccc}
    & \mathbf{Infection: yes} & \mathbf{Infection: no} & \mathbf{sum} \\ 
    \mathbf{Test: positive} & P(yes \mid pos) & P(no \mid pos ) & 1 \\ \hline
    \mathbf{Test: negative} & P(yes \mid neg) & P(no \mid neg) & 1\\ 
\end{array}
\]

To compute these probabilities, we use the definition of conditional probabilities for rows instead of columns. We divide the rows of the joint probability table in step 1 by the marginals of the test outcomes: \(P(pos)\) and \(P(neg)\). For example, for the first cell of the conditional probability table we obtain:

\[P(yes| pos)= \frac{P(pos|yes)P(yes)}{P(pos)}\]

Note that these is a new random experiment, in which all the individuals tested positive for the PCR fixed and we ask for the probability of infection.

While \(P(pos|yes)\) was the result of the study (\(0.7\)), we still need to know the the marginals \(P(yes)\) (prevalence) and \(P(pos)\).

\begin{itemize}
\item
  The prevalence \(P(yes)\) \textbf{needs to be given}. In real life, it is obtained from another study. The first prevalence study in Spain, before the summer of 2020, showed that during lock down the estimated probability of infection was \(P(yes)=0.05\), and of no infection \(P(no)=0.95\).
\item
  To find the marginal of positive tests \(P(pos)\), we can use the definition of marginal and conditional probabilities:
\end{itemize}

\(P(pos)= P(pos \cap yes) + P(pos \cap no)\)
\[= P(pos| yes)P (yes)+P(pos|no)P(no)\]
This last relation of the marginals is called \textbf{total probability rule}.

\hypertarget{bayes-theorem}{%
\section{Bayes' Theorem}\label{bayes-theorem}}

After substituting the total probability rule into \(P(yes| pos)\) , we have

\[P(yes| pos)= \frac{P(pos|yes)P(yes)}{P(pos|yes)P(yes)+P(pos|no)P(no)}\]
This expression is known as \textbf{Bayes' theorem}. It allows us to reverse the conditionals:

\[P(pos|yes) \rightarrow P(yes| pos)\]
This result is important. It allows us to \textbf{assess} a diagnostic tool in a controlled condition (infection status is a lab) and then use it to \textbf{infer} the probability of the condition (infection) when the test is positive.

\textbf{Example (COVID)}:

The test performance was:

\begin{itemize}
\item
  Sensitivity: \(P(positive| yes)= 0.70\)
\item
  False positive: \(P(positive| no)= 1- P(neg|no)=0.06\)
\end{itemize}

The study in the Spanish population gave:

\begin{itemize}
\tightlist
\item
  \(P(yes)=0.05\)
\item
  \(P(no)=1-P(yes)=0.95\).
\end{itemize}

Therefore, the probability of being infected in case of testing positive was:

\[P(yes| pos)= 0.38\]

We conclude that at that time PCR was not very good at \textbf{diagnosing} infections.

However, let us now apply Bayes' theorem to the probability of not being infected if the test was negative.

\[P(no|neg) = \frac{P(neg|no) P(no )}{ P(neg|no) P(no)+P(neg|yes)P(yes)}\]

Substituting all values gives

\[P(no| neg)= 0.98\]

Therefore, the tests were good for \textbf{ruling out} infections and a fair requirement for travelling.

In general, we can have more than two conditioning events, or controlled environments of our experiment. Therefore, Bayes' theorem says that the probability of an event \(E_i\) given the condition \(B\):

\[P(E_i| B)= \frac{P(B|E_i)P(E_i)}{P(B|E_0)P(E_0) +...+ P(B|E_k)P(E_k)}\]
when \(E_0, E_1, ..., E_k\) are \(k\) mutually exclusive and exhaustive events and \(B\) is a condition of interest.

\textbf{Example (Mosophonia)}

Consider the diagnosis of misophonia. An interesting question is whether misophonia severity \(4\) is more likely in depressed patients (\(D\)). This corresponds to the conditional probability \(P(4|D)\). In a clinical study of misophonia, patients were evaluated for four different severity levels and tested for signs of clinical depression. The probability of severity \(4\) in depressed patients, \(P(4|D)\), can be expressed using Bayes' theorem in terms of the probability of depression in severe cases (\(P(D|4)\)) and the marginal probabilities \(P(4)\) and \(P(D)\). The prevalence of depression (\(P(D)\)) can be calculated using the total probability rule:

\[
P(D) = P(D|0)P(0) + \ldots + P(D|4)P(4)
\]

We do not know whether depression or misophonia occurs first, but reverse conditional probabilities (e.g., \(P(D|4)\)) can be useful as an initial step toward identifying potential causal relationships. Depression relief could then, in some cases, be delivered by misophonia treatment. It is important to note that conditional probability reflects correlation, not causation. However, if misophonia is indeed a cause of depression, we would expect the probability of depression to increase with misophonia severity, following a dose-response relationship. Additionally, establishing causation requires biological or physical plausibility, ruling out reverse causation (e.g., depression causing misophonia), and other supporting criteria.

Even so, this may not be sufficient, as confounding factors could influence the observed relationship. For example, if misophonia severity and depression are both higher in women, the association may be due to their shared dependence on gender rather than a direct causal link. Adjustments and further conditioning of the experiment may be necessary, requiring additional control variables or the measurement of more outcome types to disentangle these effects.

\textbf{Conditional tree}

The terms in the total probability rule can also be \textbf{organized} in a conditional tree.

\includegraphics{./figures/treetot.jpg}

The \textbf{total probability rule} tells us in how many ways I can get the result \(B\) from the outcomes \(A\) or \(A'\)

\[P(B)=P(B|A)P(A)+P(B|A')P(A')\]

Trees are an important tool to determine possible ways of causation as they can incorporate several types of conditions.

\textbf{Example (Monty Hall)}

Monty Hall, the host of a 1970s television show, asked a contestant to choose one of three doors. Behind one door was a car; behind the other two, goats. After the contestant made an initial choice, Monty revealed one of the two remaining doors that hid a goat. He then asked whether the contestant wanted to stick with their original choice or switch to the other unopened door. Once the final decision was made, the chosen door was opened to reveal the prize.

The game gained widespread attention after a 1990 column by Marilyn vos Savant in Parade magazine, in which many academics incorrectly argued that switching made no difference. The controversy highlighted the common difficulty of understanding conditional probabilities and sparked a lasting debate within the statistical community: Should the player stick with their choice or switch doors?

Consider now the events of choosing a car (\(c\)), the first goat (\(g_1\)), or the second goat (\(g_2\)) in the first door choice:

\begin{itemize}
\tightlist
\item
  \(S_1=\{c, g_1, g_2\}\)
\end{itemize}

The probabilities of these events are equally likely \(P(c)=P(g_1)=P(g_2)=\frac{1}{3}\).

After Monty Hall opening a door that reveals either goat 1 or goat 2, consider the events that the unopened door has the car, the first goat, or the second one:

\begin{itemize}
\tightlist
\item
  \(S_2=\{C, G_1, G_2\}\)
\end{itemize}

The probabilities on these events are conditional probabilities, as they depend on what we have initially chosen:

\[
\begin{array}{cc|c|c}
    & \mathbf{c} & \mathbf{g_1} & \mathbf{g_2}\\ 
    \mathbf{C} & 0 & 1 & 1\\ 
    \mathbf{G_1} & \frac{1}{2} & 0 & 0\\ 
    \mathbf{G_2} & \frac{1}{2} & 0 & 0 \\ 
    \mathbf{Sum} & 1 & 1 & 1
\end{array}
\]

For instance, the probability that Hall's unopened door has goat 1 if the car is in the player's first door choice \(P(G_1|c)=\frac{1}{2}\). Or, the probability that the unopened door has goat 1 if goat 2 is in the first door \(P(G_1|g_2)=0\), since Monty Hall should reveal goat 1 in the second door.

What is the probability that in the remaining door there is a car?

The total probability rule for \(P(C)\) is:

\includegraphics{./figures/montyhall.jpg}

\[
P(C)=P(C|g_1)P(g_1)+P(C|g_2)P(g_2)=\frac{2}{3}
\]

Therefore, there is more chance of winning a car if we always switch doors and select the remaining door, as correctly indicated Selvin \citep{Selvin1975-SELAPI-2}. Note that in the case Monty Hall does not reveal a door, the player remains with two doors to switch from. Then, the player has, for instance, half chance to win a car if they selected goat 1 in the first door \(P(C|g_1)=P(G_2|g_1)=\frac{1}{2}\). Consequently, their overall chances remain the same \(P(c)=P(C)=\frac{1}{3}\) if they decide to switch doors or not.

\hypertarget{questions-2}{%
\section{Questions}\label{questions-2}}

The following data is part of John Snow's study on the London 1854 cholera epidemic \citep{snow1855mode}. He famously showed that the lack of water supply hygiene was key in the spread of cholera. The data record the date of death and the water supply source used by 3920 deceased patients.

\[
\begin{array}{|l|r|r|r|r|r|r|}
\hline
\textbf{Week Ending} & \textbf{Total} & \textbf{S and V} & \textbf{Lambeth} & \textbf{Kent} & \textbf{Other} & \textbf{Not Ascertained} \\
\hline
\text{Sept 2, 1854}  & 670 & 399 & 45  & 38  & 72 & 116 \\
\text{Sept 9, 1854}  & 972 & 580 & 72  & 45  & 62 & 213 \\
\text{Sept 16, 1854} & 856 & 524 & 66  & 48  & 44 & 174 \\
\text{Sept 23, 1854} & 724 & 432 & 72  & 28  & 62 & 130 \\
\text{Sept 30, 1854} & 383 & 228 & 25  & 19  & 24 &  87 \\
\text{Oct 7, 1854}   & 200 & 121 & 14  & 10  & 9  &  46 \\
\text{Oct 14, 1854}  & 115 &  69 & 8   & 3   & 6  &  29 \\
\hline
\textbf{Total}       & \textbf{3920} & \textbf{2353} & \textbf{302} & \textbf{191} & \textbf{279} & \textbf{795} \\
\hline
\end{array}
\]

\textbf{1)} What is the estimated probability that patient died at the on September the 2nd if use water from Southwark and Vauxhall supplier?

\textbf{\(\qquad\)a:} \(399/3920\); \textbf{\(\qquad\)b:} \(399/100\); \textbf{\(\qquad\)c:} \(399/2353\); \textbf{\(\qquad\)d:} \(399\);

\textbf{2)} John Snow reports a total of 40046 houses supplied by Southwark and Vauxhall Company and 26107 houses by Lambeth Company. Build a contingency table for deaths and no-deaths in households by company. What is the probability of dying in Southwark and Vauxhall? (compare it with that of Lambeth)

\textbf{\(\qquad\)a:} \(0.011\); \textbf{\(\qquad\)b:} \(0.058\); \textbf{\(\qquad\)c:} \(0.035\); \textbf{\(\qquad\)d:} \(0.062\)

\textbf{3)} A diagnostic test has a probability of \(8/9\) of detecting a disease if the patients are sick and a probability of \(3/9\) of detecting the disease if the patients are healthy. If the probability of being sick is \(1/9\). What is the probability that a patient is sick if a test detects the disease?

\textbf{\(\qquad\)a:} \(\frac{8/9}{8/9+3/9}\times1/9\);
\textbf{\(\qquad\)b:} \(\frac{3/9}{8/9+3/9}\times1/9\);
\textbf{\(\qquad\)c:} \(\frac{3/9\times8/9}{8/9\times1/9+3/9\times8/9}\); \textbf{\(\qquad\)d:} \(\frac{8/9\times1/9}{8/9\times1/9+3/9\times8/9}\);

\textbf{4)} As discussed in the notes, a PCR test for coronavirus had a sensitivity of 70\% and a specificity of 94\% and in Spain during confinement there was an incidence of 5\%. With these data, what was the probability of testing positive in Spain (\(P(positive)\))

\textbf{\(\qquad\)a:} \(0.035\); \textbf{\(\qquad\)b:} \(0.092\); \textbf{\(\qquad\)c:} \(0.908\); \textbf{\(\qquad\)d:} \(0.95\)

\textbf{5)} With the same data as in question 4, testing positive in the PCR and being infected are not independent events because:

\textbf{\(\qquad\) a:} Sensitivity is 70\%;
\textbf{\(\qquad\)b:} Sensitivity and false positive rate are different;
\textbf{\(\qquad\)c:} The false positive rate is 0.06\%; \textbf{\(\qquad\)d:} the specificity is 96\%

\hypertarget{exercises-2}{%
\section{Exercises}\label{exercises-2}}

\hypertarget{exercise-1-1}{%
\subsubsection{Exercise 1}\label{exercise-1-1}}

A machine is tested for its performance in producing high-quality turning rods. These are the test results

\[
\begin{array}{ccc}
 & \textbf{Rounded: Yes} & \textbf{Rounded: No} \\
\textbf{Smooth Surface: Yes} & 200 & 1 \\
\textbf{Smooth Surface: No} & 4 & 2 \\
\end{array}
\]

\begin{itemize}
\item
  What is the estimated probability that the machine will produce a rod that does not satisfy any quality control? (A: 2/207)
\item
  What is the estimated probability that the machine will produce a rod that fails at least one quality check? (A: 7/207)
\item
  What is the estimated probability that the machine will produce rods with a rounded and smooth surface? (A: 200/207)
\item
  What is the estimated probability that the bar is rounded if the bar is smooth? (A: 200/201)
\item
  What is the estimated probability that the rod is smooth if it is rounded? (A: 200/204)
\item
  What is the estimated probability that the rod is neither smooth nor rounded if it does not satisfy at least one quality check? (A: 2/7)
\item
  Are smoothness and roundness independent events? (No)
\end{itemize}

\hypertarget{exercise-2-1}{%
\subsubsection{Exercise 2}\label{exercise-2-1}}

Consider a probabilistic version of the quantum bomb tester using a double-slit experiment with a photon \citep{Elitzur1993}. A photon is sent toward a barrier with two slits, and two detectors are placed at positions a and b, as shown in the figure below.

Each time a detector captures the photon, it registers a ``tick.'' Suppose that detector a is faulty, with a probability of being ``on'' of \(4/5\). When a is ``on,'' it detects which slit the photon passed through. According to quantum theory, there are three possible outcomes in the random experiment that is performed under the condition ``a in on'': I) The photon passes through the slit asa particle where a is, and therefore is detected at a, II) The photon passes as a particle through the other slit and is detected at b with, for instance, probability \(2/3\) if not detected at a, III) The photon passes as a particle through the other slit and is not detected at b with, for instance, probability \(1/3\) if not detected at a. Now, if a is ``off,'' the photon passes through both slits as a wave, and interference occurs. In that case, there is only one possible outcome: IV) No detection occurs at b, if we cleverly placed it at a location where, in the absence of which-path detection (i.e., when a is off), destructive interference occurs --- meaning the photon will never be detected at b in that case.

Outcome II is the most curious: the photon is detected at b and, therefore, we know that a was ``on'' --- meaning we obtain information about a without any interaction. Elitzur and Vaidman attached a bomb to detector a to make the effect more dramatic: we may infer that a bomb is ``on'' without even interacting with it. Remarkably, this outcome occurs with nonzero probability.

\begin{itemize}
\item
  What is the probability that detector b ticks? (A:\(4/15\))
\item
  What is the probability that the bomb is ``on'' if the photon was detected at b? (A:\(1\))
\item
  What is the probability that the bomb is ``on'' if the photon was not detected at a (A:\(2/3\))
\end{itemize}

\includegraphics{./figures/quantum.jpg}

\hypertarget{exercise-3-1}{%
\subsubsection{Exercise 3}\label{exercise-3-1}}

We developed a test to detect the presence of bacteria in a lake. We found that if the lake contains the bacteria, the test is positive 70\% of the time. If there are no bacteria, the test is negative 60\% of the time. We implemented the test in a region where we know that 20\% of the lakes have bacteria.

\begin{itemize}
\tightlist
\item
  What is the probability that a lake that tests positive is contaminated with bacteria? (A: 0.30)
\end{itemize}

\hypertarget{exercise-4-1}{%
\subsubsection{Exercise 4}\label{exercise-4-1}}

A quality test on a random brick is defined by the events:

\begin{itemize}
\tightlist
\item
  Pass the quality test: \(E\), fail the quality test: \(E'\)
\item
  Defective: \(D\), non-defective: \(D'\)
\end{itemize}

If the diagnostic test has sensitivity \(P(E|D')= 0.99\) and specificity \(P(E'|D)=0.98\), and the probability of passing the test is \(P(E) =0.893\) then

\begin{itemize}
\item
  What is the probability that a randomly chosen brick is defective \(P(D)\)? (A: 0.1)
\item
  What is the probability that a brick that has passed the test is actually defective? (A: 0.022)
\item
  The probability that a brick is not defective \textbf{and} that it fails the test (A: 0.009)
\item
  Are \(D\) and \(E'\) statistically independent? (No)
\end{itemize}

\hypertarget{practice-2}{%
\section{Practice}\label{practice-2}}

Load misophonia data from \url{https://alejandro-isglobal.github.io/SDA/data/Misophonia.txt}

\begin{itemize}
\item
  Compute the conditional probability table of misophophonia (Misophonia) given marital status (Marital Status). What is the estimated probability of having misophonia if the patient is married?
\item
  Compute the conditional probability table of marital status (Marital Status) given misophonia (Misophonia). What is the estimated probability of being married if the patient is misophonic?
\end{itemize}

\href{https://colab.research.google.com/drive/1ncTaoBgskCJcBIb0-PdnbUFr_AKU0XRF?usp=sharing}{Solutions}

\hypertarget{discrete-random-variables}{%
\chapter{Discrete Random Variables}\label{discrete-random-variables}}

Random experiments are better understood as a two-stage processes. The first stage involves the selection of a subject, the production of a specimen, or the execution of a process, which we refer to as the observation unit. The second stage is the observation or measurement process performed on the observation unit. As a result, we acquire a characteristic or quantity from a single run of the experiment. If we perform more than one type of measurement, the observation becomes the joint result of all the measurements. For instance, the pressure, volume, and temperature of a gas.

Randomness arises in both stages. The production of an observation unit is typically subject to uncontrolled conditions, making each unit not identical to another. The experimental conditions are expected to remain consistent every time we perform the experiment so that the randomness originates from the same sources. However, if the selection of a subject is not genuinely representative of the experimental conditions, the repetition of the experiment becomes biased, and the relative frequencies may fail to converge to the expected probabilities. For example, selecting a depressed patient from a clinic introduces bias when generalizing the misophonia condition to the broader population.

Randomness also arises from measurement error. Even if the production of observation units occurs under identical conditions, additional variation in experimental outcomes may stem from the limitations of the measuring instrument. Consider, for example, measuring the position of a star in the sky using an old sextant. While the observation unit, the star, remains the same and the same sextant is used in each measurement repetition, the outcomes vary due to the instrument's precision.

In this chapter, we focus on the final outcome of a random experiment, specifically the acquisition of a numerical value from an observation unit. We define a random variable as a representation of the entire observation process, encompassing the production of the observation unit, the performance of the measurement, and the acquisition of the final numerical value.

We begin with discrete numerical outcomes, defining the probability mass function along with its main properties, such as mean and variance. Building on the abstraction of relative frequencies into probabilities, we further define the probability distribution as the limiting case of the relative cumulative frequency.

\hypertarget{definition-of-a-random-variable}{%
\section{Definition of a Random Variable}\label{definition-of-a-random-variable}}

A random variable is a symbol that represents \textbf{the process} of acquiring a \textbf{numerical outcome} from the execution of a random experiment. We write the random variable in \textbf{capitals} (i.e.~\(X\)). Think of it as the recipe to obtain an observation from the experiment. It is not the observation, it is the procedure to obtain it. A random variable can be the weight of an animal, that is the process of measuring its mass. \(10.5Kg\) is one observation.

\textbf{Definition:}

A \textbf{random variable} is a function that assigns a real \textbf{number} to an \textbf{event} from the sample space of a random experiment.

Remember than an event can be an outcome or a collection of outcomes.

When the random variable takes a \textbf{value}, it indicates the realization of an \textbf{event} and the production of a number from the random experiment.

\textbf{Example (Switch)}

If \(X \in \{0,1\}\), we then say \(X\) is a random variable that can take the values \(0\) or \(1\). \(X\) may represent the process to determine the state of a switch and encoding it as a number. If we see light in the room, the switch is on and then we assign \(X=1\) to the state of the switch. If we see no light, the switch is off and and the variable takes the value \(X=0\).

\hypertarget{the-value-of-a-random-variable}{%
\section{The value of a random variable}\label{the-value-of-a-random-variable}}

We make the distinction between variables in the model space with capital letters, as abstract entities (the state of a switch, the color of a car, the sex of a patient), and the realization of a particular outcome (whether the light of my office is on right now, the red color of my car, the first patient on the hospital today was male). For instance, we say that

\begin{itemize}
\item
  \(X=1\) is the \textbf{event} of observing the random variable \(X\) with the value of \(1\)
\item
  \(X=2\) is the \textbf{event} of observing the random variable \(X\) with the value of \(2\)
\item
  \ldots{}
\end{itemize}

\textbf{In general:}

\begin{itemize}
\tightlist
\item
  \(X=x\) is the \textbf{event} of observing the random variable \(X\) (big \(X\)) with value \(x\) (little \(x\)).
\end{itemize}

We use lower case letters to indicate the numerical \textbf{outcome} of an experiment that is obtained one day in the lab.

\hypertarget{probability-of-random-variables}{%
\section{Probability of random variables}\label{probability-of-random-variables}}

We are interested in assigning probabilities to the values that random variable can take.

For instance, for the dice, we will write the probability table as

\[
\begin{array}{cc}
\mathbf{x} & \mathbf{P_i} \\
1 & P(X=1) = 1/6 \\
2 & P(X=2) = 1/6 \\
3 & P(X=3) = 1/6 \\
4 & P(X=4) = 1/6 \\
5 & P(X=5) = 1/6 \\
6 & P(X=6) = 1/6 \\
\end{array}
\]

where we explicitly write down the events that the variable takes at given outcome value \(X=x\).

If \(X\) is observing the number after rolling a dice, then \(P(X=1)\) is the probability that the roll of a dice gives an outcome value of \(1\).

\hypertarget{probability-functions}{%
\section{Probability functions}\label{probability-functions}}

Because (little) \(x\) is a numerical quantity, the probabilities of the random variable can be plotted against the outcomes \(x\)

\includegraphics{_main_files/figure-latex/unnamed-chunk-58-1.pdf}

or written as the mathematical function

\[f(x)=P(X=x)=1/6\]
Therefore, the probabilities of a random experiment can be given in table, in a plot or as a \textbf{function}.

\hypertarget{probability-mass-functions}{%
\section{Probability mass functions}\label{probability-mass-functions}}

We can \textbf{create} any type of probability function if we satisfy Kolmogorov's probability rules.

For a discrete random variable \(X \in \{x_1 , x_2 , .. , x_m\}\), a \textbf{probability mass function} that is used to compute probabilities

\begin{itemize}
\tightlist
\item
  \(f(x_i)=P(X=x_i)\)
\end{itemize}

is always positive

\begin{itemize}
\tightlist
\item
  \(f(x_i)\geq 0\)
\end{itemize}

and its sum over all the values of the variable is \(1\):

\begin{itemize}
\tightlist
\item
  \(\sum_{i=1}^m f(x_i)=1\)
\end{itemize}

Where \(m\) is the number of possible outcomes.

Note that the definition of \(X\) and its probability mass function is general \textbf{without reference} to any experiment. The functions live in the model (abstract) space.

\textbf{Example (Physical simulator)}

In one urn put \(8\) balls following the instructions:

\begin{itemize}
\tightlist
\item
  mark \(1\) ball with number \(-2\)
\item
  mark \(2\) balls with number \(-1\)
\item
  mark \(2\) balls with number \(0\)
\item
  mark \(2\) balls with number \(1\)
\item
  mark \(1\) ball with number \(2\)
\end{itemize}

And consider performing the following random \textbf{experiment:} Take one ball and read the number.

From the classical probability, we can write the probability table, for which we do not need to run any experiment

\[
\begin{array}{cc}
\mathbf{x} & f(x)=P(X=x) \\
-2 & 1/8 \\
-1 & 2/8\\
0 & 2/8 \\
1 & 2/8 \\
2 & 1/8 \\
\end{array}
\]

The plot is

\includegraphics{_main_files/figure-latex/unnamed-chunk-59-1.pdf}

\(X\) and \(f(x)\) are mathematical objects that may or may not describe the probabilities of real random experiments. We have the freedom to construct them as we want as long as we respect their definition.

The urn experiment is an important concept. It is a simple and flexible random experiment that allows us to recreate experiments almost any kind of probability mass function. We refer to it as a physical simulator. By adjusting the number of balls, we can match the propensity of an outcome to a given precision. For example, if the probability of an outcome ``black'' is \(3/5 = 0.6\), then placing \(3\) black balls in an urn of \(5\) total balls achieves a perfect match. However, if the probability is \(2/\pi \approx 0.6366198\), a reasonable approximation could involve placing \(636\) black balls in a total of \(1000\). Clearly balls can be labeled with more than one outcome type (``black'' and ``spotted''), recreating the measurement of two different quantities on an observation unit.

A key feature of simulators is that the probabilities of the experiment are determined by its design. Once we conceive the simulator, there is no need to run the experiment to know the propensities of the outcomes. The usefulness of the simulator is to recreate the observations of the experiment. If the probabilities of the outcomes are known, the simulator allows us to inspect the likely observations of a single run or suitable finite runs of the random experiment. Clearly, a simulator is apt if we run it a large number of times (replacing each time the extracted ball back in the urn) and find that the relative frequencies it produces are close to the probabilities on which it is defined.

The simulator works from the abstract to the concrete. An essential point, however, is that only physical simulators can recreate true randomness. The production of randomness is a feature of the physical world. Mathematical functions, on the other hand, cannot produce true randomness. Instead, we use computing software to create simulators based on complex mathematical functions whose outcomes are difficult to predict. These are pseudo-random simulators, which effectively mimic a real physical simulator when the generating functions are sufficiently obscured.

\hypertarget{mean-or-expected-value}{%
\section{Mean or expected value}\label{mean-or-expected-value}}

Le us start again from a random experiment. From its relative frequencies, we defined the central tendency (average), dispersion (sample variance) and the frequency distribution (\(F_i\)) the experiment \textbf{data}. How are these quantities defined for probabilities, understood as frequencies at \(n=\infty\)?.

For a discrete random variable, we have
\[lim_{n\rightarrow \infty} f_i=f(x_i)\]
as \(f(x_i)=P_i\). The relative frequencies thus approach to the probability mass function. If we think in terms of plots, then the heights of bar plot (grey buildings) approach to the heights of the probability mass function (red needles), as we collect more data. For the urn (pseudo-random) simulator example above, we observe that the frequencies for four simulations with \(n=1000\) draws of the balls are consistently closer to the probabilities than the simulations with \(n=30\) draws.

\includegraphics{_main_files/figure-latex/unnamed-chunk-61-1.pdf}

When we discussed summary statistics of data, we defined the \textbf{center} of the observations as a value around which the observations are concentrated. We used the \textbf{average} to measure the center of mass of the \textbf{data}. The average can be computed from the relative frequencies for discrete random variables as

\(\bar{x}= \frac{1}{n} \sum_{i=1}^n x_i = \sum_{i=1}^m x_i \frac{n_i}{n}=\) \[\sum_{i=1}^m x_i f_i\]

Note the change from the number of observations \(n\) to the number of outcomes \(m\) in the summation.

\textbf{Definition}

The \textbf{mean} (\(\mu\)) or expected value of a discrete random variable \(X\), \(E(X)\), with mass function \(f(x)\) is given by

\[ \mu = E(X)= \sum_{i=1}^m x_i f(x_i) \]

\includegraphics{_main_files/figure-latex/unnamed-chunk-62-1.pdf}

It is the center of mass of the \textbf{probabilities}: The point where the probability loading on a road are balanced.

From the definition we have

\[\bar{x} \rightarrow \mu\] in the \textbf{limit} when
\(n \rightarrow \infty\) as the frequency tends to the probability mass function \(f_i \rightarrow f(x_i)\). Note that the expected value is an abstract property of the random experiment. If the probabilities of the random variable are known for all its values, then \(\mu\) is fully determined with no need to run the experiment.

While the probabilities are the propensity of the outcomes, the mean is the value at which the outcomes will gather around. Note that the expression ``expected value'' is apt, as it is a statement about the future run of the experiment. In the next run of the random experiment, we expect the observation to be close to \(\mu\).

\textbf{Example}

Find the mean of \(X\) if its probability mass function \(f(x)\) is given by

\[
\begin{array}{cc}
\mathbf{x} & f(x) \\
0 & 1/16 \\
1 & 4/16 \\
2 & 6/16 \\
3 & 4/16 \\
4 & 1/16 \\
\end{array}
\]

\includegraphics{_main_files/figure-latex/unnamed-chunk-64-1.pdf}

\[ \mu =E(X)=\sum_{i=1}^m x_i f(x_i) \]

\(E(X)=0 \times \frac{1}{16} + 1 \times \frac{4}{16} + 2 \times \frac{6}{16} + 3 \times \frac{4}{16} + 4 \times \frac{1}{16} =2\)

The \textbf{mean} \(\mu\) is the center of mass of the probability mass function. \textbf{It does not change}. It is a property of the random experiment. However, the \textbf{average} \(\bar{x}\) is the center of mass of the observations (relative frequencies). It \textbf{changes} with different data. While the mean is the value at which the observations \textbf{will gather}, the average is the value at which the observations \textbf{gathered} in the past.

\hypertarget{variance}{%
\section{Variance}\label{variance}}

While we expect the observations to gather about \(\mu\), we also expect them to distance from it due to their random behavior. When we discussed summary statistics of data, we defined the spread of the observations as an average distance from the data average.

\textbf{Definition}

The variance, written as \(\sigma^2\) or \(V(X)\), of a discrete random variable \(X\) with mass function \(f(x)\) is given by

\[\sigma^2 = V(X)= \sum_{i=1}^m (x_i-\mu)^2 f(x_i)\]
\(\sigma=\sqrt{V(X)}\) is called the \textbf{standard deviation} of the random variable and it is the expected distance from the mean of the next observation of the random experiment.

The variance is the spread of the \textbf{probabilities} about the mean. That is the moment of inertia of probability loadings about the center of mass.

\textbf{Example}

What is the variance of \(X\) if its probability mass function \(f(x)\) is given by the previous example?

\[
\begin{array}{cc}
\mathbf{x} & f(x) \\
0 & 1/16 \\
1 & 4/16 \\
2 & 6/16 \\
3 & 4/16 \\
4 & 1/16 \\
\end{array}
\]

\[\sigma^2 =V(X)=\sum_{i=1}^m (x_i-\mu)^2 f(x_i)\]

\(V(X)=(0-2)^2 \times \frac{1}{16} + (1-0)^2 \times \frac{4}{16} + (2-2)^2 \times \frac{6}{16} + (3-2)^2 \times \frac{4}{16} + (4-2)^2 \times \frac{1}{16} =1\)

\[V(X)=\sigma^2=1\]
\[\sigma=1\]

\hypertarget{probability-functions-for-functions-of-x}{%
\section{\texorpdfstring{Probability functions for functions of \(X\)}{Probability functions for functions of X}}\label{probability-functions-for-functions-of-x}}

In many occasions, we will be interested in outcomes that are function of the random variables. Perhaps, we are interested in the square of the number of flu infections, or on the square root of the number of emails in an hour.

\textbf{Definition}

For any function \(h\) of a random variable \(X\), with mass function \(f(x)\), its expected value is given by

\[ E[h(X)]= \sum_{i=1}^M h(x_i) f(x_i) \]

This is an important definition that allows us to prove three frequently used properties of the mean and variance:

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\item
  The mean of a linear function is the linear function fo the mean: \[E(a\times X +b)= a\times E(X) +b\] for \(a\) and \(b\) scalars (numbers).
\item
  The variance of a linear function of \(X\) is:\[V(a\times X +b)= a^2\times V(X)\]
\item
  The variance \textbf{about the origin} is the variance \textbf{about the mean} plus the mean squared: \[E(X^2)=E(X)^2 + V(X)\]
\end{enumerate}

The last equation is fundamental in mathematical statistics. It is the Pythagoras theorem for statistics where the expected total variation of the data from \(0\) (hypotenuse=\(E(X^2)=E[(X-0)^2]\)) is the sum of the variation \textbf{of} the mean (\(E(X)^2\)) and the variation \textbf{about} the mean (\(V(X)=E[(X-\mu)^2]\)). These last two variations are orthogonal, introducing important geometrical properties into statistical analysis. For more practical applications, the property is useful to compute the variance of a random variable.

\textbf{Example}

What is the variation of \(X\) about the origin, \(E(X^2)\), if its probability mass function \(f(x)\) is given by

\[
\begin{array}{ccc}
\mathbf{x} &\mathbf{x^2} & f(x) \\
0 & 0 &1/16 \\
1 & 1 &4/16 \\
2 & 4 &6/16 \\
3 & 9 &4/16 \\
4 & 16 &1/16 \\
\end{array}
\]

\[E(X^2) =\sum_{i=1}^m x_i^2 f(x_i)\]

\(E(X^2)=0^2 \times \frac{1}{16} + 1^2 \times \frac{4}{16} + 2^2 \times \frac{6}{16} + 3^2 \times \frac{4}{16} + 4^2 \times \frac{1}{16} =5\)

We can also verify:

\[E(X^2)=V(X)+E(X)^2\]

\(5=1+2^2\)

\hypertarget{probability-distribution}{%
\section{Probability distribution}\label{probability-distribution}}

When we discussed summary statistics of data, we also defined the frequency distribution (or the relative cumulative frequency) \(F_x\). \(F_x\) is an important quantity because it is a function on the continuous range of \(x\), even if the outcomes are discrete.

\textbf{Definition:}

The \textbf{probability distribution} function of a random variable \(X\) is defined as

\[F(x)=P(X\leq x)=\sum_{x_i\leq x} f(x_i) \]

That is the accumulated probability up to a given value \(x\)

Therefore, \(F(x)\)

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\tightlist
\item
  is bounded: \(0\leq F(x) \leq 1\); and
\item
  always increases: If \(x \leq y\), then \(F(x) \leq F(y)\)
\end{enumerate}

\textbf{Example}

For the probability mass function:

\[
\begin{array}{cc}
\mathbf{x} & f(x) \\
0 & 1/16 \\
1 & 4/16 \\
2 & 6/16 \\
3 & 4/16 \\
4 & 1/16 \\
\end{array}
\]

The probability distribution is:

\[
    F(x)=
\begin{cases}
    0, & x \leq 0\\
    1/16,& 0 \leq x < 1\\
    5/16,& 1\leq x < 2\\
    11/16,& 2\leq x < 3\\
    15/16,& 3 \leq x < 4\\
    16/16,&   4 \leq x\\
\end{cases}
\]

For\(X \in \mathbb{R}\)

\includegraphics{_main_files/figure-latex/unnamed-chunk-66-1.pdf}

\hypertarget{probability-function-and-probability-distribution}{%
\section{Probability function and probability distribution}\label{probability-function-and-probability-distribution}}

The probability function and distribution are equivalent. They encode the same information. We can get one from the other and vice-versa

\[f(x_i)=F(x_i)-F(x_{i-1})\]

with

\[f(x_1)=F(x_1)\]

for \(X\) taking values in \(x_1 \leq x_2 \leq ... \leq x_n\)

\textbf{Example}

From probability distribution:

\[
    F(x)=
\begin{cases}
    1/16,& \text{if } 0 \leq x < 1\\
    5/16,& 1\leq x < 2\\
    11/16,& 2\leq x < 3\\
    15/16,& 4\leq x < 5\\
    16/16,&  x \leq 5\\
\end{cases}
\]

We can obtain the probability mas function.

\(f(0)=F(0)=1/16\)

\(f(1)=F(1)-f(0)=5/32-1/32=4/16\)

\(f(2)=F(2)-f(1)-f(0)=F(2)-F(1)=6/16\)

\(f(3)=F(3)-f(2)-f(1)-f(0)=F(3)-F(2)=4/16\)

\(f(4)=F(4)-F(3)=1/16\)

\hypertarget{quantiles}{%
\section{Quantiles}\label{quantiles}}

Finally, we can use the probability distribution \(F(x)\) to define the median and the quartiles of the random variable \(X\).

In general, we define the \textbf{q-quantile} as the value \(x_{q}\) \textbf{under} which we have accumulated \(q\times 100\%\) of the probability

\[q=\sum_{x_i\leq x_q} f(x_i) = F (x_q)\]

\begin{itemize}
\tightlist
\item
  The \textbf{median} is the value \(x_{0.5}\) such that \(q=0.5\)
\end{itemize}

\[F(x_{0.5})=0.5\]
or

\[x_{0.5}=F^{-1}(0.5)\]

\begin{itemize}
\tightlist
\item
  The \(0.05\)-quantile is the value \(x_{0.05}\) such that \(q=0.05\)
\end{itemize}

\[x_{0.05}=F^{-1}(0.05)\]

\begin{itemize}
\tightlist
\item
  The \(0.25\)-quantile is \textbf{first quartile} the value \(x_{0.25}\) such that \(q=0.25\)
\end{itemize}

\[x_{0.25}=F^{-1}(0.25)\]
In the plots for the probability distribution \(F(X)\), the median is the distance \(x\) at which \(F(x)\) has gone up \(50\%\) of the height. And the first quartile is the distance at which \(F(x)\) has gone up \(25\%\) of the height.

\hypertarget{summary}{%
\section{Summary}\label{summary}}

This is a graphical summary of the correspondence between abstract quantities in the model space, defined on the outcomes, and observed quantities in the data space, defined on the observations.

\includegraphics{./figures/randomvarsum.JPG}

\[
\begin{array}{|c|c|}
\hline
 \textbf{Data (Observed)} & \textbf{Model (Abstract)}  \\ 
\hline
 f_i=\frac{n_i}{n} &  f(x_i)=P(X=x_i) \\ 
\hline
 F_i=\sum_{k \leq i} f_k &  F(x_i)=P(X \leq x_i) \\ 
\hline
  \bar{x}=\frac{\sum_{j=1}^n x_j}{n} & \mu= \sum_{i=1}^m x_i f(x_i)\\ 
\hline
 s^2=\frac{\sum_{j=1}^n (x_j-\bar{x})^2}{N-1} & \sigma^2=\sum_{i=1}^m (x_i-\mu)^2 f(x_i) \\ 
\hline
 s & \sigma \\ 
\hline
 m_2=\frac{\sum_{j=1}^n x_j^2}{n} & E(X^2)=\sum_{i=1}^m x_i^2 f(x_i)  \\ 
\hline
\end{array}
\]

Note that:

\begin{itemize}
\tightlist
\item
  \(j=1...n\) index the \textbf{observations} of the random variable \(X\)
\item
  \(i=1...m\) index the \textbf{outcomes} of the random variable \(X\).
\end{itemize}

Here is a useful list of properties

\begin{itemize}
\tightlist
\item
  \(\sum_{i=1}^m f(x_i)=1\)
\item
  \(f(x_i)=F(x_i)-F(x_{i-1})\)
\item
  \(E(a\times X +b)= a\times E(X) +b\); for \(a\) and \(b\) scalars.
\item
  \(V(a\times X +b)= a^2\times V(X)\)
\item
  \(E(X^2)=V(X)+E(X)^2\)
\end{itemize}

\hypertarget{questions-3}{%
\section{Questions}\label{questions-3}}

\textbf{1)} For a probability mass function is not true that

\textbf{\(\qquad\)a:} the addition of their image values is 1; \textbf{\(\qquad\)b:} its values can be interpreted as probabilities of events;
\textbf{\(\qquad\)c:} it is always positive;
\textbf{\(\qquad\)d:} cannot take value 1;

\textbf{2)} A value of a random variable is

\textbf{\(\qquad\)a:} an observation of a random experiment; \textbf{\(\qquad\)b:} the frequency of an outcome of a random experiment;
\textbf{\(\qquad\)c:} an outcome of a random experiment;
\textbf{\(\qquad\)d:} a probability of an outcome;

\textbf{3)} The estimated value of a probability \(\hat{P_i}\) is equal to the probability \(P_i\) when the number of repetitions of the random experiment is

\textbf{\(\qquad\)a:} large; \textbf{\(\qquad\)b:} infinite;
\textbf{\(\qquad\)c:} small
\textbf{\(\qquad\)d:} zero;

\textbf{4)} If a probability mass function is symmetric around \(x=0\)

\textbf{\(\qquad\)a:} The mean is lower than the median; \textbf{\(\qquad\)b:} The mean is greater than the median;
\textbf{\(\qquad\)c:} The mean and the median are equal;
\textbf{\(\qquad\)d:} The mean and the median are different from 0;

\textbf{5)} The mean and variance

\textbf{\(\qquad\)a:} are inversely proportional; \textbf{\(\qquad\)b:} are expected values of functions of \(X\);
\textbf{\(\qquad\)c:} of a linear function are the linear function of the mean and the linear function of the variance;
\textbf{\(\qquad\)d:} change when we repeat the random experiment;

\hypertarget{exercises-3}{%
\section{Exercises}\label{exercises-3}}

\hypertarget{exercise-1-2}{%
\subsubsection{Exercise 1}\label{exercise-1-2}}

Consider the following random variable \(X\) over the outcomes

\[
\begin{array}{cc}
\textbf{Outcome} & x \\
a & 0 \\
b & 0 \\
c & 1.5 \\
d & 1.5 \\
e & 2 \\
f & 3 \\
\end{array}
\]

\begin{enumerate}
\def\labelenumi{\alph{enumi})}
\item
  If each outcome is equally probable then what is the probability mass function of \(x\)?
\item
  Find:
\end{enumerate}

\begin{itemize}
\tightlist
\item
  \(P(X>3)\)
\item
  \(P(X=0\, \cup \, X=2 )\)
\item
  \(P(X \leq 2)\)
\end{itemize}

\hypertarget{exercise-2-2}{%
\subsubsection{Exercise 2}\label{exercise-2-2}}

Given the probability mass function

\[
\begin{array}{cc}
x & f(x) \\ 
10 & 0.1 \\ 
12 & 0.3 \\ 
14 & 0.25 \\ 
15 & 0.15 \\ 
17 & ? \\ 
20 & 0.15 \\ 
\end{array}
\]

\begin{itemize}
\tightlist
\item
  what is its expected value and standard deviation? (A: 14.2; 2.95)
\end{itemize}

\hypertarget{exercise-3-2}{%
\subsubsection{Exercise 3}\label{exercise-3-2}}

Given the probability distribution for a discrete variable \(X\)

\[
    F(x)= 
\begin{cases}
0, & x < -1 \\
0.2,& x \in [-1,0)\\
0.35,& x \in [0,1)\\
0.45,& x \in [1,2)\\
1,& x \geq 2\\
\end{cases}
\]

\begin{itemize}
\tightlist
\item
  find \(f(x)\)
\item
  find \(E(X)\) and \(V(X)\) (A:1; 1.5)
\item
  what is the expected value and variance of \(Y=2X+3\) (A: 6)
\item
  what is the median and the first and third quartiles of \(X\)? (A:2,0,2)
\end{itemize}

\hypertarget{exercise-4-2}{%
\subsubsection{Exercise 4}\label{exercise-4-2}}

We are testing a system to transmit digital pictures. We first consider the experiment of sending \(3\) pixels and having as \textbf{possible} outcomes events such like \((0,1,1)\). This is the event of receiving the first pixel with no error, the second with error and third with error.

\begin{itemize}
\item
  List in one column the sample space of the random experiment.
\item
  In the a second column assign the random variable that counts the number of errors transmitted for each outcome
\end{itemize}

Consider that we have a totally noisy channel, that is any outcome of three pixels is equally likely.

\begin{itemize}
\item
  What is the probability of receiving \(0\), \(1\), \(2\), or \(3\) errors in the transmission of \(3\) pixels? (A: 1/8; 3/8; 3/8; 1/8)
\item
  Sketch the probability mass function for the number of errors
\item
  What is the expected value for the number of errors? (A:1.5)
\item
  What is its variance? (A: 0.75)
\item
  Sketch the probability distribution
\item
  What is the probability of transmitting at least 1 error? (A:7/8)
\end{itemize}

\hypertarget{continous-random-variables}{%
\chapter{Continous Random Variables}\label{continous-random-variables}}

Most experiments involve the measurement of continuous quantities. Probabilities associated with continuous outcomes require special treatment. While continuous observations are constrained by the precision of the instrument, the outcomes themselves are not limited by experimental constraints. Instead, they represent the possible values a random variable can take, as if determined by an instrument with infinite precision. Continuous random variables can take values with any number of decimal places---even infinitely many. Moreover, these digits may be irregular but computable, allowing for outcomes such as
\(\pi\) or \(\sqrt{2}\).

In practice, experiments with infinite precision are impossible. Real experiments introduce randomness, either through measurement error or in the process of generating the observation unit. Small changes in the conditions can generate large outcome differences. As a result, decimal digits will inevitably become blurred beyond a certain point. The extent of this blurring depends on the limitations imposed by technology or nature, which set the boundaries for how finely we can measure.

If randomness is allowed to occur at any level of precision due to differences in observation methods, continuous outcomes can be viewed as representing the ultimate scale: the real line.

In this chapter, we will study continuous random variables. Continuity is addressed through calculus. Although we will not formally define derivatives or integrals, we will use these tools to generalize the concept of probability and provide a solid foundation for understanding random experiments with continuous outcomes. Specifically, we will define the probability density function, along with its mean and variance. Additionally, similar to discrete random variables, we will introduce the cumulative distribution function.

\hypertarget{probabilities-of-continuous-random-variables}{%
\section{Probabilities of continuous random variables}\label{probabilities-of-continuous-random-variables}}

When we talked about continuous data, we saw that we had to transform them into discrete variables (bins) to produce relative frequency tables or histograms. Let us see how to define the probabilities of continuous variables taking these partitions into account.

\textbf{Example (misophonia)}

Let us reconsider the angle of convexity of patients with misophonia (Section 2.21). The angle of convexity of 123 patients was measured. We understood each measurement as the result of a random experiment that we repeated 123 times and that we could describe in a frequency table or in a histogram.

To do this, we redefine the results as small regular intervals (bins) and calculate the relative frequency of each interval.

\[
\begin{array}{ccc}
\mathbf{outcome} & \mathbf{n_i} & \mathbf{f_i} \\
\mathrm{[-1.02,3.46]}   &  8  & 0.06504065 \\
\mathrm{(3.46,7.92]}    & 51  & 0.41463415 \\
\mathrm{(7.92,12.4]}    & 26  & 0.21138211 \\
\mathrm{(12.4,16.8]}    & 20  & 0.16260163 \\
\mathrm{(16.8,21.3]}    & 18  & 0.14634146 \\ \hline
\mathbf{sum}   & 123 & 1
\end{array}
\]

\hypertarget{relative-frequencies-2}{%
\section{Relative frequencies}\label{relative-frequencies-2}}

Following the frequentist definition of probability. The relative frequency of bin \((x_i, x_i + \Delta x)\) when \(n \rightarrow \infty\) is the probability that the random variable takes a value between \(x_i\) and \(x_i + \Delta x\)

\[lim_{n\rightarrow\infty}f_i=lim_{n\rightarrow\infty}\frac{n_i}{n} = P(x_i \leq X  \leq x_i + \Delta x)\]

The probability depends now on the length of the bins \(\Delta x\). If we make the bins smaller and smaller then the frequencies get smaller and smaller because it is unlikely to find any observation is such a small bin; that is, \(n_i \rightarrow 0\). As a consequence, the probability also vanishes

\[P(x_i \leq X  \leq x_i + \Delta x) \rightarrow 0\] when \(\Delta x \rightarrow 0\).

We now see how the frequencies get smaller when we divide the range of \(X\) into \(20\) bins

\[
\begin{array}{ccc}
\mathbf{outcome} & \mathbf{n_i} & \mathbf{f_i} \\
\mathrm{[-1.02,0.115]}   &  2 & 0.01626016 \\
\mathrm{(0.115,1.23]}    &  0 & 0.00000000 \\
\mathrm{(1.23,2.34]}     &  3 & 0.02439024 \\
\mathrm{(2.34,3.46]}     &  3 & 0.02439024 \\
\mathrm{(3.46,4.58]}     &  2 & 0.01626016 \\
\mathrm{(4.58,5.69]}     &  4 & 0.03252033 \\
\mathrm{(5.69,6.8]}      & 11 & 0.08943089 \\
\mathrm{(6.8,7.92]}      & 34 & 0.27642276 \\
\mathrm{(7.92,9.04]}     & 12 & 0.09756098 \\
\mathrm{(9.04,10.2]}     &  4 & 0.03252033 \\
\mathrm{(10.2,11.3]}     &  3 & 0.02439024 \\
\mathrm{(11.3,12.4]}     &  7 & 0.05691057 \\
\mathrm{(12.4,13.5]}     &  2 & 0.01626016 \\
\mathrm{(13.5,14.6]}     &  6 & 0.04878049 \\
\mathrm{(14.6,15.7]}     &  4 & 0.03252033 \\
\mathrm{(15.7,16.8]}     &  8 & 0.06504065 \\
\mathrm{(16.8,18]}       &  4 & 0.03252033 \\
\mathrm{(18,19.1]}       &  9 & 0.07317073 \\
\mathrm{(19.1,20.2]}     &  3 & 0.02439024 \\
\mathrm{(20.2,21.3]}     &  2 & 0.01626016 \\ \hline
\mathbf{sum}    & 123 & 1
\end{array}
\]

Let us just go to the limit and take \(\Delta =0\) then \(P(x_i \leq X \leq x_i)=P(X=x_i)=0\). This means that probability that a continuous random variable takes one single value is \(0\). While this may sound puzzling, think of the continuous real line. Imagine a random variable with possible values between \(0\) and \(5\). What is the chance that the random experiment gives the exact value of \(\pi\), with its infinite decimal places?

\hypertarget{probability-density-function}{%
\section{Probability Density Function}\label{probability-density-function}}

We define a quantity at a point \(x\) that is the amount of probability per unit distance that we would find in an \textbf{infinitesimal} bin \(dx\) at \(x\)

\[f(x)= \frac{P(x\leq X \leq x+dx)}{dx}\]

\(f(x)\) is called the probability \textbf{density} function. Density functions are how we deal with continuity. When we make the bin really small \(dx\) is small and the probability also. The ratio of these two small values, however, is constant at a given \(x\). What is the amount of mass in a particular point?. Zero. But the mass density at that point takes a finite value that can be different from zero. We can talk about probability densities as we talk about mass densities. The probability density at one point takes a finite value that can be different from zero.

Therefore, the probability that the random variable \(X\) takes a value between \(x\) and \(x+dx\)
is given by

\[P(x\leq X \leq x+dx)= f(x) dx\]

\textbf{Definition}

For a continuous random variable \(X\), a \textbf{probability density} function is such that

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\tightlist
\item
  The function is positive:
\end{enumerate}

\[f(x) \geq 0\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{1}
\tightlist
\item
  The probability that \(X\) takes \textbf{any} value is 1:
\end{enumerate}

\[\int_{-\infty}^{\infty} f(x) dx = 1\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{2}
\tightlist
\item
  The probability that \(X\) is within an interval is the \textbf{area under the curve}:
\end{enumerate}

\[P(a\leq X \leq b)=\int_{a}^{b} f(x) dx\]

The properties make sure that \(f(x)dx\) satisfies Kolmogorov's axioms of a probability measure.

The probability density function is a step forward in the abstraction of probabilities: we add the continuous limit

\[dx \rightarrow 0\]
to the already limit of infinite repetitions \(n \rightarrow \infty\) of the random experiment.

All the properties of probabilities are translated in terms of densities. From descrete random variables, we change summaions by integrations

\[\sum \rightarrow \int\]
and we change point probabilities by probabilities within an interval \[P(X=x_i)=f(x_i) \rightarrow P(x\leq X \leq x+dx)= f(x)dx\]

Similar to probability mass functions for discrete variables, probability densities are mathematical quantities that do not necessarily represent random experiments. We are free to define them as long as we respect their properties.

However, a fundamental interest in statistics is to select the densities that better describe our particular random experiment.

\hypertarget{total-area-under-the-curve}{%
\section{Total area under the curve}\label{total-area-under-the-curve}}

\textbf{Example (Needle drop)}

Consider the following random experiment: Draw a line in a sheet of paper, hold a needle vertical to the paper and let it drop. Measure the angle at which the needle drops with respect to the line.

Different angles are derived from very small variations in the conditions of the experiment. Infinite small variations from the vertical line can give large differences in the angles. This random variation of external conditions is one of the ways that randomness can arise in an experiment. Without performing the experiment, we have no reason to believe that a range of angles is more likely that another, when we aim to let it drop from the vertical line. Therefore the \textbf{probability density} is constant for every angle \(X\) is given by

\[
    f(x)= 
\begin{cases}
    \frac{1}{360},& \text{if } x\in (0,360)\\
    0,& otherwise 
\end{cases}
\]

Let us verify that the function satisfies the three properties of a probability density.

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\item
  It is evident from the definition that \(f(x) \geq 0\)
\item
  The probability of observing \textbf{any} angle; that is, that \(X\) takes any value, is summing up all the possible probabilities. That is the total \textbf{area under the curve}
\end{enumerate}

\[P(-\infty\leq X \leq \infty)= \int_{-\infty}^{\infty} f(x) dx  = 360\times \frac{1}{360}= 1\]

\includegraphics{_main_files/figure-latex/unnamed-chunk-69-1.pdf}

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{2}
\tightlist
\item
  The probability that \(X\) takes a value between \(90\) and \(180\) degrees is \textbf{area under the curve} within the interval
\end{enumerate}

\[P(90 \leq X \leq 180) = \int_{90}^{180} f(x) dx = (180-90)\times \frac{1}{360}=0.25\]

\includegraphics{_main_files/figure-latex/unnamed-chunk-71-1.pdf}

\hypertarget{probabilities-of-continous-variables}{%
\section{Probabilities of continous variables}\label{probabilities-of-continous-variables}}

For continuous variables, we compute the probability that the variable is between \(a\) and \(b\). That is

\[P(a \leq X \leq b)\]
We saw that for continuous variables, the probability that the experiment gives us a particular real number is zero: \(P(X=a)=0\). The probability \(P(a \leq X \leq b)\) is the area under the curve of \(f(x)\) between \(a\) and \(b\)

\[P(a \leq X \leq b) = \int_{a}^{b} f(x) dx\]

\includegraphics{_main_files/figure-latex/unnamed-chunk-74-1.pdf}

\hypertarget{probability-distribution-1}{%
\section{Probability distribution}\label{probability-distribution-1}}

The \textbf{probability distribution} \(F(c)\) is defined as the accumulation of probability up to the outcome \(c\)

\[F(c) = P(X \leq c)\]

That is the probability that the random variable is less or equal to \(c\). We can use \(F(c)\) to compute \(P(a \leq X \leq b)\).

Consider that:

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\tightlist
\item
  The probability accumulated up to \(b\) is given by
\end{enumerate}

\[F(b) = P(X \leq b)=\int_{-\infty}^bf(x)dx\]

\includegraphics{_main_files/figure-latex/unnamed-chunk-76-1.pdf}

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{1}
\tightlist
\item
  That the probability accumulated up to \(a\) is
\end{enumerate}

\[F(a) = P(X \leq a)\]

\includegraphics{_main_files/figure-latex/unnamed-chunk-78-1.pdf}

Then the probability that the random variable is between \(a\) and \(b\) is given by the difference in the probability distribution between \(a\) and \(b\)

\[P(a\leq X \leq b) = \int_a^b f(x)dx=F(b)-F(a)\]

\includegraphics{_main_files/figure-latex/unnamed-chunk-80-1.pdf}

\textbf{Definition}

The \textbf{probability distribution} of a continuous random variable is defined as the area under the curve up to \(a\)

\[F(a)=P(X\leq a) =\int_{-\infty} ^a f(x)dx\]

\(F(a)\) has the properties:

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\tightlist
\item
  It is between \(0\) and \(1\):
\end{enumerate}

\[F(-\infty)= 0 \,\, and \,\,F(\infty)=1\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{1}
\tightlist
\item
  It always increases:
\end{enumerate}

\[F(a)\leq F(b), \qquad \text{if} \qquad a\leq b\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{2}
\tightlist
\item
  It can be used to compute probabilities:
\end{enumerate}

\[P(a \leq X \leq b)=F(b)-F(a)\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{3}
\tightlist
\item
  It recovers the probability density:
\end{enumerate}

\[f(x)=\frac{dF(x)}{dx}\]

We use \textbf{probability distributions} to \textbf{compute probabilities} of a random variable within intervals. \(F(x)\) is derivative is the probability density function \(f(x)\).

\textbf{Example (Needle drop)}

For the uniform density function:

\[
    f(x)= 
\begin{cases}
    \frac{1}{360},& \text{if } x\in (0,360)\\
    0,& otherwise 
\end{cases}
\]

We find that the probability distribution is

\[
    F(x)= 
\begin{cases}
    0,& x \leq 0 \\
    \frac{x}{360},& \text{if } x\in (0,360)\\
    1, & 360 \leq x \\
    \\
\end{cases}
\]

\newpage

\hypertarget{probability-plots}{%
\section{Probability plots}\label{probability-plots}}

We can plot the the probability of a random variable in an interval as the \emph{area} under the \textbf{density} curve. For instance

\[P(90\leq X \leq 180)\]

\includegraphics{_main_files/figure-latex/unnamed-chunk-82-1.pdf}

Or, equivalently, we can plot the probability \(P(90 \leq X \leq 180)\) as the \emph{difference} in \textbf{distribution} values

\includegraphics{_main_files/figure-latex/unnamed-chunk-84-1.pdf}

\hypertarget{mean}{%
\section{Mean}\label{mean}}

As in the discrete case, the mean is the center of mass of probabilities and tells us the value around which we can expect the next observation of the random experiment.

\textbf{Definition}

Suppose \(X\) is a continuous random variable with probability \textbf{density} function \(f(x)\). The mean or expected value of \(X\), denoted as \(\mu\) or \(E(X)\), is

\[\mu=E(X)=\int_{-\infty}^\infty x f(x) dx\]

It is the continuous version of the center of mass.

\textbf{Example (Needle drop)}

The random variable with probability density

\[
    f(x)= 
\begin{cases}
    \frac{1}{360},& \text{if } x\in (0,360)\\
    0,& otherwise 
\end{cases}
\]

Has an expected value at

\[E(X)=\int_{0}^{360} x\frac{1}{360}dx= \frac{1}{2}\frac{x^2}{360} |_0^{360}= 180\]

\includegraphics{_main_files/figure-latex/unnamed-chunk-86-1.pdf}

\hypertarget{variance-1}{%
\section{Variance}\label{variance-1}}

As in the discrete case, the variance measures the dispersion of probabilities around the mean. It is the expected squared distance from the mean, indicating how far we expect the next observation of the random experiment to be.

\textbf{Definition}

Suppose \(X\) is a continuous random variable with probability density function \(f(x)\). The variance of \(X\), denoted as \(\sigma^2\) or \(V(X)\), is

\[\sigma^2=V(X)=\int_{-\infty}^\infty (x-\mu)^2 f(x) dx\]

It is the continuous version of the moment of inertia.

\hypertarget{functions-of-x}{%
\section{\texorpdfstring{Functions of \(X\)}{Functions of X}}\label{functions-of-x}}

In many occasions, we will be interested in outcomes that are functions of the random variables. Perhaps, we are interested in the square of the elongation of a spring, or on the square root of the temperature of an engine.

\textbf{Definition}

For any function \(h\) of a random variable \(X\), with mass function \(f(x)\), its expected value is given by

\[E[h(X)]= \int_{-\infty}^{\infty} h(x) f(x)dx\]

From this definition we recover the same properties as in the discrete case:

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\item
  The mean of a linear function is the linear function fo the mean: \[E(a\times X +b)= a\times E(X) +b\] for \(a\) and \(b\) scalars.
\item
  The variance of a linear function of \(X\) is:\[V(a\times X +b)= a^2\times V(X)\]
\item
  The expected squared distance about the origin is the mean squared plus the variance about the mean: \[E(X^2)=E(X)^2+V(X)\]
\end{enumerate}

\hypertarget{exercises-4}{%
\section{Exercises}\label{exercises-4}}

\hypertarget{exercise-1-3}{%
\subsubsection{Exercise 1}\label{exercise-1-3}}

For the probability density

\[
    f(x)= 
\begin{cases}
    \frac{1}{100},& \text{if } x\in (0,100)\\
    0,& otherwise 
\end{cases}
\]

\begin{itemize}
\tightlist
\item
  compute the mean (A:\(50\))
\item
  compute the variance using \(E(X^2)=V(X)+E(X)^2\) (A:\(100^2/12\))
\item
  compute \(P(\mu-\sigma\leq X \leq \mu+\sigma)\) (A: \(0.57\))
\item
  What are the first and third quartiles? (A: \(25\); \(75\))
\end{itemize}

\hypertarget{exercise-2-3}{%
\subsubsection{Exercise 2}\label{exercise-2-3}}

Given
\[
    f(x)= 
\begin{cases}
0, & x < 0 \\
ax, & x \in [0,3] \\
b, & x \in (3,5) \\
\frac{b}{3}(8-x),& x \in [5,8]\\
0, & x > 8 \\
\end{cases}
\]

\begin{itemize}
\item
  What are the values of \(a\) and \(b\) such that \(f(x)\) is a continous probability density function? (A: 1/15; 1/5)
\item
  what is the mean of \(X\)? (A:4)
\end{itemize}

\hypertarget{exercise-3-3}{%
\subsubsection{Exercise 3}\label{exercise-3-3}}

For the probability density

\[
    f(x)= 
\begin{cases}
    \lambda e^{-\lambda x},& \text{if } x \geq 0\\
    0,& otherwise 
\end{cases}
\]

\begin{itemize}
\tightlist
\item
  Confirm that this is a probability density
\item
  Compute the mean (A: 1/\(\lambda\))
\item
  Compute the expected value of \(X^2\) (A: 2/\(\lambda^2\))
\item
  Compute variance (A: 1/\(\lambda^2\))
\item
  Find the probability distribution \(F(a)\) (A: \(1-exp(-\lambda a)\))
\item
  Find the median (A: \(\log{2}\)/\(\lambda\))
\end{itemize}

\hypertarget{exercise-4-3}{%
\subsubsection{Exercise 4}\label{exercise-4-3}}

Given the cumulative distribution for a random variable X

\[
    F(x)= 
\begin{cases}
0, & x  < -1 \\
\frac{1}{80}(17+16x-x^2),& x \in [-1,7)\\
1,& x \geq 7\\
\end{cases}
\]

compute:

\begin{itemize}
\tightlist
\item
  \(P(X>0)\) (A:\(63/80\))
\item
  \(E(X)\) (A:\(1.93\))
\item
  \(P(X>0|X<2)\) (\(A:28/45\))
\end{itemize}

\hypertarget{discrete-probability-models}{%
\chapter{Discrete Probability Models}\label{discrete-probability-models}}

We seek to develop models to describe the probabilities of random experiments. Models are the first elements elements to consider when constructing a theory to explain such experiments. A suitable model not only allows us to describe the likelihoods of outcomes more accurately but also provides valuable insights into the experiment's properties. These properties often include intrinsic quantities that characterize the experiment, which we may attempt to influence or relate to other natural phenomena.

Consider, for instance, the distribution of deaths in London's Soho during the cholera outbreak of 1854. John Snow observed that the probability of dying from cholera was significantly higher near the Broad Street water pump supplied by the Southwark and Vauxhull Company. By identifying the location as a probabilistic property of the outbreak, he hypothesized that the disease was transmitted through contaminated water. Acting on this reasoning, he had the pump handle removed, which led to a reduction in mortality. While he ruled out airborne transmission, the germ theory of cholera was only achieved by the critical knowledge from the additional experiments of Robert Koch \citep{Lippi2014}.

In this chapter we will see some probability mass functions that are used to describe common random experiments. When we propose a model and we give the probability of an outcome, we make a \textbf{prediction} for the outcome. That is a statement about the likelihood of its future observation.

We will introduce the concept of parameter, as an intrinsic characteristic of the experiment, and of parametric models, as a set of similar experiments that differ only on that characteristic. In particular, we will discuss the uniform and Bernoulli probability functions and how they are used to derive the binomial and negative binomial probability models. We will show how to use Python and R functions to compute the probabilities of these models.

\hypertarget{probability-model}{%
\section{Probability model}\label{probability-model}}

A \textbf{probability model} is a probability mass function that may represent the probabilities of a random experiment.

\textbf{Example (Physical simulator)}

The probability mass function defined by the table

\[
\begin{array}{cc}
x & f(x) \\
-2 & 1/8 \\
-1 & 2/8 \\
0  & 2/8 \\
1  & 2/8 \\
2  & 1/8 \\
\end{array}
\]

represents the experiment of drawing \textbf{one} ball from an urn where there are two balls with labels: \(-1, 0, 1\) and one ball fore each label: \(-2, 2\).

\textbf{Example (Dice)}

The probability mass function \(f(x)=1/6\), for \(x\in\{1,2,3,4,5,6\}\), represents the throw of a dice.

\hypertarget{parametric-models}{%
\section{Parametric models}\label{parametric-models}}

When we have a random experiment with \(m\) possible outcomes, we need to find \(m\) numbers to determine the probability mass function. As in the first example above, we needed \(5\) values in the column \(f(x)\) of the probability table.

However, \textbf{in many cases}, we can formulate probability functions \(f(x)\) that depend on \textbf{very few} numbers only. As in the second example above, we only needed to know how many possible outcomes the throw of a dice has.

\textbf{Example (Classical probability)}

A random experiment with \(m\) equally likely outcomes has a probability mass function:
\[f(x)=\frac{1}{m}\]

We only need to know \(m\).

The numbers we \textbf{need to know} to fully determine a probability function are called \textbf{parameters}.

\hypertarget{uniform-probability-mass-function-one-parameter}{%
\section{Uniform probability mass function (one parameter)}\label{uniform-probability-mass-function-one-parameter}}

The previous example is the classical interpretation of probability, and defines our first parametric model.

\textbf{Definition}

A random variable \(X\) with outcomes \(\{1,...m\}\) has a discrete \textbf{uniform distribution} if all its \(m\) outcomes have the same probability

\[f(x)=\frac{1}{m}\]

\(m\) is the natural parameter of the model. Once we define \(m\) for an experiment, we choose a particular probability mass function. The function above is really a family of probability mass functions that depend on \(m\): \(f(x; m)\).

The mean and variance of a variable that follows a uniform distribution are:

\[\mu= \frac{m+1}{2}\]

and

\[\sigma^2= \frac{m^2-1}{12}\]
which are derived from the definitions.

Note that \(\mu\) and \(\sigma^2\) are also \textbf{parameters}. Remember that they are expected value, and the expected squared distance from the expected value. If we know any of them then we can fully determine the distribution. Using the equations above, we can have three different \textbf{parametrizations} of the uniform distribution

\[f(x)=\frac{1}{m}=\frac{1}{2\mu-1}=\frac{1}{\sqrt{12\sigma^2+1}}\]
The two last are cumbersome. The first one is natural, as the parameter \(m\) has the simple interpretation of the number of possible outcomes. Let us look at some probability mass functions in the family of uniform parametric models. Here are four members of the family, each characterized by a different \(m\)

\includegraphics{_main_files/figure-latex/unnamed-chunk-89-1.pdf}

\textbf{Example (Normality of \(\pi\))}

A normal number is a real number with infinitely many decimal places, where each digit appears with equal probability. In 1909, Borel hypothesized that \(\pi\) is a normal number, yet after more than a 100 years mathematicians have not been able to prove it.

\(\pi\) has infinite digits and their values can be computed sequentially but the value at any given position is difficult to \textbf{predict}. If \(\pi\) is normal, the numbers from \(0\) to \(9\) should each appear exactly \textbf{one-tenth} of the time in the infinite sequence of \(\pi\). Borel's conjecture is that the probability mass function for the digits of \(\pi\) is

\[
f(x) = \frac{1}{m}
\]

where \(m\) is the base of the numerical system used to express \(\pi\).

If \(\pi\) is expressed in \textbf{binary}, the probability of each digit (0 or 1) should be \(f(x) = \frac{1}{2}\) for \(x \in \{0,1\}\). Bailey an colleagues computed four trillion hexadecimal digits of
in \textbf{hexadecimal}, where 16 digits are used (from 0 to F), and obtained relative frequencies near \(\frac{1}{16}\) \citep{Bailey2012}. They obtained empirical evidence of what we believe is a truth that could be derived from pure mathematical reasoning.

\hypertarget{uniform-probability-mass-function-two-parameters}{%
\section{Uniform probability mass function (two parameters)}\label{uniform-probability-mass-function-two-parameters}}

Let us introduce a new \textbf{uniform} probability model with \textbf{two parameters}: The minimum and maximum outcomes.

If the random variable takes values in \(\{a, a+1, ...b\}\), where \(a\) and \(b\) are integers and all the outcomes are equally probable then

\[f(x)=\frac{1}{b-a+1}\]

as the total number of outcomes is \(m=b-a+1\).

If the random variable has the probability mass function \(f(x)\) above, we then say that \(X\) distributes uniformly between \(a\) and \(b\) and write

\[X \sim Unif(a,b)\]

\textbf{Properties}

The mean and variance of a variable that follows a two parameter uniform distribution are:

\[\mu= \frac{b+a}{2}\]
and

\[\sigma^2= \frac{(b-a+1)^2-1}{12}\]

To prove this, change variables \(X=Y+a-1\), \(y \in \{1,...m\}\), and apply the properties of the mean and variance of a linear function.

\textbf{Probability mass functions}

Let us look at some probability mass functions in the family of uniform parametric models:

\includegraphics{_main_files/figure-latex/unnamed-chunk-91-1.pdf}

\textbf{Example (Children age)}

Consider the following prediction: What is the probability that a child is 9 years old if chosen at random from a primary school?

We propose a model. Based on the setup of the experiment, and depending on the country, the minimum age in primary school is \(a = 6\) and the maximum is \(b = 11\). If all grades have the same number of students and each grade contains the same number of classes, then the appropriate probabilistic model for this random experiment is a discrete uniform distribution:

\[
X \sim \text{Unif}(a = 6, b = 11)
\]

That is,

\[
f(x) = \frac{1}{6} \quad \text{for } x \in \{6, 7, 8, 9, 10, 11\}.
\]

Therefore, our prediction is that \(1\) in \(6\) children are 9 years old.

The mean and variance of this probability mass function are:

\[
\mu = 8.5 \quad \text{and} \quad \sigma^2 = 2.916\overline{6}.
\]

These values are also \textbf{predictions of the model}: they indicate that we expect to observe ages of about \(\mu = 8.5\), with a typical deviations of approximately \(\sigma = 1.70\).

\includegraphics{_main_files/figure-latex/unnamed-chunk-93-1.pdf}

\textbf{Parametric models}

A model is \textbf{one} particular function \(f(x)\) that describes the probabilities of our experiment. If the function depends only on a few parameters to be fully determined then by changing the values of the parameters, we recreate a \textbf{family of models} \(f(x; a,b)\). A family of models are models that differ only on those parameters.

Ideally, the model and the parameters are \textbf{interpretable}. In our example, \(a\) represents the the minimum age at school and \(b\) the maximum age.The family represents experiments that differ on those intrinsic characteristics, say the family of different school systems with different levels. Knowledge of the model that represents our experiment is then reduced to knowing the value of the parameters \(a\) and \(b\). They can be considered as the \textbf{physical properties} or intrinsic characteristics of the experiment.

Note that in cases like this, we can fully specify the model parameters by \textbf{deduction} based on the design of the experiment and reasonable assumptions, without the need to run an actual experiment. Here, complete knowledge can be attained through pure reasoning alone, and predictions be made from the armchair.

\hypertarget{bernoulli-trial}{%
\section{Bernoulli trial}\label{bernoulli-trial}}

We now move beyond the equal probability case. Suppose we have a simple model with only two possible outcomes, \(A\) and \(A'\), which have \textbf{unequal} probabilities.

\textbf{Examples:}

\begin{itemize}
\item
  Recording the sex of a patient who enters a hospital emergency room (\(A: \text{male}\), \(A': \text{female}\)).
\item
  Checking whether a manufactured machine is defective or not (\(A: \text{defective}\), \(A': \text{not defective}\)).
\item
  Hitting a target (\(A: \text{success}\), \(A': \text{failure}\)).
\item
  Transmitting one pixel correctly (\(A: \text{yes}\), \(A': \text{no}\)).
\end{itemize}

Note that in all these examples, the probability of outcome \(A\) is typically \textbf{unknown} and cannot be deduced from the experimental set up.

\textbf{Probability model}

We will introduce the probability of an outcome (\(A\)) as the \textbf{parameter} of the model, yet to be determined. The model can be expressed in three equivalent forms:

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\tightlist
\item
  As a probability table
\end{enumerate}

\[
\begin{array}{cc}
\textbf{Outcome} & \textbf{Probability} \\
A' & 1 - p \\
A & p \\
\end{array}
\]

\begin{itemize}
\tightlist
\item
  Outcome \(A'\) (failure/outcome of no interest) has probability \(1 - p\).\\
\item
  Outcome \(A\) (success/outcome of interest) has probability \(p\) (the parameter).
\end{itemize}

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{1}
\tightlist
\item
  As a probability mass function of a random variable \(K\) taking values \(\{0, 1\}\) for \(A'\) and \(A\), respectively
\end{enumerate}

\[
f(k) = 
\begin{cases}
1 - p, & k = 0 \quad (event \, A') \\
p, & k = 1 \quad (event \, A)
\end{cases}
\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{2}
\tightlist
\item
  As a concise probability mass function
\end{enumerate}

\[
f(k; p) = p^k (1 - p)^{1 - k}
\]

for \(k \in \{0,1\}\).

We then say that \(X\) follows a Bernoulli probability mass function with parameter \(p\)
\[X \sim Bernoulli(p)\]

We call these types of experiments Bernoulli trials. A Bernoulli trial is the simplest form of a random experiment, designed to produce either the observation of a single event or its absence. The defining characteristic of the experiment is the probability
\(p\) of the event, represented as a parameter that is yet to be determined.

The simplicity of the trial should not be underestimated. It is the ``atom'' of probabilistic models, forming the foundation for more complex models. Moreover, it provides the insight that probabilities are intrinsic properties of the random experiment itself, existing independently of any repetition. Repetition of the experiment is simply the way we reveal these probabilities.

Understanding probabilities as abstract constructs (parameters), and demonstrating how they are revealed through the Law of Large Numbers, was Jacob Bernoulli's great achievement \citep{Bernoulli2006}---one that laid the foundation for mathematical statistics. As we will see in an example further below.

\textbf{Properties:}

The mean and variance of a variable that follows a Bernoulli distribution are:

\[\mu=p\]
and

\[\sigma^2=p(1-p)\]

Note that the parameter \(p\) is the probability of the outcome \(A\),
which is the same as the value of the probability mass function at \(1\)

\[p=P(A)=P(K=1)=f(1)\]

The parameter fully determines the probability mass function, including its mean and variance.

Let us look at some probability mass functions in the family of Bernoulli distributions

\includegraphics{_main_files/figure-latex/unnamed-chunk-95-1.pdf}

\hypertarget{binomial-experiment}{%
\section{Binomial experiment}\label{binomial-experiment}}

Now we consider the repetitions of the random experiment. When we repeat the random experiment \(n\) times, we first count the number of times an outcome was observed (\(n_A\)), we then divide by \(n\) to obtain the relative frequencies (\(f_A=n_A/n\)).

In the repetition of the experiment, we obtain one single value for \(n_A\). However, if we perform other \(n\) repetitions of the experiment \(n_A\) changes its value. Consequently, \(n_A\) is the observation of a new random variable \(X=N_A\). This random variable is a measurement of an outcome of a new random experiment: The repetition of the original experiment \(n\) times.

When the outcome \(A\) is obtained from a Bernoulli trial with probability \(p\) then the repetition of the trial \(n\) times produces an outcome \(n_A=x\) with some probability. We are interested in describing the probability that \(X\) takes a given value \(x\).

\textbf{Examples (Repetitions of Bernoulli trials):}

\begin{itemize}
\item
  Writing down the sex of \(n=10\) patients (\(A:man\) and \(A':woman\)) who go into an emergency room of a hospital. What is the probability that at a given time \(X=9\) patients are men, when the probability that a patient in the room is a man is \(p=0.8\)?
\item
  Trying \(n=5\) times to hit a target (\(A:success\) and \(A':failure\)). What is the probability that I hit the target \(X=5\) times when I usually hit it \(25\%\) of the times (\(p=0.25\))?
\item
  Transmitting a \(100\)-pixel picture with errors (\(A:error\) and \(A':correct\)). What is the probability that \(X=2\) pixels are errors, when the probability of one error is \(p=0.1\)?
\end{itemize}

\hypertarget{binomial-probability-function}{%
\section{Binomial probability function}\label{binomial-probability-function}}

In what follows, we will assume that the true value of the Bernoulli trial parameter \(p\) \textbf{is known}. The question of how to determine \(p\) is central to statistics and will be addressed later.

When we repeat a Bernoulli trial and stop at \(n\), is the value \(x\) that gives the total number of observed events \(A\) common or rare? what is its probability mass function \(P(X=x)=f(x)\)?

\textbf{Example (Transmission of pixels)}

What is the probability of transmitting \(X=2\) errors in a \(4\)-pixel picture, if the probability of a single error is \(p=0.1\)?

Regard the transmission of a \(4\)-pixel picture as a random experiment where

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\item
  The transmission of each single pixel is a \textbf{Bernoulli trial} \[K_i \sim Bernoulli(p)\] were \(k_i \in \{no\,error:0, error:1\}\) and, \(i\) is the pixel transmitted \(i=\{1,2,3,4\}\). For instance \(K_3=0\) is the event of no error in transmitting pixel \(3\).
\item
  \textbf{The random variable} of the picture is: \[(K_1, K_2, K_3, K_4)\] For instance, we may obtain the \textbf{outcome} \((K_1=0, K_2=1, K_3=0, K_4=1)\) or \((0, 1, 0, 1)\) as a picture with an error in the second and in the fourth pixels, and no errors in the first and third pixels. If we send a new picture, will obtain another 4-size vector of zeros and ones.
\item
  \textbf{The random variable} for the number of errors \[X=\sum_{i=1}^4 K_i\]
  counts the number of errors in the picture.
  Therefore, the possible number of errors in the transmission are values \(x\in \{0,1,2,3,4\}\). For example \(X\) takes the value \(2\) (\(X=2\)) for the outcome \((0, 1, 0, 1)\) because \(x=0+1+0+1\).
\end{enumerate}

Before we give the general formula of the probabilities, it is illustrative to first inspect the probabilities of particular \textbf{number of errors}:

\begin{itemize}
\tightlist
\item
  What is the probability of transmitting a picture with \(4\) \textbf{errors}?
\end{itemize}

The probability of \(4\) errors is the \textbf{joint probability} of transmitting an error in the \(1^{st}\) \textbf{and} \(2^{nd}\) \textbf{and} \(3^{rd}\) \textbf{and} \(4^{th}\) pixel:

\[P(X=4)=P(1,1,1,1)=p\times p\times p\times p=p^4\]

because \(K_i\) are \textbf{independent} the probabilities of the errors at each pixel multiply.

\begin{itemize}
\tightlist
\item
  What is the probability of transmitting a picture with \textbf{no errors}?
\end{itemize}

The probability of \(0\) errors is the joint probability of transmitting \textbf{no error} in \textbf{any} pixel:

\[P(X=0)=P(0,0,0,0)=(1-p)(1-p)(1-p)(1-p)=(1-p)^4\]

\begin{itemize}
\tightlist
\item
  What is the probability of tranmitting \(3\) \textbf{errors}?
\end{itemize}

The probability of \(3\) errors is the \textbf{addition} of the probabilities of all possible pictures with \(3\) errors:

\[P(X=3)=P(0,1,1,1)+P(1,0,1,1)+P(1,1,0,1)+P(1,1,1,0)=4p^3(1-p)^1\]
because all off these outcomes (pictures) are \textbf{mutually exclusive}.

Therefore, the probability of \(x\) \textbf{errors} is

\[
    f(x)= 
\begin{cases}
    1\times p^0(1-p)^4,&  x=0 \\
    4\times p^1(1-p)^3,&  x=1 \\
    6\times p^2(1-p)^2,&  x=2 \\
    4\times p^3(1-p)^1,&  x=3 \\
    1\times p^4(1-p)^0,&  x=4 \\
\end{cases}
\]

or more shortly

\[f(x)=\binom 4 x p^x(1-p)^{4-x}\]
for \(x=0,1,2,3,4\)

where \(\binom 4 x\) is the number of \textbf{possible} 4-pixel pictures with \(x\) errors. For instance, \(\binom 4 2=6\) in the formula above.

\textbf{Definition:}

The \textbf{binomial probability} function is the probability mass function of observing \(x\) outcomes of type \(A\) in \(n\) independent Bernoulli trials, where \(A\) has the same probability \(p\) in each trial.

The function is given by

\[f(x)=\binom n x p^x(1-p)^{n-x}\]
for \(x=0,1,...n\) and \(\binom n x= \frac{n!}{x!(n-x)!}\) is called \textbf{the binomial coefficient} and gives the number of ways one can obtain \(x\) outcomes of type \(A\) in a set of \(n\).

When a variable \(X\) has a binomial probability function, we say it distributes binomially and write

\[X\sim Binomial(n,p)\]

where \(n\) and \(p\) are parameters.

Let us look at some probability mass functions in the family of binomial parametric models:

\includegraphics{_main_files/figure-latex/unnamed-chunk-97-1.pdf}

\textbf{Properties:}

The mean and variance of a variable that follows a Binomial distribution are

\[\mu=np\]
\[\sigma^2=np(1-p)\]

Note that the mean and variance of a binomial variable is \(n\) times the mean and variance of a Bernoulli variable. These properties can be demonstrated by the fact that \(X\) is the sum of \(n\) independent Bernoulli variables. Therefore,

\(E(X)=E(\sum_{i=1}^n K_i)=np\)

and

\(V(X)=V(\sum_{i=1}^n K_i)=n(1-p)p\)

The last equation requires independence of the Bernoulli trials.

\textbf{Example (Transmission of pixels)}

\begin{itemize}
\item
  The expected value for the number of errors in the transmission of \(4\) pixels is \(np=4\times 0.1=0.4\) when the probability of an error is \(0.1\).
\item
  The variance is \(n(1-p)p=0.36\)
\item
  What is the probability of observing \(4\) errors?
\end{itemize}

Since we are repeating a Bernoulli trial \(n=4\) times and counting the number of events of type \(A\) (errors), when \(P(A)=p=0.1\) then

\[X \sim Binomial(n=4, p=0.1)\]
That is from \[f(x)=\binom 4 x 0.1^x(1-0.1)^{4-x}\]

We compte

\(P(X=4)=f(4)=\binom 4 4 0.1^4 0.9^{0}=0.1^4=10^{-4}\)

which can be done in Python and R with the following code

\[
\begin{array}{l|l}
 \textbf{Python} & \textbf{R}  \\ 
 \texttt{from scipy.stats import binom} & \texttt{dbinom(4,4,0.1)}
\\ 
 \texttt{binom.pmf(4,4,0.1)}& \\ 
\end{array}
\]

\begin{itemize}
\tightlist
\item
  What is the probability of observing \(2\) errors?
\end{itemize}

\(P(X=2)=\binom 4 2 0.1^2 0.9^2=0.0486\)

Computed with the functions:

\begin{verbatim}
Python: binom.pmf(2,4,0.1)
R: dbinom(2,4,0.1)
\end{verbatim}

\textbf{Example (Earth's rings)}

Tomkins and colleagues presented evidence suggesting that during the Ordovician period, the Earth may have had rings similar to those of Saturn. These rings were likely formed when a large asteroid became trapped in Earth's orbit and was subsequently pulverized by tidal forces. Fourteen meteorite impact craters were identified within the time window between 466 and 450 million years ago \citep{Tomkins2024}, all located within a band of \(\pm30^\circ\) latitude around the equator.

To assess the significance of this pattern, Tomkins et al.~calculated the probability of observing at least 14 meteorite impacts within that equatorial band, assuming that meteorites fall uniformly over the Earth's surface. The equatorial band covers approximately \(29.4\%\) of the Earth's surface area, so the probability of a single impact falling within it is \(p = 0.294\). What is the probability that all 14 meteorites fell in the band, when the probability of a single hit is about \(3\) in \(10\)?

Let \(X\) be the random variable denoting the number of impacts (out of 14) that fall within the equatorial band. Then,

\[
X \sim \text{Binomial}(n=14, p=0.294)
\]

with probability mass function:

\[
f(x) = \binom{14}{x} (0.294)^x (0.706)^{14 - x}
\]

The probability of observing all 14 impacts in the band is:

\[
P(X = 14) = f(14) = \binom{14}{14} (0.294)^{14} \approx 3.96 \times 10^{-8}
\]

which we can compute with the functions

\begin{verbatim}
Python: 1-binom.cdf(13, 14, 0.9)
R: 1-pbinom(13, 14, 0.296)
\end{verbatim}

As reported by Tomkins and colleagues, this probability is extremely low, suggesting that the assumption that the meteorites came from every direction with the same probability is unlikely. This supports the alternative explanation that the meteorites originated from the same direction---possibly from a disintegrating ring that once surrounded the Earth during that time. The presence of such a ring coincides with a global icehouse period, likely caused by the cooling effect of the ring's shadows.

\textbf{Quantiles}

We can ask for the values of the binomial variable \(X\) that accumulate up to a quarter, half, or three-quarters of the probability. These values correspond to the first quartile, median, and third quartile. More generally, we can determine any level of accumulation:

\[x_q = F^{-1}(q)\]

where \(q\) is a quantile, a value between \(0\) and \(1\) that refers to the proportion of probability accumulated up to it. The interquartile range is

\[\text{IQR} = x_{0.75}-x_{0.25}\]

For the 4-pixel transmission \(x_{0.25}=0\), \(x_{0.75}=1\) and \(\text{IQR}=1\), or

\[P(0 \leq X \leq 1)=0.5\]
which means that half of the probability is captured between pictures with \(0\) and \(1\) errors.

\includegraphics{_main_files/figure-latex/unnamed-chunk-99-1.pdf}

The inter quartile range can be computed as

\begin{verbatim}
Python: binom.ppf(0.75, 4, 0.1)- binom.ppf(0.25, 4, 0.1)
R: qbinom(0.75, 4, 0.1)- qbinom(0.25, 4, 0.1)
\end{verbatim}

\textbf{Example (Bernoulli's insight)}

Bernoulli's insight is that the probability \(p\) of an event \(A\) can be experimentally inferred by repeating the experiment \(n\) times and computing the relative frequency \(f_A = \frac{n_A}{n}\), where \(n_A\) is the number of times \(A\) occurs.

To formalize this idea, Bernoulli used the properties of the binomial function to compute the interquartile range (IQR) of the relative frequency \(f_A = \frac{X}{n}\). Here, \(X\) is the random variable that counts how many times we observe \(A\); its realization is \(n_A\). Since \(X \sim \text{Binomial}(n, p)\), we can define its interquartile range as the interval between the 25th and 75th percentiles:

\[
P(x_{0.25} \leq X \leq x_{0.75}) = 0.5
\]

Dividing the inequality by \(n\), we obtain:

\[
P\left(\frac{x_{0.25}}{n} \leq \frac{X}{n} \leq \frac{x_{0.75}}{n}\right) = 0.5
\]

Therefore, we can use the binomial distribution to compute the interquartile range of the relative frequency as:

\[
\text{IQR}(f_A) = \frac{x_{0.75}}{n} - \frac{x_{0.25}}{n}
\]

For the 4-pixel picture this is simply \(IQR(f_A)=1/4-0/4=1/4\). Now, with the help of the binomial function, Bernoulli saw that if we compute this range for larger and larger values of \(n\), \(\text{IQR}(f_A)\) gets smaller and smaller

\includegraphics{_main_files/figure-latex/unnamed-chunk-101-1.pdf}
Therefore, the two values that capture the middle 50\% of the relative frequencies get closer. As \(n\) grows, the distribution of \(f_A\) becomes more concentrated around \(p\) \citep{Bernoulli2006}, meaning the frequency stabilizes near the true probability. In short, he provided a formal justification of

\[
\lim_{n \rightarrow \infty} f_A = p
\]

\hypertarget{negative-binomial}{%
\section{Negative binomial}\label{negative-binomial}}

We can now repeat the Bernoulli trial in different ways. Imagine that we are interested in counting the well-transmitted pixels (\(A'\)) before a \textbf{given number} of errors occur. Say, we can \textbf{tolerate} \(r\) errors in the transmission.

Our random experiment is now: Repeat Bernoulli trials until we observe the outcome \(A\) appears \(r\) times. Stop and count how many \(A'\) you have got.

The outcome of the experiment is the number of \(A'\) events before \(r\) \(A\)'s occur, that is the frequency \(n_{A'}=y\).

We are interested in finding the probability of observing a particular number of events \(A'\), \(P(Y=y)\), where \(Y=N_{A'}\) is the random variable.

\textbf{Example (Transmission of pixels)}

What is the probability of observing \(y\) well-transmitted (\(A'\)) pixels before \(r\) errors (\(A\))?

Let us first find the probability of \textbf{one particular} picture with \(y\) number of correct pixels (\(A'\)) and \(r\) number of errors (\(A\)). One observation of the experiment can be schematically reresented as

\[(0,0,1,., 0,1,...0,1)\]

where we consider that there are \(y\) zeros and \(r\) ones. Therefore, we observe \(y\) correct pixels in a total of \(y + r\) pixels.

The probability of this picture is:

\[P(0,0,1,., 0,1,...0,1)=p^r(1-p)^y\]

Remember that \(p\) is the probability for a single pixel error (\(A\)).

How many pictures can have \(y\) correct pixels (\(0\)'s) before \(r\) errors (\(1\)'s)?

Note that

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\item
  The last pixel is fixed (marks the end of transmission)
\item
  The total number of ways in which \(y\) number of zeros can be allocated in \(y + r-1\) pixels (the last pixel is fixed with value 1) is: \(\binom {y + r-1} y\)
\end{enumerate}

Therefore, the probability of observing \(y\) \(0\)'s before \(r\) \(1\)'s (each 1 with probability \(p\)) is

\[P(Y=y)=f(y)=\binom {y+r-1} y p^r(1-p)^y\]

for \(y=0,1,...\)

We then say that \(Y\) follows a negative binomial distribution and we write

\[Y\sim NB(r,p)\]

where \(r\) and \(p\) are parameters representing the tolerance and the probability of a single error (event \(A\)).

\textbf{Properties:}

The mean and variance of a variable that follows a negative binomial distribution are

\[\mu
= r\frac{1-p}{p}\]

and

\[\sigma^2= r\frac{1-p}{p^2}\]

Let us look at some probability mass functions in the family of negative binomial parametric models:

\includegraphics{_main_files/figure-latex/unnamed-chunk-103-1.pdf}

\textbf{Example (Website)}

A website has three servers. One server operates at a time and, only when a request fails, another server is used.

If the probability of failure for a request is known to be \(p=0.0005\) then

\begin{itemize}
\tightlist
\item
  What is the expected number of successful requests before the three computers fail?
\end{itemize}

Since we are repeating a Bernoulli trial until \(r=3\) events of type \(A\) (failure) are observed (each with \(P(A)=p=0.0005\)) and are counting the number of events of type \(A'\) (successful requests) then

\[Y \sim NB(r=3, p=0.0005)\]

Therefore, the expected number of requests before the system fails is:

\(\mu=r\frac{1-p}{p}=3\frac{1-0.0005}{0.0005}=5997\)

Note that there are actually \(6000\) trials.

\begin{itemize}
\tightlist
\item
  What is the probability of dealing with at most \(5\) successful requests before the system fails?
\end{itemize}

We, therefore, want to compute the probability distribution at \(5\):

\(F(5)=P(Y\leq 5)=\Sigma_{y=0}^5 f(y)\)

\(=\sum_{y=0}^5\binom {y+2} y 0.0005^r0.9995^y\)

\(=\binom {2} 0 0.0005^3 0.9995^0 +\binom {3} 1 0.0005^3 0.9995^1\)

\(+\binom {4} 2 0.0005^3 0.9995^2 +\binom {5} 3 0.0005^3 0.9995^3\)

\(+\binom {6} 4 0.0005^3 0.9995^4 +\binom {7} 5 0.0005^3 0.9995^5\)

\(= 6.9\times 10^{-9}\)

which can be done with

\begin{verbatim}
Pthyon: nbinom.cdf(5,3,0.0005)
R: pnbinom(5,3,0.0005)
\end{verbatim}

\textbf{Example (Sex differences in COVID mortality)}

Men were three-fold more likely to die during the COVID-19 pandemic than women. The \textbf{fatality ratio} was reported as \(FR = 3.5\) \citep{DehingiaRaj2021}. The fatality ratio is defined as the ratio of death probabilities between men and women:

\[
FR = \frac{P(D \mid M)}{P(D \mid W)}
\]

From this, we can compute the probability that a deceased patient was a \textbf{woman} using \textbf{Bayes' theorem}:

\[
P(W \mid D) = \frac{P(D \mid W) \cdot P(W)}{P(D \mid W) \cdot P(W) + P(D \mid M) \cdot P(M)}
\]

Substituting the fatality ratio and assuming an equal gender distribution in the population (\(P(W) = P(M) = 0.5\)):

\[
P(W \mid D) = \frac{(1/3.5) \cdot 0.5}{(1/3.5) \cdot 0.5 + 0.5} = 0.22
\]

How likely is it that at most 175 men died before 50 women did?

We model this using the negative binomial distribution, which gives the number of male deaths required before 50 women have died, assuming each deceased individual is female with probability \(p = 0.22\).

We want to compute the probability that at most 175 men (\(X\)) died before 50 women (\(r = 50\)) did:

\[
P(X \leq 175) = F(175; r = 50, p = 0.22)
\]

This gives approximately:

\[
P(X \leq 175) \approx 0.49
\]

That is, 175 male deaths is around the \textbf{median} number expected before observing 50 female deaths.

\begin{verbatim}
Pthyon: nbinom.pmf(175, 50, 0.22)
R: pnbinom(175, 50, 0.22)
\end{verbatim}

Note that the ratio of deaths at the median, 175 men to 50 women, gives \(175/50=3.5\). This matches the fatality ratio, reinforcing the model's consistency.

\hypertarget{geometric-distribution}{%
\section{Geometric distribution}\label{geometric-distribution}}

We call \textbf{geometric distribution} to the \textbf{negative binomial} distribution with \(r=1\)

The probability of observing \(A'\) events before observing the \textbf{first} event of type \(A\) is

\[P(Y=y)=f(y)= p(1-p)^y\]

\[Y\sim Geom(p)\]

The mean and variance of a variable that follows a geometric distribution are

\[\mu= \frac{1-p}{p}\]

and

\[\sigma^2= \frac{1-p}{p^2}\]

Note that the mean and variance of a negative binomial variable is \(r\) times the mean and variance of a geometric variable.

\hypertarget{hypergeometric-model}{%
\section{Hypergeometric model}\label{hypergeometric-model}}

The \textbf{hypergeometric model} arises from the urn experiment, with a total number fo \(N\) balls and, we draw \(n\) balls without replacing them back in the urn. If the balls are labeled with \(A\) and \(A'\), what is the probability of drawing \(x\) balls of type \(A\) in a total of \(n\) draws?

The general model is to consider \(N\) total balls in a urn. Mark \(K\) with label \(A\) and \(N-K\) with label \(A'\). Take out \(n\) balls one by one with no replacement in the urn, and then count how many \(A\)'s you obtained.

The \textbf{Binomial} model can be derived from the \textbf{Hypergeometric} model when we consider that either \(N\) is infinite, or that every time we draw a ball we replace it back in the urn.

\textbf{Example (measles)}

By July 2025, the U.S. reported \(N=1288\) confirmed measles cases. Over \(90\%\) of cases involved unvaccinated individuals or those with unknown vaccination status, that is \(K=1159\) patients.

Of those reported cases, imagine we ask \(n=200\) and find that \(x=183\) did not vaccinate. What is the probability of the observing at least \(183\) unvaccinated people in our survey?

\textbf{Definition:}

The probability of obtaining \(x\) cases (type \(A\)) in a sample of \(n\), drawn from a population of \(N\) where \(K\) are cases (type \(A\)) is

\(P(X=x)=P(one\,sample) \times (Number\, of\, ways\, of\, obtaining\, x)\)

\[=\frac{1}{\binom N n}\binom K x \binom {N-K} {n-x}\]

where \(x \in \{\max(0, n-(N-K), ... \min(K, n) \}\). The first limit includes the case where we draw more balls than \(A'\)-labeled balls, and the second limit when we draw all he \(A\)-labeled balls.

We then say that \(X\) follows a hypergeometric distribution and write

\[X \sim Hypergeometric(N,K,n)\]
The hypergeometric model has three parameters.

\textbf{Properties:}

The mean and variance of a variable that follows a hypergeometric distribution are:

\[\mu =  n  \frac{K}{N} = np\]
and

\[\sigma^2 = np(1-p)\frac{N-n}{N-1}\]

\(p=\frac{K}{N}\) is the proportion of chickenpox in a population of size \(N\). Note that when \(N \rightarrow \infty\) or we replace the ball, \(K=n_A\) and then we recover the binomial properties.

Let us look at some probability mass functions in the family of hypergeometric parametric models:

\includegraphics{_main_files/figure-latex/unnamed-chunk-105-1.pdf}

\textbf{Example (measles)}

\begin{itemize}
\tightlist
\item
  what is the probability that at least \(183\) measles patients are unvaccinated in a particular survey of \(200\) patients, drawn from a population of \(1288\) patients where \(1159\) did not vaccinate?
\end{itemize}

Given \(X \sim Hypergeometric(N=1159,K=1159,n=200)\), the probability that we need to compute is
\(1-P(X \leq 183)=F(183)=0.18\)

That is about \(18\%\) of the 200-patient surveys will report more than \(187\) unvaccinated people.

\begin{verbatim}
Python: 1- hypergeom.cdf(183, 1288, 200, 1159)
R: 1- phyper(183, 1159, 1288 - 1159, 200)
\end{verbatim}

The solution is the addition of the blue needles in the plot.

\includegraphics{_main_files/figure-latex/unnamed-chunk-107-1.pdf}

\hypertarget{questions-4}{%
\section{Questions}\label{questions-4}}

\textbf{1)} What is the expected value and the variance of the number of failures in \(100\) prototypes, when the probability of a failure is \(0.25\)

\textbf{\(\qquad\)a:} \(0.25\), \(0.1875\);
\textbf{\(\qquad\)b:} \(25\), \(0.1875\);
\textbf{\(\qquad\)c:} \(0.25\), \(18.75\);\\
\textbf{\(\qquad\)d:} \(25\), \(18.75\)

\textbf{2)} The number of available tables at lunch time in a restaurant is better described by which probability model

\textbf{\(\qquad\)a:} Binomial;
\textbf{\(\qquad\)b:} Uniform;
\textbf{\(\qquad\)c:} Negative Binomial;\\
\textbf{\(\qquad\)d:} Hypergeometric

\textbf{3)} The expected value of a Binomial distribution is not

\textbf{\(\qquad\)a:} \(n\) times the expected value of a Bernoulli;
\textbf{\(\qquad\)b:} the expected value of a Hypergeometric, when the population is very big;
\textbf{\(\qquad\)c:} \(np\);\\
\textbf{\(\qquad\)d:} the limit of the relative frequency when the number of repetitions is large

\textbf{4)} Opinion polls for the USA 2022 election give a probability of \(0.55\) that a voter favors the republican party. If we conduct our own poll and ask 100 random people on the street, How would you compute the probability that in our poll democrats win the election?

\textbf{\(\qquad\)a:}binom.cdf(49, 100, 0.55)=0.13;
\textbf{\(\qquad\)b:}1-binom.cdf(49, 100, 0.55)=0.86;
\textbf{\(\qquad\)c:}binom.cdf(51, 100, 0.45)=0.90; \textbf{\(\qquad\)d:}1-binom.cdf(51, 100, 0.45)=0.095

\textbf{5)} In an exam a student chooses at random one of the four answers that he does not know. If he doesn't know \(10\) questions what is the probability that more than \(5\) (\(>5\)) questions are correct?

\textbf{\(\qquad\)a:}binom.pmf(5, 10, 0.25)\textasciitilde{} 0.05; \textbf{\(\qquad\)b:}binom.cdf(5, 10, 0.75)\textasciitilde{} 0.07; \textbf{\(\qquad\)c:}binom.pmf(5, 10, 0.75)\textasciitilde{} 0.05; \textbf{\(\qquad\)d:}1-binom.cdf(5, 10, 0.25)\textasciitilde{} 0.01

\hypertarget{exercises-5}{%
\section{Exercises}\label{exercises-5}}

\hypertarget{exercise-1-4}{%
\subsubsection{Exercise 1}\label{exercise-1-4}}

If 20\% of the bolts produced by a machine are defective, determine the probability that, out of
4 bolts chosen at random

\begin{itemize}
\tightlist
\item
  1 bolts will be defective (A:0:4096)
\item
  0 bolts will be defective (A::4096)
\item
  at most 2 bolts will be defective (A:0:9728)
\end{itemize}

\hypertarget{exercise-2-4}{%
\subsubsection{Exercise 2}\label{exercise-2-4}}

In a population, the probability that a baby boy is born is \(p=0.51\). Consider a family of 4 children

\begin{itemize}
\tightlist
\item
  What is the probability that a family has only one boy?(A: 0.240)
\item
  What is the probability that a family has only one girl?(A: 0.259)
\item
  What is the probability that a family has only one boy or only one girl?(A: 0.4999)
\item
  What is the probability that the family has at least two boys?(A: 0.7023)
\item
  What is the number of children that a family should have such that the probability of having at least one girl is more than \(0.75\)?(A:\(n=3>\log(0.25)/\log(0.51)\))
\end{itemize}

\hypertarget{exercise-3-4}{%
\subsubsection{Exercise 3}\label{exercise-3-4}}

A search engine fails to retrieve information with a probability \(0.1\)

\begin{itemize}
\item
  If we system receives \(50\) search requests, what is the probability that the system fails to answer three of them?(A:0.1385651)
\item
  What is the probability that the engine successfully completes \(15\) searches before the first failure?(A:0.020)
\item
  We consider that a search engine works sufficiently well when it is able to find information for moe than \(10\) requests for every \(2\) failures. What is the probability that in a reliability trial our search engine is satisfactory?(A: 0.697)
\end{itemize}

\hypertarget{poisson-and-exponential-models}{%
\chapter{Poisson and Exponential Models}\label{poisson-and-exponential-models}}

A single natural or artificial process may support different random experiments. Random experiments are often designed to extract information about the process, which is conveniently encoded as a parameter. They may also be designed to predict a particular type of outcome.

For instance, if we are interested in knowing the probability of a Bernoulli trial, we can estimate it using the relative frequency of a Binomial experiment. The experiment with the most repetitions will provide a better estimate of the probability.

If our goal is to determine the adverse effects of a drug, we might opt for a Binomial experiment and conduct a survey of 1,000 patients undergoing treatment. However, if we are interested in assessing the drug's safety, we might choose a Negative Binomial experiment, testing patients sequentially and calculating the probability of observing no adverse effects before the first occurrence.

The key point, however, is that different experiments can be designed from a similar process, with preferences based on efficiency or the type of outcomes we seek to predict.

An important example of experimental models linked to similar phenomena is the relationship between the Poisson and exponential models. The Poisson model describes a discrete random variable that counts the number of events in an interval, while the exponential model describes a continuous random variable that measures the distance between two consecutive events. As we will see, the same underlying phenomenon can lead to different questions, which in turn result in either the Poisson or the exponential model being more appropriate.

\hypertarget{discrete-probability-models-1}{%
\section{Discrete probability models}\label{discrete-probability-models-1}}

In the previous chapter, we built complex models from simple ones. At each stage, we introduced a novel concept.

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  \textbf{Uniform distribution}: Classical interpretation of probability.
\item
  \textbf{Bernoulli trial}: Introduction of the probability \(p\) as a \textbf{parameter} (family of models)
\item
  \textbf{Binomial distribution}: Introduction of the \textbf{repetition} of a random experiment (\(n\)-times Bernoulli trials)
\item
  \textbf{Poisson distribution}: Repetition of a random experiment within a continuous interval, with \textbf{no control} over when or where the Bernoulli trial occurs.
\end{enumerate}

The final stage is the Poisson process, which describes the repetition of a random experiment with additional randomness in the timing of occurrences.

\hypertarget{poissson-experiment}{%
\section{Poissson experiment}\label{poissson-experiment}}

Imagine that we are observing events that \textbf{depend} on time or distance \textbf{intervals}.

for example:

\begin{itemize}
\tightlist
\item
  cars arriving at a traffic light
\item
  getting messages on a mobile phone
\item
  impurities occurring at random in a copper wire
\end{itemize}

Suppose that the events are outcomes of \textbf{independent} Bernoulli trials each appearing randomly on a continuous interval, and we want to \textbf{count} them.

What is the probability of observing \(X\) events in an interval's unit (time or distance)?

\textbf{Example (Impurities in a copper wire)}

Imagine that, for a single-cell probe, we need to design a nanowire of high-purity copper of level \(5N\) (\(\sim 99.999\%\) purity), corresponding to approximately \(10^{15}\) impurities per \(cm^3\). The wire has a diameter of \(0.1\mu m\) and a length of \(10\mu m\), giving an expected \(78\) impurities per wire. Impurities will deposit randomly along a copper wire and we expect that the specimens we produce will change in the number of impurities.

We would like to determine how likely we fail to produce a high-purity wire with a maximum tolerance of a \(100\) impurities.

What is the probability that the the next wire we produce has more than \(100\) impurities?

\hypertarget{poisson-probability-mass-function}{%
\section{Poisson probability mass function}\label{poisson-probability-mass-function}}

To calculate the probability mass function \(f(x)=P(X=x)\) of the previous example we divide the length of the wire of \(10\mu n\) into \(10,000nm\).

Nanometers are small enough so

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\item
  either there is or there is not an impurity in each nanometer.
\item
  each nanometer can be considered as a \textbf{Bernoulli trial}.
\end{enumerate}

\includegraphics{./figures/Pois.JPG}

\textbf{From the Binomial to the Poisson probability function}

The probability of observing \(X\) impurities in \(n=10000nm\) (\(10\mu m\)) approximately follows a binomial distribution

\[f(x) \sim \binom n x p^x(1-p)^{n-x}\]

where \(p\) is the probability of finding an impurity in a micrometer.

The expected value of a Binomial variable is:
\(E(X)=np\). The expected number of impurities in \(10\mu\) is a number \(\lambda=np\) that we will assume to know. \(\lambda\) is a property of the material.

If we substitute \(p=\lambda/n\) in the binomial function we obtain

\[f(x) \sim \binom n x \big(\frac{\lambda}{n}\big)^x(1-\frac{\lambda}{n})^{n-x}\]

Now, there \textbf{could} still be two impurities in a micrometer. Therefore, we need to increase the partition of the wire and take \(n \rightarrow \infty\).

In the limit:

\[f(x)= \frac{e^{-\lambda}\lambda^x}{x!}\]

where \(\lambda\) is constant. It is the density of impurities per centimeter, a \textbf{physical property} of the system. \(\lambda\) is therefore the \textbf{parameter} of the probability model.

\textbf{Derivation details:}

For \(f(x) \sim \binom n x \big(\frac{\lambda}{n}\big)^x(1-\frac{\lambda}{n})^{n-x}\)

in the limit (\(n \rightarrow \infty\))

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\tightlist
\item
  \(\frac{1}{n^x}\binom n x =\frac{1}{n^x}\frac{n!}{x! (n-x)!}=\frac{(n-x)!(n-x+1)...(n-1)n}{n^x x! (n-x)!}=\frac{n(n-1)..(n-x+1)}{n^x x!} \rightarrow \frac{1}{x!}\)
\item
  \((1-\frac{\lambda}{n})^{n} \rightarrow e^{-\lambda}\) (definition of exponential)
\item
  \((1-\frac{\lambda}{n})^{-x} \rightarrow 1\)
\end{enumerate}

Putting everything together then:

\[f(x)= \frac{e^{-\lambda}\lambda^x}{x!}\]

\textbf{Definition}

Given

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\tightlist
\item
  an interval in the real numbers
\item
  events occur at random in the interval
\item
  the average number of events on the interval is known (\(\lambda\))
\item
  if one can find a small regular partition of the interval such that each of them can be considered a Bernoulli trial.
\end{enumerate}

Then the random variable \(X\) that counts events across the interval is a \textbf{Poisson} variable with probability mass function\\
\[f(x)= \frac{e^{-\lambda}\lambda^x}{x!}, \lambda>0\]
which we write as

\[X \sim Poiss(\lambda)\]

\textbf{Properties:}
When \(X\) follows a Poisson model, it has mean and variance

\[\mu= \lambda\]

and

\[\sigma^2= \lambda\]

\textbf{Example (Patient hospitalization)}

In a medium-sized hospital during a typical influenza season, the average number of patients hospitalized with influenza is \(1\) per day. What is the probability that exactly four patients will be hospitalized on a given day?

We have that the random variable that counts the number of influenza patients hospitalized follows a Poisson model \[X \sim Poiss(\lambda)\] with \(\lambda=1\). That is; it has probability mass function

\[f(x)= \frac{e^{-1}1^x}{x!}\]
Therefore, the probability that the variable takes value 4 is \(P(X=4)\) is

\[f(4; \lambda=1)= \frac{e^{-1}1^4}{4!}=0.01532831\]

Which can be computed with

\begin{verbatim}
Python: poisson.pmf(4,1)
R: dpois(4,1)
\end{verbatim}

What is the probability of hospitalizing four patients in \textbf{three days}, when the average number of hospitalized patients with influenza is \(1\) per day?

The unit in which we do the counts has changed from \(1\) day to \(3\) days, so we have to \textbf{re-scale} \(\lambda\). If before the average number of patients in one day was \(\lambda=1\), the average number of patients in three days is now 3: \(\lambda_{3h}=3\lambda_{1h}=3\times 1=3\)

We have that the variable is Poisson: \(X \sim Poiss(\lambda_{3h})\) with \(\lambda_{3h}=3\) and its probability mass function is:

\[f(x)= \frac{e^{-3}3^x}{x!}\]
Therefore the probability that the variable takes value 4 is \(P(X=4)\):

\[f(4; \lambda=3)= \frac{e^{-3}3^4}{4!}=0.1680314\]

computed as:

\begin{verbatim}
Python: poisson.pmf(4,3)
R: dpois(4,3)
\end{verbatim}

\textbf{Example (Impurities in a copper wire)}

What is the probability of counting \textbf{at least} \(100\) impurities in a \(5N\) copper wire with \(0.1\mu m\) of diameter and \(10\mu m\) long, for which \(78\) impurities are expected.

We have that the variable is Poisson: \(X \sim Poiss(\lambda)\) with \(\lambda=78\) and its probability mass function is:

\[f(x)= \frac{e^{-78}78^x}{x!}\]
and then, we want to compute

\(P(X\geq 100)=1-P(X < 100)=1-P(X \leq 99)=1-F(99; \lambda=78)=0.00934\)

where we use the probability distribution \(F(x,\lambda)\) of the Poisson model. We are then likely to discard \(0.9\%\) our production of cables that do not pass the quality control measure.

\begin{verbatim}
Python: 1-poisson.cdf(99,78)
R: 1-ppois(99, 78)
\end{verbatim}

Let us look at some probability mass functions in the family of parametric Poisson models:

\includegraphics{_main_files/figure-latex/unnamed-chunk-109-1.pdf}

\hypertarget{continuous-probability-models}{%
\section{Continuous probability models}\label{continuous-probability-models}}

Continuous probability models are \textbf{probability density functions} \(f(x)\) of a continuous random variables that we \textbf{believe} describe real random experiments.

A probability density function \(f(x)\)

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\tightlist
\item
  is positive
\end{enumerate}

\[f(x) \geq 0\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{1}
\tightlist
\item
  allows us to compute probabilities using the area under the curve
\end{enumerate}

\[P(a\leq X \leq b)=\int_{a}^{b} f(x) dx\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{2}
\tightlist
\item
  is such that the probability that the outcome is in sample space is \(1\)
\end{enumerate}

\[\int_{-\infty}^{\infty} f(x) dx = 1\]

\hypertarget{exponential-process}{%
\section{Exponential process}\label{exponential-process}}

Let us go back to a \textbf{Poisson process} defined by the probability

\[f(k)=\frac{e^{-\lambda}\lambda^k}{k!}, \lambda>0\]

for the number of events (\(k\)) in an interval.

Consider now that we are interested in the length/time we should wait for the \textbf{first} event to occur.

We can ask about the probability that the first event occurs after length/time \(X\).

Therefore, since \(X\) is a \textbf{continuous} random variable, we are looking for its probability density function \(f(x)\).

\hypertarget{exponential-probability-density}{%
\section{Exponential probability density}\label{exponential-probability-density}}

Consider a Poisson variable \(K\) that counts the number of events in an interval. The probability of no events (\(K=0\) counts) \textbf{if} an interval has length \(a\) is

\[P(K=0)=\frac{e^{-\lambda a}(\lambda a)^0}{0!}\]

where we re-scale the units of \(\lambda\) to \(a\). Simplifying, we obtain

\[P(K=0)=e^{-\lambda a}\]

Now consider \(a\) as a possible outcome of a random variable \(X\) that measures the survival time/distance until the first event. The probability that the first event occurs after \(a\) is the probability that there are no events in the \(a\) interval

\[P(X>a)=P(K=0)=e^{-\lambda a}\]
This probability is called the survival function of \(X\)

\[S(a)=e^{-\lambda a}\]
In the copper wire example this is the probability that a wire of length \(a\) is completely pure, with no impurities. In a clinical study this represents probability that a patient survives beyond time \(a\).

The cumulative distribution function, taking \(a=x\), is

\[F(x)=1-S(x)= 1- e^{-\lambda x},\, x\geq0.\]
Differentiating gives the probability density function

\[f(x)=\lambda e^{-\lambda x},\, x\geq0\]

for the waiting time/length \(X\).

\textbf{Definition}

In a Poisson process with parameter \(\lambda\) the probability of measuring a length/time \(X\) between any two consecutive events is given by the \textbf{Exponential probability density}

\[f(x)=\lambda e^{-\lambda x}, x\geq 0\]

\(\lambda\) is the parameter of the model, also known as the \textbf{decay rate}, or the hazard in clinical studies. In general survival experiments \(\lambda\) may depend on \(x\) (for instance, the hazard of dying may depend on age: \(h(x)=\lambda\)). However, when \(\lambda\) is constant the exponential distribution is memoryless,

\[P(X>x+a|X>a) = P(X>x)\]
the waiting-time distribution does not depend on the elapse time (\(a\)). For example, if we cut a wire just after one impurity, the distance until the next impurity follows the same exponential law. Similarly, in survival analysis, without aging effects, the probability of surviving an additional time \(x\) is the same regardless of how long the patient has already survived. This is analogous to coin tossing: the probability of heads on the next toss does not depend on previous outcomes.

\textbf{Properties:}

When \(X\) follows an exponential model, we write \[X \sim Exp(\lambda)\]

and its mean and variance are

\[\mu=\frac{1}{\lambda}\]

and

\[\sigma^2=\frac{1}{\lambda^2}\]

Let us look at a couple of the probability densities in the exponential family

\includegraphics{_main_files/figure-latex/unnamed-chunk-111-1.pdf}

\hypertarget{exponential-distribution}{%
\section{Exponential Distribution}\label{exponential-distribution}}

Consider the following questions:

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\tightlist
\item
  In a Poisson process ¿What is the probability that an interval is \textbf{maximum} of size \(a\) until the first event?
\end{enumerate}

If \(K \sim Poisson(\lambda)\) counts independent events in an interval then the distance/time until the first event is \(X \sim Exp(\lambda)\). Note that the same parameter \(\lambda\) is used for both models because they represent different random experiments on the same process. The first one measures a discrete variable and the second a continuous one.

The probability until the first event is computed with the probability distribution of the exponential model

\(F(a)=P(X \leq a)\)

\[F(a)=\lambda \int_\infty^a e^{-\lambda x}dx=1-e^{-\lambda a}\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{1}
\tightlist
\item
  In a Poisson process What is the probability of observing an interval \textbf{minimum} of size \(a\) until the first event?
\end{enumerate}

Using the probability distribution this can be computed as:

\[P(X > a)=1- P(X \leq a)= 1- F(a) = e^{-\lambda a}\]

Let us look at a couple of exponential distributions from the exponential family

\includegraphics{_main_files/figure-latex/unnamed-chunk-113-1.pdf}

The median \(x_m\) is such that \(F(x_m)=0.5\). That is \(x_m=\frac{\log(2)}{\lambda}\).

\textbf{Example (Radioactive decay)}

What is the probability that the waiting time to detect the first radioactive particle from Americium-241 (commonly used in smoke detectors) with a Geiger counter is less than one second, assuming a decay rate of
\(\lambda = 3.33\) particles per second?

This is modeled using the exponential distribution, which describes the waiting time between events in a Poisson process. The probability is given by:

\[P(X \le 1)=F(1,\lambda=3.33)=0.998\]

computed as

\begin{verbatim}
Python: expon.cdf(2,0,1/3.33)
R: pexp(3.33,2)
\end{verbatim}

So, there is approximately a \(99.9\%\) chance that we will detect a particle within 2 seconds from the radioactive material of a smoke detector.

Note than Python uses the the decay rate \(1/\lambda\) as argument, rather than the hazard \(\lambda\).

\textbf{Example (Earthquake)}

According to data from VolcanoDiscovery, Japan experiences approximately \(12.7\) earthquakes of magnitude \(6\) or higher per year. This translates to about one earthquake of such magnitude per month.

What is the probability that we have to wait less than one month for an earthquake of magnitude \(6-7\) in Japan?

\(X\) is the time in years for the next earthquake and therefore \(X \sim Exp(\lambda=1)\). Therefore, we can compute

\[P(X < 1)= P(X \le 1) = F(1,\lambda=1)=0.63\]
Where \(F(1)\) is the exponential distribution function above and computed as

\begin{verbatim}
Python: expon.cdf(1,0,1)
R: pexp(1,1)
\end{verbatim}

There is about a \(63\%\) chance of an earthquake of magnitude \(6-7\) in Japan within a month.

\hypertarget{questions-5}{%
\section{Questions}\label{questions-5}}

\textbf{1)} During WWII in London, the expected number of bombs that hit an area of \(3km^2\) was \(0.92\). The probability that, in one day, one area received two bombs was

\textbf{\(\qquad\)a:}1-poisson.cdf(x=2, lambda=0.92);
\textbf{\(\qquad\)b:}poisson.cdf(x=2, lambda=0.92); \textbf{\(\qquad\)c:}1-poisson.pmf(x=2, lambda=0.92); \textbf{\(\qquad\)d:}poisson.pmf(x=2, lambda=0.92)

\textbf{2)} The probability that a passenger has to wait less than 20 minutes until the next bus arrives at her stop is better described by

\textbf{\(\qquad\)a:} A poisson model on the number of buses per 20 minutes;\\
\textbf{\(\qquad\)b:} An exponential distribution at 20 minutes with a given expectation of buses per minute;
\textbf{\(\qquad\)c:} A binomial model that counts the number of buses per 20 minutes
\textbf{\(\qquad\)d:} A normal distribution with an average of buses per 20 minutes and a given standard deviation;

\textbf{3)} From the exponential probability distribution in the figure below, what is the most likely value of the median?

\textbf{\(\qquad\)a:} \(2\); \textbf{\(\qquad\)b:} \(3\); \textbf{\(\qquad\)c:} \(4\); \textbf{\(\qquad\)d:} \(5\)

\includegraphics{_main_files/figure-latex/unnamed-chunk-114-1.pdf}

\hypertarget{exercises-6}{%
\section{Exercises}\label{exercises-6}}

\hypertarget{exercise-1-5}{%
\subsubsection{Exercise 1}\label{exercise-1-5}}

The average number of phone calls per hour coming into the switchboard of a company is \(150\). Find the probability that during one particular minute there will be

\begin{itemize}
\tightlist
\item
  0 phone calls (A:0.082)
\item
  1 phone call (A:0.205)
\item
  4 or fewer calls (A:0.891)
\item
  more than 6 phone calls (A:0.0141)
\end{itemize}

\hypertarget{exercise-2-5}{%
\subsubsection{Exercise 2}\label{exercise-2-5}}

The average number of radioactive particles hitting a Geiger counter in a nuclear energy plant under control is \(2.3\) per minute.

\begin{itemize}
\item
  What is the probability of counting exactly \(2\) particles in a minute? (A:0.265)
\item
  What is the probability of detecting exactly \(10\) particles in \(5\) minutes? (A:0.112)
\item
  What is the probability of at least one count in two minutes? (A:0.9899)
\item
  What is the probability of having to wait less than \(1\) second to detect a radioactive particle, after we switch on the detector? (A:0.037)
\item
  We suspect that a nuclear plant has a radioactive leak if we wait less than \(1\) second to detect a radioactive particle, after we switch on the detector. What is the probability that when we visit in \(5\) plants that are under control, we suspect that at least one has a leak? (A:0.1744).
\end{itemize}

\hypertarget{normal-distribution}{%
\chapter{Normal Distribution}\label{normal-distribution}}

In random experiments involving continuous outcomes, randomness can arise from changes in the units of measurement or from measurement error. In the first case, randomness can be understood as variability in the production of the unit. Under similar conditions, production may be influenced by uncontrolled factors. In the second case, the unit can be separated from the measuring process, and randomness can be understood as variations during the acquisition of the observed value.

Randomness can be described in a probabilistic model by a parameter, with the variance being the natural choice. Simple assumptions about how outcomes should deviate from the mean led Gauss to discover the normal distribution. While studying the trajectory of a celestial body, he realized that the randomness in measuring its position was due to instrumental error. The normal distribution was therefore first conceived as an error distribution. Later, when applied to anthropometric measurements, it was shown to effectively describe variations among individuals, paving the way for its applications in the life sciences.

The importance of the normal distribution stems from the fact that it is the continuous limit of the binomial distribution. Coupled with Bernoulli's insight that relative frequency approaches the expected value in a Bernoulli trial, the normal distribution supports this behavior for any type of distribution. This principle forms the basis of the central limit theorem, which we will cover in the following chapters.

In this chapter, we will introduce the normal probability distribution as originally discovered by Gauss to represent errors in astronomical data and later applied by Galton to represent variations in human height.

\hypertarget{history}{%
\section{History}\label{history}}

In 1801 Gauss analyzed the data that obtained by Giuseppe Piazzi for the position of the Ceres, a large asteroid between Mars and Jupiter.

At the time Piazzi suspected it was a new planet, as it moved day to day against the fixed stars. In January, it could be seen at the horizon just before dawn. However, as days passed Ceres will rise latter and latter until it could not longer be seen because of the Sun rise. The Sun caught up with it.

Gauss understood that the measurements for the position of Ceres had errors.

He was therefore interested in finding how the observations were \textbf{distributed} so he could find the most \textbf{likely} orbit. With the orbit, we could derive the mass of the object and then decide whether it was massive as a planet or just a big asteroid.

Data was available only for the month of January. After which Ceres would disappear. He wanted to \textbf{predict} six moths latter, once it had slowly transited behind the Sun, where astronomers should point their telescopes just after at dusk.

Gauss had to account for the errors in the position of Ceres at a given day due to measurement

\includegraphics{./figures/ceres.JPG}

Gauss supposed that

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\item
  small errors were more likely than large errors.
\item
  error were symmetrical. The error at a distance \(-\epsilon\) from the real position of Ceres was equally probable than the error at \(+\epsilon\).
\item
  the most \textbf{likely} position (the one that we believe the most) of Ceres at any given time in the sky was the \textbf{average} of multiple measurements.
\end{enumerate}

That was enough to show that the deviations of the observations \(y\) \textbf{from the orbit} satisfied the equation \citep{Stahl2006}

\[\frac{df(y)}{dy}=-Cyf(y)\]
with \(C\) a positive constant. The solution of this differential equation is:

\[f(y)=\frac{\sqrt{C}}{\sqrt{2\pi}}e^{-\frac{Cy^2}{2}}\]

\hypertarget{normal-probability-density}{%
\section{Normal probability density}\label{normal-probability-density}}

Normal, or Gaussian, probability density gives the distribution of measurement errors from the \textbf{actual} but \textbf{unknown} position of Ceres in the sky. Let us make a couple of changes to the function.

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\tightlist
\item
  We write the error density from the horizon using the random variable \(X\), that is, \(y=x-\mu\). \(\mu\) is the \textbf{actual} but \textbf{unknown} position of Ceres from the horizon. After a change of variable we find the probability density function:
\end{enumerate}

\[f(x)=\frac{\sqrt{C}}{\sqrt{2\pi}}e^{-C(x-\mu)^2}\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{1}
\tightlist
\item
  We rename the variable \(C\) to \(\frac{1}{\sigma^2}\)
\end{enumerate}

We then arrive at the following definition.

\textbf{Definition:}

A random variable \(X\) defined on the real numbers has a \textbf{normal} probability density function if it takes the form

\[f(x; \mu, \sigma^2)=\frac{1}{\sqrt{2\pi}\sigma}e^{-\frac{(x-\mu)^2}{2\sigma^2}}, x \in {\mathbb R}.\]

A normally distributed variable \(X\) has mean

\[E(X) = \mu\]

which for Gauss represented the real position of Ceres from the horizon; and variance

\[V(X) = \sigma^2\]
which represented the dispersion of the error in the observations from the mean, and depended on the quality of the telescope and the skill of Piazzi.

\(\mu\) and \(\sigma\) are the natural parameters of the model, which fully describe the normal density function. Their interpretation depends on the nature and design of the random experiment.

It is important to underline from the outset that the parameters are not observable: they are abstract properties of the experiment. This is true not only for the normal model but for any statistical model. For instance, the true position of Ceres (\(\mu\)) was unknown; instead, Gauss obtained repeated measurements of Ceres's position \((x_1, x_2, \dots, x_n)\). How could he infer likely values of an abstract quantity such as \(\mu\) from empirical data \((x_1, x_2, \dots, x_n)\)? This is the central question we will address in the following chapters.

Let us pause to appreciate the depth of the problem. For empiricists, all knowledge comes from experience, and thus \(\mu\) is regarded as an abstraction derived from data. For idealists, knowledge arises from intellectual insight or reasoning, so \(\mu\) represents a fundamental aspect of reality, while the observations are imperfect reflections of it. The experimental scientist finds reassurance in the fact that the sample mean \(\bar{x}\), an empirical quantity, converges to \(\mu\), the underlying abstraction, as \(n\) increases.

For the moment, we are going to \textbf{assume} that we know the parameters \(\mu\) and \(\sigma\), so we can use the normal model to make predictions.

When \(X\) follows normal probability density function, we say that \(X\) is normally distributed with mean \(\mu\) and variance \(\sigma^2\) and write

\[X \sim N(\mu, \sigma^2) \]

Let us look at some probability densities in the normal parametric model

\includegraphics{_main_files/figure-latex/unnamed-chunk-115-1.pdf}

For Gauss, the change from the red to the yellow curve means that Ceres moved, and the change from the red to the blue curve means that the telescope was less accurate.

\hypertarget{probability-distribution-2}{%
\section{Probability distribution}\label{probability-distribution-2}}

The probability distribution of the normal density:

\[F(a)=P(Z \leq a)\]

is the \textbf{error} function defined by the area under the curve from \(-\infty\) to \(a\)

\[F(a)=\int_{-\infty}^{a}\frac{1}{\sqrt{2\pi}\sigma}e^{-\frac{(x-\mu)^2}{2\sigma^2}} dx\]
The function is found in most computer programs, and it does not have a closed form in terms of known functions.

\textbf{Example (Female height)}

Adolphe Quetelet applied the normal distribution to anthropometric data, proposing that the average of body measurements represented the ``l'homme moyen'' (the average man), which he saw as an idealized standard. Francis Galton, influenced by Quetelet, developed the concept of human variability as a defining property of populations. However, he also believed that selective breeding and eugenics could be used to shape populations toward an ideal. While both views are now considered ethically and scientifically incorrect and outdated, the introduction of normal distributions of traits was fundamental to the development of quantitative genetics.

In Galton's famous study on human height \citep{Galton1886}, he estimated that the mean of female height was \(163cm\) (\(64inch\)) with standard deviation of \(5.8cm\) (\(2.30inch\)).

What is the probability that a woman in Galton's population is at most \(150cm\)?

\[P(X\le 150)=F(150, \mu=165, \sigma=8)=0.012\]

which can be computed with the code

\begin{verbatim}
Python: norm.cdf(150, 163, 5.8)
R: pnorm(150, 163, 5.8)
\end{verbatim}

Therefore, the type of woman that Galton measured, presumably 19th century middle-class English, had only \(1.2\%\) chance to be smaller than \(150cm\).

What is the probability that a woman's height in the Galton's population is between \(165cm\) and \(170cm\)?

\[P(165 \le X \le 170)=F(170, \mu=165, \sigma=8)-F(165, \mu=165, \sigma=8)=0.2513\]

\begin{verbatim}
Python: norm.cdf(170, 163, 5.8)-norm.cdf(165, 163, 5.8)
R: pnorm(170, 163, 5.8)-pnorm(165, 163, 5.8)
\end{verbatim}

Let us look at the probability distribution function

\includegraphics{_main_files/figure-latex/unnamed-chunk-117-1.pdf}

The red bar in the left is the probability in example above.

What is the first quartile for female height?

The first quartile is defined as:

\[F(x_{0.25}, \mu=163, \sigma=5.8)=0.25\]

or

\[x_{0.25}=F^{-1}(0.25, \mu=163, \sigma=5.8)=159.088\]

\begin{verbatim}
Python: norm.ppf(0.25, 163, 5.8)
R: qnorm(0.25, 163, 5.8)
\end{verbatim}

Let us make the following prediction: If we could measure many women in Galton's population, then about \(25\%\) of them would be at most \(160cm\) tall. Note that, while we cannot perform this experiment, as Galton's population is no longer available, the probabilities, as outcome propensities, reasonably describe the intrinsic properties of that experiment.

\textbf{Example (Temperature)}

The statistical theory of gases provides another key example where the variance is interpreted as an intrinsic property of the system, rather than as measurement error.

Maxwell proposed that the velocity \(V\) of a particle in an ideal gas at equilibrium follows a \textbf{normal distribution}:

\[
f(v; \mu, \sigma^2) = \frac{1}{\sqrt{2\pi} \, \sigma} \, e^{- \frac{(v - \mu)^2}{2\sigma^2}}
\]

Assuming the gas is at rest, the mean velocity is \(\mu = 0\). For simplicity, we consider only one spatial dimension.

Earlier, Rudolf Clausius had related temperature to the average kinetic energy of gas particles, using the ideal gas law. He showed that:

\[
\frac{1}{2} k_B T = E\left(\frac{1}{2} m V^2\right)
\]

where \(k_B\) is Boltzmann's constant, and \(m\) is the mass of a particle. Given Maxwell's statistical description of molecular velocities, the kinetic energy becomes:

\[
E\left(\frac{1}{2} m V^2\right) = \frac{1}{2} m \, E(V^2) = \frac{1}{2} m \, \text{Var}(V) = \frac{1}{2} m \, \sigma^2
\]

Solving for \(T\), we obtain:

\[
T = \frac{m}{k_B} \sigma^2
\]

Thus, temperature is directly proportional to the variance of particle velocities. This gives a powerful statistical interpretation of thermodynamic temperature: a measure of the spread in molecular motion.

Although Maxwell could not directly measure particle speeds, his probabilistic velocity model made accurate predictions about macroscopic properties of gases. For instance, it predicted that the viscosity of a dilute gas should be independent of pressure, a result he later confirmed experimentally.

\hypertarget{quantiles-of-the-normal-distribution}{%
\section{Quantiles of the normal distribution}\label{quantiles-of-the-normal-distribution}}

The outcomes of a normal random variable are often described in three ranges:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  Since the mean \(\mu\) is also the median, it divides the measurements into two equal halves.
\item
  Values of \(x\) that lie more than \(2\sigma\) from the mean are considered \textbf{rare} (occurring in less than \(5\%\) of cases).
\item
  Values of \(x\) that lie more than \(3\sigma\) from the mean are considered \textbf{extremely rare} (occurring in less than \(0.2\%\) of cases).
\end{enumerate}

\includegraphics{_main_files/figure-latex/unnamed-chunk-118-1.pdf}

\textbf{Example (Female height)}

We can describe the limits of the common heights for the distribution of female height in Galton's population.

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\tightlist
\item
  at a distance of one standard deviation from the mean, between \((157.2, 168.8)\), we find \(68\%\) of the population
\end{enumerate}

\[P(163-5.8 \leq X \leq 163-5.8)=P(157.2 \leq X \leq 168.8)=F(168.8)-F(157.2)=0.68\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{1}
\tightlist
\item
  at a distance of two standard deviations from the mean, between \((151.4, 174.6)\), we find \(95\%\) of the population
\end{enumerate}

\[P(163-2 \times 5.8 \leq X \leq 163+2\times 5.8)=F(174.6)-F(151.4)=0.95\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{2}
\tightlist
\item
  at a distance of three standard deviations from the mean, between \((145.6, 180.4)\), we find \(99.7\%\) of the population
\end{enumerate}

\[P(163-3 \times 5.8 \leq X \leq 163+3\times 5.8)=F( 180.4)-F(145.6)=99.7\]

It is customary to say that the distribution of the random variable \(X\) is the population distribution. However, it is important to emphasize that this population is not a concrete reality but an abstract property of the random experiment. It is the map, not the territory.

When we speak of a population, we really mean the hypothetical repetition of the random experiment an infinite number of times, with the idea that we could eventually exhaust all possible measurement units. For example, if Galton could measure the entire population of women at a given time, the histogram of those heights would be very close to the distribution of \(X\). While the distribution could thus be approximated by the population histogram, it would never be identical to it---if only because exhausting all observational units is impossible.

Note, however, that it is still reasonable to talk about the probability that a hypothetical woman living in Galton's time would have a given height, even though she may never have existed and the population in question no longer exists. Probabilities can therefore be understood as statements about future or hypothetical outcomes of the experiment.

\hypertarget{standard-normal-density}{%
\section{Standard normal density}\label{standard-normal-density}}

The \textbf{standard} normal density is one member of the normal family, such that

\[f(x; \mu=0, \sigma^2=1)=\frac{1}{\sqrt{2\pi}}e^{-\frac{x^2}{2}}, x \in {\mathbb R}\]

The mean and variance are

\[E (X)= \mu = 0\]

and

\[V (X)=  \sigma^2 =1\]

When a random variable follows a standard normal probability density, we say that the variable is \textbf{standard normal} and write

\[X \sim N(0,1)\]

\hypertarget{standard-distribution}{%
\section{Standard distribution}\label{standard-distribution}}

The probability distribution of a standard normal variable is

\[\Phi(x)=F(x)=P(X \leq a)\]

Because the \textbf{standard} distribution is special and will appear often, we use the letter \(\Phi\) for it. This is the \textbf{error} function defined by

\[\Phi(x)=\int_{-\infty}^{x} \frac{1}{\sqrt{2\pi}}e^{-\frac{z^2}{2}} dz\]

You can find it in most computer programs.

\begin{verbatim}
Python: norm.cdf(x)
R: prnorm(x)
\end{verbatim}

We usually define the limits of the most common outcomes for the normal standard variable

\includegraphics{_main_files/figure-latex/unnamed-chunk-120-1.pdf}

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\tightlist
\item
  The interquartile range \[P(-0.67 \leq X \leq 0.67)=0.50\]
\end{enumerate}

\begin{verbatim}
Python: [norm.ppf(0.25), norm.ppf(0.75)]
R: c(qnorm(0.25),qnorm(0.75))
\end{verbatim}

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{1}
\tightlist
\item
  \(95\%\) range \[P(-1.96 \leq X \leq 1.96)=0.95\]
\end{enumerate}

\begin{verbatim}
Python: [norm.ppf(0.025), norm.ppf(0.975)]
R: c(qnorm(0.025),qnorm(0.975))
\end{verbatim}

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{2}
\tightlist
\item
  \(99\%\) range \[P(-2.58 \leq X \leq 2.58)=0.99\]
\end{enumerate}

\begin{verbatim}
Python: [norm.ppf(0.005), norm.ppf(0.995)]
R: c(qnorm(0.005),qnorm(0.995))
\end{verbatim}

\hypertarget{standardization}{%
\section{Standardization}\label{standardization}}

All normal variables can be \textbf{standardized}. This means that if \(X \sim N(\mu, \sigma^2)\), then we can transform the variable to
a \textbf{standardized variable}

\[Z=\frac{X-\mu}{\sigma}\]

which will have density:

\[f(z)=\frac{1}{ \sqrt{2\pi}}e^{-\frac{z^2}{2}}\]
Therefore, for any \(X \sim N(\mu, \sigma^2)\)

\[Z=\frac{X-\mu}{\sigma} \sim N(0, 1) \]

\includegraphics{_main_files/figure-latex/unnamed-chunk-121-1.pdf}

We can demonstrate this by replacing \(x=\sigma z+\mu\) and \(dx=\sigma dz\) in the probability equation

\(P(x\leq X \leq x +dx)=P(z\leq Z \leq z +dz)\)
\[=\frac{1}{\sqrt{2\pi}\sigma}e^{-\frac{(x-\mu)^2}{2\sigma^2}}dx\] \[=\frac{1}{ \sqrt{2\pi}}e^{-\frac{z^2}{2}} dz\]

One of the advantages of standardization if that we can compute the probability of any normal variable \(X\sim N(\mu, \sigma^2)\) using the standard distribution

\(F(a)=P(X<a)=P(\frac{X-\mu}{\sigma}<\frac{a-\mu}{\sigma})\)
\[=P(Z < \frac{a-\mu}{\sigma})= \Phi \big(\frac{a-\mu}{\sigma}\big)\]

For computing \(P(a\leq X \leq b)\), we use the property of the probability distributions

\(F(b)-F(a)=P(X\leq b)-P(X\leq a)\)

\[=\Phi \big(\frac{b-\mu}{\sigma}\big)-\Phi \big(\frac{a-\mu}{\sigma}\big)\]

Therefore \(\Phi(x)\) is the only function we need to compute probabilities for any normal random variable. Before computers, this property let statisticians rely on a single set of tables, and it still shows how all the randomness of a normal experiment can be captured by a probability function that is free of model‑specific parameters.

\hypertarget{questions-6}{%
\section{Questions}\label{questions-6}}

\textbf{1)} It is not true that for a normally distributed variable

\textbf{\(\qquad\)a:} its mean and median are the same; \textbf{\(\qquad\)b:} the standard probability distribution can be used to compute its probabilities; \textbf{\(\qquad\)c:} its interquartile range is twice its standard deviation; \textbf{\(\qquad\)d:} \(5\%\) of its observations are a distance greater than twice its standard deviation

\textbf{2)} For a normal standard variable

\textbf{\(\qquad\)a:} \(50\%\) of its observations are between \((-0.67,0.67)\);
\textbf{\(\qquad\)b:} \(2\%\) of its observations are lower than \(-2.58\);
\textbf{\(\qquad\)c:} \(5\%\) of its observations are greater than \(1.96\);
\textbf{\(\qquad\)d:} \(25\%\) of its observations are between \((-1.96,-0.67)\)

\textbf{3)} if we know that \(\Phi(-0.8416212)=0.2\) then what is \(\Phi(0.8416212)\)

\textbf{\(\qquad\)a:} \(0.1\);
\textbf{\(\qquad\)b:} \(0.2\);
\textbf{\(\qquad\)c:} \(0.8\);
\textbf{\(\qquad\)d:} \(0.9\)

\textbf{4)} the third quartile of a normal variable with mean \(10\) and standard deviation \(2\) is

\textbf{\(\qquad\)a:} norm.ppf(1/3, 10, 2)=9.138545;
\textbf{\(\qquad\)b:} norm.ppf(1-0.75, 10, 2)=8.65102 ;
\textbf{\(\qquad\)c:} norm.ppf(1-1/3, 10, 2)=10.86145 ;
\textbf{\(\qquad\)d:} norm.ppf(0.75, 10, 2)= 11.34898

\textbf{5)} probability that a normal variable with mean \(10\) and standard deviation \(2\) is in \((-\infty,10)\) is

\textbf{\(\qquad\)a:} 0.25;
\textbf{\(\qquad\)b:} 0.5;
\textbf{\(\qquad\)c:} 0.75;
\textbf{\(\qquad\)d:} 1:

\hypertarget{exercises-7}{%
\section{Exercises}\label{exercises-7}}

\hypertarget{exercise-1-6}{%
\subsubsection{Exercise 1}\label{exercise-1-6}}

Find the area under the standard normal curve in the following cases:

\begin{itemize}
\tightlist
\item
  Between \(z=0.81\) and \(z=1.94\) (A:0.182)
\item
  To the right of \(z=-1.28\) (A:0.899)
\item
  To the right of \(z=2.05\) or to the left of \(z=-1.44\) (A:0.0951)
\end{itemize}

\hypertarget{exercise-2-6}{%
\subsubsection{Exercise 2}\label{exercise-2-6}}

\begin{itemize}
\item
  What is the probability that a man's height is at least
  \(165\)cm if the population mean is \(175\)cm y the standard deviation is \(10\)cm? (A:0.841)
\item
  What is the probability that a man's height is between
  \(165\)cm and \(185\)cm? (A:0.682)
\item
  What is the height that defines the \(5\%\) of the smallest men? (A:158.55)
\end{itemize}

\hypertarget{sampling-distributions}{%
\chapter{Sampling distributions}\label{sampling-distributions}}

When making predictions, the probabilities of the outcomes of an experiment are considered fundamental properties of the system or process under study. To carry out these predictions, we require a model and its parameters to fully specify the probability mass (or density) function that describes the propensities of the outcomes.

These parameters may be derived from the experimental design and the underlying theory supporting the experiment. However, when our goal is not just to make predictions but to test a theory or model, we must be able to estimate the parameters from experimental data and compare them to the theoretical values. It is also common to encounter situations where no established theory predicts the value of a parameter---particularly when it represents an unknown physical property of a system. In such cases, the primary objective is to estimate this parameter and draw conclusions based on its value.

As with probabilities, the parameters of a probabilistic model are abstract and unobservable, but they can be partially revealed through repeated experimentation.

In this chapter, we study the estimation of the mean and variance of normal distributions using random samples. These two quantities are the natural parameters of the normal distribution. The mean represents the expected value of the experiment and often directly corresponds to the physical property of interest---such as the mass or charge of the electron, or the longevity of a pacemaker. The variance, on the other hand, can reflect the uncertainty introduced by the experimental process itself (as in the case of J.J. Thomson's measurements), or variability among observational units (such as patients in a clinical trial of pacemakers).

We will introduce the sample mean (average) and the sample variance as random variables used to estimate these parameters. Both the sample mean and the sample variance have their own probability density functions, known as sampling distributions. We will use these distributions to quantify how precisely we can estimate the mean and variance of the random experiment from data.

\hypertarget{random-sample}{%
\section{Random sample}\label{random-sample}}

\textbf{Example (Electron's charge to mass ratio)}

In 1897, J. J. Thomson was the first to measure the charge-to-mass ratio (\(q/m\)) of electrons \citep{Thomson1897eoverm}, demonstrating that they were the constituents of cathode rays and significantly smaller than hydrogen atoms. Although Thomson originally reported his results as the mass-to-charge ratio (\(m/e\)), here we present them as \(q/m\) values, expressed in units of \(Coulomb/Kg \times 10^{11}\), as illustrated below

\includegraphics{_main_files/figure-latex/unnamed-chunk-123-1.pdf}

We say that J. J. Thomson took a \textbf{random sample} of size 26, meaning he performed 26 independent repetitions of the random experiment: measuring the \(q/m\) ratio under varying experimental conditions.

The experiments were conducted using different types of gases, yet the observations appeared to cluster around a single \textbf{unknown} value.

\textbf{Definition:}

A \textbf{random sample} of size \(n\) is the \textbf{repetition} of a random experiment \(n\) \textbf{independent} times.

\begin{itemize}
\item
  A random sample is an \(n\)-dimensional \textbf{random variable}:

  \[(X_1, X_2, \dots, X_n)\]
\end{itemize}

where \(X_i\) is the random variable from \emph{i-th} repetition of the random experiment that measures the \(q/m\) ratio, each subject to measurement error and with the same probability density function \(f(x)\) for all \(i\).

\begin{itemize}
\item
  \textbf{An observation} of a random sample is the set of \(n\) values obtained from performing the experiment:

  \[(x_1, x_2, \dots, x_n)\]
\end{itemize}

\textbf{Example (Electron Measurement)}

The \textbf{observation} of a sample of size \(26\) for the \(q/m\) ratio is a vector with \(26\) components:

(1.75, 2.94, 2.32, 3.12, 2.08, 2.50, 2.50, 2.85,
2.00, 2.50, 2.50, 2.56, 1.88, 2.12, 2.12, 1.88,
1.96, 1.85, 1.58, 1.88, 2.17, 1.63, 2.08, 1.11,
1.42, 1.00)

Clearly, the quantity \(q/m\) is a parameter of the physical system---an intrinsic property, a constant. However, the process of measuring this quantity---namely the random variable representing each specific experimental setup and the formulas used to compute the values---varied in each repetition, producing different outcomes due to measurement error or experimental noise.

J.J. Thomson did not directly observe the value of \(q/m\) but rather measurements---realizations of the random variables \(X_i\)---each subject to experimental variability, with the expected value of these measurements equal to \(q/m\). At the time, there was no theoretical model predicting its value. The experiments were exploratory, and knowledge emerged not from theory, but from empirical observation.

How did J.J. Thomson estimate the value of \(q/m\) from the data?

To move from raw measurements to a meaningful scientific conclusion, J.J. Thomson needed a systematic way to combine these varied observations into a single best estimate of the expected charge-to-mass ratio. In the next section, we will explore how statistical methods allow us to estimate unknown parameters from random samples like this one.

\hypertarget{parameter-estimation}{%
\section{Parameter estimation}\label{parameter-estimation}}

To find likely values for parameters, we use data. Therefore, we take a \textbf{random sample}---that is, we repeat the experiment \(n\) times, collect data, and use it to \textbf{estimate} the parameters.

\textbf{Estimates of the Mean and Variance}

Recall that for a discrete random variable, the mean (or expected value) is defined using the number of possible outcomes \(m\) and the probability mass function \(f(x)\):

\[
\mu = \sum_{i=1}^m x_i f(x_i)
\]

This formula represents the \textbf{center of mass of the probabilities}, where \(f(x_i)\) is the probability of the outcome \(x_i\).

This concept is closely related to the \textbf{sample average}, which we define as:

\[
\bar{x} = \frac{1}{n} \sum_{i=1}^n x_i
\]

Here, the \(x_i\)'s are the \textbf{observed values} in a random sample of size \(n\). We can also express this average as a weighted sum over the distinct observed values \(x_i\) with their associated \textbf{relative frequencies} \(f_i\) (i.e., the proportion of times each outcome appears in the sample):

\[
\bar{x} = \sum_{i=1}^m x_i f_i
\]

If the relative frequencies \(f_i\) converge to the probabilities \(f(x_i)\) as \(n \to \infty\), then the average \(\bar{x}\) will converge to the mean \(\mu\). For finite \(n\), we treat the average as an estimate of the mean:

\[
\hat{\mu} = \bar{x}
\]

Note that \(\hat{\mu}\) is not the same as \(\mu\). The value \(\hat{\mu}\) is a \textbf{statistic}---a quantity derived from the data---while \(\mu\) is a \textbf{parameter}, an inherent (but typically unknown) property of the underlying probability distribution. They are conceptually equal only in the limit as \(n \to \infty\), i.e., if we could repeat J.J. Thomson's experiment an infinite number of times under identical conditions.

When the average is calculated from a small sample, we know it will contain some error, which we will try to quantify latter.

Similarly, the \textbf{variance} of a discrete random variable is defined in terms of the probability mass function as:

\[
\sigma^2 = \sum_{i=1}^m (x_i - \mu)^2 f(x_i),
\]

which represents the expected squared deviation from the mean.

From the observed values of a sample of size \(n\), we compute the \textbf{sample variance} as:

\[
s^2 = \frac{1}{n - 1} \sum_{i=1}^n (x_i - \bar{x})^2,
\]

The use of \(n - 1\) will be explained later. If we express the sample variance in terms of the distinct values \(x_i\) and their relative frequencies \(f_i\), we obtain:

\[
s^2 = \frac{n}{n - 1} \sum_{i=1}^m (x_i - \bar{x})^2 f_i.
\]

As the sample size \(n\) increases, the sample variance \(s^2\) converges to the variance \(\sigma^2\). Thus, we take \(s^2\) as an estimate of \(\sigma^2\) when working with finite samples.

\textbf{Example (Electron)}

We \textbf{assume} that the value obtained for the \(q/m\) experiment follows a normal probability density function.

\[X \sim N(x; \mu, \sigma^2)\]
Where the parameters \(\mu=q/m\) and \(\sigma\) are unknown.

The average of J. J. Thomson's sample is \(\bar{x}=2.09\)

\begin{verbatim}
Python: np.mean(x)
R: mean(x)
\end{verbatim}

and the sample variance is \(s^2=0.51^2=0.26\)

\begin{verbatim}
Python: np.var(x, ddof=1)
R: var(x)
\end{verbatim}

that we can take as the estimated values of \(\mu\) and \(\sigma^2\). So the \textbf{fitted} model is

\[X \sim N(x; \mu=2.09, \sigma^2=0.51^2)\]
\includegraphics{_main_files/figure-latex/unnamed-chunk-124-1.pdf}

The fitted distribution represents J.J. Thomson's experiment with its mean and its variance. The sample average was not expected to exactly equal the value of \(q/m\), as it is subject to random error. Had he taken another sample, the average would likely have been different. By contrast, \(q/m\) appears to be a physical invariant with no associated dispersion. Today's more accurate value of \(q/m\) is

\(q/m \sim \frac{1.602176634\times 10^{-19}}{9.1093837015\times 10^{−31}} C/Kg =\) \[1.758820024\times 10^{11} C/Kg\]

The error that J.J. Thomson's made in his 1897 paper was \(0.33\times 10^{11} C/kg\), about \(18\%\) in relation to the current estimate.

Is this an expected error of his experiment? If he was to re-do his experiment how close to the current value of \(q/m\) he would typically be?

\hypertarget{law-of-large-numbers}{%
\section{Law of Large Numbers}\label{law-of-large-numbers}}

When we estimate parameters using data---for example, by using\\
\[
\hat{\mu} = \bar{x}
\]\\
as an estimate of \(\mu\), or\\
\[
\hat{\sigma}^2 = s^2
\]\\
as an estimate of \(\sigma^2\)---we know that we are introducing an error. If we were to take a different sample of \(q/m\) measurements (say, of size 24), the estimate would likely change, because the sample average \(\bar{x}\) depends on the data and is therefore variable.

This raises an important question:

When we perform an experiment, can we quantify how good our estimate of the parameter is?

\textbf{The Sample Mean as a Random Variable}

The first thing to recognize is that the numerical value we calculate, \(\bar{x}\), is just one observation of a random variable, which we denote by \(\bar{X}\).

\textbf{Definition:}

The \textbf{sample mean} (or average) of a random sample of size \(n\) is defined as:
\[
\bar{X} = \frac{1}{n} \sum_{i=1}^n X_i
\]
Here, \((X_1, X_2, \dots, X_n)\) are independent and identically distributed (i.i.d.) random variables with mean \(\mu\) and variance \(\sigma^2\), and constitute the \(n\)-dimensional random variable of the sample.

The average \(\bar{X}\) is itself a random variable. For instance, in our sample of 24 measurements, we observed:
\[
\bar{x} = 2.09
\]
Had we taken another sample of size 24, the average would likely have been different.

\textbf{Why Use \(\bar{X}\) to Estimate \(\mu\)?}

The value \(\bar{x}\) can be used to estimate the unknown parameter \(\mu = q/m\) because it is an observation of the random variable \(\bar{X}\) with two very important and general properties:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  Unbiasedness:\\
  \[
  E(\bar{X}) = \mu
  \]
\item
  Consistency as \(n \to \infty\):\\
  \[
  lim_{n \to \infty} V(\bar{X}) = 0
  \]
\end{enumerate}

\textbf{Demonstration of the Properties}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  Expected value of the mean:
  \[
  E(\bar{X}) = E\left(\frac{1}{n} \sum_{i=1}^n X_i\right) = \frac{1}{n} \sum_{i=1}^n E(X_i) = \mu
  \]
  This holds because the \(X_i\) are i.i.d. random variables with expected value \(\mu\).
\item
  Variance of the mean:
  \[
  V(\bar{X}) = V\left(\frac{1}{n} \sum_{i=1}^n X_i\right) = \frac{1}{n^2} \sum_{i=1}^n V(X_i) = \frac{\sigma^2}{n}
  \]
  Since the variance shrinks as \(1/n\), the sample mean becomes more stable as the sample size increases.
\end{enumerate}

\textbf{Estimating \(\mu\)}

These two properties imply that \(\bar{x}\), the observed average, converges closer and closer to the value \(\mu\) as the sample size \(n\) increases.

This phenomenon is known as the \textbf{law of large numbers}. It holds regardless of the underlying probability distribution of the \(X_i\), provided they are independent and identically distributed with finite variance.

In other words:

\begin{itemize}
\item
  The estimation error in using \(\bar{x}\) as an approximation for \(\mu\) decreases as \(n\) increases.
\item
  This is because the variability (variance) of \(\bar{X}\) decreases with \(n\).
\end{itemize}

So when we write:
\[
\bar{x} = \hat{\mu}
\]
we understand that this estimate becomes increasingly reliable as the sample grows.

\hypertarget{inference}{%
\section{Inference}\label{inference}}

We know that when we take large samples, the estimation error tends to be small. However, for a given sample size \(n\), we would like to quantify how far our estimate might be from the parameter value. This gives us an idea of how accurate our estimate is. Specifically, we may ask:

What is the probability of making an error of a given size when estimating \(\mu\) with \(\bar{x}\)?

When we calculate probabilities related to an estimator, we are performing \textbf{inference}. Inference problems often involve questions such as:

\begin{itemize}
\tightlist
\item
  What is the probability that the estimate \(\bar{x}\) differs from the parameter \(\mu\) by no more than some margin \(m\)?
\end{itemize}

Mathematically, this is expressed as:
\[
P(-m \leq \bar{X} - \mu \leq m)
\]
This represents the probability that the difference between the random variable \(\bar{X}\) (the sample mean) and the parameter \(\mu\) lies within a range of size \(2m\).

To calculate such probabilities, we need:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  A probability model for \(\bar{X}\) (i.e., its probability distribution)
\item
  The parameters of that model (e.g., the mean and variance of the distribution)
\end{enumerate}

What, then, is the probability distribution of \(\bar{X}\), and what about the distribution of \(S^2\) (the sample variance), so that we can calculate the relevant probabilities?

These probability functions are known as \textbf{sampling distributions} because they describe the behavior of statistics like \(\bar{X}\) and \(S^2\) when we take repeated samples. Understanding these distributions allows us to calculate their probabilities and make informed statements about how close our estimates are likely to be to the parameters.

\textbf{Example (Electron)}

We can pose an inference question based on J.J. Thomson's experiment. the mean \(\mu\) is unknown, but the sample average from 24 repetitions is \(\bar{x} = 2.09\) (in \(10^{11} C/kg\) units). Assuming that each experimental measurement follows a normal distribution and that the standard deviation is approximately \(\hat{\sigma} = 0.51\), what is the probability that the sample mean \(\bar{X}\) lies within \(0.33\) of the mean \(\mu\)?

In other words, we want to find the probability that the sample mean is within a \textbf{margin of error} of \(0.33\) from \(\mu\):

\[
P(-0.33 \leq \bar{X} - \mu \leq 0.33)
\]

To calculate this probability, we need to know the probability distribution of \(\bar{X}\).

\hypertarget{sample-mean}{%
\section{Sample mean}\label{sample-mean}}

\textbf{Theorem:} If the random variable \(X\) follows a normal distribution

\[
X \sim N(\mu, \sigma^2),
\]

then the sample mean \(\bar{X}\) is also normally distributed as

\[
\bar{X} \sim N\left(\mu, \frac{\sigma^2}{n}\right).
\]

This fits perfectly with the law of large numbers because:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  The \textbf{mean} of \(\bar{X}\) is exactly the parameter \(\mu\) we want to estimate:
\end{enumerate}

\[
E(\bar{X}) = \mu,
\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{1}
\tightlist
\item
  The \textbf{variance} of \(\bar{X}\) gets smaller as the sample size \(n\) grows:
\end{enumerate}

\[
V(\bar{X}) = \frac{\sigma^2}{n}.
\]

We say that \(\bar{X}\) is an \emph{unbiased estimator} of \(\mu\), meaning that on average it hits the value---like a dart player who on average hits the bull's-eye.

It is also a \emph{consistent estimator} because as \(n\) increases, the variance (or ``spread'') of \(\bar{X}\) gets smaller---like a dart player who improves their aim and gets closer to the bull's-eye every time.

The quantity

\[
se = \sqrt{V(\bar{X})} = \frac{\sigma}{\sqrt{n}}
\]

is called the \textbf{standard error} of the sample mean. It tells us how much we expect our sample mean \(\bar{x}\) to typically vary from the mean \(\mu\). Sometimes it is written as \(\sigma_{\bar{x}}\), to emphasize that it is the standard deviation of \(\bar{X}\).

Because \(\bar{X}\) is normal when \(X\) is normal, we can use the normal distribution to calculate probabilities related to \(\bar{X}\).

For example, suppose we want to find the probability that our estimate \(\bar{X}\) lies within a margin of error \(m\) from \(\mu\):

\[
P(-m \leq \bar{X} - \mu \leq m).
\]

Using the fact that \(\bar{X} \sim N\left(\mu, \frac{\sigma^2}{n}\right)\), we can standardize this:

\[
P\left(-\frac{m}{\sigma/\sqrt{n}} \leq \frac{\bar{X} - \mu}{\sigma/\sqrt{n}} \leq \frac{m}{\sigma/\sqrt{n}}\right) = P(-z \leq Z \leq z),
\]

where

\[
z = \frac{m}{\sigma/\sqrt{n}},
\]

and \(Z\) is a standard normal variable.

This allows us to calculate exactly how likely it is that our sample mean is close to the mean within the margin \(m\).

\textbf{Example (Electron)}

Suppose we want to know the probability that the sample mean \(\bar{X}\) is within a margin of error \(m = 0.33\) from the mean \(\mu\).

We assume the sample variance estimate \(\hat{\sigma} = 0.51\) is close to \(\sigma\), and we have a sample size \(n = 24\). (Later, Gosset corrected for small samples using the t-distribution, but for now we treat \(n=24\) as large enough.)

The standardized margin is

\[
z = \frac{0.33}{0.51 / \sqrt{24}} \approx 3.17.
\]

The probability that the sample mean is within \(\pm 0.33\) of \(\mu\) is then

\[
P\left(-3.17 \leq Z \leq 3.17\right) = \Phi(3.17) - \Phi(-3.17) \approx 0.99,
\]

where \(\Phi\) is the cumulative distribution function (CDF) of the standard normal distribution.

In code:

\begin{verbatim}
Python: norm.cdf(3.17) - norm.cdf(-3.17)
R: pnorm(3.17) - pnorm(-3.17)
\end{verbatim}

This means about 99\% of the averages from repeated samples of size 24 would lie within 0.33 of the mean \(\mu\).

If we take the most recent and precise value of \(q/m \approx 1.75\) (in \(10^{11} C/kg\) units) as a better estimate to the mean, this tells us that most samples similar to J.J. Thomson's would produce averages close to that value with a precision of about \(0.33\).

Note that the value of \(q/m\) is not directly observable and likely has infinite, unpredictable decimal places (i.e.~is a normal number). What J.J. Thomson discovered was a consistent pattern in his measurements --- a regularity that later researchers could replicate with increasing accuracy in other experiments.

\hypertarget{prediction}{%
\section{Prediction}\label{prediction}}

While some random experiments aim to infer a property of the system under study, others aim to make predictions about particular outcomes. However, if no theoretical knowledge is available about the parameters of the probabilistic model, these parameters must first be estimated---often by performing a large number of repetitions of the experiment.

\textbf{Example (Pacemaker Prediction)}

The Micra Transcatheter Pacemaker is a miniaturized pacemaker delivered percutaneously with a high success rate. A key measure of its clinical efficacy is the prediction of \textbf{minimum virtual battery longevity (VBL)}.

In an early study of 630 patients, the estimated battery longevity---based on usage conditions observed at 12 months---was \textbf{12.1 years}, with \textbf{89\%} of patients projected to have a longevity of \textbf{more than 10 years} (corresponding to a standard deviation of \(1.71\)) \citep{Duray2017Micra12mo}.

Devices with projected longevity less than 5 years are unlikely to be marketed or adopted, especially for young or high-risk patients. Fortunately, \textbf{none of the patients} in this study had a projected longevity below 5 years.

\textbf{Question:} How can we compute the probability that a Micra pacemaker has a projected longevity of less than 5 years?

We \textbf{assume} the longevity of pacemakers follows a \textbf{normal distribution}:

\[
X \sim \mathcal{N}(\mu, \sigma^2)
\]

To compute \(P(X \leq 5)\), we need values for the parameters \(\mu\) and \(\sigma^2\).

The reported mean is:\\
\[
  \bar{x} = 12.1
  \]

\begin{verbatim}
Python: np.mean(x)
R: mean(x)
\end{verbatim}

and \(s^2=1.71^2=2.9\)

\begin{verbatim}
Python: np.var(x, ddof=1)
R: var(x)
\end{verbatim}

that we can take as the estimated values of \(\mu\) and \(\sigma^2\). So the \textbf{fitted} model is

\[X \sim N(x; \mu=12.1, \sigma^2=1.71^2)\]

\includegraphics{_main_files/figure-latex/unnamed-chunk-125-1.pdf}

The probability that a patient is implanted with a pacemaker less than 5 years longevity can be computed using the cumulative distribution function of a normal distribution:

\[
P(X \leq 5) = F(5; \mu = 12.1, \sigma^2 = 1.71^2) = 0.00001
\]

\begin{verbatim}
Python: norm.cdf(5, 12.1, 1.71)
R: pnorm(5, 12.1, 1.71)
\end{verbatim}

That is, the estimated probability is about \(0.001\%\).

This provides a probabilistic argument supporting the adoption of the pacemaker, even for children and high-risk patients. As of 2025, it is estimated that 200,000 Micra devices have been implanted. Therefore, we expect only about 2 patients with pacemaker longevity under 5 years.

If such rare cases occur, they may be considered unfortunate but extremely low-probability events---comparable to or even lower than other everyday risks, such as dying in a car accident (roughly 1 in 100).

When inferring the parameters of a random experiment, we require as many repetitions as possible to produce reliable estimates. When parameters are entirely unknown, inferences from large studies are treated as the \textbf{gold standard}.

\hypertarget{validation}{%
\section{Validation}\label{validation}}

When the parameters of a random experiment have been estimated using random variables with reasonable distributional properties---such as clustering around the expected value---the stability of those estimates can be tested by performing a new set of experiments under different conditions. Repeated validation of parameters that represent properties of a system strengthens the notion that the property in question reflects a fundamental aspect of the system.

\textbf{Example (Pacemaker Validation)}

Random experiments are characterized by some experimental conditions. In the pacemaker study there are some inclusion and exclusion criteria, for example patients who are entirely pacemaker dependent were excluded for safety reasons. Pacemaker configuration can change for those patients. Imagine that we want to run a new study on 10 of those patients and measure the VBL for each of them. We then need to calculate the probability that the estimate we obtain for \(\mu\) in our new study is within a margin of error from the gold standard taken as true values \(\mu\) and \(\sigma^2\). We can compute, for instance, the probability that \(\bar{X}\) is within 1 year from \(\mu\).

\emph{Probability densities for \(X\) and \(\bar{X}\)}

Remember that we have \textbf{two probability functions}:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  The probability function of \(X\) that is also the probability function of the \textbf{random experiment}.
\item
  The probability function of \(\bar{X}\) that is probability function of the \textbf{sample} (a collection of independent random experiments).
\end{enumerate}

Let us assume that the the longevity of the pacemakers is

\[X \sim N(\mu=12.1, \sigma^2=1.70^2)\]

\includegraphics{_main_files/figure-latex/unnamed-chunk-126-1.pdf}

In our new study, we plan to obtain a sample of \(10\) patients (like the blue crosses on the plot). Since \(X\) is normally distributed, the sample mean \(\bar{X}\) is also normally distributed. Therefore, we know the probability distribution of the sample mean \(\bar{X}\)

\[\bar{X} \sim N(12.1 , \frac{1.70^2}{10})\]

\includegraphics{_main_files/figure-latex/unnamed-chunk-127-1.pdf}

We can see that the outcomes of \(\bar{X}\) (like the black dot in the plot) will be more tightly concentrated around \(\mu\) than those of the random variable \(X\). In fact, when \(n\) is large, the sample means become so close to \(\mu\) and the distribution so sharply peaked that a single observed average (black dot) becomes a sufficiently accurate estimate of \(\mu\).

We want to calculate \textbf{the probability} that our estimate is within a margin of error of \(1\) year. That is, within a distance of \(1\) from the mean:

\(P(-1 \leq \bar{X} - 12.1 \leq 1) = P(11.1 \leq \bar{X} \leq 13.1)\)

Remember the standard error is: \(se=\frac{\sigma}{\sqrt{n}}=\frac{1.7}{\sqrt{10}}= 0.53\).

\[=\Phi(13.1; 14.1, 0.53)-\Phi(10.1; 14.1, 0.53)=0.93\]

\begin{verbatim}
Python: norm.cdf(14.1, 12.1, 0.53)-norm.cdf(10.1, 12.1, 0.53)
R: pnorm(13.1, 12.1, 1.7/sqrt(10))-pnorm(11.1, 12.1, 1.7/sqrt(10))
\end{verbatim}

We can, therefore, expect that \(93\%\) of the sample means from new studies with a sample size of 10 will fall within the interval \((11.1, 13.1)\), providing reassurance about the safety of the device for high-risk patients in our new study.

\hypertarget{sample-sum}{%
\section{Sample Sum}\label{sample-sum}}

Some random experiments consist of the collection of several identical trials, where the aim is to measure the combined contributions of each of them. Think, for example, of the collective effect of identical non-interacting molecules in a gas. In such cases, we may be interested in the \textbf{sum} of the outcomes of each experiment.

The \textbf{sample sum} is the \textbf{random variable}

\[
Y = n \bar{X} = \sum_{i=1}^n X_i
\]

which is a function of the random sample \((X_1, \ldots, X_n)\).

\textbf{Example (Cables)}

Imagine that a client asks a metallurgical company to supply 8 cables that together can carry up to 96 tons---that is, 12 tons each. The engineer needs to guarantee that none of the cables will break when loaded with this weight.

In \textbf{stock}, there are cables, all produced under the same specifications, that might be suitable. So, the engineer selects 8 of these cables at random and loads each until it breaks. Here are the results (in tons):

13.34642, 13.32620, 13.01459, 13.10811, 12.96999, 13.55309, 13.75557, 12.62747

If the company intends to use all 8 cables simultaneously to carry a total of 96 tons, we should consider \textbf{adding} their individual capacities. The observed value for the sample sum is:

\[
y = 13.21268 \times 8 = 105.70144 \text{ tons}
\]

What is the probability that when we put all the cables together, they can carry a total weight within 2 tons of the total expected weight that the cables can support?

\textbf{Distribution of the sample sum}

\textbf{Theorem:} if \(X\) follows a normal distribution
\[X \sim N(\mu, \sigma^2)\]

then \(Y\) is normal

\[Y \sim N(n\mu, n\sigma^2)\]

and \(Y\) has

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\tightlist
\item
  mean \[E(Y)=n\mu\]
\item
  variance \[V(Y)=n\sigma^2\]
\end{enumerate}

As we have the distribution of \(Y\) we can compute probabilities on it.

\textbf{Example (Cables)}

If the breaking load of the cables have been previously certified with a gold standard experiment to be a normal variable

\[X \sim N(\mu=13, \sigma^2=0.35^2)\]

then the sample sum of size \(8\) is normal

\[Y \sim N(n\mu=104, n\sigma^2=8\times 0.35^2)\]

with mean and variance

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\tightlist
\item
  \(E(Y)=n\mu=104\)
\item
  \(V(Y)=n\sigma^2=8\times 0.35^2=0.98\); \(\sqrt{V(Y)}=0.9899495\)
\end{enumerate}

The probability that when we put all the cables together, they can carry a total weight between \(102=104-2\) and \(106=104+2\) can be computed using the normal distribution function

\(P(102 \leq Y \leq 106)=\Phi(102; 104, 8\times 0.35^2)-\Phi(106; 104, 8\times 0.35^2)=0.956\)

We can calculate it as:

\begin{verbatim}
Python: norm.cdf(106, 104, 0.9899495)-norm.cdf(102, 104, 0.9899495)
R: pnorm(106, 104, 0.9899495)-pnorm(102, 104, 0.9899495)
\end{verbatim}

Therefore, we can expect that \(95.6\%\) of the total weight that 8 cables can carry lies between \(102\) and \(106\) Tons, which represents a margin of error of \(2\) Tons around the total mean \(n\mu = 104\) Tons.

This probability is equal to the probability that a single cable breaks between \(13-\sqrt{2/8}\) and \(13-\sqrt{2/8}=13.25\) Tons.

The margin of error does not scale linearly with the number of cables; instead, it increases with the square root of the sample size, \(\sqrt{n}\). This is because the individual variations tend to cancel each other out when added---some cables break above the average strength, others below.

This slower growth of uncertainty ensures that collective measurements (like the total strength) become more accurate and reliable as the number of independent components increases, which is a fundamental principle of statistical inference.

\hypertarget{sample-variance-1}{%
\section{Sample Variance}\label{sample-variance-1}}

The variance is another key parameter of a random experiment. In some cases, it may reflect an intrinsic property of the system; in others, it may relate to the precision of the measurement process. When we estimate the variance of a random experiment using the observed value of the sample variance

\[
s^2 = \hat{\sigma}^2
\]

we are also introducing an estimation error. So how can we quantify the error we make?

\textbf{Definition}
Just as we defined the \textbf{sample mean} \(\bar{X}\) as an estimator of the population mean \(\mu\), we now turn to estimating the population variance \(\sigma^2\).

The \textbf{sample variance} \(S^2\) of a random sample of size \(n\) is defined as

\[
S^2 = \frac{1}{n - 1} \sum_{i=1}^n (X_i - \bar{X})^2
\]

It measures the dispersion of the sample values around the sample mean \(\bar{X}\). The denominator \(n - 1\), rather than \(n\), ensures that \(S^2\) is an unbiased estimator of the population variance \(\sigma^2\). This correction, known as Bessel's correction, accounts for the fact that we have used the sample mean \(\bar{X}\) --- an estimate of \(\mu\) --- in computing the deviations.

\hypertarget{example-cables}{%
\subsection{Example (Cables)}\label{example-cables}}

In a sample of \(8\) cables, the sample variance took the value:

\[
s^2 = \frac{1}{n - 1} \sum_{i=1}^n (x_i - \bar{x})^2 = 0.1275608
\]

The sample variance \(S^2\) is:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  \textbf{Unbiased}: Its expected value equals the true variance:\\
  \[
  E(S^2) = V(X) = \sigma^2
  \]
\item
  \textbf{Consistent}: Its variance decreases as the sample size increases:\\
  \[
  V(S^2) \to 0 \quad \text{as} \quad n \to \infty
  \]
\end{enumerate}

Therefore, \(S^2\) is both an \textbf{unbiased} and \textbf{consistent} estimator of \(\sigma^2\). This justifies using the observed sample variance \(s^2\) as an estimate of the true variance:

\[
s^2 = \hat{\sigma}^2
\]

Just like \(\hat{\mu} = \bar{x}\), the error in estimating \(\sigma^2\) with \(s^2\) gets smaller as \(n\) becomes larger.

\textbf{Why Do We Divide by \(n - 1\)?}

One might propose estimating \(\sigma^2\) using:

\[
S_n^2 = \frac{1}{n} \sum_{i=1}^n (X_i - \bar{X})^2
\]

However, this version is:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  \textbf{Biased}:\\
  \[
  E(S_n^2) = \sigma^2 - \frac{\sigma^2}{n} \neq \sigma^2
  \]
\item
  but \textbf{Consistent}:\\
  \[
  V(S_n^2) \to 0 \quad \text{as} \quad n \to \infty
  \]
\end{enumerate}

The bias appears because \(S_n^2\) measures the spread around the sample mean \(\bar{X}\), not the true mean \(\mu\). The error introduced when substituting \(\bar{X}\) for \(\mu\) contributes an extra variance term \(\sigma^2 / n\).

We correct for this bias by multiplying \(S_n^2\) by \(\frac{n}{n - 1}\):

\[
E\left( \frac{n}{n - 1} S_n^2 \right) = \sigma^2
\]

This gives us the corrected (unbiased) \textbf{sample variance}:

\[
S^2 = \frac{n}{n - 1} S_n^2 = \frac{1}{n - 1} \sum_{i=1}^n (X_i - \bar{X})^2
\]

which satisfies:

\[
E(S^2) = \sigma^2
\]

\textbf{Example (Quality Control)}

We encounter inference problems when we are interested in the probability of observing a certain value for the \textbf{sample variance} \(S^2\).

Consider a quality control process in which cables are expected to have breaking loads close to a specified target value \(\mu\). We want to avoid producing cables whose breaking strength varies too much from the mean.

Suppose we take a sample of 8 cables. If the sample variance is too high (specifically, \(S^2 > 0.3\)), we stop production and declare that the process is out of control.

What is the probability that the sample variance from a sample of 8 cables exceeds the threshold of \(0.3\)?

\hypertarget{distribution-of-the-sample-variance}{%
\section{Distribution of the Sample Variance}\label{distribution-of-the-sample-variance}}

\textbf{Theorem:} If \(X \sim N(\mu, \sigma^2)\), then the random variable

\[
W = \frac{(n - 1) S^2}{\sigma^2}
\]

follows a chi-squared distribution with \(df = n - 1\) degrees of freedom:

\[
W \sim \chi^2(n - 1)
\]

The probability density function (PDF) of the chi-squared distribution with \(n - 1\) degrees of freedom is given by:

\[
f(w) = C_n \, w^{\frac{n - 3}{2}} e^{-w/2}
\]

where:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  \(C_n = \frac{1}{2^{(n-1)/2} \, \Gamma\left(\frac{n - 1}{2}\right)}\) is a normalizing constant to ensure \(\int_0^\infty f(w) \, dw = 1\),
\item
  \(\Gamma(x)\) is the gamma function, a generalization of the factorial for real numbers.
\end{enumerate}

If the value of \(\sigma^2\) is known, we can use the distribution of \(W\) to compute probabilities related to the sample variance \(S^2\).

\hypertarget{the-chi2-distribution}{%
\section{\texorpdfstring{The \(\chi^2\) Distribution}{The \textbackslash chi\^{}2 Distribution}}\label{the-chi2-distribution}}

The chi-squared distribution has a parameter called the \textbf{degrees of freedom} \(df = n - 1\), which depends on the sample size. The shape of the distribution changes with different values of \(df\). The chi-squared distribution underpins the t-distribution used when inferring the mean with an unknown \(\sigma\) and small \(n\), as we see in the following chapter. Below, we explore how the probability density function behaves for different values of \(df\).

\includegraphics{_main_files/figure-latex/unnamed-chunk-130-1.pdf}

\textbf{Example (Variations in Cable Strength)}

If we \textbf{know} that our cables are certified as
\[X \sim N(\mu=13, \sigma^2=0.35^2)\]

so

\[W=\frac{(n-1)S^2}{\sigma^2}= \frac{7S^2}{0.35^2} \sim \chi^2(n-1)\]

we can calculate \[P(S^2 > 0.3)=P(\frac{(n-1)S^2}{\sigma^2} > \frac{(n-1)0.3}{\sigma^2 } )\]
\(=P(W > \frac{7\times0.3}{0.35^2})=P(W > 17.14286)\)

\(=1-P(W \leq 17.14286)\)

\(= 1- F_{\chi^2,df=7}(17.14286)=0.016\)

\begin{verbatim}
Python: 1-chi2.cdf(17.14286, df=7)
R: 1-pchisq(17.14286, 7)
\end{verbatim}

There is only a \(1\%\) chance of getting a value greater than \(0.3\). So \(s^2>0.3\) seems to be a good criteria to stop production and review the process.

Consider that sample of \(8\) cables that the engineer took was to perform a quality control test of the production line

13.34642, 13.32620, 13.01459, 13.10811, 12.96999, 13.55309, 13.75557, 12.62747

and the observed sample variance was \(s^2=0.1275608\). Therefore, the sample is not very dispersed because \(s^2 < 0.3\) and we believe that all is well and manufacturing is under control.

\hypertarget{questions-7}{%
\section{Questions}\label{questions-7}}

\textbf{1)} The sample mean is an unbiased estimator of the population mean because

\textbf{\(\qquad\)a:} The expected value of the sample mean is the population mean;
\textbf{\(\qquad\)b:} The expected value of the population mean is the sample mean;
\textbf{\(\qquad\)c:} The standard error approaches zero as \(n\) approaches infinity;
\textbf{\(\qquad\)d:} The variance of the sample mean approaches zero as \(n\) approaches infinity;

\textbf{2)} Why is the statistic \(S^2=\frac{1}{n-1}\sum_{i=1}^{n}(X_i -\bar{X})^2\) used? instead of \(S_n^2=\frac{1}{n}\sum_{i=1}^{n}(X_i -\bar{X})^2\) to estimate the variance of a random variable?

\textbf{\(\qquad\)a:} because its variance is \(0\);
\textbf{\(\qquad\)b:} because it is a consistent estimator of \(\sigma^2\);
\textbf{\(\qquad\)c:} because it is an unbiased estimator of \(\sigma^2\);
\textbf{\(\qquad\)d:} because it is the mean square distance to the sample mean (\(\bar{X}\));

\textbf{3)} What is the variance of the sample mean \(\bar{X}=\frac{1}{n}\sum_{i=1}^n X_i\)?

\textbf{\(\qquad\)a:}\(\sigma\);
\textbf{\(\qquad\)b:}\(\frac{\sigma}{\sqrt{n}}\);
\textbf{\(\qquad\)c:}\(\sigma^2\);
\textbf{\(\qquad\)d:}\(\frac{\sigma^2}{n}\);

\textbf{4)} What is the mean and variance of the sample sum?

\textbf{\(\qquad\)a:}\(\mu\), \(n\sigma\);
\textbf{\(\qquad\)b:}\(n\mu\),\(n\sigma\);
\textbf{\(\qquad\)c:}\(\mu\), \(n\sigma^2\);
\textbf{\(\qquad\)d:}\(n\mu\), \(n\sigma^2\);

\textbf{5)} An inference question requires to

\textbf{\(\qquad\)a:} calculate the expected value of an estimator;
\textbf{\(\qquad\)b:} estimate the value of a parameter;
\textbf{\(\qquad\)c:} calculate a probability of an estimator;
\textbf{\(\qquad\)d:} fit a probability model;

\hypertarget{exercises-8}{%
\section{Exercises}\label{exercises-8}}

\hypertarget{exercise-1-7}{%
\subsubsection{Exercise 1}\label{exercise-1-7}}

Imagine that in the cable example the client asks to sell them one cables that can carry \(12\) Tons.

The engineer takes \(8\) sample of cables in stock at random, and load them until they break with the results

13.34642, 13.32620, 13.01459, 13.10811, 12.96999, 13.55309, 13.75557, 12.62747

If he does not know the certified breaking load, but he assumes that it is a normal variable, is it reasonable to sell one cable from the stock? (A: \(P(X<12)=0.0003\)).

\hypertarget{from-estimation-to-inference}{%
\subsection{From Estimation to Inference}\label{from-estimation-to-inference}}

So far, we have seen how to use a sample to estimate population parameters such as the mean and variance. But these so called \textbf{point estimates} alone are not enough. Scientific conclusions require an understanding of how reliable those estimates are. This leads us to \textbf{inference} questions such as:

\begin{itemize}
\tightlist
\item
  How precise is our estimate?
\item
  What range of values is consistent with the data?
\item
  Could the observed estimate have occurred by chance?
\end{itemize}

In the following chapters, we explore how the sampling distributions of estimators form the basis for answering these questions through \textbf{confidence intervals} and \textbf{hypothesis tests}.

\hypertarget{exercise-2-7}{%
\subsubsection{Exercise 2}\label{exercise-2-7}}

An electronics company manufactures resistors that have an average resistance of 100 ohms and
a standard deviation of 10 ohms. The resistance distribution is normal.

\begin{itemize}
\item
  What is the sample mean of \(n=25\) resistors? (R:100)
\item
  What is the variance of the sample mean of \(n=25\) resistors? (R:4)
\item
  What is the standard error of the sample mean of \(n=25\) resistors? (R:2)
\item
  Find the probability
  that a random sample of \(n = 25\) resistors have an average resistance of less than \(95\) ohms (R: 0.0062)
\end{itemize}

\hypertarget{exercise-3-5}{%
\subsubsection{Exercise 3}\label{exercise-3-5}}

A battery model charges an average of \(75\%\) of its capacity in one hour with a standard deviation of \(15\%\).

\begin{itemize}
\item
  If the battery charge is a normal variable, what is the probability that the charge difference between the sample mean of \(25\) batteries and the mean charge is at most \(5\%\)? (R:0.9044)
\item
  If we charge \(100\) batteries, what is that probability? (R:0.9991)
\item
  If instead we only charge \(9\) batteries, what charge \(c\) is exceeded by the sample mean with probability \(0.015\)? (A:85.850)
\end{itemize}

\hypertarget{exercise-4-4}{%
\subsubsection{Exercise 4}\label{exercise-4-4}}

On January 28, 1986, the space shuttle Challenger exploded just seconds after launch, killing all seven crew members. The night before, concerns were raised about the potential damage that the low temperature forecast at launch time---\(29°F\) (\(–1.6°C\))---could cause to two rubber O-rings that sealed sections of the solid rocket boosters. The failure of engineers to properly analyze available data, combined with pressure to avoid further delays in the mission, contributed to the disaster.

Engineers had access to data from previous shuttle launches where damage to the O-rings and they the temperatures on the days of those launches were recorded \citep{tufte1997visual}. Crucially ignoring the severity of the the damage and the launches with no damage, some arguments against delaying the mission were based on the temperatures of the launches with some damage only.

These are the the temperatures of the launches that registered some damage

53, 57, 58, 63, 70, 75

What is the probability that a launch with some damage is observed at temperature \(29°F\)? (A: \(P(X< 29)=3.10\times 10^{-5}\)).

\hypertarget{central-limit-theorem}{%
\chapter{Central limit theorem}\label{central-limit-theorem}}

In this chapter we will discuss more fully the \textbf{margin of errors} when estimating the mean of the population distribution with the average. In the design of components, usually a predefined tolerance in the acceptability of the component's properties is conventionally given for certification. When testing the components with a testing sample, this puts limits to how much variability we can tolerate in the sample average. Statistically, we may set the margin of error such that the probability that the average is within the error is at least a fixed number, usually \(95\%\).

When observing an average of a natural process and some theoretical or gold-standard is known for the mean and variance we can compute the margin of error to have a prediction boundary on where to find the average when we perform the data collection for the sample. Decisions and inferences when the sample data is obtained will be the subject of confidence intervals and hypothesis testing.

Before that, in this chapter and the next chapter, we will first deal with random experiments that are not modeled with the normal distribution. We will discuss how the \textbf{central limit theorem} will allow us to compute the margin of error for any type of distribution if the sample is large.

We will also introduce the t-statistic, for computing the margin of error when the sample is small but the population distribution is normal.

\hypertarget{margin-of-error}{%
\section{Margin of error}\label{margin-of-error}}

When deciding whether the error of estimation of \(\mu\) by the sample mean \(\bar{x}\) is large or not, we usually compare it with a \textbf{predefined} tolerance.

The \textbf{margin of error} at \(5\%\) level is the distance from \(\mu\) that captures \(95\%\) of the averages \(\bar{X}\):

\[P(-m \leq \bar{X}-\mu \leq m)=P(\mu-m \leq \bar{X} \leq\mu + m)=0.95\]

This means that \(95\%\) of the outcomes of \(\bar{X}\) from a random sample are a distance \(m\) from \(\mu\).

\hypertarget{averages-of-normal-variables}{%
\section{Averages of normal variables}\label{averages-of-normal-variables}}

We want to know the number \(m\) in the equation

\[P(\mu-m \leq \bar{X} \leq\mu + m)=0.95\]

To solve this equation we need two steps. \textbf{First}, we need to know the \textbf{distribution} of \(\bar{X}\).

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  When \(X\) is normal (\(X \sim N(\mu, \sigma^2)\)) then
\end{enumerate}

\[\bar{X} \sim N(\mu, \frac{\sigma^2}{n})\]
We \textbf{then} need to \textbf{standardize} \(\bar{X}\). Remember that to standardize a normal variable, we subtract its mean an divide it by its standard deviation.

\[Z=\frac{\bar{X}-E(\bar{X})}{\sqrt{V(\bar{X})}} =\frac{\bar{X}-\mu}{\frac{\sigma}{\sqrt{n}}}  \sim N(0,1)\]

\(Z\) is then standard normal. Remember the probability density function for a standard normal variable and its quantiles

\includegraphics{_main_files/figure-latex/unnamed-chunk-134-1.pdf}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{1}
\tightlist
\item
  Substituting the mean of \(\bar{X}\) and its standard deviation into the equation for the margin of error, we have:
\end{enumerate}

\(P(\mu-m \leq \bar{X} \leq\mu + m)=P(-\frac{m}{\sigma/\sqrt{n}} \leq \frac{\bar{X}-\mu}{\frac{\sigma}{\sqrt{n}}}\leq\frac{m}{\sigma/\sqrt{n}})\)
\[=P(-\frac{m}{\sigma/\sqrt{n}} \leq Z \leq\frac{m}{\sigma/\sqrt{n}})=0.95\]

Compare it with the plot above, where

\[P(-1.96 \leq Z \leq 1.96)=0.95\]
Therefore,

\[\frac{m}{\sigma/\sqrt{n}}=1.96\]
is the distance from \(z=0\) that captures \(95\%\) of the distribution of the standard normal variable. Threfore, the margin of error at \(5\%\) is

\[m=1.96\frac{\sigma}{\sqrt{n}}\]

The limits of the interval \((-1.96, 1.96)\) for \(Z\) are computed from

\[\Phi^{-1}(0.975)=1.96\]
where \(\Phi^{-1}\) is the inverse of the standard normal distribution (norm.ppf(0.975)), and retrieves the value of \(z\) that has accumulated \(97.5\%\) of probability. \(2.5\%\) are left out at the higher tail of the distribution, that add to the other \(2.5\%\) left out at the left of \(\Phi^{-1}(0.025)=-1.96\), because the distribution is symmetric.

\textbf{Example (Resistors)}

Imagine we want to implement an established manufacturing process for producing \(2.5\,\Omega\) resistors. We want to certify that our implementation is highly capable at the 1\% tolerance level. This means that the standard deviation of the implemented process should be no more than one-sixth of \(2.5 \times 0.01\), following the criteria for a so-called Six Sigma process capability (Cp).

In other words:

\begin{itemize}
\tightlist
\item
  The mean resistance should be close to \(2.5\,\Omega\)
\item
  The standard deviation should be less than or equal to \(0.004\,\Omega\)
\item
  Nearly all resistors should fall within the range \((2.49, 2.51)\,\Omega\)
\end{itemize}

If the specifications of the process have established from previous gold-standard experiments that the resistance follows a normal distribution:

\[
X \sim N(\mu = 2.5, \sigma^2 = 0.004^2)
\]

Although large samples are required for formal certification, we take a pilot sample of 10 resistors to quickly test our implementation of the process. Since we will estimate the mean \(\mu\) using the sample mean \(\bar{X}\), and this estimate is subject to sampling variability, we are interested in computing the margin of error at the 5\% level for \(\bar{X}\).

The sample mean \(\bar{X}\) has:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Expected value: \(E(\bar{X}) = \mu = 2.5\)
\item
  Standard error:
  \[
  se= \sqrt{V(\bar{X})} = \frac{\sigma}{\sqrt{n}} = \frac{0.004}{\sqrt{10}}
  \]
\end{enumerate}

Then, the 95\% margin of error is:

\[
m = 1.96 \times \frac{0.004}{\sqrt{10}} = 0.00248
\]

Therefore, we can expect that in 95\% of such pilot samples of size 10, the sample mean resistance \(\bar{x}\) will lie within the interval:

\[
(2.5 - 0.00248,\ 2.5 + 0.00248) = (2.49752,\ 2.50248)\,\Omega
\]

Obtaining a sample average within the specified interval provides preliminary reassurance that the implementation may meet the required precision, supporting the decision to pursue a larger-scale certification study.

\hypertarget{central-limit-theorem-1}{%
\section{Central Limit Theorem}\label{central-limit-theorem-1}}

We could solve the margin of error because we assumed that that variable \(X\) was normal. What if \(X\) follows any other probability distribution?

\textbf{Theorem:} For any random variable \(X\) with any type of probability function

\[X \sim f(x)\]
the standardized statistic

\[Z=\frac{\bar{X}-E(\bar{X})}{\sqrt{V(\bar{X})}}\]

approximates to a standard distribution

\[Z \sim_d N(0,1)\] when \(n\rightarrow \infty\)

\textbf{Consequence:} We can compute probabilities for \(\bar{X}\) if \(n\) is large, using the normal distribution:

\[\bar{X} \sim_{aprox}  N(\mu, \frac{\sigma^2}{n})\]
This is tipically a good approximation when \(n\geq 30\).

\textbf{Example (Number of Individuals per Species)}

In their foundational work in ecology R.A. Fisher et al.~analysed the number of individuals for each moth species that was captured over 4 years at Rothamsted Experimental Station (Rothamsted Research) \citep{fisher1943species}.

They showed that most species were observed only once and that fewer species were observed with large number of individuals. While the last model they proposed was a Log-Series model

\[X \sim \frac{-\theta^x}{x\ln(1-\theta)}\]

Where \(X\) is the number of individuals of a species in a survey. The random experiment is to take a survey of a given size, identify a species and count the number of individuals in the species. The Poisson model underestimated rare species but showed that the observed patterns in species abundances can arise from random sampling processes --- not necessarily from deterministic biological interactions.

\includegraphics{_main_files/figure-latex/unnamed-chunk-135-1.pdf}

The mean and variance are:

\begin{itemize}
\tightlist
\item
  \(\mu= {\frac {-1}{\ln(1-\theta)}}{\frac {\theta}{1-\theta}}=65.05513\)
\item
  \(\sigma^2= -{\frac {\theta^{2}+\theta\ln(1-\theta)}{(1-\theta)^{2}(\ln(1-\theta))^{2}}}=21071.27\)
\end{itemize}

Imagine, we go to Rothamsted Research and plan to roll out a campaign of moth collection in a surrounding area for 4 years. We will count each at each station the number of species collected with 1 specimen, 2 specimens, and so on. What would be the expected number of individuals in a randomly chosen species and its margin of error, if we assume that the biodiversity of moths has not changed since Fisher's time?

Fisher's mean and the standard error of the average species abundance \(\bar{X}\) assuming that we sample a total of 240 species (as Fisher did):

\begin{itemize}
\tightlist
\item
  \(\mu_{\bar{x}}=65.05513\)
\item
  \(\sigma_{\bar{x}}=\sqrt{\frac{\sigma}{n}}=9.37\)
\end{itemize}

As \(n \geq 30\)

\[Z=\frac{\bar{X}-\mu_{\bar{x}}}{\sigma_{\bar{x}}}\]

is almost standard normal variable and: \[\bar{X} \sim_{aprox}  N(\mu_{\bar{x}}, \sigma_{\bar{x}})\]

The margin of error at \(5\%\) level can be computed again with the standard distribution

\[m=\Phi^{-1}(0.975)\times \sigma_{\bar{x}}=1.96\times 9.37=18.36\]

We can expect \(95\%\) of the averages of the abundance over \(240\) species in four years to fall between
\((65.05-18.36, 65.05+18.36)= (46.69, 83.41)\)

If the observed average falls outside this interval, it may indicate a statistically significant change in moth biodiversity since Fisher's study, assuming the Poisson model and historical parameters still hold.

\includegraphics{_main_files/figure-latex/unnamed-chunk-137-1.pdf}

\hypertarget{sample-sum-and-clt}{%
\section{Sample sum and CLT}\label{sample-sum-and-clt}}

The \textbf{sample sum} is the \textbf{statistic}

\[Y=X_1+X_2+...X_n=\sum_{i=1}^n X_i=n \bar{X}\]

with

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\tightlist
\item
  mean \[E(Y)=n\mu\]
\item
  variance \[V(Y)=n\sigma^2\]
\end{enumerate}

The CLT tells us that for any random variable \(X\) with \textbf{unknown} (any type of) distribution

\[X \sim f(x)\]

the standardized statistic

\[Z=\frac{\bar{X}-E(\bar{X})}{\sqrt{V(\bar{X})}}\]

approximates to a standard distribution

\[Z \sim_d N(0,1)\] when \(n\rightarrow \infty\). \(Z\) can also be written as

\[Z=\frac{n\bar{X}-nE(\bar{X})}{\sqrt{n^2V(\bar{X})}}=\frac{Y-E(Y)}{\sqrt{V(Y)}}=\frac{Y-n\mu}{\sqrt{n}\sigma}\]

\textbf{Consequence:} We can compute probabilities for the sample sum \(Y=n\bar{X}\) if \(n\) is large, using the normal distribution:

\[Y \sim_{aprox}  N(n\mu, n\sigma^2)\]

\textbf{Example (De Moivre-Laplace's insight)}

For a Bernoulli trial \(X \sim Bernoulli(p)\), its mean is \(\mu=p\) and variance \(\sigma^2=p(1-p)\).

Now, the sample sum \(Y=n\bar{X}\) is a random variable that counts the number of events with probability \(p\) in a repetition of \(n\) trials, therefore

\[Y \sim Binom(n, p)\]
with mean \(E(Y)=np\) and variance \(V(Y)=np(1-p)\). If we apply the CLT for the sample sum of Brenouilli trials then we have

\[Y \sim  Binom(n, p) \sim_{aprox}  N(np, np(1-p))\]
we can then proximate the binomial probability mass function with the normal probability density when \(n\) is big. This approximation is good when both \(np\) and \(n(1-p)\) are greater than \(5\).

The significance of De Moivre-Laplace's discovery cannot be overstated. It provided a practical method for approximating probabilities of binomial variables using the normal distribution, laying the groundwork for modern statistical inference. More profoundly, it offered a deep insight into how regular patterns observed in continuous (analog) systems can emerge from the randomness of discrete (digital) processes when the sample size is large. An early application of this insight was in genetics, where continuous traits such as height or skin color were explained by the cumulative effect of many discrete mutations. A striking modern example is how the discrete activations of many individual neurons can collectively produce continuous patterns of behavior---an insight that has been fundamental to modeling in artificial intelligence.

\hypertarget{unknown-sigma}{%
\section{\texorpdfstring{Unknown \(\sigma\)}{Unknown \textbackslash sigma}}\label{unknown-sigma}}

Gold-standards for the values of the mean are usually established using \(\bar{x}\) large samples sizes, as the law of large numbers tells us that the average will be close to the mean. If we are repeating the experiment in a new sample, the margin of error can be computed with the standard deviation \(s\) of the gold-standard, as an estimate of the variance.

For any random variable \(X\) with any type or unkown distribution

\[X \sim f(x)\]

and \textbf{unknown} variance \(\sigma\), we can still estimate the standard error like

\[\hat{se}=\frac{s}{\sqrt{n}}\] when \(n\) is large and write the standardized statistic

\[Z=\frac{\bar{X}-\mu}{\frac{s}{\sqrt{n}}}\sim N(0,1) \]
that allows us to use the margin of error at \(5\%\)

\[m= 1.96 \frac{s}{\sqrt{n}}\]
for the average of a new \(n\) sample of the gold-standard experiment.

\textbf{Example (Pacemaker Prediction)}

The Micra Transcatheter Pacemaker is a miniaturized pacemaker. A study of 630 patients, established the expected virtual battery longevity at \(\mu=12.1\) years with standard deviation of \(s=1.71\) \citep{Duray2017Micra12mo}.

If we want to run a new study on 10 new patients and measure the VBL for each of them then the estimated standard error is

\[\hat{se}=\frac{s}{\sqrt{n}}=\frac{1.7}{\sqrt{10}}= 0.53\]

and the margin of error

\[m= 1.96\times se= 1.0388\]
thus \(95\%\) of the samples of size 10 will have averages within a year from the gold-standard mean.

\hypertarget{t-statistic}{%
\section{T-statistic}\label{t-statistic}}

Substitution of the standard deviation \(\sigma\) for the standard deviation of the sample \(s\) is only reasonable when size of the gold-standard sample is large. Small sample expetiments of size \(n\) are often called pilots where both \(\mu\) and \(\sigma^2\) are unknown. In this cases we cannot use the central limit theorem for establishing gold-standards

However, if \(X\) is normal

\[X \sim N(\mu, \sigma^2)\]
then the standardized statistic

\[T=\frac{\bar{X}-\mu}{\frac{S}{\sqrt{n}}} \sim t_{n - 1}\]

Follows a \(t\)-distribution with \(n-1\) degrees of freedom, that allow us to compute probabilities on \(\bar{X}\), and obtain a margin of error for the average if we were to repeat the pilot study.

The probability density function (PDF) of the \textbf{Student's t-distribution} with \(n - 1\) degrees of freedom is given by:

\[
f(t) = C_n \left(1 + \frac{t^2}{n - 1}\right)^{-\frac{n}{2}}
\]

where:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  \(C_n = \frac{\Gamma\left(\frac{n}{2}\right)}{\sqrt{(n - 1)\pi} \, \Gamma\left(\frac{n - 1}{2}\right)}\) is a normalizing constant ensuring \(\int_{-\infty}^{\infty} f(t) \, dt = 1\),
\item
  \(\Gamma(x)\) is the gamma function, which generalizes the factorial to real (and complex) numbers.
\end{enumerate}

The t-distribution resembles the standard normal distribution but has heavier tails, reflecting the added uncertainty from estimating the population variance.

\includegraphics{_main_files/figure-latex/unnamed-chunk-139-1.pdf}

To compute the margin of error \(m\) at \(5\%\) level when \(n\) is small, \(\sigma\) unknown but \(X\) normal

\(P(\mu-m \leq \bar{X} \leq\mu + m)=P(-\frac{m}{s/\sqrt{n}} \leq \frac{\bar{X}-\mu}{\frac{s}{\sqrt{n}}} \leq\frac{m}{s/\sqrt{n}})\)
\[=P(-\frac{m}{s/\sqrt{n}} \leq T \leq\frac{m}{s/\sqrt{n}})=0.95\]

We use the \(t\)-distribution

\[m=t_{0.025, n-1} \frac{s}{\sqrt{n}}\]
where \(t_{0.025, n-1}\) is the value \(T\) that leaves \(2.5\%\) at each side of \(t\)-distribution with \(n-1\) degrees of freedom (\(0.025\)-quantile)

\textbf{Example (Cables)}

A sample of 8 cables was taken to perform a quality control test on the breaking load (in tons) of the cables.

13.34642, 13.32620, 13.01459, 13.10811, 12.96999, 13.55309, 13.75557, 12.62747

The observed mean and standard deviation of the sample were \(\bar{x}=13.21\) and \(s=0.36\). Therefore, the margin of error is \[m=t_{0.025, n-1} \frac{s}{\sqrt{n}}=2.36\times \hat{se}=2.36\frac{0.36}{\sqrt{8}}=0.1\]
where \(t_{0.025, n-1}=2.36\).

In subsequent samples of \(8\) cables we can expect \(95\%\) of the averages to fall between \((13.21-0.1,13.21+0.1)\).

\begin{verbatim}
Python: t.sf(0.025, 8-1) 
R: qt(0.025, 8-1, lower.tail = FALSE)
\end{verbatim}

\hypertarget{questions-8}{%
\section{Questions}\label{questions-8}}

\textbf{1)} A magnetic resonance imaging of the brain's hippocampus has \(100\) pixels. We expect \(90\%\) of the pixels to be white (brain tissue). According to the central limit theorem, what is the probability that the scanning of a patient has at most \(85\%\) of white pixels?

\textbf{\(\qquad\)a:}norm.cdf(0.9, 0.85, sqrt(0.85*0.15)/10); \textbf{\(\qquad\)b:}norm.pdf(0.85, 0.9,
sqrt(0.9*0.1)/10); \textbf{\(\qquad\)c:}norm.cdf(0.85, 0.9, sqrt(0.9*0.1)/10); \textbf{\(\qquad\)d:}norm.pdf(0.9, 0.85, sqrt(0.85*0.15)/10)

\textbf{2)} For a standard normal variable, if we the number \(z_{0.025}\) in the definition of the margin of error \(m=z_{0.025} \frac{\sigma}{\sqrt{n}}\), then it will refer to

\textbf{\(\qquad\)a:} The first quartile; \textbf{\(\qquad\)b:} The number at which the distribution has accumulated \(0.975\) of probability; \textbf{\(\qquad\)c:} The number at which the distribution has accumulated \(0.025\) probability; \textbf{\(\qquad\)d:} The third quartile;

\textbf{3)} The importance of the central limit theorem is that is applyes to the standardization of

\textbf{\(\qquad\)a:} A random variable;
\textbf{\(\qquad\)b:} The sample mean of a normal variable;
\textbf{\(\qquad\)c:} The sample mean of a random variable;
\textbf{\(\qquad\)d:} A normal variable;

\hypertarget{exercises-9}{%
\section{Exercises}\label{exercises-9}}

\hypertarget{exercise-1-8}{%
\subsubsection{Exercise 1}\label{exercise-1-8}}

An electronic component is needed for the correct functioning of a telescope. It needs to be replaced immediately when it wears out.

The mean life of the component (\(\mu\)) is \(100\) hours and its standard deviation \(\sigma\) is \(30\) hours.

\begin{itemize}
\item
  what is the probability that the average of the mean life of \(50\) components is within \(1\) hour from the mean life of a single component? (A: 0.1863)
\item
  How many components do we need such that the telescope is operational \(2750\) consecutive hours with at least \(0.95\) probability? (A: 31)
\end{itemize}

\hypertarget{exercise-2-8}{%
\subsubsection{Exercise 2}\label{exercise-2-8}}

The probability that a particular mutation is found in the population is \(0.4\). If we test \(2000\) people for the mutation:

\begin{itemize}
\tightlist
\item
  What is the probability that the total number of people with the mutation is between \(791\) and \(809\)? (A: 0.31)
\end{itemize}

hint: Use the CLT with a sample of \(2000\) Bernoulli trials. This is known as the normal approximation of the binomial distribution.

\hypertarget{exercise-3-6}{%
\subsubsection{Exercise 3}\label{exercise-3-6}}

An automated machine fills test tubes with biological samples with mean \(\mu=130\)mg and a standard deviation of \(\sigma=5\)mg. For a random sample of size \(50\)

\begin{itemize}
\item
  What is the probability that
  the sample mean (average) is between \(128\) and \(132\)gr? (A: 0.995)
\item
  what is the margin of error at \(5\%\)? (A: 1.385929)
\item
  what should be the size of the sample \(n\) such that the margin of error at \(5\%\) is \(1\)?
  (A: 97)
\end{itemize}

\hypertarget{exercise-4-5}{%
\subsubsection{Exercise 4}\label{exercise-4-5}}

In the Caribbean, there appears to be an average of \(6\) hurricanes per year. Considering that hurricane formation is a Poisson process, meteorologists plan to estimate the mean time between the formation of two hurricanes. They plan to collect a sample of size \(36\) for the times between two hurricanes.

\begin{itemize}
\item
  What is the probability that their sample average is between \(50\) and \(60\) days? (A: 0.385975)
\item
  Which should be the sample size such that they have a probability of \(0.025\) that the sample mean is greater than \(70\) days? (A: 169)
\end{itemize}

\hypertarget{maximum-likelihood}{%
\chapter{Maximum likelihood}\label{maximum-likelihood}}

n 1922, R.A. Fisher published a seminal paper that laid the foundations of modern mathematical statistics \citep{Fisher1922}. While important statistical techniques---such as regression analysis and tests of goodness-of-fit---had already been developed, their mathematical underpinnings had remained largely unformalized. This lack of rigor, Fisher argued, was partly due to the nature of the field, which deals with uncertainty and error, and where precise definitions were often not considered practically necessary.

Another major source of confusion was linguistic: the same terms were often used to refer both to the unknown quantities that researchers sought to estimate (parameters) and to the values computed from data (estimates). For example, think of the distinction between \(\mu\), the expected value of a normal random variable, and \(\bar{x}\), the sample mean used to estimate it. Clarifying such distinctions was a crucial step in the formalization of statistical inference, enabling the generalization of estimation methods to a wide variety of models.

In this chapter, we will explore what an estimator is and illustrate its use in general applications such as radioactive detection, biodiversity estimation, and the analysis of system complexity. We will introduce a general framework for obtaining estimators of model parameters, focusing in particular on the method of maximum likelihood, first introduced by Fisher. Fisher's key insight was to treat the likelihood of the observed values of random samples as a function of the unknown parameters, allowing for a principled method to derive estimates.

\hypertarget{statistic}{%
\section{Statistic}\label{statistic}}

It is time to formalize some of the notions that we have already dealt with. The intention of using a more precise language is to be able to generalize the basic concepts to any type of probability models. In particular, we need to formalize the notion of statistics, estimator and their properties, such as the error we can make when estimating a parameter with the observed value of an estimator.

To extract useful information of a random experiment from the sample we use statistics.

\textbf{Definition}

A \textbf{statistic} is any function of a random sample
\[T(X_1,X_2, ..., X_n)\]

It usually returns a number, but frequency tables and plots like histograms can also be considered statistics. Statistics are random variables and their probability distributions are called \textbf{sampling distributions}.

Statistics have different functions:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  \textbf{Description} of a sample's data
\end{enumerate}

\begin{itemize}
\tightlist
\item
  location: \(\bar{X}\)
\item
  Minimum: \(\min\{X_i\}\)
\item
  Maximum: \(\max\{X_i\}\)
\end{itemize}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{1}
\tightlist
\item
  \textbf{Estimation} of a probability model's \textbf{parameters}
\end{enumerate}

\begin{itemize}
\tightlist
\item
  We use \(\bar{X}\) to estimate \(\mu\)
\item
  We use \(S^2\) to estimate \(\sigma^2\)
\end{itemize}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{2}
\tightlist
\item
  \textbf{Inference} for predicting the outcomes of the estimators
\end{enumerate}

\begin{itemize}
\tightlist
\item
  for the mean we will use the statistics \(Z\), \(T\)
\item
  for the variance we will use \(\chi^2\)
\end{itemize}

This last point will be developed in the following chapters.

The key fact is that statistics are random variables whose observed values we take as information about the experiment. However, every time we take a new sample their value change.

\textbf{Definition of estimators}

An \textbf{estimator} is a statistic whose observed values are used to estimate the \textbf{parameters} of the probability model used to describe the outcome of the random experiments under study. When we use an estimator, the information that we are interested in extracting is the properties of the random experiment encoded in the parameters.

If we write the probability model of the random experiment as the probability function

\[X \rightarrow f(x; \theta)\]

then \(\theta\) is a parameter and the statistics \(\Theta\) is a random variable whose observations \(\hat{\theta}\) we take as estimations of \(\theta\). The estimate \(\hat{\theta}\) is a numerical value obtained by evaluating the estimator \(\Theta\) on the data and is used as an approximation of the unknown parameter \(\theta\).

\[\hat{\theta} \approx \theta\]

Let us unpack this statement. There are three different quantities that we must consider:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  \(\theta\) is a \textbf{parameter} of the probability model that describes the experiment \(f(x; \theta)\).
\item
  \(\Theta\) is an \textbf{estimator} of \(\theta\): A random variable.
\item
  \(\hat{\theta}\) is the point \textbf{estimate} of \(\theta\): An outcome or a realized value of \(\Theta\).
\end{enumerate}

\includegraphics{./figures/estimator.JPG}

\textbf{Example (Sample mean)}

When the probability density function for the outcomes of a random experiment is modeled by a normal density

\[X \rightarrow N(\mu, \sigma^2)\]

we can identify the three different quantities:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  The mean: \(\mu\) is a \textbf{parameter} of the \textbf{population} distribution \(N(\mu, \sigma^2)\).
\item
  The average: \(\bar{X}\) is an \textbf{estimator} of \(\mu\).
\item
  The point estimate of the mean: \(\bar{x}=\hat{\mu}\) is the \textbf{estimate} of \(\mu\).
\end{enumerate}

\textbf{Example (Sample variance)}

When we have a normal random variable

\[X \rightarrow N(\mu, \sigma^2)\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  \(\sigma^2\) is a \textbf{parameter} of the population distribution
\item
  \(S^2\) is an \textbf{estimator} of \(\sigma^2\)
\item
  \(s^2=\hat{\sigma}^2\) is the \textbf{estimate} of \(\sigma^2\)
\end{enumerate}

\hypertarget{properties}{%
\section{Properties}\label{properties}}

The estimators have important properties that allow us to determine the type of errors we can make when using them to estimate parameters. There are two types of error: the systematic error measured by the bias, and the random error measured by the variability of the sample. When there is low systematic error we say that the estimate is \textbf{accurate} (low bias) and if the observations from the sample do no vary much between then we say that the estimate is \textbf{precise}.

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  An estimator \(\Theta\) is \textbf{unbiased} if its expected value is the parameter
\end{enumerate}

\[E(\Theta)=\theta\]

Therefore, the estimator is accurate and does not have systemic error.

For example:

\begin{itemize}
\item
  \(\bar{X}\) is an \textbf{unbiased} estimator of \(\mu\) because \(E(\bar{X})=\mu\)
\item
  \(S^2\) is an \textbf{unbiased} estimator of \(\sigma^2\) because \(E(S^2)=\sigma^2\)
\end{itemize}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{1}
\tightlist
\item
  An estimator is \textbf{consistent} when its observed values get closer and closer as the sample size is increased
\end{enumerate}

\[lim_{n\rightarrow \infty} V(\Theta) = 0\]

Therefore the estimator is increases precision when the sample size increases.

For example:

\begin{itemize}
\tightlist
\item
  \(\bar{X}\) is \textbf{consistent} because \(V(\bar{X})=\frac{\sigma^2}{n}\rightarrow 0\) when \(n \rightarrow \infty\).
\end{itemize}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{2}
\tightlist
\item
  The mean squared error \(mse\) of \(\Theta\) is its expected squared difference from the parameter
\end{enumerate}

\[mse(\Theta)=E([\Theta - \theta]^2)\]

or equivalently is the sum of the errors in consistency and bias

\[mse(\Theta)=se^2 + bias^2\]

where \(se=\sqrt{V(\Theta)}\) is the standard error.
\textbf{Efficient estimators} are those with the lowest variance among unbiased estimators.

\hypertarget{maximum-likelihood-1}{%
\section{Maximum likelihood}\label{maximum-likelihood-1}}

Now that we know what estimators are and what makes them good, we need a method to derive them from data. The method of maximum likelihood does this by choosing an observed value of the estimator as a likely value of a parameter that which makes the observed data most probable. Remember that the parameter is un-observable and, if we do not know its theoretical value, we are trying to learn its most likely value using experimental data.

How can then we obtain \textbf{estimators} of the parameters of \textbf{any} probability model? The method of maximum likelihood offers a strategy to propose estimators.

\textbf{Example (Alpha particles)}

In 1910, Rutherford y Geiger counted the number of \(\alpha\) particles that a specimen of polonium would emit each \(7.5\) seconds. The absolute frequencies that they reported for the number of particles each \(7.5\) seconds from a total of \(2608\) observations were

\[
\begin{array}{cc}
\mathbf{outcome} & \mathbf{n_i}  \\
0 & 57 \\
1 & 203 \\
2 & 383 \\
3 & 525 \\
4 & 532 \\
5 & 408 \\
6 & 273 \\
7 & 139 \\
8 & 45 \\
9 & 27 \\
10 &  10 \\
11 &   4 \\
12 &   0 \\
13 &   1 \\
14 &   1 \\ \hline
\mathbf{sum} & 2608 \\
\end{array}
\]

\includegraphics{_main_files/figure-latex/unnamed-chunk-141-1.pdf}

They observed, for instance, that \(57\) of the times they did not observed any particle within \(7.5\) seconds, \(203\) of the times they counted one particles within \(7.5\) seconds, etc. They wanted to show that particles were emitted from independent sources (atoms) and find a probabilistic model for the counts.

What is a probability function that can describe the data?

\textbf{Proposing a probability model}

To answer the question on how to select a probabilistic model when we have some data, we follow the steps:

\textbf{Step 1.} we propose \textbf{probability function} on the nature of the random variable (continuous or discrete) that depends on parameters,

\textbf{Step 2.} we derive the \textbf{estimators} for the parameters, by maximum likelihood,

\textbf{Step 3.} finally, we use the estimator to \textbf{estimate the parameters} with the data.

In many applications, we can propose parametric models that is a model that belongs to a family of a probability functions that depends on some parameters. Proposing a probability model is done by following the \textbf{general properties} of the observations, or by what we expect to observe. Modelling requires experience, skill and knowledge of several mathematical functions. However, in most cases \textbf{well known models} are typically applied.

\textbf{Example (Alpha particles)}

In the appendix of Rutherford and Geiger's paper, Bateman showed that if there were independent sources alpha particles, the number of particles detected in a time interval would follow a Poisson distribution. That is

\[X \sim Poisson (\lambda)\]

Therefore, once selected the theoretical model, they needed to find the value of the parameter \(\lambda\) for their data.

\includegraphics{_main_files/figure-latex/unnamed-chunk-142-1.pdf}

Many values of the parameters could explain the data. We are interested in \textbf{one criterion} to choose one particular value.

The \textbf{maximum likelihood} method will gives us the estimator for \(\lambda\)

\[\hat{\lambda}_{ml}\]
using an optimal probabilistic argument.

\hypertarget{maximum-likelihood-2}{%
\section{Maximum likelihood}\label{maximum-likelihood-2}}

The objective is to find the value of the parameter that we \textbf{believe} can \textbf{best} represent the data.

The method of maximum likelihood is based on the search for the parameter value that makes the \textbf{observation} of the sample the most \textbf{probable}.

Remember, a random sample is a random variable
\[M=(X_1,...X_n)\]
an observed sample is an outcome of \(M\), a set of numerical values that we actually obtained when we repeated the random experiment

\[m=(x_1,...x_{n=2608})=c(10, 14, ... 0, 4)\]

\textbf{Maximum likelihood step 1}

First, we calculate the probability of having observed the particular \(n\)-sample: \(x_1,...x_n\). This is the product of probabilities for each observation because the repetitions of the random experiment are independent of one another. If the probabilistic model of is

\[X \sim f(x, \theta)\]
Then the probability of observing the data is

\(P(M=x_1,...x_n)=P(X=x_1)P(X=x_2)...P(X=x_n)\)
\[=f(x_1;\theta)f(x_2;\theta) ...f(x_n;\theta)\]
We call this function the \textbf{likelihood function} and we consider that:

\begin{itemize}
\tightlist
\item
  Once the data are observed, they are \textbf{fixed}
\item
  The unknown is the parameter \(\theta\)
\end{itemize}

\[L(\theta)= \Pi_{i=1..n} f(x_i; \theta)\]

\textbf{Example (Alpha particles)}

For the alpha particle experiment the model in Poisson

\[X \sim  f(x; \lambda)= \frac{e^{-\lambda}\lambda^x}{x!}\]
and the unknown the parameter \(\lambda\). Therefore, the likelihood is

\(L(\lambda;x_1,..x_n)= \frac{e^{-\lambda}\lambda^{x_1}}{x_1!}\frac{e^{-\lambda}\lambda^{x_2}}{x_2!}...\frac{e^{-\lambda}\lambda^{x_n}}{x_1!}=\)

\[\frac{e^{-n\lambda}\lambda^{\sum {x_i}}}{x_1!x_2!..x_n!}\]

\textbf{Maximum likelihood step 2}

We then ask: what is the value of \(\theta\) that makes the observed sample the most probable event? We thus want to maximize \(L(\theta)\) with respect to \(\theta\). Since we have the multiplication of many factors, it is easier to maximize the logarithm of \(L(\theta)\). This is called the log-likelihood function:

\[\ln L(\theta;x_1,..x_n)\]

\textbf{Example (Alpha particles)}

In the alpha particles example, we therefore take the logarithm and obtain the \textbf{Log-likelihood}

\[\ln L(\alpha;x_1,..x_n)= -n\lambda - \sum{x_i}\ln(\lambda)-\ln(x_1!x_2!...x_3!)\]

\textbf{Maximum likelihood step 3}

Finally we \textbf{maximize} the log-likelihood with respect to the parameter. Therefore, we differentiate the log-likelihood with respect to the parameter \(\theta\), equate to zero and solve for the maximum.

\[\frac{d \ln L(\theta)}{d \theta} \big|_{\hat{\theta}}=0 \]
The value of the parameter at the maximum is called the \textbf{maximum likelihood estimate} for the parameter and it is written with a hat \(\hat{\theta}\).

\textbf{Example (Alpha particles)}

We derive the log-likelihood

\[\frac{d \ln L(\lambda)}{d \lambda}= -n + \frac{\sum{x_i}}{\lambda}\]
The maximum is where the derivative is \(0\). This maximum is the value of our estimator \(\hat{\lambda}_{ml}\).

\[\hat{\lambda}_{ml}=\frac{1}{n}\sum_{i=1}^{n}\]

The estimator of the parameter is therefore (note the capital letters)

\[\Lambda=\frac{1}{n}\sum_{i=1}^{n}\]

Which is a the sample mean, a random variable function of the random sample

\[(X_1, X_2, ... X_n)\]

We then have the fact that the \textbf{sample mean} is the \textbf{maximum likelihood} estimator of the parameter \(\lambda\) of a Poisson model.

\emph{Estimating the parameters with the data}

In our example, we then have the observation of the random sample as a set of 2608 numbers \((x_1, x_2, ...x_{2608})\), we therefore substitute the numbers in the estimator and this will give us its observed value.

\[\hat{\lambda}_{ml}=-\frac{1}{n}\sum_{i=1}^{n} =\sum_{i=1}^{n} x_if_i =3.871549\]

Using the relative frequency table. Therefore, the maximum likelihood estimate of the parameter is \(3.871549\), as reported by Rutherford and Gaiger. If we substitute this value in the probability function, and overlay it with the bar plot, we can see that it gives us a suitable description of the data.

\includegraphics{_main_files/figure-latex/unnamed-chunk-143-1.pdf}

Let us look at the log-likelihood function for the \(2608\) observations. Remember, data is fixed the data of the experiment and that \(\lambda\) varies. The function has a maximum. However, if we take another sample this function changes and so does its maximum.

\includegraphics{_main_files/figure-latex/unnamed-chunk-144-1.pdf}

\textbf{Example (Number of Individuals per Species)}

In another influential work, Fisher and colleagues analysed the number of individuals for each moth species that was captured over 4 years at Rothamsted Experimental Station \citep{fisher1943species}. This is their data aggregated in absolute frequencies

\[
\begin{array}{cccccc}
\mathbf{outcome} & \mathbf{n_i} & \mathbf{outcome} & \mathbf{n_i} & \mathbf{outcome} & \mathbf{n_i} \\
1 & 35 & 34 & 2 & 141 & 1 \\
2 & 11 & 36 & 3 & 148 & 1 \\
3 & 15 & 37 & 2 & 149 & 1 \\
4 & 14 & 43 & 2 & 151 & 1 \\
5 & 10 & 44 & 2 & 154 & 1 \\
6 & 11 & 45 & 1 & 177 & 1 \\
7 & 5  & 49 & 3 & 181 & 1 \\
8 & 6  & 51 & 4 & 187 & 1 \\
9 & 4  & 52 & 1 & 190 & 1 \\
10 & 4 & 53 & 1 & 199 & 1 \\
11 & 2 & 54 & 2 & 211 & 1 \\
12 & 2 & 57 & 1 & 221 & 1 \\
13 & 5 & 58 & 2 & 226 & 1 \\
14 & 2 & 60 & 3 & 235 & 1 \\
15 & 4 & 61 & 1 & 239 & 1 \\
16 & 3 & 64 & 1 & 244 & 1 \\
17 & 3 & 67 & 1 & 246 & 1 \\
18 & 3 & 73 & 1 & 282 & 1 \\
19 & 3 & 76 & 2 & 305 & 1 \\
20 & 4 & 78 & 1 & 306 & 1 \\
21 & 1 & 84 & 1 & 333 & 1 \\
22 & 3 & 89 & 1 & 464 & 1 \\
23 & 3 & 96 & 1 & 560 & 1 \\
24 & 1 & 99 & 1 & 572 & 1 \\
25 & 3 & 109 & 1 & 589 & 1 \\
26 & 1 & 112 & 1 & 604 & 1 \\
27 & 1 & 120 & 1 & 743 & 1 \\
28 & 3 & 122 & 1 & 823 & 1 \\
29 & 2 & 129 & 1 & 2349 & 1 \\
30 & 32 & 135 & 1 & 2349 & 1 \\
\hline
\end{array}
\]

that represents the number of individuals observed for a species during the four years campaign. This is \(35\) species were observed only once, \(11\) species twice and so on. The random variable \(X\) is then the number of events in a period of time on a particular species. One species was observed \(23\), another \(24\) and so on up to another secies observed \(49\) times. Here the species is the observation unit or the repetition of the random experiment. The random experiment was repeated 167 times (total number of species), and Fisher proposed that the random variable \(X\) followed a log-series distribution

\[X \sim \frac{-\theta^x}{x\ln(1-\theta)}\]

where \(\theta \in (0,1)\).

To find the value of \(\theta\) we can use the method of Maximum Likelihood, where we first propose the likelihood of the data

\[
L(\theta) = \prod_{i=1}^{n} \left( -\frac{\theta^{x_i}}{x_i \log(1 - \theta)} \right)
\]

Where \(x\) is the dis-aggregated data, one observation (number of counts in 4 years) per species \((1, 1, ...1_{35}, 2, ...2_{11}, ..., 49, 50, 50)\). We can then optimize the logarithm of \(L(\theta)\).

\[
\ln L(\theta) 
= \sum_{i=1}^{n} \ln\left( -\frac{\theta^{x_i}}{x_i \ln(1 - \theta)} \right)
\]

While the solution cannot be written in terms of known functions, we can solve it numerically, giving the maximum at

\[\hat{\theta}=0.9974\]

as reported by Fisher.

\includegraphics{_main_files/figure-latex/unnamed-chunk-145-1.pdf}

He also defined the biodiversity index \[\alpha=N \frac{1-\theta}{\theta}\sim 40.68\]
Were \(N=15609\) is the total number of individuals observed. This value represents a system of high richness-many species are present- and high evenness-individuals are fairly well distributed among species.

\textbf{Example (The Normal Distribution)}

Gauss used astronomical data from Giuseppe Piazzi to infer the true position of Ceres at a given time, Gauss derived the error function

\[f(x; \mu, \sigma^2)= \frac{1}{\sigma \sqrt{2 \pi}} e^{-\frac{1}{2\sigma^2} (x-\mu)^2}\]

Where the \textbf{true} position of Ceres was the mean \(\mu\), or the expected value of the outcomes. How did Gauss combined the data for having the best estimate for the position of Ceres?

What is the statistic that can describe best its position?

\includegraphics{./figures/cerestime.JPG}

In modern times, this question can be formulated as: What is the maximum likelihood estimator of \(\mu\) for a random normal variable?

\textbf{Maximum likelihood of the normal distribution}

For a random normal variable

\[X \rightarrow N(\mu, \sigma^2)\].

What are the estimators of \(\mu\) and \(\sigma^2\) that maximize the probability of the observed data?

We follow the maximum likelihood method:

\textbf{Step 1.} The likelihood function, or the probability of having observed the sample \((x_1, ....x_n)\) is

\(L(\mu, \sigma^2)=\Pi_{i=1..n} f(x_i;\mu,\sigma)\)

\[=\big( \frac{1}{\sigma \sqrt{2 \pi}}\big)^n e^{-\frac{1}{2\sigma^2} \sum_i(x_i-\mu)^2}\]

\textbf{Step 2.} We take the log of \(L\), and compute the \textbf{log-likelihood}

\[\ln L(\mu, \sigma^2)=-n \ln(\sigma \sqrt{2 \pi})-\frac{1}{2\sigma^2} \Sigma_i(x_i-\mu)^2\]

The estimates of \(\mu\) and \(\sigma^2\) are where the likelihood is maximum. They give the highest probability for the data.

\textbf{Step 3.} We differentiate with respect to \(\mu\) and \(\sigma^2\). These two derivatives give us two equations, one for each of the parameters. For deriving respect to \(\sigma^2\), it is easier to make a substitution \(t=\sigma^2\).

\begin{enumerate}
\def\labelenumi{\alph{enumi})}
\item
  \(\frac{d \ln L(\mu, \sigma^2)}{d\mu}=\frac{1}{\sigma^2} \sum_i(x_i-\mu)\)
\item
  \(\frac{d \ln L(\mu, \sigma^2)}{d\sigma^2}=-\frac{n}{2 \sigma^2}+\frac{1}{2\sigma^4} \sum_i(x_i-\mu)^2\)
\end{enumerate}

The derivatives are \(0\) at the maxima

\begin{enumerate}
\def\labelenumi{\alph{enumi})}
\tightlist
\item
  \(\frac{1}{\hat{\sigma}^2} \sum_i(x_i-\hat{\mu})=0\)
\item
  \(-\frac{n}{2 \hat{\sigma}^2}+\frac{1}{2\hat{\sigma}^4} \sum_i(x_i-\hat{\mu})^2=0\)
\end{enumerate}

solving both equations for the parameters we find for \(\mu\)

\[\hat{\mu}_{ml}=\frac{1}{n}\sum_i x_i=\bar{x}\]

and for \(\sigma^2\)

\[\hat{\sigma}^2_{ml}=\frac{1}{n}\sum_i(x_i-\bar{x})^2\]

Therefore, the sample mean (average) \(\bar{X}\) is the maximum likelihood estimator of the mean \(\mu\). Gauss showed that the statistics that we should trust most (that with highest likelihood) for the real position of the Ceres was the \textbf{average}. Gauss solving the position of Ceres, not only discovered the normal distribution, but also created the regression analysis and showed the importance of the average. It is due to him that we often use the average as a preferred statistics, assuming that the observations follow a normal distribution.

In addition, the maximum likelihood estimator of \(\sigma^2\) is a \textbf{biased} estimator because it can be shown that \[E(\hat{\sigma}^2_{ml})=\sigma^2-\frac{\sigma^2}{n}\neq \sigma^2\]

It was Fisher who showed that this estimator was important, as he used it to generalize the central limit theorem.

\hypertarget{questions-9}{%
\section{Questions}\label{questions-9}}

\textbf{1)} An estimator is not

\textbf{\(\qquad\)a:} a statistic;
\textbf{\(\qquad\)b:} a random variable;
\textbf{\(\qquad\)c:} discrete;
\textbf{\(\qquad\)d:} an observation of the parameter;

\textbf{2)} An estimator is unbiased if

\textbf{\(\qquad\)a:} it is the parameter that it estimates;
\textbf{\(\qquad\)b:} depends on \(1/n\);
\textbf{\(\qquad\)c:} its variance is small;
\textbf{\(\qquad\)d:} its expected value is the parameter it estimates;

\textbf{3)} An estimator is consistent if

\textbf{\(\qquad\)a:} it is the parameter that it estimates;
\textbf{\(\qquad\)b:} depends on \(1/n\);
\textbf{\(\qquad\)c:} its variance is small;
\textbf{\(\qquad\)d:} its expected value is the parameter it estimates;

\textbf{4)} The maximum likelihood method

\textbf{\(\qquad\)a:} Produces estimators based on the probability of the observations;
\textbf{\(\qquad\)b:} produces unbiased estimators;
\textbf{\(\qquad\)c:} produces consistent estimators;
\textbf{\(\qquad\)d:} produces estimators equal to those of the method of moments;

\hypertarget{exercises-10}{%
\section{Exercises}\label{exercises-10}}

\hypertarget{exercise-1-9}{%
\subsubsection{Exercise 1}\label{exercise-1-9}}

Take a random variable with the following probability density function

\[
f(x)=
\begin{cases}
    (1+\theta)x^\theta,& \text{if } x\in (0,1)\\
    0,&  x\notin (0,1)
\end{cases}
\]

\begin{itemize}
\item
  What is the maximum likelihood estimator for \(\theta\)?
\item
  If we take a \(5\)-sample with observations
  \(x_1 = 0.92; \qquad x_2 = 0.79; \qquad x_3 = 0.90; \qquad x_4 = 0.65; \qquad x_5 = 0.86\)
\end{itemize}

What is the estimated value of the parameter \(\theta\)?

\begin{itemize}
\tightlist
\item
  Compute \(E(X)=\mu\) as a function of \(\theta\). What is the maximum likelihood estimator for \(\mu\)?
\end{itemize}

\hypertarget{exercise-2-9}{%
\subsubsection{Exercise 2}\label{exercise-2-9}}

For a random variable with a binomial probability function

\[f(x; p)=\binom n x p^x(1-p)^{n-x}\]

\begin{itemize}
\item
  What is the maximum-likelihood estimator of \(p\) for a sample of size \(1\) of this random variable?
\item
  In \textbf{one} exam of \(100\) students we observed \(x_1=68\) students that passed the exam. What is the estimate of the \(p\)?
\end{itemize}

\hypertarget{exercise-3-7}{%
\subsubsection{Exercise 3}\label{exercise-3-7}}

Take a random variable with the following probability density function

\[
    f(x)= 
\begin{cases}
    \lambda e^{-\lambda x},& \text{if } 0 \leq x\\
    0,& otherwise 
\end{cases}
\]

\begin{itemize}
\item
  What is the maximum likelihood estimator for \(\lambda\)?
\item
  If we take a \(5\)-sample with observations
  \(x_1 = 0.223 \qquad x_2 = 0.681; \qquad x_3 = 0.117; \qquad x_4 = 0.150; \qquad x_5 = 0.520\)
\end{itemize}

What is the estimated value of the parameter \(\lambda\)?

\begin{itemize}
\item
  What is the maximum likelihood estimator of the parameter \(\alpha=\frac{n}{\lambda}\)
\item
  Is \(\alpha\) an unbiased and consistent estimator of the mean of the sample sum \(E(Y)\), where \(Y=\sum_1^n X_i\)?
\end{itemize}

\hypertarget{interval-estimation}{%
\chapter{Interval estimation}\label{interval-estimation}}

We perform repeated measurements of a random experiment to learn about the invariant properties that define it. To separate the signal from random variation, we can propose a probabilistic model whose parameters represent abstract properties of the underlying process. These parameters are assumed to remain fixed, while the randomness in the observations is described by the probability distribution over possible outcomes.

The assumption of parameter invariance is often reasonable. For instance, we do not expect the electric charge of an electron to change. However, other quantities---such as the average height of a population---may vary across generations. In more complex or extreme systems, such as city size distributions or gene connectivity networks, parameters like the mean or variance may not even be well defined; these systems may exhibit scale-free behavior.

Given the diversity of experimental settings, the nature of the parameters we estimate can vary widely. To begin, we will consider simpler cases where, under controlled conditions, we can assume that the parameters are fixed. This allows us to define estimators that will tend to cluster around the value of the parameter. In this framework, the only source of variation in parameter estimation comes from the randomness in the sample. For example, when using the average to estimate the mean of a random variable, it is the average---not the mean of the variable---that changes across samples.

This setup naturally leads to a central question: How confident can we be that the unknown (but fixed) parameter is close to its estimated value? This is a challenging question, since we do not know the parameter's location. How can we answer it using only the data we have?

The key idea is to consider what would happen if we repeated the sampling process many times. If we assume a probability model for the data, we can ask: How far would the new estimates typically fall from the original one? If the estimator is unbiased, the spread of its sampling distribution can give us a sense of how close the estimates are likely to be to the true parameter value---because repeated estimates will cluster around it. The approach is known as frequentist, imaging repetitions of the sample.

In this chapter, we introduce the concept of \textbf{confidence intervals} for means, proportions, and variances. We will derive formulas for confidence intervals under various conditions, including when the population variance is known or unknown, and when the sample size is large.

\hypertarget{revisiting-parameter-estimation-and-marging-of-error}{%
\section{Revisiting parameter estimation and marging of error}\label{revisiting-parameter-estimation-and-marging-of-error}}

The expected value or the mean is a fundamental parameter of the random experiment. The researchers typically ask, if an experiment is repeated, the observations will gather about a number in a consistent manner?

\textbf{Estimation of the mean}

We have seen that whenever we take a random sample \((X_1, X_2, ... X_n)\), the sample mean
\[\bar{X}=\frac{1}{n}\sum_{i=1}^n X_i\]

is an estimator of the mean \(\mu\) of the random variable \(X\). That is

\[\bar{x}=\hat{\mu}\]
We take the average (the observation of the sample mean), as the value of \(\mu\) we trust the most given our data (maximum likelihood).

The estimator is \textbf{unbiased} because its values center about the parameter it is estimating

\begin{itemize}
\tightlist
\item
  \(E(\bar{X})=\mu\)
\end{itemize}

and \textbf{consistent} because as \(n\) increases then it is closer to the parameter, as its variance gets smaller

\begin{itemize}
\tightlist
\item
  \(V(\bar{X})=\frac{\sigma^2}{n}\)
\end{itemize}

where \(\sigma^2\) is the variance of \(X\). We call the quantity \(\sigma_{\bar{x}}=\frac{\sigma}{\sqrt{n}}\) the \textbf{standard error} (\(se\)).

Since \(\bar{X}\) is a random variable the estimation of the mean changes when we take another sample. Therefore, we know that we are making an error every time we take \(\bar{x}\) for \(\mu\).

\textbf{Margin of error}

When deciding whether the \textbf{error} in estimation \[\bar{X}-\mu\] is large or not, we defined the \textbf{margin of error}. The margin of error at \(5\%\) is the distance \(m\) from \(\mu\) such that \(\bar{X}\) has a probability of \(95\%\) to be observed within:

\[P(-m \leq \bar{X}-\mu \leq m)=0.95\]

\textbf{If we assume} that the distribution of \(X\) is normal, \(X \sim N(\mu, \sigma^2)\), then the \textbf{standardized error} from the mean

\[T=\frac{\bar{X}-\mu}{\frac{S}{\sqrt{n}}}\]
follows a \(t\)-distribution with \(n-1\) degrees of freedom

\[T \sim t_{n-1}\]

where \(S^2=\frac{1}{n-1} \sum_{i=1}^n (X_i-\bar{X})^2\) is the sample variance. The margin of error \(m\) at \(5\%\) is given by the

\(P(\mu-m \leq \bar{X} \leq\mu + m)\)
\[=P(-\frac{m}{S/\sqrt{n}} \leq T \leq\frac{m}{S/\sqrt{n}})=0.95\]

Since \(T\) is contained in the interval \((-t_{0.025, n-1},t_{0.025, n-1})\) with \(95\%\) of the probability, then the margin of error is

\[m=t_{0.025, n-1} \frac{s}{\sqrt{n}}\]
where the limit of the interval \(t_{0.025, n-1}\) is the value of \(T\) that leaves \(2.5\%\) of probability at the right hand side of the \(t\)-distribution with \(n-1\) degrees of freedom

\begin{verbatim}
Python: t.sf(0.025, n-1) 
R: qt(0.025, n-1, lower.tail = FALSE)
\end{verbatim}

Note that the margin of error is the expectation of how far the average will be observed from the mean with \(95\%\) of probability. Clearly if the observed average is luckily the mean then the error is zero, the errors will be symmetrical about zero and higher errors will be less probable.

\includegraphics{_main_files/figure-latex/unnamed-chunk-146-1.pdf}

\hypertarget{interval-estimation-for-the-mean}{%
\section{Interval estimation for the mean}\label{interval-estimation-for-the-mean}}

The problem is that in many experiments we \textbf{do not know} \(\mu\). If we are going to collect the first sample of the experiment, we do not have an expectation of how far the average is going to be observed from the mean, nor an expectation of its deviation from it.

\textbf{Example (CRISPR)}

CRISPR-Cas9 is a technology that enables gene editing. CRISPR-Cas system uses a guide RNA to direct the Cas9 enzyme to a specific DNA sequence, where it creates a cut. This allows researchers to disrupt, correct, or insert genetic material at targeted sites in the genome. The first-in-human study using CRISPR-edited cells, assessed the safety and feasibility of editing immune T cells to combat a form of lung cancer that had not responded to standard therapies \citep{Lu2020}. This is the reported efficiency of editing in the study

\[
\begin{array}{cc}
 \mathbf{Patient} & \mathbf{Editing Efficiency (\%)} \\
 C-03      & 16                     \\
 C-02      & 34                     \\
 C-01      & 10                     \\
 B-03      & 16                     \\
 B-02      & 12                     \\
 B-01      & 30                     \\
 A-04      & 22                     \\
 A-03      & 10                     \\
 A-02      & 20                     \\
 A-01      & 18                     \\
 PreA-02   & 8                      \\
 PreA-01   & 16                     \\
\end{array}
\]

This data is the repetition of a random experiment 12 times, where the efficiency was the random variable and the observation unit a patient. As the same experimental procedure was applied, it is expected that the efficiency will cluster around some value that represents the CRISPR experiment of the study.

\includegraphics{_main_files/figure-latex/unnamed-chunk-147-1.pdf}

The observations (crosses) and the average (dot) is all we have. We can assume that if the researchers were to repeat the sample on 12 different patients, the average will not be far from the value that they observed, because the average clusters around the mean. However, we do not know where the mean is.

Nonetheless, we can assume that editing efficiency is normally distributed

\[X \sim N(\mu, \sigma^2)\]
at unknown \(\mu\) and \(\sigma\).

Data suggest that the expected editing efficiency \(\mu\) is about \(\bar{x}=17.66\). But, how \textbf{confident} are we? after all, we know that when we take the value of \(\bar{x}\) for \(\mu\), we know we are making a mistake but do not know how big it actually is. The key is to note that the dispersion of sample (\(s\)) will give us an idea on how the average will disperse (\(s/\sqrt{n}\)).

\textbf{Definition}

To address the question of where \(\mu\) is, we define the \textbf{confidence interval} for \(\mu\). From the margin of error equation

\[P(-m \leq \bar{X} - \mu \leq  m)=0.95\]
We solve for \(\mu\), which is indeed \textbf{the real unknown}

\[P(\bar{X} - m \leq \mu \leq \bar{X} + m)=0.95\]

The left and right limits of the inequality are random variables which motivate the definition for the \textbf{random confidence interval at \(95\%\):}

\[(L,U)=(\bar{X} - m,\bar{X} + m)\]

This interval is a new \textbf{random variable} and it has by definition a probability of \(0.95\) to contain \(\mu\).

The \textbf{observed interval} that we obtain from the experiment is (lower case)

\[(l,u)=(\bar{x} - m,\bar{x} + m)\]

This interval either contains or it does not contain the parameter \(\mu\): we will \textbf{never know}!

However, we can still say that we have a \textbf{confidence} of \(95\%\) that the interval \((l,u)\) has captured the true unknown parameter \(\mu\). Think of a lottery scratch ticket that we are not allow to scratch to see the prize before buying it. The ticket either has or does not have the prize, only that we do not know which case it is. However, we can be confident that the ticked does not have the main prize, because the probabilities of the game (not the instance in my hand) are low.

\hypertarget{confidence-interval-estimation}{%
\section{Confidence Interval Estimation}\label{confidence-interval-estimation}}

We can \textbf{estimate} confidence intervals when we are able find an standardization of an estimator with a known distribution.

Standardization allows separating the random variation from the parameters, which is not always possible but can be done for some useful and widely applied examples.

Notice that the interval depends of the margin error, which is a parameter (property) of the estimator. As such, the observed interval needs to be computed with a required statistical interpretation and, consequently, it is not a pure datum of the experiment. Hence, confidence intervals are estimated rather then directly observed from the experiment.

\hypertarget{estimation-of-the-mean-for-normal-variables}{%
\subsection{Estimation of the mean for normal variables}\label{estimation-of-the-mean-for-normal-variables}}

When the random variable \(X\) is a normal variable, we can readily write the confidence interval at \(95\%\)

\[(l,u)=(\bar{x} - m, \bar{x} + m)\]
where \[m=t_{0.025, n-1} \frac{s}{\sqrt{n}}\]

That is:

\[(l,u)=(\bar{x} - t_{0.025, n-1} \frac{s}{\sqrt{n}}, \bar{x} + t_{0.025, n-1} \frac{s}{\sqrt{n}})\]

\textbf{Example (CRISPR)}

In the CRISPR example, we assume that \(X\) is normally distributed and from the data we can compute \(\bar{x}= 17.66\) and \(s=7.9\).

Then the margin of error is

\[m=t_{0.025, 11} \frac{s}{\sqrt{n}}=2.2\frac{7.9}{\sqrt{12}}=5.04\]
and the \(95\%\) confidence interval is

\((l,u)=(\bar{x} - m, \bar{x} + m)=\) \[(12.61, 22.71)\]

Which means that we are \(95\%\) confident that the efficiency of the T-cell editing can be found between \(12.6\%\) and \(22.7\%\)

\begin{verbatim}
Python:
from scipy.stats import ttest_1samp
x = [16, 34, 10, 16, 12, 30, 22, 10, 20, 18, 8, 16]
res = ttest_1samp(x, popmean=0)
print(res.confidence_interval)

R:
x <- c(16, 34, 10, 16, 12, 30, 22, 10, 20, 18, 8, 16)
t.test(x)$conf.int
\end{verbatim}

We can also write the interval as

\[\hat{\mu}=\bar{x}  \pm m =  17.6 \pm 5.0\]
This means that, when estimating the mean by the average, we are confident that the efficiency is between the tenths and twenties of percentage points, and less confident in the figures on percentage units. We have no confidence on the decimal places.

Remember that the confidence interval \((l,u)\) (black below) is an observation of the random confidence interval \((L,U)\). Therefore, if we change the sample then \((l,u)\) changes (grey below). The frequentist statistician encourages us to imagine selecting a different group of \(12\) patients, compute the confidence intervals, and repeat this process an infinite amount of times. Then, about \(95\%\) of the confidence intervals will contain the unknown \(\mu\). We just do not know which intervals, including the estimated one, will contain \(\mu\)!

\includegraphics{_main_files/figure-latex/unnamed-chunk-148-1.pdf}

We can say that Lu and colleagues achieved a T-cell editing efficiency of about \(17\%\) as they repeated the experiment 12 times in similar conditions. It is clear, however, that other laboratories may obtain different efficiencies, as some of the conditions will change. Therefore, we expect that the parameter \(\mu\) will change between laboratories in addition to sampling error. At \(17\%\) efficiency, they showed that the treatment appear to be safe for 12 cancer patients, but as efficiency is improved and more patients are recruited then patient safety will need to be updated.

\textbf{Confidence level}

We can change our confidence from \(95\%\) to \(99\%\). When we computed the margin of error at \(95\%\), we left out \(\alpha=0.05\) probability, \(0.025\) on each side of the standardized error.

Now, we can leave out \(\alpha=0.01\) probability, \(0.005\) on each side. Therefore the \(99\%\) confidence interval is

\[(l,u) = (\bar{x} - t_{0.005, n-1}\frac{s}{\sqrt{n}},\bar{x} +  t_{0.005, n-1}\frac{s}{\sqrt{n}})\]

\includegraphics{_main_files/figure-latex/unnamed-chunk-149-1.pdf}

where \(t_{0.005, n-1}=F^{-1}(0.995)\) is the inverse of the probability density function for the \(t\)-distribution.

\begin{verbatim}
Python: t.ppf(1-0.005, n-1) 
R: qt(0.005, n-1, lower.tail = FALSE)
\end{verbatim}

\textbf{Example (Impact energy)}

A metallic material is tested for impact to measure the energy required to cut it at a given temperature. Ten specimens of A238 steel were cut at 60ºC at the following impact energies (J):

64.1, 64.7, 64.5, 64.6, 64.5, 64.3, 64.6, 64.8, 64.2, 64.3

If we \textbf{assume} that the impact energy is normally distributed we can estimate the \(99\%\) confidence interval for the cutting energy, as an intrinsic propioerty of the material. From the data we have

\begin{itemize}
\tightlist
\item
  \(\bar{x}=64.46\)
\item
  \(s=0.227\)
\end{itemize}

we assume

\begin{itemize}
\tightlist
\item
  \(\alpha=0.01\) (the confidence limit)
\item
  \(t_{0.005,9}=3.24\) obtained from \(t_{0.005,9}=\) t.ppf(1-0.005, 9)
\end{itemize}

The confidence interval is then

\((l,u)=(\bar{x}- t_{0.005,9}\frac{s}{\sqrt{n}},\bar{x}+t_{0.005,9} \frac{s}{\sqrt{n}})\)

\[=(64.22, 64.69)\]

\begin{verbatim}
Python:
from scipy import stats
x = [64.1,64.7,64.5,64.6,64.5,64.3,64.6,64.8,64.2,64.3]
res = stats.ttest_1samp(x, popmean=0)
res.confidence_interval(confidence_level=0.99)

R:
x <- c(64.1,64.7,64.5,64.6,64.5,64.3,64.6,64.8,64.2,64.3)
t.test(x, conf.level = 0.99)$conf.int
\end{verbatim}

Note that the confidence interval at \(95\%\) is \((64.29, 64.62)\), which is smaller than the \(99\%\) interval. The wider intervals with increasing confidence shows the trade-off between confidence and precision.

\hypertarget{estimation-of-the-proportion-for-dichotomic-variables}{%
\subsection{Estimation of the proportion for dichotomic variables}\label{estimation-of-the-proportion-for-dichotomic-variables}}

\textbf{Example (Mendel's peas)}

Mendel first produced a large number of hybrid pea plants. These were plants for which one parent descended from a long line of plants that produced round peas, and the other from a long line that produced wrinkled peas. He then took pairs of these hybrid plants and produced 253 pea plants. He collected a total of 7,324 seeds from the hybrid offspring and observed that 5,474 were round and 1,850 were wrinkled \citep{mendel1901experiments}. What is the estimated proportion of round seeds in the offspring observed by Mendel?

The outcome \(X_i\) that the \(i\)-th seed is a round pea is a Bernoulli trial

\[X_i \sim Bernoulli(p)\]
with mean \(\mu=p\) and variance \(\sigma^2=p(1-p)\).

Mendel's sample would look something like
\[(x_1,x_2, x_3, ...x_{n=7324})=(0,1,0,.. 1, 0)\] with \(5474\) ones and in a total of \(7324\) repetitions of the trial. The sample has an average \[\bar{x}=\frac{1}{7324}\sum_{i=1}^{7324} x_i=\frac{5474}{7324}=0.747\]
Which is nothing but the relative frequency of the round peas. Since, in general, the sample mean is an unbiased estimator of \(\mu\), then, for the Bernoulli trial, this the relative frequency is a point estimate of the parameter \(p\)

\[\hat{p}=\bar{x}=0.747\]

This makes sense because \(\bar{x}\) is the observed relative frequency of ones \(f_1\) in the sample. And as such, it is an estimator of the probability of observing a one in a Bernoulli trial

\[f_1 =\hat{P}(X=1)\]
However, how confident are we about this estimation? That is, how confident are we to take the relative frequency as the value of the probability? We want a confidence interval for \(p\).

\textbf{Confidence intervals for proportions}

When \(n\hat{p}>5\) and \(n(1-\hat{p})>5\), the \textbf{standardized error} of estimating \(p\) with \(\bar{X}\) can be approximated to a standard normal variable with the help of the central limit theorem, using De Moivre' normal approximation of the Binomial.

\[Z=\frac{\bar{X}-\mu}{\sigma/\sqrt{n}}= \frac{\bar{X}-p}{\big[\frac{p(1-p)}{n} \big]^{1/2}}\sim N(0,1)\]
Therefore the margin of error \(m\) at \(95\%\) is

\[m=  z_{0.025}\frac{\sqrt{p(1-p)}}{\sqrt{n}}\]
Since we do not know what the value of \(p\) is then we will estimate it with \(\bar{x}\) and then
\[m=  z_{0.025}\frac{\sqrt{\bar{x}(1-\bar{x})}}{\sqrt{n}}\]
Which is a good approximation when the central limit theorem starts to hold. Therefore, an approximate \(95\%\) CI interval of \(p\) is:

\[CI=(l,u)=(\bar{x}-z_{0.025}\big[\frac{\bar{x}(1-\bar{x})}{n} \big]^{1/2},  \bar{x}+z_{0.025}\big[\frac{\bar{x}(1-\bar{x})}{n} \big]^{1/2})\]
Note that we could not solve exactly the margin of error without reference to the parameter \(p\). We were able to introduce an estimate because the central limit theorem achieves a suitable standardization when \(n\) is large. More accurate confidence intervals for small \(n\) or far from \(p=1\) and \(p=0\) have been developed.

\textbf{Example (Mendel's peas)}

In Mendel's experiments, he counted \(5474\) rounded peas in a total of \(7324\) total seeds and he therefore obtained

\begin{itemize}
\item
  \(\bar{x}=0.747\) (relative frequency as a point estimate of the proportion)
\item
  \(z_{0.025}=1.96\) (using norm.sf(0.025)).
\end{itemize}

Therefore the \(95\%\) confidence interval for \(p\) is

\((l,u)=(0.747-0.013, 0.747+0.013)\)

\[=(0.734, 0.760)\]

\begin{verbatim}
Python:
from statsmodels.stats.proportion import
proportion_confint(5474, 7324)

R:
prop.test(5474, 7324)$conf.int
\end{verbatim}

The estimated probability of observing a round pea is

\[\hat{p} = 0.747 \pm 0.013\]

which validates Mendel's first law, as it aligns closely with the theoretical value \textbf{deduced} for \(p\). Let \(A\) represent the allele for round peas. In Mendel's experiments, all offspring were produced from hybrid parents carrying alleles \((A, A')\). Since the round trait is dominant, a pea will be round if it inherits at least one \(A\) allele. This corresponds to the event \((A_m \cup A_f)\), which occurs in three out of four possible allele combinations. Therefore, the theoretical probability of a round pea is \(p = 3/4 = 0.75\), a value that is well captured by the \textbf{inferred} confidence interval of the relative frequency.

Mendel's success lay in identifying a simple biological system where probabilities could be reasoned and predictions made about where relative frequencies would cluster. However, it is important to remember that this reasoning was grounded in prior observation, experimentation, and reflection on empirical data. A map cannot be drawn without first exploring the territory.

\hypertarget{estimation-of-the-variance}{%
\section{Estimation of the variance}\label{estimation-of-the-variance}}

The variance is another fundamental parameter of the random experiment. The random experiment will be characterized by a variability that may be intrinsic of the observation unit or a characteristic of the experimental error when measuring an outcome. Researchers in this case may ask, if an experiment is repeated, how far the observations will be from the mean?

We have seen that whenever we take a random sample \((X_1, X_2, ... X_n)\), the sample variance
\[S^2=\frac{1}{n-1}\sum_{i=1}^n (X_i-\bar{X})^2\]
is an estimator of the variance \(\sigma^2\) of the random variable \(X\). The estimator is \textbf{unbiased} because its values are centered about the parameter it is estimating

\[E(S^2)=\sigma^2\]

and it is also \textbf{consistent}. We can then take a the value of \(s^2\) from a particular sample as the value of \(\sigma^2\), which is a characteristic of random experiment. That is

\[s^2=\hat{\sigma}^2\]

Since \(S^2\) is a random variable the estimation of the variance changes when we take another sample.

\textbf{Example (impact energy)}

A metallic material is tested for impact to measure the energy required to cut it at a given temperature. Ten specimens of A238 steel were cut at 60ºC at the following impact energies (J):

64.1, 64.7, 64.5, 64.6, 64.5, 64.3, 64.6, 64.8, 64.2, 64.3

While the cutting energy appears to cluster about a value, the specimens show variability on their energies. Some are cut easier than others. How stable is the cutting energy of the specimens? what is estimation of the variance of these data?

\begin{itemize}
\tightlist
\item
  The sample variance is \(s^2=0.051\)
\end{itemize}

\begin{verbatim}
Python:
import numpy as np
x = [64.1, 64.7, 64.5, 64.6, 64.5, 64.3, 64.6, 64.8, 64.2, 64.3]
np.var(x, ddof=1)

R: 
x <- c(64.1, 64.7, 64.5, 64.6, 64.5, 64.3, 64.6, 64.8, 64.2, 64.3)
var(x)
\end{verbatim}

How confident can we be on the decimals places of this estimation? What is the confidence interval for the variance?

\hypertarget{confidence-interval-for-the-variance}{%
\section{Confidence interval for the variance}\label{confidence-interval-for-the-variance}}

To compute a confidence interval of the variance, we need a statistics that is a function of \(S^2\) and allows to measure the standardize error we make when we estimate \(\sigma^2\) by \(s^2\).

\textbf{If we assume} that the distribution of \(X\) is normal, \(X \sim N(\mu, \sigma^2)\), then the \textbf{standardized error} from the variance

\[W=\frac{S^2(n-1)}{\sigma^2}\]
follows a \(\chi^2\) distribution with \(n-1\) degrees of freedom

\[W \sim \chi^2_{n-1}\]

This is an error rate that if the observed sample variance \(s^2\) is luckily the variance \(\sigma^2\) then the rate is \((n-1)\). The errors will distribute asymmetrical about \((n-1)\) and higher errors and those close to \(0\) will be less probable.

Using this error rate, we will look for a \(95\%\) confidence interval \((L,U)\) of \(\sigma^2\) such that the random interval \[P(L \leq \sigma^2 \leq U)=0.95\]

captures \(\sigma^2\) with \(95\%\) of probability.

We start by determining the values that capture the \(95\%\) of the \(\chi^2\)-distribution

\[P(\chi^2_{0.975,n-1} \leq W \leq \chi^2_{0.025,n-1})=0.95\]

\includegraphics{_main_files/figure-latex/unnamed-chunk-150-1.pdf}

Replacing the value of \(W\)

\[P(\chi^2_{0.975,n-1} \leq \frac{S^2}{\sigma^2}(n-1) \leq \chi^2_{0.025,n-1})=0.95\]

and solving for \(\sigma^2\)

\[P(\frac{S^2 (n-1)}{\chi^2_{0.025,n-1}}\leq \sigma^2 \leq \frac{S^2(n-1)}{\chi^2_{0.975,n-1}})=0.95\]

We find a random interval that captures \(\sigma^2\)
with \(95\%\) confidence

\[(L,U) = (\frac{S^2 (n-1)}{\chi^2_{0.025,n-1}},\frac{S^2(n-1)}{\chi^2_{0.975,n-1}})\]

The \textbf{observed} \(95\%\) confidence interval (script size) is

\[(l,u) = (\frac{s^2 (n-1)}{\chi^2_{0.025,n-1}},\frac{s^2(n-1)}{\chi^2_{0.975,n-1}})\]

where

\begin{itemize}
\item
  \(\chi^2_{0.975,n-1}=\) chi2.sf(0.975, df=n-1) is the value of \(W\) that leaves \(0.975\) probability to the right hand side.
\item
  \(\chi^2_{0.025,n-1}=\)chi2.sf(0.025, df=n-1) is the value of \(W\) that leaves \(0.025\) probability to the right hand side.
\end{itemize}

\textbf{Example (impact energy)}

Ten specimens of A238 steel were cut at 60ºC at finding a sample variance \(s^2=0.05155556\). To estimate the confidence interval for the variance we compute

\begin{itemize}
\item
  \(\chi^2_{0.975,n-1}=2.700389\)
\item
  \(\chi^2_{0.025,n-1}=19.02277\)
\end{itemize}

\begin{verbatim}
Python:
chi2.ppf(1 - 0.975, df=9) 
chi2.ppf(1 - 0.025, df=9)

R: 
qchisq(0.975, 9, lower.tail = FALSE)
qchisq(0.025, 9, lower.tail = FALSE)
\end{verbatim}

Therefore

\[(l,u)= (\frac{0.227^2 (10-1)}{19.02277},\frac{0.227^2(10-1)}{2.700389})=(0.02,0.17)\]

We find that we can be \(95\%\) confident to have captured \(\sigma^2\) within this interval.

Note that the interval for the variance is \textbf{not symmetric} and we cannot formulate it as an estimate \(\pm\) margin of error. Given an average, we can then estimate with \(s\) the expected distance of a subsequent observation of the random experiment, that is its standard deviation \(\sigma\). The confidence interval for the standard deviation will be the square root of the the interval for the variance, and will give us the best and the worst case at which we can expect the observation to fall (orange) from either side of the mean.

\includegraphics{_main_files/figure-latex/unnamed-chunk-151-1.pdf}

\hypertarget{questions-10}{%
\section{Questions}\label{questions-10}}

\textbf{1)} The margin of error at \(95\%\) confidence of a normal variable is

\textbf{\(\qquad\)a:} \(\frac{s}{\sqrt{n}}\);
\textbf{\(\qquad\)b:} \(1.96\times se\);
\textbf{\(\qquad\)c:} \(\frac{\sigma}{\sqrt{n}}\);
\textbf{\(\qquad\)d:} \(\sigma\)

\textbf{2)} when we talk about \(z_{0.025}\) we mean:

\textbf{\(\qquad\)a:} The value of a normal standard variable that has accumulated up to \(99.75\%\) of probability;
\textbf{\(\qquad\)b:} The value of a normal standard variable that has accumulates up to \(0.25\%\) of probability;
\textbf{\(\qquad\)c:} The probability of a standard variable up to \(99.75\%\);
\textbf{\(\qquad\)d:} The probability of a standard variable up to \(0.25\%\)

\textbf{3)} The random confident interval \((L,U)\) for the mean at \(95\%\)

\textbf{\(\qquad\)a:} is a two dimensional parameter of the sample distribution;
\textbf{\(\qquad\)b:} gives the limits where \(\mu\) has a probability of occurring \(95\%\) of the times;
\textbf{\(\qquad\)c:} is an estimate of the average;
\textbf{\(\qquad\)d:} captures \(\mu\) \(95\%\) of the times

\textbf{4)} A confidence interval for the mean written as \(\hat{\mu}=56.99 \pm 0.01\)

\textbf{\(\qquad\)a:} indicates that we are \(\%99\) confident that the mean is \(56.99\);
\textbf{\(\qquad\)b:} indicates that we cannot trust the last decimal place on the estimation of the mean;
\textbf{\(\qquad\)c:} indicates that the mean of the population is at \(56.99\) with error \(0.01\);
\textbf{\(\qquad\)d:} indicates that we can trust the unit figure (\(6\)) on the estimation of the mean

\textbf{5)}If we know the value of \(\mu\) and find that the confidence interval did not catch it then

\textbf{\(\qquad\)a:} the confidence interval is not well computed;
\textbf{\(\qquad\)b:} it is a rare observation of the confidence interval;
\textbf{\(\qquad\)c:} the confidence interval does not estimate the mean;
\textbf{\(\qquad\)f:} there is little probability of finding the mean in the confidence interval

\hypertarget{exercises-11}{%
\section{Exercises}\label{exercises-11}}

\hypertarget{exercise-1-10}{%
\subsubsection{Exercise 1}\label{exercise-1-10}}

In a scientific paper, the authors report a \(95\%\) confidence interval of \((228, 232)\) for the natural frequency (Hz) of a metallic beam. They used a sample of size \(25\) and considered that the measurements were distributed normally.

\begin{itemize}
\item
  What is the mean and the standard deviation of the measurements?
\item
  Compute the \(99\%\) confidence interval.
\end{itemize}

hints:

\begin{itemize}
\item
  in R \(t_{0.025, 24}=\) t.ppf(1-0.025, 24)\(\sim 2\)
\item
  in R \(t_{0.005, 24}=\)t.ppf(1-0.005, 24)\(\sim 2.8\)
\end{itemize}

\hypertarget{exercise-2-10}{%
\subsubsection{Exercise 2}\label{exercise-2-10}}

compute \(95\%\) CI the mean of a normal variable with known variance \(\sigma^2=9\) and \(\bar{x}=22\), using a sample of size \(36\).

\hypertarget{exercise-3-8}{%
\subsubsection{Exercise 3}\label{exercise-3-8}}

This year, \(17\) of \(1000\) of patients with influenza developed complications.

\begin{itemize}
\item
  Compute the \(99\%\) confidence interval for the proportion of complications.
\item
  The previous year \(2\%\) showed complications. Can we say with \(99\%\) confidence that this year there is a significant drop in influenza complications?
\end{itemize}

\hypertarget{exercise-4-6}{%
\subsubsection{Exercise 4}\label{exercise-4-6}}

What is the confidence interval for population variance of a normal variable if we take a random sample of size \(n=10\) and observe a sample variance of \(0.5\)?

\hypertarget{practice-3}{%
\section{Practice}\label{practice-3}}

Load misophonia data \url{https://alejandro-isglobal.github.io/SDA/data/data_0.txt}

\begin{itemize}
\item
  Compute the confidence interval for the mean of the cephalometric measures. (``Angulo\_convexidad'', ``protusion.mandibular'', ``Angulo\_cuelloYtercio'', ``Subnasal\_H'')
\item
  Compute the confidence interval for the proportion of misophonic (``Misofonia''), and depression (``depresion.dic'').
\item
  Compute the confidence interval for the variance of the age (``Edad''). What is the confidence interval for the standard deviation of the population?
\end{itemize}

\href{https://colab.research.google.com/drive/13El5aoycT_6Wasvyx427TvglygiSPMpF?usp=sharing}{Solutions}

\hypertarget{hypothesis-testing}{%
\chapter{Hypothesis Testing}\label{hypothesis-testing}}

When we perform an experiment, we often want to determine whether changes we introduce have a real effect. For instance, we might want to know if we can influence the outcome of the experiment, or whether a new condition alters the results in a meaningful way. Often, we already have expectations---based on prior knowledge---of how the data should behave when these new conditions are \textbf{not present}. But since the outcomes of experiments are subject to random variation in any case, how can we distinguish a real effect from random noise?

The strategy is to construct a probability model for the experiment and assess how the model's \textbf{parameters} change under different conditions. We use random samples to estimate the parameters and by comparing the estimations under different conditions, we can decide whether the parameters shift in a way that provides evidence of a significant effect.

The key challenge is to determine whether the observed variation in the parameter estimation is genuinely due to the changes in conditions or simply a result of sampling variability.

In this chapter, we introduce the framework of \textbf{hypothesis testing} for means, proportions, and variances. We will define the \textbf{null hypothesis} (which assumes no change or effect) and the \textbf{alternative hypothesis} (which assumes a meaningful change). Using data, we will learn how to decide whether the evidence supports rejecting the null hypothesis in favor of the alternative.

Finally, we will discuss the types of errors that can occur in hypothesis testing---\textbf{false positives} (Type I errors) and \textbf{false negatives} (Type II errors)---and how they affect our conclusions.

\hypertarget{hypothesis-formulation}{%
\section{Hypothesis formulation}\label{hypothesis-formulation}}

\textbf{Example (Lithium-ion (Li-ion) batteries)}

Li-ion batteries are expected to play a pivotal role in supporting the global transition to clean energy and electrified transportation. Their performance, measured in discharged capacity (in milliampere-hours, mAh), is highly sensitive to storage conditions such as temperature and state of charge (SOC). The CALCE battery group systematically studied how different storage temperatures and SOC levels affect long-term battery degradation \citep{CALCE_StorageData2025}.

The discharged capacity refers to the total amount of electric charge a battery can deliver during a full discharge cycle after being fully charged. For example, batteries stored fully charged for three weeks at 50°C show a discharged capacity of 1.561 mAh after being recharged. This raises a practical and important question:

Does storing the battery in a fully discharged state instead improve its discharged capacity after storage?

Or more broadly:

Should a battery be stored fully charged or fully discharged to minimize degradation over time?

We are interested in testing whether the mean discharged capacity of batteries stored fully discharged for three weeks differs from 1.561 mAh, after they have been fully recharged.

To answer this, we can formulate two mutually exclusive statements:

\begin{itemize}
\item
  \begin{enumerate}
  \def\labelenumi{\alph{enumi}.}
  \tightlist
  \item
    The mean discharged capacity of the discharged-storage batteries is equal to 1.561 mAh\\
  \end{enumerate}
\item
  \begin{enumerate}
  \def\labelenumi{\alph{enumi}.}
  \setcounter{enumi}{1}
  \tightlist
  \item
    The mean discharged capacity of the discharged-storage batteries is different from 1.561 mAh
  \end{enumerate}
\end{itemize}

Only one of these statements can be true. The goal of the experiment is to collect data and decide which of the two better reflects the behavior of the batteries.

It is important to note that due to natural variability, some individual batteries may show capacities greater than 1.561 mAh, and some lower. The key question is about the mean performance of the group, not individual observations.

Let \(\mu\) represent the mean discharged capacity of batteries stored fully discharged. Then, the above statements can be reformulated as statistical hypotheses:

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\tightlist
\item
  \(H_0: \mu = 1.561\,\text{mAh}\) (null hypothesis)\\
\item
  \(H_1: \mu \neq 1.561\,\text{mAh}\) (alternative hypothesis)
\end{enumerate}

Here, \(H_0\) assumes no effect of storage condition---that is, fully discharged storage results in the same capacity as fully charged storage. In contrast, \(H_1\) suggests that the storage condition does influence the battery's performance. These two statements constitute the basis for hypothesis testing.

\textbf{Definition}

In statistics, a statement (conjecture) about the probability function of a random variable is called a \textbf{hypothesis}.

The hypothesis is usually written in two dichotomous statements

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\item
  The \textbf{null} hypothesis \(H_0\): when the conjecture is false. It usually refers to the \textbf{status quo}. The data may be explained by the status quo.
\item
  The \textbf{alternative} hypothesis \(H_1\): when the conjecture is true. It usually refers to \textbf{research hypothesis}. The data may be explained by the alternative to the status quo.
\end{enumerate}

\textbf{Example (Fertilizer)}

What are the null and the alternative hypothesis for the following situation?

Fertilizer developers want to test whether their new product has a real effect on the growth of plants.

Being \(\mu_0\) the mean growth of the plants \textbf{without} fertilizer (known) and \(\mu\) the mean growth of the plants with the fertilizer (unknown)

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\tightlist
\item
  \(H_0:\mu \leq \mu_0\) (The fertilizer may do nothing: status quo)
\item
  \(H_1:\mu > \mu_0\) (The fertilizer may have the desired effect: research interest)
\end{enumerate}

\textbf{Example (Chemotherapy)}

Pharmaceutical companies need to know if a novel chemotherapy can cure 90\% of cancer patients.

Being \(p_0\) the proportion of patients that are cured \textbf{without} the chemotherapy (known) and \(p\) the proportion that are cures \textbf{with} the chemotherapy (unknown)

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\tightlist
\item
  \(H_0:p \leq p_0\) (The chemotherapy may do nothing: status quo)
\item
  \(H_1: p > p_0\) (The chemotherapy may have the desired effect: research interest)
\end{enumerate}

Note that our new improved experiment has the parameter \(p\) and we want to know how it compares to the experiment without \textbf{any additional improvement} that has the parameter \(p_0\).

We want to decide between \(H_0\) and \(H_1\). There are two options:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  We \textbf{reject} the alternative hypothesis \(H_1\); that is, we accept the null hypothesis \(H_0\).
\item
  We \textbf{accept} the alternative hypothesis \(H_1\) (our interest); that is, we reject the null hypothesis \(H_0\).
\end{enumerate}

\textbf{Example (Li-ion Batteries)}

Suppose we want to store Li-ion batteries but are unsure whether to store them fully charged or fully discharged to minimize long-term degradation.

To address this, we formulate a hypothesis test:

\begin{itemize}
\tightlist
\item
  \(H_0: \mu = 1.561\,\text{mAh}\). The mean discharge capacity is the same whether batteries are stored fully charged or fully discharged.\\
\item
  \(H_1: \mu \neq 1.561\,\text{mAh}\). The mean discharge capacity differs when batteries are stored fully discharged compared to fully charged.
\end{itemize}

To decide between \(H_0\) and \(H_1\), we measure the discharge capacity of 12 batteries that were stored fully discharged for three weeks.

\includegraphics{_main_files/figure-latex/unnamed-chunk-152-1.pdf}

From the data, the average discharge capacity of the 12 batteries stored fully discharged is\\
\(\bar{x} = 1.569232\) mAh. At first glance, this might suggest that storing batteries discharged reduces degradation compared to storing them fully charged.

However, we must consider that \(\bar{x}\) is the result of a random sample and may vary. Sometimes it will be higher than 1.561mAh, and sometimes lower. Therefore, the observation \(\bar{x} \ne 1.561\) is not, by itself, sufficient evidence to reject \(H_0\).

This leads us to ask:

If \(H_0\) is true, what range of values would we typically expect for \(\bar{x}\), based on a sample of 12 batteries?

In other words, is \(\bar{x} = 1.569232\) a typical result, assuming the expected value of the new experiment is indeed 1.561 mAh?

To answer this, we need a formal decision process, which we introduce in the following sections.

\hypertarget{hypothesis-testing-1}{%
\section{Hypothesis Testing}\label{hypothesis-testing-1}}

Let us summarize the different \textbf{cases}, \textbf{methods}, and \textbf{types} of hypothesis testing. We will then discuss each case using a specific example.

Hypotheses can be tested --- or decided upon --- using confidence intervals. Therefore, we will focus on the same \textbf{cases} we encountered when constructing confidence intervals, namely:

\begin{itemize}
\tightlist
\item
  Hypothesis test for the mean \(\mu\), when \(X \sim N(\mu, \sigma^2)\)
\item
  Hypothesis test for the proportion \(p\), when \(X \sim \text{Bernoulli}(p)\) and both \(np > 5\) and \(n(1-p) > 5\)
\item
  Hypothesis test for the variance \(\sigma^2\), when \(X \sim N(\mu, \sigma^2)\)
\end{itemize}

There are three \textbf{methods} for testing hypotheses:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Using \textbf{confidence intervals}
\item
  Using a \textbf{rejection region}
\item
  Using a \textbf{p-value}
\end{enumerate}

All three methods are mathematically equivalent and lead to the same decision.

Finally, there are three \textbf{types} of hypothesis tests:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  \textbf{Two-tailed} test
\item
  \textbf{Upper-tailed} test
\item
  \textbf{Lower-tailed} test
\end{enumerate}

\hypertarget{hypothesis-testing-for-the-mean}{%
\section{Hypothesis testing for the mean}\label{hypothesis-testing-for-the-mean}}

A \textbf{tow tailed} hypothesis contrast is of the form

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\tightlist
\item
  \(H_0:\mu = \mu_0\) (status quo)
\item
  \(H_1:\mu \neq \mu_0\) (research interest)
\end{enumerate}

This is called two tailed because the alternative hypothesis \(H_1\) requires that the mean \(\mu\) of the new experiment is either lower or higher than the mean \(\mu_0\) of the old experiment, or gold standard. This hypothesis can be tested in different cases. For these hypothesis we will assume that \(X\) is a normal variable.

\hypertarget{hypothesis-test-with-a-confidence-interval}{%
\subsection{Hypothesis test with a confidence interval}\label{hypothesis-test-with-a-confidence-interval}}

Remember that the confidence interval at \(95\%\) for the mean is

\[(l,u)=(\bar{x}-t_{0.025, n-1} \frac{s}{\sqrt{n}},\bar{x}+t_{0.025, n-1} \frac{s}{\sqrt{n}})\]

when the outcomes of the random experiment distribute normally \(X\sim N(\mu, \sigma^2)\).

\textbf{Testing Criteria:}

We can have only two options:

\begin{itemize}
\tightlist
\item
  If the confidence interval \textbf{contains} the null hypothesis \(\mu_0\):
\end{itemize}

\[\mu_0\in (l,u)\] then we \textbf{accept} \(H_0\) with \(95\%\) confidence. We do not know where the mean the experiment under the new conditions is but we are confident that it is within the confidence interval. Since \(\mu_0\) is within the interval we cannot discard that \(\mu_0\) is the expected value of our experiment. We then say that the data could have been produced by the old experiment, rendering the new conditions irrelevant.

\begin{itemize}
\tightlist
\item
  If the confidence interval does \textbf{not contain} the null hypothesis\[\mu_0\notin (l,u)\] then we \textbf{reject} \(H_0\) with \(95\%\) confidence. Since we are confident that the mean of the new experiment is within the interval and \(\mu_0\) is not within it, then we can discard that the mean is \(\mu_0\). We then say that the data could not have been produced by the old experiment, rendering the new conditions meaningful.
\end{itemize}

\textbf{Example (Li-ion Batteries)}

Twelve fully discharged Li-ion batteries were stored for three weeks, then they were re-charged and their discharged capacity was measured

1.5681, 1.5578, 1.572, 1.5637, 1.5769, 1.5627, 1.5757, 1.5782, 1.5806, 1.5645, 1.5484, 1.5821

From the histogram of the data we see that we can assume that the discharge capacity distributes normally

\includegraphics{_main_files/figure-latex/unnamed-chunk-153-1.pdf}

Therefore, we can compute the confidence interval for the mean \(\mu\) of this sample

\[(l,u)= (1.562, 1.575)\]
The confidence interval tells us that we trust with \(95\%\) confidence that the \(\mu\) is in the interval. We don not know the true value of \(\mu\) but we see that \(\mu_0=1.561\) is not in it.

\[\mu_0\notin (1.562, 1.575)\]

our conclusion is to reject that \(H_0\) could have produced our \textbf{observed interval}. We also say that the data supports the research question that storing discharged batteries prevents their deterioration. More technically, we say that we \textbf{do not reject} \(H_0\).

\includegraphics{_main_files/figure-latex/unnamed-chunk-154-1.pdf}

\hypertarget{hypothesis-test-with-acceptancerejection-zones}{%
\subsection{Hypothesis test with acceptance/rejection zones}\label{hypothesis-test-with-acceptancerejection-zones}}

An equivalent way to test the hypothesis is to see if our set of observations are either common or rare \textbf{if} we assume that the \textbf{null hypothesis is true}. Let us remember the hypothesis contrast

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\tightlist
\item
  \(H_0:\mu = \mu_0\) (status quo)
\item
  \(H_1:\mu \neq \mu_0\) (research interest)
\end{enumerate}

To test the hypothesis with a \textbf{rejection zone} we compute the standardized statistic

\[T=\frac{\bar{X}-\mu_0}{\frac{S}{\sqrt{n}}}\]
when the null hypothesis is true. This is the standardized error that we will make if we estimate \(\mu_0\) with the average \(\bar{X}\) of the experiment under the new conditions.

Note that we are standardizing with respect to \(\mu_0\) (the null hypothesis). Remember that the interval

\[(-t_{0.025, n-1}, t_{0.025, n-1})\]
defines the most common values of \(T\) since they capture \(95\%\) of the probability ouf the outcomes of T

\[P(-t_{0.025, n-1} \leq T \leq t_{0.025, n-1})=0.95\]

\textbf{Testing criteria:}

\begin{itemize}
\tightlist
\item
  If the observed statistics \(t_{obs}\) under the null hypothesis \textbf{is} in the acceptance region
\end{itemize}

\[t_{obs}=\frac{\bar{x}-\mu_0}{\frac{s}{\sqrt{n}}} \in (-t_{0.025, n-1}, t_{0.025, n-1})\]
we \textbf{accept} \(H_0\) with \(95\%\) confidence. In this case, the observed error that we make when we estimate \(\mu_0\) by \(\bar{x}\) is a typical sampling error.

The interval \((-t_{0.025, n-1}, t_{0.025, n-1})\) is called \textbf{acceptance interval} of \(H_0\) at \(95\%\) confidence level.

\begin{itemize}
\tightlist
\item
  If the observed statistics \(t_{obs}\) under the null hypothesis \textbf{is not} in the acceptance region
\end{itemize}

\[t_{obs} \notin (-t_{0.025, n-1}, t_{0.025, n-1})\] then we \textbf{reject} \(H_0\) with \(95\%\) confidence. In this case, the observed error that we make when we estimate \(\mu_0\) by \(\bar{x}\) is too large for sampling error, and therefore the assumption that the null hypothesis was true does not hold.

The region \((-t_{0.025, n-1}] \cup[t_{0.025, n-1})\) is called the \textbf{rejection zone}.

\includegraphics{_main_files/figure-latex/unnamed-chunk-155-1.pdf}

\textbf{Example (Li-ion Batteries)}

To test whether storing discharged batteries is better than storing them fully charged, we calculate the \textbf{the standardized error} that we make when we estimate \(\mu_0\) with \(\bar{X}\)

\[t_{obs}=\frac{\bar{x}-\mu_0}{\frac{s}{\sqrt{n}}}=2.7921 \notin (-t_{0.025, n-1}=-2.20, t_{0.025, n-1}=2.20)\]
Think of this regions as a region of tolerance for the error. Because our observation is not within, we have enough evidence to distrust the that the changing the storage conditions is irrelevant. We conclude that our observed average is not a typical observation of \(\bar{x}\) when the null hypothesis \(\mu_0\) is true. Therefore, we again reject that the data is consistent \(H_0\) and say that we have evidence that support the notion that storing discharged batteries is better.

\hypertarget{hypothesis-test-with-a-p-value}{%
\subsection{Hypothesis test with a P-value}\label{hypothesis-test-with-a-p-value}}

We can also contrast the \textbf{two tail} hypothesis by calculating the probability that the average of another sample from the null hypothesis will be even rarer than the average we just observed. We define the \(pvalue\) as

\[pvalue = P(T \leq -t_{obs}) + P(t_{obs} \geq T) = 2 (1-F(|t_{obs}|))\]

That is the probability that if we were to we take another sample of the same size from the null hypothesis, we are able to obtain at least the error that we observed. If our observed error is rare for the null hypothesis then this probability value will be small.

\textbf{Testing criteria:}

\begin{itemize}
\tightlist
\item
  If the observed \(pvalue\) is
\end{itemize}

\[pvalue \geq \alpha =1-0.95=0.05\]

then we \textbf{accept} the null hypothesis \(H_0\) with \(95\%\) confidence.

\begin{itemize}
\tightlist
\item
  If the observed \(pvalue\) is
\end{itemize}

\[pvalue < \alpha =1-0.95=0.05\]

then we \textbf{reject} \(H_0\) and accept our research question with \(5\%\) significance level.

\(\alpha\) is the significance level. It gives us how much of the distribution we are leaving out, and defines the region that we consider estimation errors to be beyond the tolerance of the null hypothesis.

Remember: We always trust our data. If the null hypothesis says that our data is a \textbf{rare} observation we then distrust the null hypothesis, and reject it.

\textbf{Example (Li-ion Batteries)}

Let us think that the average \(\bar{x}=1.569232\) could be achieved by batteries that were stored fully charged. Perhaps, we were lucky and select a sample made of good batteries with high discharge capacity, no matter the storage conditions. If that is so then how lucky were we? For our data the observed statistic was \(t_{obs}=2.7921\) and its \textbf{p-value} is

\[pvalue=2\times (1-F(2.7921))=0.017\]

\begin{verbatim}
Python: 
2*(1- t.sf(2.7921, df=11))

R: 
2*(1- pt(2.7921, 11))
\end{verbatim}

\includegraphics{_main_files/figure-latex/unnamed-chunk-156-1.pdf}

We conclude that if we test \(12\) batteries and store them fully charged, it is unlikely to obtain an average discharge capacity this far from the \(1.561mAh\). Only about \(1\%\) of fully charged battery samples will get this far. Since this probability is lower than our tolerance, or significance level \(alpha=0.05\), again conclude that storing discharged batteries is a better option.

The entire analysis in Python and R can be done with the following code

\begin{verbatim}
Python:
from scipy import stats
data = [1.5681, 1.5578, 1.572, 1.5637, 1.5769,
        1.5627, 1.5757, 1.5782, 1.5806, 1.5645,
        1.5484, 1.5821]
stats.ttest_1samp(data, popmean=1.561)
R:
x <- c(1.5681, 1.5578, 1.572, 1.5637, 1.5769,
        1.5627, 1.5757, 1.5782, 1.5806, 1.5645,
        1.5484, 1.5821)
t.test(observaciones, mu=1.561)
\end{verbatim}

\hypertarget{upper-tail-hypothesis}{%
\subsection{Upper tail hypothesis}\label{upper-tail-hypothesis}}

We may be interested in only testing for the fact that our experiment's mean has a higher mean than the null hypothesis' mean.

The upper-tailed test is

\begin{itemize}
\tightlist
\item
  \(H_0:\mu \leq \mu\). The new conditions of the experiment have a mean that is \textbf{at most} the null hypothesis mean.
\item
  \(H_1:\mu > \mu\). The new conditions of the experiment have a mean that is \textbf{higher} the null hypothesis mean.
\end{itemize}

This is called \textbf{upper-tailed} test because the alternative hypothesis \(H_1\) requires that the mean \(\mu\) is \textbf{higher} than \(\mu_0\). The testing criteria are the same as those for the \textbf{two tail} hypothesis.

\textbf{Testing criteria:}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  \emph{Confidence interval:} If the \textbf{upper-tailed} confidence interval \textbf{contains} the null hypothesis
\end{enumerate}

\[\mu_0\in (l,u)=(\bar{x}-t_{0.05} \frac{s}{\sqrt{n}}, \infty)\]
where \(t_{0.05, n-1}=F^{-1}(0.95)=\)t.ppf(1-0.05, df=n-1), then we \textbf{accept} \(H_0\) with \(95\%\) confidence. Otherwise we reject it.

Note that this test is from the point of view of the \textbf{data}, we are not centering the confidence interval around \(\bar{x}\), instead we are leaving all the \(5\%\) of the rare errors to the left of the average. We are, therefore, asking if \(\mu_0\) is significant lower than the average.

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{1}
\tightlist
\item
  \emph{Rejection/acceptance region:} If the observed statistics \(t_{obs}\) under the null hypothesis \textbf{is} in the acceptance region
\end{enumerate}

\[t_{obs}=\frac{\bar{x}-\mu_0}{\frac{s}{\sqrt{n}}} \in (-\infty, t_{0.05, n-1})\]

then we \textbf{accept} \(H_0\) with \(95\%\) confidence. Otherwise, we reject it.

Note that this test is from the point of view of the \textbf{null hypothesis}. We are leaving all the \(5\%\) of the rare errors to the right of the null hypothesis and therefore ask if the average is significantly higher than \(\mu_0\).

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{2}
\tightlist
\item
  \(pvalue\): If the observed \textbf{upper-tailed} \[pvalue= 1-F(t_{obs})\]
\end{enumerate}

1-t.cdf(zobs, df=n-1) is greater than \(\alpha=1-0.95=0.05\)

\[pvalue \geq \alpha =0.05\]

then we \textbf{accept} \(H_0\) with \(95\%\) confidence. Otherwise we reject it. Note that this test is again from the point of view of the \textbf{null hypothesis}. We are asking: if we were to take another average from sampling the experiments of the null hypothesis, what is the probability that it will be higher than the observed one?

\textbf{Example (Li-ion Batteries)}

In the example of the Li-ion batteries, we may be interested to test whether storage discharged batteries improves discharge capacity. Therefore, the upper-tailed hypothesis is

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\tightlist
\item
  \(H_0:\mu \leq 1.561\) (storing discharged batteries is \textbf{at most} as good as storing them fully charged)
\item
  \(H_1:\mu > 1.561\) (storing discharged batteries is \textbf{better} then storing them fully charged: \textbf{research interest})
\end{enumerate}

We will test the higher tail of the distribution.
For the data that we discussed before, we then \textbf{reject} \(H_0\) at \(95\%\) confidence because of any of the three equivalent contrasts:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  The \textbf{upper tailed} confidence interval does not contain the null hypothesis \(\mu_0=1.561\)
\end{enumerate}

\[\mu_0=1.561 \notin (\bar{x}-t_{0.05, n-1} \frac{s}{\sqrt{n}}, \infty)=(1.563937, \infty)\]
where \(t_{0.05, n-1}=\)t.ppf(0.95, 11)= 1.795885

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{1}
\tightlist
\item
  We have that the acceptance region for \(H_0\) is:
\end{enumerate}

\[(-\infty, t_{0.05, n-1})=( -\infty,  1.795885)\]

and that the observed standardized error is not in the region
\[t_{obs} =  \frac{1.569232-1.561}{\frac{0.01}{\sqrt{12}}}=2.7921 \notin ( -\infty,  1.795885)\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{2}
\tightlist
\item
  The upper tail \(pvalue\) is lower than \(\alpha=0.05\)
  \[pvalue=1-F(2.7921)=0.0087 <0.05\]
\end{enumerate}

where \(pvalue=\)t.sf(2.7921, 11).

The hypothesis test is performed in Python and R like

\begin{verbatim}
Python:
from scipy import stats

data = [1.5681, 1.5578, 1.572, 1.5637, 1.5769,
        1.5627, 1.5757, 1.5782, 1.5806, 1.5645,
        1.5484, 1.5821]

stats.ttest_1samp(data, popmean=1.561, alternative='greater')
R:
x <- c(1.5681, 1.5578, 1.572, 1.5637, 1.5769,
        1.5627, 1.5757, 1.5782, 1.5806, 1.5645,
        1.5484, 1.5821)
t.test(observaciones, mu=1.561, alternative="greater")
\end{verbatim}

We then conclude that evidence show that storing battering fully discharge improves discharge capacity.

\includegraphics{_main_files/figure-latex/unnamed-chunk-157-1.pdf}

\hypertarget{paired-t-test}{%
\subsection{Paired t-test}\label{paired-t-test}}

\textbf{Example (Gosset's Soporific)}

In some cases, we are not sure about the numerical value of a parameter under the null hypothesis, but we know that we want to improve the value of a parameter in two different conditions.

In the famous 1908 paper of Gosset, titled ``The probable error of the mean'', were he proposed the \(T\) statistics \citep{student1908probable}, he analyzed the soporific effect of the hyoscyamine hydrobromide. It was already known at the time that the change of the structural configuration from dextro (deviating polarized light to the right) to leaevo (deviating polarized light to the left) of the molecule would hchange its biological effect. To test the differences\\
- Ten individuals were given the \textbf{dextro} version of the molecule and wrote down the additional hours slept under this treatment

0.7, -1.6, -0.2, -1.2, -0.1, 3.4, 3.7, 0.8, 0, 2

with average \(0.75\)

\begin{itemize}
\tightlist
\item
  The same 10 individuals were given the \textbf{laevo} version of the molecule and wrote down the additional hours slept under this treatment
  1.9, 0.8, 1.1, 0.1, -0.1, 4.4, 5.5, 1.6, 4.6, 3.4
\end{itemize}

with average \(2.33\)

The scientific hypothesis was that the laevo version of the molecule was better than the dextro one. For each individual, Gosset computed the difference between the treatments. Taking \(X\) as the \textbf{difference} between treatments, this was the sample observed for \(X\)

1.2, 2.4, 1.3, 1.3, 0, 1, 1.8, 0.8, 4.6, 1.4

The average hours gained by laevo with respect to dextro was \(1.58\), and its standard deviation \(s=1.229995\).

The scientific question can be stated as \textbf{upper-tailed} paired t-test:

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\tightlist
\item
  \(H_0:\mu \leq 0\) (no treatment difference: \(\mu_2-\mu_1=0\))
\item
  \(H_1:\mu > 0\) (treatment 2-laeveo have higher gains in sleeping hours than treatment 1-dextro: \(\mu_2-\mu_1>0\))
\end{enumerate}

Where \(\mu\) is the mean of the \textbf{differences} between treatments, and the null hypothesis states that there is no difference.

If we suppose that \(X\) is normal, the \textbf{standardized error} that we make when we estimate the null difference \(\mu_0=0\) by the average of the differences is

\[T=\frac{\bar{X}}{\frac{S}{\sqrt{n}}}\]

and its observation

\[t_{obs}=\frac{\bar{x}}{\frac{s}{\sqrt{n}}}\]
which is also known as the \textbf{signal} to \textbf{noise} ratio.

we can test the hypothesis for the difference \(X=treatmet_1-treatment_2\) using a upper tailed test which give us

\[\bar{x}=1.58\] and \[pvalue=0.0014\]
This is obtained from the following code in Python and R

\begin{verbatim}
Python:
from scipy import stats
x = [1.2, 2.4, 1.3, 1.3, 0, 
     1, 1.8, 0.8, 4.6, 1.4]
stats.ttest_1samp(x, popmean=0, alternative='greater')

R:
x <- c(1.2, 2.4, 1.3, 1.3, 0, 
     1, 1.8, 0.8, 4.6, 1.4)
t.test(x, mean=0, alternative="greater") 
\end{verbatim}

Gosset therefore observed a significant increase in \(1.58\) hours of sleep between laevo and dextro-hyoscyamine hydrobromide. We thus reject that the difference between soporific means is \(0\) or that their means are equal. Equivalently, we can test the hypothesis using a paired t-test, where we introduce the observation for each separate condition, and state that the observations are paired.

\begin{verbatim}
Python:
from scipy import stats
medicine1 <- [0.7, -1.6, -0.2, -1.2, -0.1,
              3.4, 3.7, 0.8, 0, 2]
medicine2 <- [1.9, 0.8, 1.1, 0.1, -0.1, 
              4.4, 5.5, 1.6, 4.6, 3.4]

stats.ttest_rel(medicine2, medicine1, alternative='greater')

R:
medicine1 <- c(0.7,-1.6,-0.2,-1.2,-0.1,3.4,3.7,0.8,0,2)
medicine2 <- c(1.9,0.8,1.1,0.1,-0.1,4.4,5.5,1.6,4.6,3.4)

t.test(medicine2, medicine1, alternative="greater", paired = TRUE) 
\end{verbatim}

The experiment not only confirmed the biological consequences of isomers but it was the first ever \textbf{cross-over design} of a clinical trial. Gosset thus designed and used a statistical tool with key consequences in the analysis and design of pharmacological experiments, among others.

In the following plot, we show all the statistical elements for this example. In red, we show the null hypothesis of no difference between treatments. The \(95\%\) confidence interval for the difference is shown in the upper part. The CI does not contain the null. In the second row, we see the data represented in a box plot. At the bottom row, we see the raw data, that is the individual observations for the change in hours for each patient.

\includegraphics{_main_files/figure-latex/unnamed-chunk-160-1.pdf}

\hypertarget{lower-tail-hypothesis}{%
\subsection{Lower tail hypothesis}\label{lower-tail-hypothesis}}

\textbf{Example (NaCl)}

When \(11.6g\) of NaCl is dissolved in \(100 g\) of water it has a molar concentration of \(1.92 mol/L\). Imagine we design a process to remove salt from this concentration and obtain the following results

1.716, 1.889, 1.783, 1.849, 1.891

If we are interested only in the case that we are able to remove salt from the concentration then we rather propose a \textbf{lower tail} hypothesis.

The lower tail hypothesis are of the form

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\item
  \(H_0:\mu \geq \mu_0\). The new conditions of the experiment give an expectation that is \textbf{at most} as the initial one -status quo.
\item
  \(H_1:\mu < \mu_0\). The new conditions of the experiment give an expectation that is \textbf{lower than} the initial one -research interest
\end{enumerate}

\textbf{Testing criteria:}

Note that the lower tail is given by the alternative \(H_1\). We want to test that the average concentration after the process is lower than the initial concentration. The contrast criteria are the same as for the other types of hypothesis. For this type of hypothesis, we will accept the null hypothesis if

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  \(\mu_0\) is in the confidence interval:
  \[\mu_0\in (l,u)=(-\infty, \bar{x}+t_{0.05,n-1} \frac{s}{\sqrt{n}})\]
\item
  or, \(t_{obs}\) is in the aceptance region:
\end{enumerate}

\[t_{obs}\in (t_{0.05,n-1}, \infty)\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{2}
\tightlist
\item
  or, the \(pvalue\) on the lower tail of the distribution.\\
  \[pvalue=F_t(t_{obs},n-1)\]
  is higher than \(\alpha=0.05\)
\end{enumerate}

In any other case, we reject \(H_0\) and accept the alternative hypothesis.

\textbf{Example(NaCl)}

If we are efficient at removing salt then we would like to provide evidence for the alternative lower tail hypothesis.

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\tightlist
\item
  \(H_0:\mu \geq 1.92\) (null)
\item
  \(H_1:\mu < 1.92\) (alternative)
\end{enumerate}

We can assume that the concentration is normal. Therefore, we only need to change the ``alternative'' argument to ``less'' in the t-test functions

\begin{verbatim}
Python:
from scipy import stats
data = [1.716, 1.889, 1.783, 1.849, 1.891]

stats.ttest_1samp(data, popmean=1.92, alternative='less')

R:
x <- c(1.716, 1.889, 1.783, 1.849, 1.891)
t.test(x, mu=1.92, alternative="less")
\end{verbatim}

We see that the expected value of the null hypothesis \(\mu_0=1.92\) is not within the \(95\%\) lower-tail confidence interval \((-\infty, 1.897376)\), and that the \(pvalue=0.002\) is lower than the significance level \(\alpha=0.05\). Therefore, we can reject the null hypothesis and say that our eveidence support the alternative, meaning that the prototipe is able to exctract salt from the mix.

If we compare result with that of the two tail test, we will see that the \(pvalue\) is reduced in half, and therefore we have more confidence in rejecting the lower tail hypothesis than the two-sided hypothesis.

\textbf{Hypothesis Testing with Large \(n\) and Any Distribution}

On many occasions, \(X\) is not normally distributed, but if we can take large samples (\(n \ge 30\)), then we can use the \textbf{Central Limit Theorem (CLT)}. The CLT tells us that the sampling distribution of the sample mean \(\bar{X}\) is approximately normal, regardless of the distribution of \(X\), provided \(X\) has finite variance.

The \textbf{standardized error} from the null hypothesis can then be approximated by a standard normal distribution:

\[
Z = \frac{\bar{X} - \mu_0}{\frac{\sigma}{\sqrt{n}}} \approx N(0, 1)
\]

If the population standard deviation \(\sigma\) is unknown, we replace it with the sample standard deviation \(S\), and use the \textbf{t-statistic}:

\[
T = \frac{\bar{X} - \mu_0}{\frac{S}{\sqrt{n}}} \sim t_{n-1}
\]

For large \(n\), the t-distribution approaches the standard normal distribution, so we may approximate:

\[
T \approx N(0, 1) \quad \text{for large } n
\]

\textbf{Note:} The use of the normal approximation for the \(T\)-statistic is common when \(n \ge 30\), but strictly speaking, the statistic follows a \(t\)-distribution with \(n-1\) degrees of freedom.

\hypertarget{hypothesis-testing-for-the-proportion}{%
\section{Hypothesis testing for the proportion}\label{hypothesis-testing-for-the-proportion}}

If our random experiment is a Bernoulli trial \(X \sim Bernoulli(p)\), we can formulate hypothesis contrasts for the probability \(p\) of an event in the trial, or the proportion. Consider an upper tailed hypothesis

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\tightlist
\item
  \(H_0: p \leq p_0\) (status quo)
\item
  \(H_1: p> p_0\) (research interest)
\end{enumerate}

In this case, we test a hypothesis for the proportion if

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  \(X\) is a Bernoulli trial, and
\item
  \(np\), \(n(1-p)\) are both greater than \(5\), so we can apply the central limit theorem.
\end{enumerate}

Remember that if we take a the sample of \(n\) Bernoulli trials \((1,0,1,...0)\), \[\bar{X}=\frac{1}{n}\sum_{i=1}^n X_i\] is the relative frequency for the number of `'ones'' obtained in the sample. This is an estimator of \(p\).

If we assume that the null hypothesis is true then \(X \sim Bernoulli(p_0)\) and the standardized error that we make when we estimate \(p_0\) with \(\bar{X}\) is

\[Z=\frac{\bar{X}-p_0}{\frac{\sqrt{p_0(1-p_0)}}{\sqrt{n}}}  \sim N(0,1)\]

\(\sigma=\sqrt{p_0(1-p_0)}\) is the standard deviation of \(X\) when the null hypothesis is true. Remeber that \(V(X)=\sigma^2=p_0(1-p_0)\). With this \(Z\) statistic, we can accept or reject the null hypothesis using any of the three testing criteria.

\textbf{Example (Process Improvement)}

We may be satisfied with a new process if \(90\%\) of the times we improve the previous process. If we run a sample of \(200\) new processes and find that \(188\) times we improved the old process. This makes a relative frequency of \(0.94\) or we managed to improve the process \(94%
\) of the times. Can we be satisfied with the new process at \(95\%\) confidence?

To answer this questions, we can formulate an upper-tailed hypothesis contrast for the null hypothesis \(p_0=0.9\). Therefore, the null and alternative hypotheses are

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\tightlist
\item
  \(H_0: p \leq p_0=0.9\) (The new process is not satisfactory to the requires reliability: status quo)
\item
  \(H_1: p> p_0=0.9\) (The new process improves the older one reliably: research interest)
\end{enumerate}

We assume that if the null hypothesis is true then

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  The probability model of a random experiment is
\end{enumerate}

\[X \sim Bernoulli (p_0)\]
2. and check that when \(np_0=180>5\) and \(n(1-p_0)=20>5\)

Therefore, we can apply the test for the proportions.

We can use any of the three criteria to test the hypothesis. For this example, we believe that have a satisfactory process because, we \textbf{reject} \(H_0\) at \(95\%\) following

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  The upper tail confidence interval for the \(p\) does not include \(p_0\)
\end{enumerate}

\(p_0=0.9 \notin (\bar{x}-z_{0.05}\big[\frac{p_0(1-p_0)}{n} \big]^{1/2},1)= (0.905,1)\)

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{1}
\tightlist
\item
  The observed standardized error from the null is not in the acceptance region
\end{enumerate}

\[z_{obs}= \frac{\bar{X}-p_0}{\big[\frac{p_0(1-p_0)}{n} \big]^{1/2}} =\frac{0.94-0.90}{\sqrt{0.00045}}=1.88563 \notin (-\infty, z_{0.05})=(-\infty, 1.644)\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{2}
\tightlist
\item
  The upper tail \(pvalue\) is lower than \(\alpha=0.05\):
\end{enumerate}

\[pvalue=1-\Phi(1.885618)=0.02967323<0.05\]

\begin{verbatim}
Python: 
1-norm.cdf(1.885618)
R:
1-pnorm(1.885618)
\end{verbatim}

\includegraphics{_main_files/figure-latex/unnamed-chunk-162-1.pdf}

The test can be performed in Python and R using the code

\begin{verbatim}
Python:
from statsmodels.stats.proportion import proportions_ztest
proportions_ztest(188, 200, value=0.9, alternative='larger')

R:
prop.test(188, 200, p=0.9, alternative="greater", correct = FALSE)
\end{verbatim}

\hypertarget{hypothesis-testing-for-the-variance}{%
\section{Hypothesis Testing for the Variance}\label{hypothesis-testing-for-the-variance}}

In many cases, experiments are run to test the dispersion of data. Such as, when complying with strict design standards where measurements must be between certain values. Or when different treatments are applied to different groups, we want to see the dispersion of outcomes between the groups. Also the dispersion of the data may be a property that offer important insights into the system under study.

A \textbf{tow tailed} hypothesis contrast for the variance is of the form

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\tightlist
\item
  \(H_0:\sigma = \sigma_0\) (status quo)
\item
  \(H_1:\sigma \neq \sigma_0\) (research interest)
\end{enumerate}

This hypothesis for \(\sigma\), or equivalently for \(\sigma^2\), can be tested when the random variable \(X\) is a normally distributed.

Remember that if we take a random sample \[S^2=\frac{1}{n-1}\sum_{i=1}^n (X_i-\bar{X})^2\] is the sample variance. This is an unbiased and consistent estimator of \(\sigma^2\).

If we assume that the null hypothesis is true and \(X \sim N(\mu, \sigma_0)\) then the \textbf{error ratio} that we make when we estimate \(\sigma^2\) with \(s^2\) is

\[W=\frac{(n-1)S^2}{\sigma_0^2}\]

Note that when \(W=n-1\), we make no error. \(W\) follows a \(\chi^2\) (chi-squared) distribution with \(n-1\) degrees of freedom.

\[W \sim \chi^2(n-1)\]

With the observation of \(W\), we can accept or reject the null hypothesis using any of the three testing criteria.

\textbf{Example (Hart Rate Variability)}

The heart rate, as measured by an electrocardiogram (ECG), shows a distinct periodic pattern, with the peak signal corresponding to the R-wave of the QRS complex. However, the periodicity of the hart rate is not exact. In fact, heart rates that get close to perfect periodicity is bad news. The lack of variability in the intervals between successive R-peaks---known as RR intervals---is indicative of cardiac dysfunction and is a known predictor of mortality. Conversely, high variability in the heart rate is associated with better cardiovascular health. The fractal and nonlinear complexity of this variability is the focus of ongoing research in cardiology, as it reflects the dynamic interplay of autonomic regulation and physiological adaptability.

Irurzun and colleagues measured RR intervals by 24 hour Holter monitoring on \textbf{147 healthy} patients to determine RR change at different ages and between sexes\citep{irurzun2021rr}. Patient ``00'' was, for instance, a 53 year old male with the following distribution of pre-processed RR values during waking hours in milliseconds.

\includegraphics{_main_files/figure-latex/unnamed-chunk-163-1.pdf}

The sample variance of the patient is \(s^2=8029.751\). The expected RR healthy variability of a 53 year is \(\sigma^2_0=8000ms\), according to the reference Framingham Heart Study. Is patient ``00'' within the range of the expected value of healthy RR variability and a suitable candidate for the study?

We therefore want to contrast the two tailed tail hypotheses

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\tightlist
\item
  \(H_0:\sigma^2 \leq \sigma_0^2=8000\) (Patient has a very normal RR variability)
\item
  \(H_1:\sigma^2 \neq \sigma_0^2=8000\) (Patient has a RR variability different than the expected value)
\end{enumerate}

Let us test the hypothesis using the \textbf{acceptance region}.

The contrast statistics is \[W=\frac{(n-1)S^2}{\sigma_0^2} \sim \chi^2(n-1)\]

and the threshold limit \(\alpha=0.05\). Therefore, the acceptance region \(P(\chi^2_{0.975,2999} \leq W\leq \chi^2_{0.025,29999})=0.95\) is

\[(\chi^2_{0.975,2999}, \chi^2_{0.025,2999})=(2849.11,3152.678)\]

\begin{verbatim}
Python:
chi2.ppf(0.975,19)
chi2.ppf(0.025,19)

R:
qchisq(0.025, 2999)
qchisq(0.975, 2999)
\end{verbatim}

For our data, the observed \textbf{standardized error ratio} is:

\[w_{obs}=\frac{2999 \times8029.751}{8000}=3010.153\]

which falls within the acceptance region

\[w_{obs}=3010.153\in (2849.11,3152.678)\]

and therefore we do not reject the null hypothesis and conclude that RR data of patient ``00'' supports that he has a very normal RR variability.

\includegraphics{_main_files/figure-latex/unnamed-chunk-164-1.pdf}

The \(95\%\) CI interval for his RR variance is

\((l,u) = (\frac{s^2 (n-1)}{\chi^2_{0.025,n-1}},\frac{s^2(n-1)}{\chi^2_{0.975,n-1}})\)
\[( 7902.782, 8159.819)\]

which clearly contains the expected value \(\sigma^2_0=8000\) for his age and sex. This patient, has a RR variance that it is very close to the expected variance measured as a gold reference in a large population sample. The healthy range, contrasted with clinical data, is, however, much larger \((3600, 14400)ms^2\). Obtaining a RR variance within the specified interval provides reassurance that the patient is healthy, supporting the decision to be included in the study.

\hypertarget{errors-in-hypothesis-testing}{%
\section{Errors in hypothesis testing}\label{errors-in-hypothesis-testing}}

The result of an upper tail hypothesis test may be to \textbf{reject} the null hypothesis:

\[H_0: \mu\leq\mu_0\]

when \(H_0\) is actually \textbf{true}. In the case of the batteries, we rejected the null hypothesis that the discharge capacity of stored batteries will not change if they were stored fully charged or discharged. Data made us believe that storing discharged batteries was better and can recommend this type of storage.

We must bear in mind that the decisions are based on the data. It may well be that the observed statistic has fallen, \textbf{by chance}, far from the null hypothesis, in the rejection zone of \(H_0\) even when this hypothesis is true. The statistic is a random variable and one observation can have a large value by chance.

In our experiment, we selected, without knowing, batteries that had a previously high discharge capacity in any condition. Larger than the null \(\mu_0=1.561\). If the selection error is systematic we call it \textbf{selection bias}.

Another possible source of the error could be that the type of batteries that we are testing do not have a discharge capacity of \(\mu_0=1.561\) when fully charged. They have a higher discharged capacity equal to that when they are stored fully discharged. The null hypothesis is badly represented by \(\mu_0\), which we really do not know in our experiment. We need a \textbf{control group} to assess it. A solution is to test the same battery in two different conditions, randomly prescribed, and perform a paired test. In the case that the first test influences the outcome of second test on the same specimen, two random samples of each condition can be taken and its difference compared.

When we perform the hypothesis test, we do not know if \(H_0\) is true. Let us imagine that we found by other means that \(H_0\) is really true. The probability of rejecting the truth (\(H_0\)) is precisely the level of statistical significance \(\alpha\). We call this probability the probability of making a \textbf{type 1} error.
Taking an upper tailed hypothesis test for the mean at significance level \(5\%\), we have that the probability that the \textbf{standardized error} falls in the rejection zone is

\[\alpha = P(Z> t_{0.05, n-1})=0.05\]
\includegraphics{_main_files/figure-latex/unnamed-chunk-166-1.pdf}

A type 1 error is also called a \textbf{false positive} because our research interest is in \(H_1\). When we reject \(H_0\), we accept \(H_1\) and say that our test is \textbf{positive}. Accepting \(H_1\) translates to announcing a discovery, so the type 1 error is announcing a discovery that is not true: we falsely claimed a discovery because the data suggested it, we were fooled by it.

There is another type of error. The result of an upper tail hypothesis test can be \textbf{accepting} the null hypothesis:

\[H_0: \mu\leq\mu_0\]

when this is \textbf{not true}. Again for the batteries, imagine now that our data made us accept that the changing conditions of the batteries do not affect their discharged capacity (\(H_0\)) when actually it does. We chose by chance batteries that had already a lower discharged capacity no matter the condition.

In this case, it may be that the observed random error fell, due to randomness, close to the null hypothesis, in the acceptance zone of \(H_0\), when really \(H_1\) is true. If we found out somehow that, for example, \(\mu\) really does have a value of \(\mu_1\) (blue dot in the plot) then the alternative hypothesis would be exactly:

\[H_1: \mu=\mu_1\]
\includegraphics{_main_files/figure-latex/unnamed-chunk-167-1.pdf}

If \(H_1\) is indeed true (blue line, which we do not know when we perform the hypothesis test) then the statistic is \textbf{really} a random variable \(Y\) that has mean

\[E(Y)=\frac{\mu_1-\mu_0}{\frac{s}{\sqrt{n}}}\]

and most of them will fall close to this value, therefore, in the rejection area of \(H_0\), validating the hypothesis test. However, there are cases in which the observed statistic falls within the acceptance zone of \(H_0\) due to randomness (in the blue shaded area), despite the fact that the statistics are produced by \(H_1\). In these cases we accept \(H_0\) when it is not true. This error is called \textbf{a type 2 error} or a \textbf{false negative}. Since our research interest is in \(H_1\) we failed to accept it. rejecting \(H_1\) translates to discarding a discovery, so the type 2 error is ignoring a discovery that is actually true.

For an upper tailed test for the mean and a significance level \(\alpha\) this probability is

\[\beta= P(Y < t_{0.05, n-1})\]
Where \(Y \sim T(n-1, ncp=\frac{\mu_1-\mu_0}{s/\sqrt{n}})\) is the \textbf{true distribution} of the observed statistics, and corresponds to a t-distributions shifted to the right by the parameter \(ncp\). We also say that the statistical \textbf{power} of the test is \(1-\beta\).

\textbf{Example (Power Calculation of Gossets' Soporifics)}

The results of pre-clinical findings, like those of Gosset, would require extensive validation to be approved for clinical use. A clinical trial, aimed to validate an effect, needs to compute the statistical power expected to reject the null to be approved. Imagine we aim to validate Gossets experiments, replicating his findings. The hypotheses that we want to contrast are

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\tightlist
\item
  \(H_0 : \mu = 0\) (no soporific difference between treatments)
\item
  \(H_1 : \mu > 0\) (higher soporific effect of the laevo isomer than the dextro one)
\end{enumerate}

and prove that the mean difference between treatments is positive on \(100\) patients, with a statistical significance level of \(5\%\). Assuming the values of the previous Gosset's study that suggested that the difference between the means was \(\mu=\mu_1=1.58\) hours and \(\sigma=1.23\) hours, what is the type 2 error we should expect if we repeat Gosset's experiment using a sample size of \(100\) patients?

The probability of accepting the null hypothesis is

\[\alpha = P(T> t_{0.05, 100-1})=0.05\]
where \(t_{0.05, 99}\)t.ppf(1-0.05, 99)\(=1.66\). The type 2 error is therefore

\[\beta= P(Y > 1.66)\]

that is, the probability of accepting that the laevo version of the molecule does not produce higher soporific effects that the dexo one, when in fact it does. For the upper tailed test for the mean, the observed statistics actually distributes as

\[Y \sim T(100-1, ncp=\frac{1.58}{1.23/\sqrt{10}}= 4.06)\] and the type 2 error is

\[\beta = F_{t(n-1, ncp)}(4.06)=0.008\]

\begin{verbatim}
Python:
nct.cdf(1.66, 99, ncp=12.84553)
R:
pt(1.66, 99, ncp=12.84553)
\end{verbatim}

Therefore, if were were to repeat Gosset's experiment using \(100\) patients an infinite amount of times, only \(\alpha=5\%\) of the times we would announce that the laevo molecule is more soporific than the dextro one when it really is not, while \(\beta=0.8\%\) of the times we would announce that we do no see a difference when there is one. In this case, the statistical power to detect a biological effect of changing the molecule's chirality is \(1-\beta=0.991\) or \(99.1\%\).

When designing an experiment is customary to fix the power at \(80\%\) and then compute the sample size \(n\) required to achieve this power. For Gosset's experiment, the effect is so strong that with only \(4\) patients we will achieve a statistical power of \(80\%\).

\hypertarget{sensitivity-and-specificity}{%
\subsection{Sensitivity and Specificity}\label{sensitivity-and-specificity}}

When we carry out a hypothesis test we have two possibilities for each condition:

\begin{itemize}
\tightlist
\item
  \(H_1\) is actually : \textbf{true} (\(\mu=\mu_1\)) or \textbf{false} (\(\mu=\mu_0\)).
\end{itemize}

We also have two outcomes of the hypothesis test:

\begin{itemize}
\tightlist
\item
  The test for \(H_1\) is: \textbf{positive} (\(t_{obs}\) in the acceptance zone of \(H_1\)) or \textbf{negative} (\(t_{obs}\) in the acceptance zone of \(H_0\)).
\end{itemize}

\textbf{Example (PCR)}

We do \textbf{a} PCR to test for an infection. The hypothesis test is

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\tightlist
\item
  \(H_0\) no infection
\item
  \(H_1\) there is infection
\end{enumerate}

We do the PCR test and it gives us

\begin{enumerate}
\def\labelenumi{\roman{enumi}.}
\tightlist
\item
  negative: we reject the infection (\(H_1\))
\item
  positive: we accept the infection (\(H_1\))
\end{enumerate}

We can write the contingency table for the probabilities of the results of the hypothesis test as

\begin{longtable}[]{@{}ccc@{}}
\toprule\noalign{}
& \(H_1\) is true & \(H_1\) is false \\
\midrule\noalign{}
\endhead
\bottomrule\noalign{}
\endlastfoot
\textbf{The test on \(H_1\) is positive} & \(1-\beta\) & \(\alpha\) \\
\textbf{The test on \(H_1\) is negative} & \(\beta\) & \(1-\alpha\) \\
\textbf{sum} & 1 & 1 \\
\end{longtable}

we therefore have

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  The \textbf{type 2 error} rate: probability of a false negative (ignore a finding when it is true)
\end{enumerate}

\[\beta=P(negative|H_1)\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{1}
\item
  The \textbf{True positive} rate: This is the power or sensitivity of a test (claiming a discovery when it is true, the main objective)
  \[1-\beta=P(positive|H_1)\]
\item
  The \textbf{Type 1 error} rate: probability of a false positive (state a discovery when it is false)
  \[\alpha=P(positive|H_0)\]
\item
  The \textbf{True Negative} rate: This is the specificity of a test (ignore a finding when it is false)
  \[1-\alpha=P(negative|H_0)\]
\end{enumerate}

\hypertarget{exercises-12}{%
\section{Exercises}\label{exercises-12}}

\hypertarget{exercise-1-11}{%
\subsubsection{Exercise 1}\label{exercise-1-11}}

Imagine we take a random sample of size \(n = 41\) of a normal random variable \(X\), and find that the sample average is \(10\) and the sample variance is \(1.5\).

\begin{itemize}
\tightlist
\item
  What is then the confidence interval for the mean of \(X\) at \(95\%\) confidence level?
\end{itemize}

Consider that \(t_{0.025,40}=\) t.ppf(0.975, 40) \(\sim 2\).

\begin{itemize}
\item
  Test the hypothesis that the mean of \(X\) is \textbf{different} than \(10.5\), using a \(5\%\) significance threshold.
\item
  Write the code to calculate the P-value to test the hypothesis that the mean of \(\mu\) is \textbf{lower} than \(10.5\), using a \(5\%\) significance threshold.
\end{itemize}

Consider that the code for the T probability distribution with \(n-1\) degrees of freedom is t.cdf(tobs, n-1).

\hypertarget{exercise-2-11}{%
\subsubsection{Exercise 2}\label{exercise-2-11}}

\(10\) gas condensates showed the following concentrations of mercury (in \(ng/ml\)):

\(23.3\), \(22.5\), \(21.9\), \(21.5\), \(19.9\), \(21.3\), \(21.7\), \(23.8\), \(22.6\), \(24.7\)

Assuming that the mercury concentration is distributed normally across gas condensates, test the hypothesis that condensate does not surpass the toxicity limit established at \(24 ng/ml\).

\hypertarget{exercise-3-9}{%
\subsubsection{Exercise 3}\label{exercise-3-9}}

The manufacturer of gene expression microarrays guarantees that at least \(97\%\) of the microarrays they produce have high-quality signals. A customer receives a batch of \(200\) pieces and finds that \(8\) unperformed.

Should the customer return the lot due to poor quality?

\hypertarget{practice-4}{%
\section{Practice}\label{practice-4}}

Load misophonia data \url{https://alejandro-isglobal.github.io/SDA/data/data_0.txt}

We have four measures of anxiety:

\begin{itemize}
\tightlist
\item
  Trait: ansiedad.rasgo (are you an anxious person?) continuous:0-100
\item
  State: ansiedad.estado (are you currently feeling anxious?) continuous:0-100
\item
  Diagnosed: ansiedad.medicada (have you been diagnosed with an anxiety disorder?) binary (si, no)
\item
  Excess: ansiedad.dif (difference between State and Trait)
\end{itemize}

We are interested in the variable misofonia.dif, that is the observed \textbf{excess} of anxiety from the trait

\(excess = state - trait\)

Test the following hypotheses

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\item
  Is excess in anxiety different from 0? is it higher?
\item
  Is excess in anxiety higher than 0 for men and women separately?
\item
  Is the proportion of anxious patients different from \(0.03\)?
\end{enumerate}

\href{https://colab.research.google.com/drive/13El5aoycT_6Wasvyx427TvglygiSPMpF?usp=sharing}{Solutions}

\hypertarget{contingency-tables}{%
\chapter{Contingency tables}\label{contingency-tables}}

Our aim is often to compare the estimate of a parameter---describing a property of a random experiment---to a reference value. This reference value defines the probability distribution of the experiment under previously known or established conditions. If we change the conditions of the experiment to improve that property, our goal is to demonstrate that the new estimate is significantly different from the reference. On the other hand, if we aim to validate the reference by repeating the experiment under the same conditions, our goal is to show that the estimate remains close to the reference value.

In either case, the reference must be known without uncertainty prior to the experiment. Reference values can sometimes be derived unambiguously from theoretical considerations. For example, Mendel's 0.75 probability that the offspring of hybrid pea plants produce round seeds is supported by a theoretical genetic model. When no such model exists---for instance, in the case of the mass-to-charge ratio of the electron---then a reference may be based on a gold standard established by highly accurate previous measurements.

But what can we do when no such reference value exists for the null hypothesis? Suppose we only know the two different conditions we want to compare. Gosset's experiment on testing the soporific effects of two structural configurations of the same molecule provides a solution. He proposed using the same individual to test both variants, and measuring the difference in effect. The reference value then becomes the null difference in soporific effect between the two molecular structures across individuals.

A further complication arises in experiments where each specimen (or experimental unit) can only be tested under a single condition. In such cases, two independent samples must be taken---one per condition. Equivalently, each trial of the random experiment produces two measurements: one of the outcome, and one of the condition. Here, the null hypothesis is formulated as the statistical independence between the outcome and the conditions.

In this chapter, we will examine how to test for statistical independence between a random variable with discrete outcomes and two or more conditions of the experiment. We will use the contingency table of conditional probabilities to define the null hypothesis, and test it using the chi-squared \(\chi^2\) statistic. We will also introduce Fisher's exact test, especially useful when sample sizes are small. We will illustrate both methods using historical data: the link between smoking and lung cancer, and the effect of natural selection on humans through changes in the frequency of the lactase persistence mutation.

Finally, when the goal is to test whether observed relative frequencies match those expected from a theoretical model, the \(\chi^2\) test can also be applied. We will illustrate this important use in the linear dependence of the lactase mutation with the latitude of European human populations.

\hypertarget{difference-between-proportions}{%
\section{Difference between proportions}\label{difference-between-proportions}}

\textbf{Example (Smoking and Lung Cancer)}

In 1950 Doll and Hill established smoking as a potential cause of lung cancer \citep{doll1950smoking}. Before them, smoking was not considered as a potential cause of the increasing incidence in lung cancer. The study became a template in epidemiological studies and help to change the public view on the use of tobacco. The control measures that came decades later, based of scientific evidence encourage by this work, helped to save millions of lives.

Doll and Hill took a sample of lung cancer patients and controls and recorded whether they were smokers or not. We can imagine that the random sample of the smoking status (yes:1, no:0) for the \textbf{lung cancer} patients

1, 0, 0, 1, 0, \ldots{} 0

where the first patient was a smoker and and the last one was not.

Smoking status is a Bernoulli trial \(Y\) with outcomes \((0, 1)\) and has a probability mass function

\[Y_A \sim Bernoulli (p_A)\]
The parameter \(p_A\) is the probability of smoking under the lung cancer condition \(A\).

We also write down the smoking status of a patient who is a \textbf{control} (non-smoker), and observe

0, 0, 1, 0, 0, \ldots{} 1

Smoking status is a Bernoulli trial with probability \(p_B\)

\[Y_B \sim Bernoulli (p_B)\]
The parameter \(p_B\) is the probability of smoking under the control condition \(B\).

\hypertarget{difference-between-proportions-1}{%
\section{Difference between proportions}\label{difference-between-proportions-1}}

In this study \textbf{one} random experiment has a two-value outcome: (smoking status, diagnosis). That is the observation of a patient is determined by two variables \textbf{both of which are categorical}

These are the possible outcomes of each variable:

\begin{itemize}
\tightlist
\item
  Smoking \(\in\{yes, no\}\)
\item
  Cancer diagnosis \(\in \{A:cancer,B:control\}\)
\end{itemize}

Repeating the experiment \(n\) times, the data matrix for the patients would look like

\[
\begin{array}{ccc}
\mathbf{Individual} & \mathbf{Condition} & \mathbf{Outcome} \\ 
c_1 & \text{A} & \text{1} \\ 
c_2 & \text{A} & \text{0} \\
c_3 & \text{A} & \text{1} \\
\vdots & \vdots & \vdots \\ 
c_{n-2} & \text{B} & \text{1} \\ 
c_{n-1} & \text{B} & \text{1} \\ 
c_{n}& \text{B} & \text{1} \\ 
\end{array}
\]

The first patient was diagnosed for lung cancer and was a smoker.

\textbf{Question:}

Our research question is to find whether the outcome and the condition are \textbf{statistically dependent} variables. We want to know if the probability of smoking is higher in lung cancer patients than in controls.

\hypertarget{contingency-table-of-conditional-probabilities}{%
\section{Contingency table of conditional probabilities}\label{contingency-table-of-conditional-probabilities}}

Our experiment is such that we make diagnosis \textbf{conditioned} to the exposure. We then have the conditional probability table

\[
\begin{array}{cc|c}
    & \mathbf{A:cancer} & \mathbf{B:control} \\ 
    \mathbf{smoking: yes} & P(yes \mid A) & P(yes \mid B) \\ 
    \mathbf{smoking: no} & P(no \mid A) & P(no \mid B) \\ 
    \mathbf{sum} & 1 & 1
\end{array}
\]

We want to know if the probability of smoking in condition \(A\) is different from the probability of smoking in condition \(B\), and therefore the hypothesis contrast is

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\tightlist
\item
  \(H_0: p_A=p_B\), no differences of smoking between cases and controls.
\item
  \(H_1: p_A\neq p_B\), the probabilities of smoking are different between cases and controls.
\end{enumerate}

\hypertarget{test-for-the-difference-between-proportions}{%
\section{Test for the difference between proportions}\label{test-for-the-difference-between-proportions}}

If we consider that the \textbf{null hypothesis} is true, then the cancer tag (``A'' or ``B'') is statistically independent from whether an individual in the study was a smoker or not. Remember the definition of statistical independence: if smoking is independent from diagnosis status
\[p=P(yes)=P(yes|A)=P(yes|B)\]
Therefore the hypothesis contrast can be written as:

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\tightlist
\item
  \(H_0: p=p_A=p_B\)
\item
  \(H_1: p\neq p_A \neq p_B\)
\end{enumerate}

We do \textbf{not know} the values of \(p\), \(p_A\) and \(p_B\). Therefore we need to \textbf{estimate} them from the data.

\textbf{Example (Smoking and Lung Cancer)}

Doll and Hill reported the following observations for females in their study:

\begin{itemize}
\item
  They included \(n_A=60\) cancer patients and observed \(41\) smokers.
\item
  They included \(n_B=60\) controls and observed \(28\) smokers.
\end{itemize}

For long cancer patients (\(A\)), we have that the estimated probability of smoking is the \textbf{relative frequency} of smokers, that is

\begin{itemize}
\tightlist
\item
  \(\bar{y}_A=\hat{p}_A=41/60=0.68\)
\end{itemize}

Likewise, for controls (\(B\)), we have that the estimated probability of smoking is

\begin{itemize}
\tightlist
\item
  \(\bar{y}_B=\hat{p}_B=28/60=0.47\)
\end{itemize}

If the null hypothesis is true then the overall probability of smoking \(p\) can be estimated from either condition, or from both of them \textbf{taken together}, as if they all had the same diagnosis:

\[\hat{p}=\frac{n_A\hat{p}_A+n_B\hat{p}_B}{n_A+n_B}=\frac{41+28}{60+60}=0.575\]
And \(\hat{p}\) is called the pooled proportion, an estimate of the marginal probability of smoking \(p\).

\hypertarget{chi2-test}{%
\section{\texorpdfstring{\(\chi^2\) test}{\textbackslash chi\^{}2 test}}\label{chi2-test}}

The \textbf{standardized error} that we make when we estimate the pooled probability \(\hat{p}\) with the relative frequency of smokers using the lung cancer patients only is

\[Z_A= \frac{\bar{Y}_A-\hat{p}}{\sqrt\frac{\hat{p}(1-\hat{p})}{n_A}}\]

Remember that one observation of \(\bar{Y}_A\) is what we call \(\hat{p}_A\), an estimate of the conditional probability of smoking if receiving a cancer diagnosis. Likewise for condition \(B\), the error made in using controls only (\(B\)) to estimate the pooled proportion is

\[Z_B= \frac{\bar{Y}_B-\hat{p}}{\sqrt\frac{\hat{p}(1-\hat{p})}{n_B}}\]

\includegraphics{_main_files/figure-latex/unnamed-chunk-169-1.pdf}

In the plot, each cross is a smoker (either \(1\) or \(0\)) the dotted line is the estimated probability \(\hat{p}\) of all subjects taken together (the pooled proportion) and the dots are the estimated probabilities of smoking in the lung cancer patients (\(A\)) and controls (\(B\)).

The conditional probability table by cancer and control condition is

\[
\begin{array}{cc|c}
    & \mathbf{A:cancer} & \mathbf{B:control} \\ 
    \mathbf{smoking: yes} & 0.68 & 0.47 \\ 
    \mathbf{smoking: no} & 0.32 & 0.53 \\ 
    \mathbf{sum} & 1 & 1
\end{array}
\]

Which we can illustrate in a barplot

\includegraphics{_main_files/figure-latex/unnamed-chunk-170-1.pdf}

The probabilities of smoking within each diagnosis are different but look similar. Is the difference significant? Can we reject the null hypothesis taking into account the variation on the frequencies about the pooled proportion?

Using the CLT, the total \textbf{standardized squared error} of estimating the proportion of smoking using each condition separately is the sum of the errors

\[W= Z_A^2+Z_B^2\sim \chi^2(1)\]

which is a random variable that follows a \(\chi^2\) distribution with 1 degree of freedom. If the null hypothesis is true, we can use any condition to estimate the pool proportion.

\textbf{Using the marginals}

We formulated the null hypothesis from the statistical independence given by conditional probabilities. \textbf{Another alternative} is to formulate it from the product of the marginals.

Therefore, the hypothesis test can \textbf{also} be written as

\begin{itemize}
\tightlist
\item
  \(H_0:P(yes, A)=P(yes)P(A)\). Cancer diagnosis and smoking are statistically \textbf{independent} as the joint probability is the product of the marginals.
\item
  \(H_1:P(yes, A)\neq P(yes)P(A)\). Cancer diagnosis and smoking are statistically \textbf{dependent}.
\end{itemize}

If diagnosis and smoking status are truly independent from each other then one can show that the variable \(W\) above \textbf{can also} be computed as the sum of the squared errors that we make when we estimate the \textbf{joint probabilities} using the \textbf{marginals} (i.e.~\(\hat{P}(yes,A)=\hat{p}\hat{P}(A)\)), which we can if the null hypothesis is true. Therefore, the total \textbf{standardized squared error} of before can be re-written as:

\[W= \sum_{j\in\{yes,no\}} \sum_{i\in\{A,B\}} \frac{(O_{i,j} - E_{i,j})^2}{E_{i,j}}\]
where, for instance,
\(O_{yes, A}=\hat{P}(yes,A)\) is the \textbf{observed} joint frequency of smoking and lung cancer, and \(E_{yes, A}=\hat{P}(yes)\hat{P}(A)\) is the \textbf{expected} frequency derived from the product of the marginals. From the joint probability table this statistics is the sum of the squared differences between the cells in the table and the products of the marginals, and easy to compute.

If the observed value for \(W\) is a large, its probability under the null hypothesis is low, and we then may reject this hypothesis. The value of \(W\) is exactly the same however we compute it.

\textbf{Example (Smoking and Lung Cancer)}

When we compute the observed value of \(W\), using the first form, we find

\(w_{obs}= \frac{(\hat{p}_A-\hat{p})^2}{\frac{\hat{p}(1-\hat{p})}{n_A}} +\frac{(\hat{p}_B-\hat{p})^2}{\frac{\hat{p}(1-\hat{p})}{n_B}}\) \[=\frac{(0.6833-0.575)^2}{{\frac{0.575(1-0.575)}{60}}}+ \frac{(0.4666-0.575)^2}{{\frac{0.575(1-0.575)}{60}}}= 5.76\]

In terms of the expected value of \(W\) (\(E(W)=1\) -degrees of freedom) when we make no mistake) the null hypothesis is rewritten:

\begin{itemize}
\item
  \(H_0:w=1\). Lung cancer and smoking are statistically \textbf{independent} and the error in the estimation of the pooled proportion is low using each condition separately.
\item
  \(H_1:w>1\). Lung cancer and smoking are statistically \textbf{dependent} and the error in the estimation of the pooled proportion is high using each condition separately.
\end{itemize}

We can test whether this value is large under the null hypothesis. If the null is true this statistics is close to zero, otherwise is large. We can then compute the \(pvalue\) given by

\[pvalue=P(W \geq w_{obs}) =0.016\]

\begin{verbatim}
Pyhton:
chi2.sf(5.76, 1) 
R:
pchisq(5.76, 1, lower.tail = FALSE)
\end{verbatim}

This \(pvalue\) is not lower than the significance level \(\alpha=0.05\) and therefore Doll and Hill \textbf{reject} \(H_0\), and concluded that

\begin{itemize}
\item
  the frequencies of smoking are \textbf{different} between cancer patients and controls,
\item
  or, equivalently, that the frequency of smoking statistically \textbf{depends} from patient diagnosis,
\item
  or, equivalently, that the frequency of smoking \textbf{is significantly associated} lung cancer diagnosis.
\end{itemize}

The analysis can be done with the following code, with the contingency table for the absolute frequencies.

\begin{verbatim}
import numpy as np
from scipy.stats import chi2_contingency

observed = np.array([[19, 32], [41, 28]])

chi2, p, dof, expected = chi2_contingency(observed, correction=False)
print(f"Chi-squared statistic: {chi2:.4f}")
print(f"P-value: {p:.4f}")

R:
conttable <- cbind(c(19, 32),c(41,28))
chisq.test(conttable, correct = FALSE)
\end{verbatim}

The analysis presented here was only one of the evidences of the effect of smoking and cancer. Doll and Hill had to argue against the possibility that lung cancer patients started to smoke after diagnosis (reverse causation) and show that the size of the effect depended on the amount of cigarettes smokes (dose response). Hill famously listed 9 criteria to help support a causal relationship underlying a statistical association \citep{hill1965environment}.

\hypertarget{fishers-exact-test}{%
\section{Fisher's exact test}\label{fishers-exact-test}}

Another approach is \textbf{Fisher's exact test}. When the conditions of the CLT break, often when one of the cells in the contingency table is very small, or the relative frequencies are close to 1 or 0, then the \(\chi^2\) test cannot be applied.

\textbf{Example(Smoking and Lung Cancer in Males)}

Doll and Hill also reported the following observations for males in their study:

\begin{itemize}
\item
  They included \(n_A=649\) cancer patients and observed \(647\) smokers. Only \(2\) where non-smokers.
\item
  They included \(n_B=649\) controls and observed \(622\) smokers. \(27\) were non-smokers.
\end{itemize}

The hypothesis contrast is

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\tightlist
\item
  \(H_0: p_A \leq 1269/1298=0.977\)
\end{enumerate}

For cancer patients (condition \(A\)) the probability of smoking is at most the same as in the patients and controls together.

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\setcounter{enumi}{1}
\tightlist
\item
  \(H_1: p_A < 1269/1298=0.977\)
\end{enumerate}

The alternative hypothesis assumes that the probability of smoking in cancer patients \(A\) is higher than for all subjects together, indicating that there is an increase in the number of smokers in the cancer patients. The observed frequency of smokers in lung cancer is \(\hat{p}_A=647/649=0.997\), but is it statistically significant greater than \(0.977\)?

Smoking was widely spread among males in the 50's. To test the statistical association under this conditions where the CLT does not hold then we use Fisher's exact test.

\hypertarget{hypergeometric-distribution}{%
\section{Hypergeometric distribution}\label{hypergeometric-distribution}}

Take a ballot for each of the \(N=1298\) patients and controls into an urn:

There are a total of \(N\) ballots:

\begin{itemize}
\tightlist
\item
  There are \(K=1269\) ballots marked as smokers
\item
  \(N-K=29\) ballots marked as not smokers.
\end{itemize}

Then, if we take a \(n=649\) randomly selected ballots (similar in number to lung cancer patients), we may ask: What is the probability of observing more than \(647\) patients who are smokers?

The probability of obtaining \(Y\) smokers in a sample of \(n\) drawn from a population of \(N\) where \(K\) are smokers is

\(P(Y=y)=P(one\,sample) \times (Number\, of\, ways\, of\, obtaining\, x)\)

\[=\frac{1}{\binom N n}\binom K y \binom {N-K} {n-y}\]
Therefore, \(Y\) follows a hypergeometric distribution
\[Y \sim Hypergeometric(N,K,n)\]

\includegraphics{_main_files/figure-latex/unnamed-chunk-171-1.pdf}

Remember that \(647\) smokers were actually observed. Therefore, we are asking how rare that observation is in the case where diagnosis conditioning plays no role, or that cases and controls come from the same urn. For this probabilistic model, we compute the probability \(P(Y>647)\), or put simply the \(pvalue\).

If the observed value for \(y_{obs}=647\) is a rare high \textbf{observation} for the null hypothesis, we then reject the probabilistic urn model for the null. The upper tail \(pvalue\) for an observation \(647\) is

\[pvalue=P(Y > 647)= 1- F_{hyper(N,K,n)}(647) = 0.00000064\]
In Python and R

\begin{verbatim}
from scipy.stats import hypergeom
1-hypergeom.cdf(18, 600, 64, 200)

R:
1-phyper(646,  1269, 29, 649)
\end{verbatim}

Which is lower than the significance level \(\alpha=0.05\) and therefore we also reject \(H_0\) for males and conclude:

\begin{itemize}
\tightlist
\item
  that the frequency of smoking \textbf{is significantly associated} with cancer diagnosis also in males.
\end{itemize}

The \textbf{odds ratio} gives us the \textbf{strength} of the of the change in the in the smoker proportion between controls and lung cancer:

\[OR=\frac{p_A/(1-p_A)}{p_B/(1-p_B)}=14.04\]

It is a \textbf{magnitude} of the observed statistical association between smoking and lung cancer. We use the \(OR\) to distinguish between the different cases

\begin{itemize}
\tightlist
\item
  When \(p_A\) is equal to \(p_B\) then \(OR=1\) (independence of smoking and lung cancer)
\item
  When \(p_A\) is lower than \(p_B\) then \(OR<1\)
\item
  When \(p_A\) is greater than \(p_B\) then \(OR>1\)
\end{itemize}

We say that for an \(OR=14\) in the case of the smoking, there was \textbf{an increase} of \(14\)-fold on the risk of cancer due to smoking.

\begin{verbatim}
Python:
import numpy as np
from scipy.stats import fisher_exact
observed = np.array([[27, 622], [2, 647]])

odds_ratio, p_value = fisher_exact(observed, alternative="greater")

print(f"Odds ratio: {odds_ratio:.4f}")
print(f"P-value: {p_value:.4f}")

R:
matrix <- cbind(c(27, 622),c(2, 647))
fisher.test(matrix, alternative = "greater")
\end{verbatim}

\hypertarget{difference-between-several-proportions}{%
\section{Difference between several proportions}\label{difference-between-several-proportions}}

We would also like to to know if the outcome of a random experiment may vary across multiple condition.

We then formulate the hypothesis contrast:

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\tightlist
\item
  \(H_0: p=p_A=p_B,...=p_M\)
\end{enumerate}

The null hypothesis assumes that conditions (\(i=\{A,B,... ,M\}\)) and the outcome (\(j=\{1,0\}\)) are all independent and then they have the same probability of the outcome \(j=1\).

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\setcounter{enumi}{1}
\tightlist
\item
  \(H_1:\) at least for one \(i\), \(p\neq p_i\)
\end{enumerate}

The alternative hypothesis assumes that the outcome is statistical independent from \textbf{at least one condition}.

For deciding between hypothesis, we use the the so called \(\chi^2\)-test for the homogeneity between conditions.

\textbf{Example (Selection of the Lactase Mutant)}

En el 2004 Bersaglieri et al \citep{bersaglieri2004genetic} showed that the the human populations have been under the selection of the lactase alelle --13910*T in which a citocine was replaced by a tiamine in the DNA sequence of the lactase gene \emph{LCT}. The change of a nucleotide at the given position gave the carriers tactate tolerance, promoting herding 5000--10000 years ago. Here are the number of chromosomes observed for each population and the allele frequencies (in a transposed contingency table) that they reported on eight European populations only

\[
\small
\begin{array}{c|c|ccc}
\textbf{Population} & \textbf{Chromosomes (n)} & \textbf{lactase (T)} & \textbf{lactase (C)} & \textbf{sum} \\
\hline
\textbf{Sardinian} & 56 & 0.071 & 0.929 & 1 \\
\textbf{French Basque} & 48 & 0.667 & 0.333 & 1 \\
\textbf{Tuscan} & 16 & 0.063 & 0.937 & 1 \\
\textbf{Adygei} & 34 & 0.118 & 0.882 & 1 \\
\textbf{French} & 58 & 0.431 & 0.569 & 1 \\
\textbf{North Italian} & 28 & 0.357 & 0.643 & 1 \\
\textbf{Orcadian} & 32 & 0.688 & 0.312 & 1 \\
\textbf{Swedish Finnish} & 360 & 0.815 & 0.185 & 1 \\
\end{array}
\]

We have that the \textbf{standardized squared error} from the null hypothesis can be written as:

\[W= Z_A^2+Z_B^2+Z_C^2+Z_D^2+Z_E^2+Z_F^2+Z_G^2+Z_H^2\sim \chi^2(8-1)\]

which is a random variable that follows a \(\chi^2\) distribution with \(7=8-1\) degrees of freedom (number of populations \(-1\)).

Each term in \(W\) is the squared \textbf{standardized error} observed when we estimate the pooled proportion of the mutant allele \(\hat{p}\) using each population separately.

The estimated probability of the mutant allele in all populations together is the pooled proportion

\(\hat{p}=\frac{n_A\hat{p}_A+n_B\hat{p}_B+n_C\hat{p}_C+n_D\hat{p}_D+n_E\hat{p}_E+n_F\hat{p}_F+n_G\hat{p}_G+n_H\hat{p}_H}{n_A+n_B+n_C+n_D+n_E+n_F+n_G+n_H}\) \[=0.62\]

and the computation of the observed \(W\) is
\[w_{obs}=204.28\]

We can now test how rare this observation is under the null hypothesis that all the probabilities of the mutant allele across the populations are equal (i.e.~homogeneous). We compute the \(pvalue\)

\[pvalue=P(W \geq w_{obs}) =1-F_{\chi^2(7)}(204.28)= 1.0\times 10^{-40}\]

The observed \(\chi^2\) statistic and the corresponding \(pvalue\) can be obtained in Python and R with the following code:

\begin{verbatim}
import numpy as np
from scipy.stats import chi2_contingency

observed = np.array([[4, 32, 1, 4, 25, 10, 22, 293 ],
[52, 16, 15, 30, 33, 18, 10, 67]])

chi2_stat, p_value, dof, expected = chi2_contingency(observed)

print(f"Chi-squared statistic: {chi2_stat:.4f}")
print(f"P-value: {p_value:.4f}")
print(f"Degrees of freedom: {dof}")

R: 
observed <- rbind(c(4, 32, 1, 4, 25, 10, 22, 293),
c(52, 16, 15, 30, 33, 18, 10, 67))

chisq.test(observed)
\end{verbatim}

where the \(\chi^2\)-test is computed with the number of chromosomes observed for each allele and population. That is the product of the total number of chromosomes in each population and their relative frequencies.

We see that the \(pvalue\) is much lower than the significance level \(\alpha=0.05\) and therefore we \textbf{reject} \(H_0\) and conclude that: The frequency of the T allele of the \emph{LCT} gene \textbf{is significantly associated} the differences in the European populations, providing evidence for an underlying selection of lactate tolerance.

We can illustrate the result a conditional frequency table by European population or with a bar plot.

\includegraphics{_main_files/figure-latex/unnamed-chunk-173-1.pdf}

The \(\chi^2\)-test tests the homogeneity of all the proportions of the mutant allele across the populations in relation to the overall mutant frequency \(\hat{p}=0.62\). Note that the pooled proportion takes into account the number of individuals in each population. As the Swedish are the population that contribute the most to the pooled proportion, it strongly leans toward this population.

In the figure showing the frequency of the lactase mutant across European populations, we have ordered the populations according their South-North latitude. We see that, except for the French Basque, the frequency increases with latitude. This remark supports the hypothesis that lactase persistence has been favored by positive selection in high-latitude regions of Europe, where sunlight is scarce and the synthesis of vitamin D is limited. In these environments, milk consumption provided a valuable source of calcium and vitamin D, both important for maintaining bone health. How can provide further evidence of this observation?

\hypertarget{goodness-of-fit}{%
\section{Goodness of fit}\label{goodness-of-fit}}

A key insight of the \(\chi^2\) test is to realize that the overall proportion offers a model to the entire data. If we split the data into categories (conditions), each of them can be used to estimate the overall proportion \(p\). If the model is a good fit to the data, the estimate from each category will fall close to the proportion, distributing around its value as a normal variable. A good fit means low accumulated error from all the estimates and, therefore, a low value of the observed \(\chi^2\)-statistic, \(w_{obs}\).

Under this perspective, the \(pvalue\) offers a measure of how good the data fits the null hypothesis. If it is large, we say that the data fits the null probability model, if it is small then the null probability model is not a good model for the data.

\textbf{Example (Selection of the Lactase Mutant)}

In the figure showing the frequency of the lactase mutant across European populations, we have ordered the populations according their South-North latitude. We see that, except for the French Basque, the frequency increases with latitude. The constant overall proportion \(\hat{p}=0.62\) does not appear to be a good fit across the populations. The \(\chi^2\)-test gave a very low \(pvalue\) and therefore, we discard that the null hypothesis is a suitable probabilistic model for the entire data.

How can we improve the model?

An option is to plot the bars at the estimated latitudes for each population (\(i\)) and make the proportion \(p=p(x_i)\) depend of the latitude \(x_i\) as a linear function

\[p(x_i)=\alpha+\beta\times x_i\]

instead of being equal for all the populations. We may leave the French basques out because of their high frequency that could be due to a combination of genetic isolation, cultural continuity, and historical reliance on pastoralism.

A popular method to estimate \(\alpha\) and \(\beta\) is to find the values that minimize the squared differences between the observed frequency \(\hat{p}_i\) and its expected value by the latitude model \(\hat{p}(x_i)\)

\[Q(\alpha, \beta)= \sum_{i=1}^{M} (\hat{p}_i -\hat{p}(x_i))^2\]
where \(M\) are the number of conditions (population). The values \(\hat{\alpha}\) and \(\hat{\beta}\) minimize the sum of squares \(Q\).

\includegraphics{_main_files/figure-latex/unnamed-chunk-174-1.pdf}

As such the error of the estimates to the new line are

\[Z_i= \frac{\bar{Y}_i-\hat{p}(x_i)}{\sqrt\frac{\hat{p}(x_i)(1-\hat{p}(x_i))}{n_i}}\]

each evaluated at their own latitude \(\hat{p}(x_i)\). If we sum all the errors we have

\[W= \sum_{i=1}^{M} Z^2 \sim\chi^2(M-2)\]
This statistic distributes as \(\chi^2\) with \(M-2\) degrees of freedom, and we can use it to measure the goodness of fit of the model to the data. The model has two parameters that reduce the degrees of freedom of the statistic. Each term, assumes that the error from the condition \(i\) is a standard normal variable.

\[
\small
\begin{array}{c|c|cc}
\textbf{Population} & \textbf{Chromosomes (n)} & \textbf{Observed: } \hat{p}_i & \textbf{Expected: } \hat{p}(x_i) \\
\hline
\textbf{Sardinian} & 56 & 0.071 & 0.053\\
\textbf{Tuscan} & 16 & 0.063 & 0.163\\
\textbf{Adygei} & 34 & 0.118 & 0.200\\
\textbf{French} & 58 & 0.431 & 0.311\\
\textbf{North Italian} & 28 & 0.357 & 0.274 \\
\textbf{Orcadian} & 32 & 0.688 & 0.752\\
\textbf{Swedish Finnish} & 360 & 0.789  & 0.185\\
\end{array}
\]

We see for our lactase example that the observed statistic, computed from the table, is

\(w_{obs}=\sum_{i=1}^M Z_i^2= \sum_{i=1}^M \frac{(\hat{p}_i-\hat{p}(x_i))^2}{\frac{\hat{p}(x_i)(1-\hat{p}(x_i))}{n_i}}\) \[=10.01\]

with \(7-2\) degrees of freedom (removing the Basques). The p-value is \(pvalue=0.07\), indicating that the linear is consistent with the data. We typically accept suitable fits to the data that have \(pvalue>0.05\). We thus offer additional evidence to show that the lack of homogeneity of the frequencies may be explained by the changes the environment that go along with different latitudes.

The \(\chi^2\)-test is a general goodness of fit test for curve fitting. The expected value of an outcome at a point \(x\) can be derived from fully theoretical models and compared against the observed data that is assumed to distribute normally about its theoretical value.

\hypertarget{questions-11}{%
\section{Questions}\label{questions-11}}

\textbf{1)} The \(\chi^2\) test for contingency table is used to

\textbf{\(\qquad\)a:} test the associations of a categorial variable;
\textbf{\(\qquad\)b:} test statistical independence between two categorical variables;
\textbf{\(\qquad\)c:} test statistical independence between a continuous and a categorical variable;
\textbf{\(\qquad\)d:} test significance of a categorial variable

\textbf{2)} In the Fisher test we compute the \(pvalue\) using

\textbf{\(\qquad\)a:} the central limit theorem;
\textbf{\(\qquad\)b:} a conditional probability;
\textbf{\(\qquad\)c:} the relative frequencies;
\textbf{\(\qquad\)d:} a parametric model

\textbf{3)} The probability to heal from an injury without treatment is 0.6. The probability to heal from an injury with treatment is 0.8. What is the odd ratio of healing with treatment with respect of healing without it.

\textbf{\(\qquad\)a:} 0.376;
\textbf{\(\qquad\)b:} 1.5;
\textbf{\(\qquad\)c:} 2.666;
\textbf{\(\qquad\)d:} 0.6666

\textbf{4)} The null hypothesis in testing for differences in proportions across different conditions assume

\textbf{\(\qquad\)a:} different conditions for different proportions;
\textbf{\(\qquad\)b:} the same condition across different proportion;
\textbf{\(\qquad\)c:} different proportions across the conditions;
\textbf{\(\qquad\)d:} the same proportion across the conditions\\
\textbf{5)} if the \(pvalue\) is lower than \(0.05\) in a test of several proportions then

\textbf{\(\qquad\)a:} we accept that at least one proportion is different from the rest;
\textbf{\(\qquad\)b:} We accept that all proportions are different between each other;
\textbf{\(\qquad\)c:} We accept the all proportions are equal;
\textbf{\(\qquad\)d:} We accept that there is one single proportion

\hypertarget{practice-5}{%
\section{Practice}\label{practice-5}}

Load hospital data (\url{https://alejandro-isglobal.github.io/SDA/data/hospital.txt})

\begin{itemize}
\item
  Test the hypothesis that the infection proportion at hospitals C and D are different. Use a \(\chi^2\) test and a exact Fisher test. Interpret the OR.
\item
  Make a bar plot
\item
  Test the hypothesis that the infection proportion at hospitals A, B and E are different.
\item
  Make a bar plot
\end{itemize}

\href{https://colab.research.google.com/drive/1qHNmibyg9L7v8NcO3HHmknIGxt6_77KK?usp=sharing}{Solutions}

\hypertarget{mean-differences-between-two-samples}{%
\chapter{Mean differences between two samples}\label{mean-differences-between-two-samples}}

Testing statistical dependence lies at the core of statistical inference. In many scientific studies, we want to know whether changes in one factor are associated with changes in an outcome. To answer this, experiments are planned with great care so that we can identify and measure the sources of variation in the data.

Variation in an outcome can come from several sources. Some of it is due to measurement error, which improved techniques can often reduce. Some comes from intrinsic differences between the observational units (people, animals, samples, etc.). If these differences are irrelevant to the main question, we can control for them --- for example, by grouping similar units together and repeating the experiment under the same conditions. Sometimes, however, controlling all conditions is impractical or undesirable. In those cases, we can use randomization so that uncontrolled influences become part of the random error, which can then be handled statistically.

In the kind of experiments we consider here, we deliberately change one condition of interest, repeat the random experiment, and record the outcome. Each experimental unit is observed under only one condition --- we do not (or cannot) measure the same unit under multiple conditions.

When the outcome is a continuous variable, we may ask how its expected value changes as the condition changes. The theoretical question is: Does changing the condition alter the probability distribution of the outcome, and therefore its mean? Experimentally, if the averages differ between conditions, we want to know: To what extent can we conclude that the outcome is statistically dependent on the condition?

In this chapter, we will focus on testing statistical independence between one continuous variable and two discrete conditions (or groups). We will start with the case where \(n\) is large and the central limit theorem applies, and illustrate its use using the transformative finding in preventive health from the Framingham cohort that established hypertension as a risk factor for stroke.

Next, we will introduce the \(t\)-test, which applies when \(n\) is small and the outcomes are normally distributed with equal variances. This will be illustrated with James Clerk Maxwell's counterintuitive discovery that gas viscosity is independent of pressure --- a key finding in the foundation of statistical mechanics.

Finally, we will consider the case where the conditions also change the variances of normally distributed outcomes. Here, we will present the two-sample \(t\)-test for unequal variances and demonstrate its application to comparing the weights of leptin knocked out mice, demonstrating that hormone encoded by the gene plays a role in obesity.

\hypertarget{difference-in-means-between-two-groups}{%
\section{Difference in means between two groups}\label{difference-in-means-between-two-groups}}

We will consider an outcome of a random experiment that follows a normal probability model:

\[ Y \sim N(\mu, \sigma^2) \]

We aim to measure this outcome several times under two different conditions, or in two groups, called \(A\) and \(B\). Our goal is to determine whether the expected value (mean) of the random variable changes between these conditions.

When testing a hypothesis about the mean, but without knowing a fixed reference value \(\mu_0\) for comparison, we often estimate it by repeating the experiment under a reference condition \(A\), often referred to as the control condition. We then compare this estimate to the mean obtained under a new condition \(B\), where we may have applied a new treatment or selected units from a specific case. Examples include testing clinical differences between cases and controls, comparing a drug to a placebo, or changing the conditions of a physical system.

\textbf{Example (Framingham cohort)}

In 1948, 5209 men and women from Fremingham, Massachusetts, were recruited to be followed during their life course to find the causes of hart disease. The cohort, was transform epidemiological research with unprecedented insights into cardiovascular disease. From this investigations we now know, for example, that hypertension is a risk factor for stroke \citep{kannel1961factors}, which has changed clinical practice, ever since.

Let us explore this finding with available data from the cohort. While access to three generations of full data can be obtained to perform comprehensive analysis, here we will illustrate the fining with the teaching data set from the Framingham Heart Study.

In this data matrix, we have can take for instance two conditions: \(A:Stroke\) and \(B:No \,stroke\) and assume that

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\tightlist
\item
  the level of systolic blood preasure (SYBP) in a randomly chosen man between 40 and 59 years of age who reported a stroke has a probability density
\end{enumerate}

\[Y_A \sim N(\mu_A, \sigma_A^2)\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi})}
\setcounter{enumi}{1}
\tightlist
\item
  the level in a man with the same characteristics but who has not suffer a stroke has a probability density for SYBP
\end{enumerate}

\[Y_B \sim N(\mu_B, \sigma_B^2)\]

\hypertarget{data-1}{%
\section{Data}\label{data-1}}

One random experiment has two variables: \((c, y)\):

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  \(C \in \{A,B\}\) is a discrete variable with two possible outcomes, one for each experimental condition.
\item
  \(Y\) a continuous variable, whose observation produces the outcome of interest
\end{enumerate}

The repetition of the random experiment \(n\) times is therefore a table such as

\[
\begin{array}{ccc}
\mathbf{Individual} & \mathbf{Condition} & \mathbf{Outcome} \\ 
c_1 & \text{A} & \text{121.0} \\ 
c_2 & \text{A} & \text{127.5} \\
c_3 & \text{A} & \text{141.0} \\
\vdots & \vdots & \vdots \\ 
c_{n-2} & \text{B} & \text{153.5} \\ 
c_{n-1} & \text{B} & \text{160.0} \\ 
c_{n}& \text{B} & \text{173.0} \\ 
\end{array}
\]

For example, subject \(1\) was under condition \(A\) and had outcome \(121.0\) for \(Y\).

\textbf{Example (Framingham cohort)}

In the Framingham cohort the SYBP is a continuous outcome in 40-59 year old man

\begin{itemize}
\tightlist
\item
  \(SYBP \in (83.5, 217)\)
\end{itemize}

And the condition is whether the same individual reported having a stroke or not:

\begin{itemize}
\tightlist
\item
  \(stroke \in \{yes:A,no:B\}\)
\end{itemize}

\(201\) men reported having a stroke and \(2760\) reported not having one. We want to know whether the probability of having high systolic blood pressure changes between stroke patients or controls. We may also ask whether the expected value of SYBP is significantly higher in stroke patients. Our research interest is to show that SYSBP is statistically dependent from stroke status.

\hypertarget{difference-between-means}{%
\section{Difference between means}\label{difference-between-means}}

We take the SYSBP levels \textbf{conditioned to} each stroke, and observed:

\begin{itemize}
\item
  \(n_A=201\) stroke patients had a mean of \(\bar{y}_A=140.91\) and \(s=21.79\)
\item
  \$n\_B=2760 \$ non-stroke controls had a mean of \(\bar{y}_B=133.99\) and \(s=18.42\)
\end{itemize}

We can show a plot for the mean in each condition overlying the histogram of the data.

\includegraphics{_main_files/figure-latex/unnamed-chunk-175-1.pdf}

Or we can overlay the histograms horizontally

\includegraphics{_main_files/figure-latex/unnamed-chunk-176-1.pdf}

The plots suggest that stroke patients will have more probability of high SYBP than controls, however some stroke patients will have very low SYBP, and some controls high. The dispersion of the outcome is large in both groups but, \textbf{on average} stroke patients have more SYPD. Is the difference between the averages \(\bar{y}_A\) and \(\bar{y}_B\) (the dots at the bottom of the histogram) \textbf{statistically significant}?

How confident are we on the magnitude of this difference? That is, could have we got this lower higher levels of SYBP in stroke patients by chance alone (null hypothesis) or there is biological difference that cannot be fully explained by chance (alternative hypothesis)?

\hypertarget{hypothesis-test}{%
\section{Hypothesis test}\label{hypothesis-test}}

Let us formulate the hypothesis contrast

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\item
  \(H_0: \mu_A=\mu_B\) The null hypothesis is that both groups have the same mean.
\item
  \(H_1: \mu_A \neq \mu_B\) The alternative hypothesis (research interest) is that the means are different between groups.
\end{enumerate}

Only one can be true. How would the data decide? To make things easier let us redefine the contrast.

If we consider \(\delta\), the difference between means \(\delta=\mu_A-\mu_B\), then the hypotheses can be written as

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\tightlist
\item
  \(H_0: \delta=0\)
\item
  \(H_1: \delta\neq 0\)
\end{enumerate}

Either the difference between the means is zero or not. \(\delta\) is the \textbf{parameter of interest} in our study, and to test hypotheses on it, we need to find an \textbf{estimator} for it.

\hypertarget{estiamtor-of-the-mean-difference}{%
\section{Estiamtor of the mean difference}\label{estiamtor-of-the-mean-difference}}

The statistic \(D=\bar{Y}_A-\bar{Y}_B\), that is the difference in averages, is an estimator of \(\delta\). In particular, we can show that the the estimator is

\begin{itemize}
\tightlist
\item
  unbiased because its expected value is the parameter
\end{itemize}

\[E(D)=E(\bar{Y}_A-\bar{Y}_B)=\mu_A-\mu_B=\delta\]

\begin{itemize}
\tightlist
\item
  and consistent because its variance gets smaller when \(n=n_A+n_B\) gets bigger
\end{itemize}

\[\sigma_D^2=V(\bar{Y}_A-\bar{Y}_B)=\frac{\sigma^2_A}{n_A}+\frac{\sigma^2_B}{n_B}\]

This means that we can take one observed value of \(D\), that is \(d_{obs}\), for the estimation of the parameter \(\delta\).

It is important to have a clear idea of the different components in the estimation and the hypothesis test, as they are shown below. Those are the distributions of the data, the distributions of the averages and the means for each condition, and their difference.

\includegraphics{_main_files/figure-latex/unnamed-chunk-177-1.pdf}

Remember that, for instance, the average \(\bar{Y}_A\) is also called the sample mean, and it has a distribution that is centered on \(\mu_A\) and has variance \(\frac{\sigma_A^2}{n_A}\).

\hypertarget{standardized-error}{%
\section{Standardized error}\label{standardized-error}}

If \(Y_A\) and \(Y_B\) are both normal independent variables, their averages are normal and so it is their difference \(D\). Therefore, the standardization of \(D\) is approximately standard normal

\[Z=\frac{D-\delta}{\sqrt{V(D)}}=\frac{\bar{Y}_A-\bar{Y}_B -\delta}{\sqrt{\frac{s^2_A}{n_A}+\frac{s^2_B}{n_B}}} \sim_{approx} N(0,1)\]

when both \(n_A\) and \(n_B\) are large. It is approximate because as we do not know the variances \(\sigma^2_A\) and \(\sigma^2_B\), we estimated them with \(s_A^2\) and \(s_B^2\).

We can also use this approximation when \textbf{we do not know the distributions} of \(Y_A\) and \(Y_B\), as a result of the central limit theorem, when both groups have large sample sizes.

\hypertarget{standardized-error-for-the-null}{%
\section{Standardized error for the null}\label{standardized-error-for-the-null}}

If the null hypothesis is true (\(\delta=0\)), then when we estimate \(\delta\) with \(D\) we make an error. When \(n\) is large, the \textbf{standardized error} from the null hypothesis is the statistic

\[Z=\frac{\bar{Y}_A-\bar{Y}_B}{\sqrt{\frac{s_A^2}{n_A}+\frac{s^2_B}{n_B}}}\]

that follows a standard normal distribution because of the central limit theorem.

To test the hypothesis we then ask if the observed \(z_{obs}\) is within the acceptance region of the null hypothesis.

In particular, we ask: is the probability of observing a more extreme value than \(z_{obs}\) lower than \(\alpha=0.05\) if the null hypothesis is true?

The value of \(z_{obs}\) is the standardized value of the observed difference between the averages:

\[z_{obs}=\frac{d_{obs}}{\sqrt{\frac{s_A^2}{n_A}+\frac{s^2_B}{n_B}}}\]

For our data, we have that the \textbf{observed} standardized mean difference is

\[z_{obs}=\frac{\bar{y}_A-\bar{y}_B }{\sqrt{\frac{s^2_A}{n_A}+\frac{s^2_B}{n_B}}}=\frac{140.91-131.99}{\sqrt{\frac{21.79}{201}+\frac{18.42}{2760}}}=5.95\]

Since our hypothesis is two tailed then the the two-tailed \(pvalue\) is

\[pvalue=2(1-\Phi(5.95))=2.68 \times 10^{-9}\]

which is much lower than \(\alpha\).

Therefore, we reject the null hypothesis that \(H_0:\delta=0\) that is that the SYBP in stroke patients and controls are equal. In other words, we have strong evidence that the mean SYBP between conditions are different.

In Python and R, we use the fact that the standardized statistic \(Z\) when \(n\) is large is close to a \(t\) distribution with large degrees of freedom, even when the outcome distributions are not normal. We can use the \(t\)-test as an approximation for the normal standard in this cases

\begin{verbatim}
Python
import pandas as pd
from scipy import stats

# Assuming dat is already a pandas DataFrame
# Example: dat = pd.DataFrame({"SYSBP": [...], "STROKE": [...]})

# Split into two groups
group_no_stroke = dat.loc[dat["STROKE"] == "No Stroke", "SYSBP"]
group_stroke = dat.loc[dat["STROKE"] == "Stroke", "SYSBP"]

# Perform independent two-sample t-test (equal_var=False is Welch's t-test)
t_stat, p_value = stats.ttest_ind(group_no_stroke,
group_stroke, equal_var=False)

##Framingham Data
library(riskCommunicator)
data(framingham)
sel <- framingham$SEX == 1 &
       framingham$AGE > 40 &
       framingham$AGE < 59

dat <- data.frame(
  SYSBP = framingham$SYSBP[sel],
  STROKE = factor(framingham$STROKE[sel], labels = c("No Stroke", "Stroke"))
)

t.test(SYSBP~STROKE, data=dat)
\end{verbatim}

We may conclude that systolic blood pressure is different between stroke patients and control, suggesting high SYBP as a potential factor for the disease. However, this simple preliminary approach needs to be taken with care. The data analyzed did not record whether the SYBP was measured after or before the stroke. In addition, we do not know which of those control patients had a stroke after they reported their status. To determine whether SYBP is an independent cause of stroke, the Framingham cohort followed the patients for several years recording SYBP and cardiovascular health. Their more refined analysis took the continuous values of SYBP as conditions for the occurrence of stroke during follow up.

We can illustrate the results of the comparison between two group means in three different ways:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  Using a bar plot with confidence intervals for the estimate in the mean of each group. Note that in a report we usually add the confidence intervals for the means. Non-overlapping confidence intervals also indicate that the difference between means is \textbf{statistically significant}, as this is an equivalent criteria to reject the null hypothesis.
\item
  Using a boxplot. In this plot, we do not show the means but a summary of the distribution properties of the data at each condition. Remember that the properties are the median (the middle line), the quartiles (the box edges) and the \(5\%\) and \(95\%\) quantiles (the whiskers).
\item
  Using a violin plot. These are smoothed mirrored histograms overlaid. Here, you do not see the averages of each group but the modes (peaks of the histograms). However, you get the idea that the probability of high SYBP is higher in stroke patients that in controls.
\end{enumerate}

\includegraphics{_main_files/figure-latex/unnamed-chunk-178-1.pdf}

\hypertarget{mean-differences-when-n-is-small}{%
\section{\texorpdfstring{Mean differences when \(n\) is small}{Mean differences when n is small}}\label{mean-differences-when-n-is-small}}

\textbf{Example (Gas viscosity)}

James C. Maxwell grounded the field of statistical mechanics with a remarkable prediction. He developed the theory, designed the experiment and measure in the laboratory the fact that the viscosity of gases do not depend on their pressure \citep{maxwell1866viscosity}.

This counter-intuitive phenomena was derived from the molecular properties of gasses and help convince the scientific community that statistical properties of molecules underlie some properties of macroscopic objects.

Theoretically, we showed that gas viscosity depends on two phenomena that exactly cancel out. He deigned a rotating pendulum in a container that could be subject to different gas pressures. When the pendulum rotates then its period to complete a full swing gets longer as the number of molecules in the gas get larger because it hits more molecules slowing it down. However, in gases, as the pendulum hits the molecules, if the pressure is increased, then those molecules would not travel far and get dragged along with the pendulum, creating a gas pocked around it diminishing the drag force. These to phenomena cancel out making the viscosity independent of the pressure.

We how the results of Maxwell's experiment, dividing the pressures reported into two conditions low (\(<6\))mmHg and high (\(>19\))mmHg and the time the pendulum took for a full swing (s). While the pressures were low to examine rarefied gases, the change of in pressure in the experiment was threefold. Maxwell pre-processed the time periods to compute the rate of decay of arc lengths to compare it with theoretical values. Here, we illustrate the raw measurements of gas viscosity throughout pendulum periods.

\[
\begin{array}{ccc}
\mathbf{Experiment} & \mathbf{Pressure} & \mathbf{Period} \\ 
1 &      \text{Low}& 362.66\\
2 &      \text{Low}& 362.80\\
3 &      \text{Low}& 364.04\\
4 &      \text{Low}& 362.72\\
5 &      \text{Low}& 362.94\\
6 &      \text{Low}& 363.80\\
7 &     \text{High}& 362.64\\
8 &     \text{High}& 362.50\\
9 &     \text{High}& 362.86\\
10&     \text{High}& 363.80\\
11&     \text{High}& 362.89\\
12&     \text{High}& 363.90\\
\end{array}
\]

The experiment was then repeated a small number of times \(12\). We can make a boxplot for each condition.

\includegraphics{_main_files/figure-latex/unnamed-chunk-180-1.pdf}

We assume that

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  the period of the pendulum under high pressure is normal random variable
\end{enumerate}

\[Y_A \sim N(\mu_A, \sigma^2)\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{1}
\tightlist
\item
  the period of the pendulum under low pressure is also normally distributed
\end{enumerate}

\[Y_B \sim N(\mu_B, \sigma^2)\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{2}
\tightlist
\item
  both distributions have \textbf{the same} variance \(\sigma^2\).
\end{enumerate}

\hypertarget{data-2}{%
\section{Data}\label{data-2}}

One random experiment has two outcomes: \((preassure, period)\).

The pressure is a categorical variable and determines the conditions of the experiment:

\begin{itemize}
\tightlist
\item
  \(Pressure \in \{High:A, Low:B\}\)
\end{itemize}

The period is a continuous variable and it is the outcome of interest.

\begin{itemize}
\tightlist
\item
  \(T \in (360, 365)\) (seconds)
\end{itemize}

\hypertarget{difference-between-means-1}{%
\section{Difference between means}\label{difference-between-means-1}}

Maxwell measured the periods \textbf{conditioned to} two different pressures, and observed:

\begin{itemize}
\item
  \(n_A=6\) experiments under high pressure had period average \(\bar{y}_A=363.0983\) an standard deviation \(s_A=0.600547\)
\item
  \(n_B=6\) experiments under low pressure had period average \(\bar{y}_B=363.16\) an standard deviation \(s_B=0.6009326\)
\end{itemize}

We can draw violin plots per group

\includegraphics{_main_files/figure-latex/unnamed-chunk-181-1.pdf}

We see that the distributions are very similar similar and their means differ in 0.06 seconds. Can we have a statistical test for this difference?

\hypertarget{hypothesis-test-1}{%
\section{Hypothesis test}\label{hypothesis-test-1}}

We can again formulate the hypothesis on the difference between means \(\delta=\mu_{high} -\mu_{low}\)

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\item
  \(H_0: \delta=0\). The null hypothesis (research interest) assumes that the pendulum will take the same time to swing in high and low pressures.
\item
  \(H_1: \delta \neq 0\). Therefore, the alternative hypothesis is that the mean periods under different pressures are different.
\end{enumerate}

\hypertarget{estimator-of-the-mean-difference}{%
\section{Estimator of the mean difference}\label{estimator-of-the-mean-difference}}

The statistic \(D=\bar{Y}_A-\bar{Y}_B\) is again an unbiased estimator of \(\delta\)

\[E(D)=\delta\]

Since the periods for \(A\) and \(B\) are normal variables with the same variance \(\sigma^2\) each sample variance in each group is an estimator of \(\sigma^2\). That is \[\hat{\sigma}^2=s^2_A=s^2_B\]

then (Theorem) the \textbf{standardized error}

\[T=\frac{\bar{Y}_A-\bar{Y}_B -\delta}{\sqrt{s_p^2(\frac{1}{n_A}+\frac{1}{n_B})}} \sim T(n_A+n_B-2)\]

follows exactly a T-distribution with \(n_A+n_B-2\) degrees of freedom.

The \textbf{pooled variance} \(s_p^2\), is an estimator of \(\sigma^2\)

\[\hat{\sigma}^2=s_p^2= \frac{(n_A-1) s^2_A+(n_B-1) s^2_B}{n_A+n_B-2}\]

\hypertarget{standardized-error-for-the-null-1}{%
\section{Standardized error for the null}\label{standardized-error-for-the-null-1}}

If the null hypothesis is true (\(\delta=0\)), then when we estimate \(\delta\) with \(D\) we make an error. The \textbf{standardized error} from the null hypothesis is the statistic

\[T=\frac{\bar{Y}_A-\bar{Y}_B }{\sqrt{s_p^2(\frac{1}{n_A}+\frac{1}{n_B})}} \sim T(n_A+n_B-2)\]

that follows a T-distribution.

To test the hypothesis, we then ask if the observed \(t_{obs}\) falls within the acceptance region of the null hypothesis.

In particular, we ask: is the probability of observing a more extreme value than \(t_{obs}\) lower than \(\alpha=0.05\), if the null hypothesis is true?

The value of \(t_{obs}\) is the standardized value of the observed difference between the averages:

\(t_{obs}=\frac{d_{obs}}{\sqrt{s_p^2(\frac{1}{n_A}+\frac{1}{n_B})}}\)

\[=\frac{\bar{y}_A-\bar{y}_B }{\sqrt{\frac{s^2_p}{n_A}+\frac{s^2_p}{n_B}}}=\frac{363.0983-363.16}{\sqrt{\frac{0.3608883}{6}+\frac{0.3608883}{6}}}=-0.178\]

The two-tailed \(pvalue\) of \(t_{obs}\) is

\[pvalue=2(1-F_{t,10}(0.138))=0.89\]

which is higher than \(\alpha=0.05\). Therefore, the data shows that there are no significant differences between the periods under low or high pressure, which is consistent with Maxwell's theoretical prediction, and gave experimental support to a counter-intuitive gas property that could be satisfactory explained from the statistical properties of its molecular components.

We can compute this in Python and R

\begin{verbatim}
Pyhton:
import pandas as pd
from scipy import stats
# Create the dataframe
dat = pd.DataFrame({
    "Pressure": ["Low"]*6 + ["High"]*6,
    "Period": [362.66, 362.8, 364.04, 362.72, 362.94, 363.8,
               362.64, 362.5, 362.86, 363.8, 362.89, 363.9]})
# Split into groups
low = dat.loc[dat["Pressure"] == "Low", "Period"]
high = dat.loc[dat["Pressure"] == "High", "Period"]

# Two-sample t-test with equal variances
t_stat, p_val = stats.ttest_ind(low, high, equal_var=True)
print("t-statistic:", t_stat)
print("p-value:", p_val)

R:
dat <- data.frame(Pressure=c("Low", "Low", "Low", "Low", "Low", "Low",
"High", "High", "High", "High", "High", "High"), 
Period=c(362.66, 362.8, 364.04, 362.72, 362.94, 363.8, 
362.64, 362.5, 362.86, 363.8, 362.89, 363.9))

t.test(Period ~Pressure, data=dat, var.equal=TRUE)
\end{verbatim}

\hypertarget{mean-differences-with-unequall-variances}{%
\section{Mean differences with unequall variances}\label{mean-differences-with-unequall-variances}}

\textbf{Exemple (Leptin knockouts)}

Leptin is an adipose tissue hormone that creates the sensation of satiety after eating. It is believe to play an important role into the development of obesity at early ages. Ramos-Lobos and colleagues tested the additional effect of leptin during neurodevelopment in mice \citep{Ramos-Lobo2019}. They produced 7 male mice for which their leptin gene was knocked out, and could not produce the hormone. While 16 mice were left with normal leptin function. These are called wild type.

We can use this data to show that the levels leptin can produce weight gains in the animals. We therefore ask whether the mean weight of the animals is different between wild types and knock-outs. As the event of knocking out a gene is clearly before the development of obesity, this type of genetic experiments are considered to suggest a causal link between a gene and a trait.

We assume that

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  the weight of the control animals (wild type) has a probability density
\end{enumerate}

\[Y_A \sim N(\mu_A, \sigma^2)\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{1}
\tightlist
\item
  the weight of the animals with no leptin gene has a probability density
\end{enumerate}

\[Y_B \sim N(\mu_B, \sigma^2)\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{2}
\tightlist
\item
  both distributions have the same variance \(\sigma^2\).
\end{enumerate}

\hypertarget{data-3}{%
\section{Data}\label{data-3}}

One random experiment in this study has two outcome types: \((leptin, weight)\).

Leptin is categorical variable that determines the condition of the experiment whether the animal has a functioning gene (leptin+) or not (leptin-)

\begin{itemize}
\tightlist
\item
  \(leptin \in \{leptin+:A,leptin-:B\}\)
\end{itemize}

The outcome of interest is the weight of the animal, and it is a continuous variable

\begin{itemize}
\tightlist
\item
  \(weigth \in (20, 60)\)
\end{itemize}

The data looks like

\[
\begin{array}{ccc}
\mathbf{Mouse} & \mathbf{Leptin} & \mathbf{Weigth} \\ 
1 &\text{Leptin}  &27.67\\
2 &\text{Leptin+}  &27.40\\
3 &\text{Leptin+}  &25.77\\
4 &\text{Leptin+}  &25.60\\
5 &\text{Leptin+}  &25.03\\
6 &\text{Leptin+}  &25.90\\
7 &\text{Leptin+}  &26.67\\
8 &\text{Leptin+}  &25.60\\
9 &\text{Leptin+}  &28.93\\
10 &\text{Leptin+}  &31.83\\
11 &\text{Leptin+}  &25.90\\
12 &\text{Leptin+}  &26.30\\
13 &\text{Leptin+}  &27.90\\
14 &\text{Leptin+}  &26.77\\
15 &\text{Leptin+}  &25.83\\
16 &\text{Leptin+}  &20.87\\
17 &\text{Leptin-}  &46.57\\
18 &\text{Leptin-}  &40.43\\
19 &\text{Leptin-}  &41.97\\
20 &\text{Leptin-}  &41.17\\
21 &\text{Leptin-}  &41.57\\
22 &\text{Leptin+}  &46.17\\
23 &\text{Leptin+}  &53.83
\end{array}
\]

The box plot suggests that the variances for each group are different because the box for the leptin- group is thinner (less dispersed) than the box for the leptin+ group. It appears that knocking out the gene not only reduces the mean weight but also the weight variance.

\includegraphics{_main_files/figure-latex/unnamed-chunk-183-1.pdf}

Therefore it is better to assume that

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  the weight of the wild type animal (leptin+) has a probability density
\end{enumerate}

\[Y_A \sim N(\mu_A, \sigma_A^2)\]

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{1}
\tightlist
\item
  the weight of the animal with no leptin has a probability density
\end{enumerate}

\[Y_B \sim N(\mu_B, \sigma_B^2)\]

When we assume unequal variances in each group, the \textbf{standardized error} for the null hypothesis (\(\delta=0\))

\[T=\frac{\bar{Y}_A-\bar{Y}_B }{\sqrt{\frac{s_A^2}{n_A}+\frac{s_B^2}{n_B}}} \sim_{aprox} T(\nu)\]

approximately follows a t-distribution with
\[\nu=\frac{(\frac{s_A^2}{n_A}+\frac{s_B^2}{n_B})^2}{\frac{(s_A^2/n_B)^2}{n_A-1}+\frac{(s_B^2/n_B)^2}{n_B-1}}\]
degrees of freedom. The brilliant idea of Welsh was to fix the form of the variance of \(D\) (denominator of \(T\)), and then ask which was the closest \(t\)-distribution that would describe \(T\). For that he needed to adjust the degrees of freedom. This is called a Welsh test.

In Python and R, the Welsh test is obtained by setting the parameter of equal variances to false in the t-test functions

\begin{verbatim}
Python:
stats.ttest_ind(groupA, groupB, equal_var=False)

R:
t.test(groupA, groupB, var.equal=FALSE)
\end{verbatim}

For the mouse data, we observe a very significant increase in \(18.03\)gr (\(pvalue=2.444 \times 10^{-5}\)) in weight between the wild-type mice and leptin knockouts, demonstrating the effect of leptin hormone in mouse weight and a possible role in human obesity.

Additional studies also required to supplement the knockout mice with leptin hormone and observe the reduction of weight. Such interventions demonstrated the causal role of the hormone on weight. For human studies, similar evidence could be obtained from randomized drug-placebo studies of patients with leptin deficiency who are given leptin supplementation or placebo.

Note that if we had use the equal variances \(t\)-test, we would have obtained a more significant result \(pvalue=3.376854 \times 10^{-11}\) but this model is less appropriate based on how the data distributes.

\hypertarget{questions-12}{%
\section{Questions}\label{questions-12}}

\textbf{1)} We test for the difference between the means of two random variables when we have measured

\textbf{\(\qquad\)a:} two continuous random variables;
\textbf{\(\qquad\)b:} two categorical random variables;
\textbf{\(\qquad\)c:} one dichotomic variable and one continuous random variable;
\textbf{\(\qquad\)d:} any categorical variable and one continuous random variable;

\textbf{2)} The statistic \(D=\bar{Y}_A-\bar{Y}_B\) estimates

\textbf{\(\qquad\)a:} The difference between the averages of \(Y_A\) and \(Y_B\);
\textbf{\(\qquad\)b:} The difference between the means of \(Y_A\) and \(Y_B\);
\textbf{\(\qquad\)c:} The variation of \(\bar{Y}_A\) with respect to \(\bar{Y}_B\);
\textbf{\(\qquad\)d:} \(d_{obs}\)

\textbf{3)} We can use a \(Z\)-test only when

\textbf{\(\qquad\)a:} we have a large sample of the continuous variable in one of the conditions;
\textbf{\(\qquad\)b:} we have large samples of the continuous variable in each condition;
\textbf{\(\qquad\)c:} the variances of the continuous variable are equal in each conditions;
\textbf{\(\qquad\)d:} when the distributions of the continuous variable in each condition are normal

\textbf{4)} We can use a \(t\)-test only when

\textbf{\(\qquad\)a:} we have small samples of the continuous variable in
each condition;
\textbf{\(\qquad\)b:} the distributions of the continuous variable in each condition are normal;
\textbf{\(\qquad\)c:} \(D\) is unbiased;
\textbf{\(\qquad\)d:} we cannot use a \(Z\)-test

\textbf{5)} For the data

\[
\begin{array}{ccc}
\mathbf{Subject} & \mathbf{Y} & \mathbf{C} \\ 
1&1.1&A\\ 
2&0.9&A\\ 
3&0.8&B\\ 
4&0.6&B\\ 
\end{array}
\]

assuming that \(Y\) is normally distributed, which is the best test?

\textbf{\(\qquad\)a:} \(t\)-test with equal variances
\textbf{\(\qquad\)b:} \(t\)-test with unequal variances
\textbf{\(\qquad\)c:} \(z\)-test with equal variances
\textbf{\(\qquad\)d:} \(z\)-test with unequal variances

\hypertarget{practice-6}{%
\section{Practice}\label{practice-6}}

Load leptin data \url{https://alejandro-isglobal.github.io/SDA/data/dataleptin.txt}

\begin{itemize}
\item
  Test the hypothesis that in the control animals, the weight of females is different than the weight of males
\item
  Test the hypothesis that the leptin KO animals have higher weight than the KO animals with supplemented leptin
\item
  Make a bar plot and a boxplot for the latter case. Compute the confidence intervals for the weight, in each group.
\end{itemize}

\href{https://colab.research.google.com/drive/1F61puLHyUBuS5ERbbruy4002bQ3C3DjY?usp=sharing}{Solutions}

\hypertarget{mean-differences-across-several-groups}{%
\chapter{Mean differences across several groups}\label{mean-differences-across-several-groups}}

Experiments may be subjected to more than two conditions. We will first consider experiments with categorical and mutually exclusive conditions: if an experimental unit belongs to one condition, it cannot belong to another. For instance, in a study of all-cause mortality, patients may die from coronary disease, cancer, or respiratory conditions; or in a population genetics study, we may examine allele frequencies across different global populations. In both cases, we would like to know whether the set of conditions under study influences the outcome. If at least one condition changes the outcome, that is enough to conclude that the categorical random variable encoding the conditions (as different levels) influences the properties of the experiment, and therefore its outcomes.

If the outcome is a continuous variable, we are often interested in how its \textbf{mean} changes across the categories of the condition variable. The categories of the condition variable are usually referred to as \emph{groups}. In this chapter, we will generalize the test for differences between two means to the case of many groups, that is, many levels of a categorical variable. We will do this using \textbf{analysis of variance (ANOVA)}, which compares the variance \emph{within groups} against the variance \emph{between groups}. We will illustrate this analysis with data showing that leptin deficiency can cause obesity.

In ANOVA, we explain the observed variation by decomposing it into additive sources. Some sources of variation are of primary interest and define the hypotheses we want to test, while others correspond to systematic error that we try to control for. Using Michelson and Morley's classic data, we will see that accounting for multiple sources of variation in the measurement of the speed of light was key to showing that light travels at the same speed in any direction.

Conditions may also arise from different types of categorical variables, and in such cases they are not mutually exclusive. For example, when studying mortality, \emph{disease type} (cause of death) may be one condition type, and \emph{sex} another; in genetics, we may study allele frequencies across different \emph{regions} (first type of condition) and across different \emph{species} (second type). To handle these situations, we will introduce the \textbf{two-way ANOVA}, which allows us to measure the independent contributions of two different condition types.

Finally, we will use two-way ANOVA with \textbf{interaction}, which tests whether at least one combination of the two categorical variables produces effects that cannot be explained by the independent contributions of each variable alone. We will illustrate this by analyzing the differential effect of leptin deficiency in the weight of females.

\hypertarget{different-means-among-several-conditions}{%
\section{Different means among several conditions}\label{different-means-among-several-conditions}}

\textbf{Example (Leptin knockouts)}

Leptin is a hormone that regulates satiety. Ramos-Lobo and colleagues tested the additional effect of leptin during neurodevelopment in mice \citep{Ramos-Lobo2019}. In their study, they reported the weight of 16 male mice with normal leptin function, 7 male mice in which the leptin gene was knocked out, and 10 knockout mice that received leptin hormone injections. The inclusion of this third group demonstrates that lean weight can be partially restored by leptin supplementation, showing that leptin deficiency itself causes weight gain.

For this data, we assume three experimental conditions:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  The weight of the control animals (wild type, Leptin+) follows

  \[
  Y_A \sim N(\mu_A, \sigma^2)
  \]
\item
  The weight of the knockout animals injected with leptin follows (Leptin-(sup))

  \[
  Y_B \sim N(\mu_B, \sigma^2)
  \]
\item
  The weight of the knockout animals without leptin follows (Leptin-)

  \[
  Y_C \sim N(\mu_C, \sigma^2)
  \]
\end{enumerate}

\hypertarget{data-4}{%
\section{Data}\label{data-4}}

One random experiment in this study can be considered to have two outcome components: \((\text{leptin}, \text{weight})\).

\begin{itemize}
\item
  \textbf{Leptin status} is a categorical variable determining the experimental condition:\\
  \[
  leptin \in \{ \text{Leptin+ : A},\ \text{Leptin-(sup) : B},\ \text{Leptin- : C} \}
  \]
\item
  \textbf{Weight} is the continuous outcome of interest:\\
  \[
  weight \in (20, 60)
  \]
\end{itemize}

The data look like:

\[
  \begin{array}{ccc}
  \mathbf{Mouse} & \mathbf{Leptin} & \mathbf{Weight} \\ 
  1 & \text{Leptin+} & 27.67 \\
  2 & \text{Leptin+} & 27.40 \\
  3 & \text{Leptin+} & 25.77 \\
  4 & \text{Leptin+} & 25.60 \\
  5 & \text{Leptin+} & 25.03 \\
  6 & \text{Leptin+} & 25.90 \\
  7 & \text{Leptin+} & 26.67 \\
  8 & \text{Leptin+} & 25.60 \\
  9 & \text{Leptin+} & 28.93 \\
  10 & \text{Leptin+} & 31.83 \\
  11 & \text{Leptin+} & 25.90 \\
  12 & \text{Leptin+} & 26.30 \\
  13 & \text{Leptin+} & 27.90 \\
  14 & \text{Leptin+} & 26.77 \\
  15 & \text{Leptin+} & 25.83 \\
  16 & \text{Leptin+} & 20.87 \\
  17 & \text{Leptin-(sup)} & 24.33 \\
  18 & \text{Leptin-(sup)} & 22.37 \\
  19 & \text{Leptin-(sup)} & 26.10 \\
  20 & \text{Leptin-(sup)} & 17.50 \\
  21 & \text{Leptin-(sup)} & 35.17 \\
  22 & \text{Leptin-(sup)} & 25.97 \\
  23 & \text{Leptin-(sup)} & 27.67 \\
  24 & \text{Leptin-(sup)} & 23.37 \\
  25 & \text{Leptin-(sup)} & 31.83 \\
  26 & \text{Leptin-(sup)} & 22.37 \\
  27 & \text{Leptin-} & 46.57 \\
  28 & \text{Leptin-} & 40.43 \\
  29 & \text{Leptin-} & 41.97 \\
  30 & \text{Leptin-} & 41.17 \\
  31 & \text{Leptin-} & 41.57 \\
  32 & \text{Leptin-} & 46.17 \\
  33 & \text{Leptin-} & 53.83 \\
  \end{array}
\]

\hypertarget{difference-between-means-2}{%
\section{Difference between means}\label{difference-between-means-2}}

We compute the mean and standard deviation of weights \textbf{within each group}:

\begin{itemize}
\tightlist
\item
  \(n_A = 16\) control mice (Leptin+) had a mean \(\bar{y}_A = 26.50\) and \(s_A = 2.25\)
\item
  \(n_B = 10\) knockout mice with leptin replacement (Leptin-(sup)) had a mean \(\bar{y}_B = 25.67\) and \(s_B = 5.03\)
\item
  \(n_C = 7\) knockout mice without leptin (Leptin-) had a mean \(\bar{y}_C = 44.53\) and \(s_C = 4.77\)
\end{itemize}

We can visualize the distributions with bar plots and violin plots:

\includegraphics{_main_files/figure-latex/unnamed-chunk-185-1.pdf}

The wild type mice (Leptin+) and the knockout mice that received leptin supplementation (Leptin-(sup)) show similar mean weights. In contrast, knockout mice without leptin (Leptin-) have a much higher average weight. Since the confidence interval for this group does not overlap with those of the other two groups, we conclude that the mean weight of leptin-deficient mice is significantly higher. The violin plots further illustrate differences in the distribution of weights across groups.

When testing differences in means between two groups, the null hypothesis assumes that the means are equal. When comparing more than two groups, the null hypothesis is that all groups share the same mean. The alternative is that at least one group has a different mean. This motivates the use of analysis of variance (ANOVA).

\hypertarget{hypothesis-test-2}{%
\section{Hypothesis test}\label{hypothesis-test-2}}

Let us now formulate the hypothesis contrast:

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\item
  \textbf{Null hypothesis}\\
  \[
  H_0: \mu=\mu_A = \mu_B = \mu_C
  \]\\
  The null hypothesis assumes that there are no differences among the group means. In this case, all bars in the bar plot would align at the same height, within the margin of their confidence intervals.
\item
  \textbf{Alternative hypothesis}\\
  \[
  H_1: \text{at least one } \mu \neq \mu_i \quad \text{for some $j$} 
  \]\\
  The alternative hypothesis assumes that at least one group mean is different from the others. At least one bar in the bar plot is clearly distinct from the rest.
\end{enumerate}

How can we test these hypotheses? We seek a statistic whose value allows us to decide whether to reject or retain \(H_0\).

For simplicity, assume that all groups have the same number of observations,\\
\[
n_A = n_B = n_C = n
\]\\
This means each condition contains the same number of mice in the leptin example, for a total of \(3n.\) mice. Such a situation is called a \textbf{balanced design}. The results we will derive still hold approximately if we replace \(n\) by the \textbf{average number of observations per group},\\
\[
n = \frac{1}{k} \sum_{i=1}^k n_i.
\]

\hypertarget{distribution-of-group-means-under-h_0}{%
\subsection{\texorpdfstring{Distribution of group means under \(H_0\)}{Distribution of group means under H\_0}}\label{distribution-of-group-means-under-h_0}}

If the null hypothesis is true, there are no differences between group means and they all coincide with a single common mean \(\mu\):

\[
\mu_A = \mu_B = \mu_C = \mu.
\]

Therefore, each group average follows the same distribution:\\
\[
\bar{Y}_A, \bar{Y}_B, \bar{Y}_C \sim N\!\left(\mu, \frac{\sigma^2}{n.}\right).
\]

The observed vector of means,\\
\[
m = (\bar{y}_A, \bar{y}_B, \bar{y}_C),
\]\\
can be interpreted as the result of an \textbf{averaging experiment}: we repeatedly sample \(n\) weights in each of the three conditions and take their averages. If \(H_0\) holds, these three averages are independent estimates of the same underlying mean \(\mu\).

\hypertarget{sources-of-variation}{%
\subsection{Sources of variation}\label{sources-of-variation}}

We can distinguish two sources of dispersion in the obervatons of the experiment:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  \textbf{Within-group variance} (\(\sigma^2\))\\
  -- the natural dispersion of individual outcomes in each condition, which reflects the variability of the original random experiment.
\item
  \textbf{Between-group variance} (\(\sigma^2_{\bar{Y}}\))\\
  -- the dispersion of the group averages, which reflects how different the sample means are from one another.
\end{enumerate}

Under the null hypothesis, changing conditions does not affect the group averages, because the outcomes are independent of the conditions. By the law of large numbers, as the number of experiments \(n\) in each group increases, the averages converge to \(\mu\). Hence, the variance of the averages decreases as\\
\[
H_0: \quad \sigma^2_{\bar{Y}} = \frac{\sigma^2}{n}.
\]

If, however, one group has a true mean different from \(\mu\), its average will not get close to \(\mu\) as \(n.\) grows. In that case, the between-group variance will be \textbf{larger than expected} under the null:\\
\[
H_1: \quad \sigma^2_{\bar{Y}} > \frac{\sigma^2}{n}.
\]

\hypertarget{variance-components-estimators}{%
\section{Variance components estimators}\label{variance-components-estimators}}

To test the null hypothesis, we need estimators for the two variance components:

\begin{itemize}
\tightlist
\item
  the \textbf{within-group variance} \(\sigma^2\),\\
\item
  the \textbf{between-group variance} \(\sigma^2_{\bar{Y}}\).
\end{itemize}

\textbf{Estimator of the within-group variance}

Within each group, the data vary around their group mean. For example, in group \(A\), the sample variance is

\[
S_A^2=\frac{1}{n-1} \sum_{i=1}^{n} (Y_{Ai}-\bar{Y}_{A})^2.
\]

where \(i\) denotes the \(i\)-th repetition of the experiment under condition \(A\). If we only considered the control mice (group \(A\)), this sample variance \(S_A^2\) would be an unbiased estimator of the population variance \(\sigma^2\).

Since we assume all groups share the same variance \(\sigma^2\), we can combine information from all groups by averaging their sample variances:

\[
S_e^2 = \frac{1}{k}\sum_{j=1}^k S_j^2.
\]

where \(k\) is the number of groups, or conditions of the experiment. This pooled estimator \(S_e^2\) is called the \textbf{mean square error (MSE)}. It represents the variability of the random experiment \emph{within} conditions.

\includegraphics{_main_files/figure-latex/unnamed-chunk-186-1.pdf}

\textbf{Estimator of the between-group variance}

Now let us look at the variability of the \textbf{group means}. This is the right-hand side of the plot, where we see how the group means disperse around the overall mean (black point in in the plot).

For each group mean \(\bar{Y}_j\):

\[
V(\bar{Y}_j) = \frac{\sigma^2}{n}, \quad \text{if } H_0 \text{ is true}.
\]

Thus, under the null hypothesis, the group means behave like repeated estimates of the same mean \(\mu\).

The sample variance of the group means is

\[
S^2_{tr}=\frac{1}{k-1} \sum_{j=1}^k (\bar{Y}_{j}-\bar{Y})^2,
\]

where \(\bar{Y}\) is the overall mean across all observations:

\[
\bar{Y}=\frac{1}{k}\sum_{j=1}^k \bar{Y}_j = \frac{1}{kn}\sum_{j=1}^k \sum_{i=1}^n Y_{ji}.
\]

This statistic \(S^2_{tr}\) estimates the variance of the group means, \(\sigma^2_{\bar{Y}}\).

\textbf{The F-ratio}

If the null hypothesis is \textbf{true} and the number of observations per group grows large, then all the group means converge to the same overall mean \(\mu\). Consequently, the variance of the group means should be close to its expected value \(\sigma^2/n\). Remember that \(V(\bar{Y})=\sigma^2/n\) which is estimated by \(S_{tr}^2\).

On the other hand, \(S_e^2\) estimates \(\sigma^2\) and therefore \(S_e^2/n\) is another estimation of the group variance \(\sigma^2/n\).

This leads to the \textbf{F statistic}:

\[
F=\frac{S_{tr}^2}{S_e^2/n}.
\]

\begin{itemize}
\tightlist
\item
  Under \(H_0\), \(F\) is close to \(1\). As \(n\) increases the outcomes get close to their means, which get close to the overall mean.
\end{itemize}

If the null hypothesis is \textbf{not true} and we take more and more observations in each group, then at least one group mean will not converge to the common mean. For example, if \(\mu_B=\mu_C=\mu \neq \mu_A\), the averages from groups \(B\) and \(C\) will get closer to \(\mu\), but the average from group \(A\) will concentrate around \(\mu_A \neq \mu\). As a result, the expected value of \(S^2_{tr}\) will be larger than \(\frac{\sigma^2}{n}\), because there is an irreducible component of variability due to the difference of the group \(A\) with the rest.

In this situation, we say that the data contain \textbf{treatment variance} (variance explained by group differences) in addition to the random within-group variance. Consequently, the \(F\) statistic will tend to be greater than 1, providing evidence to reject the null hypothesis.

\begin{itemize}
\tightlist
\item
  Under \(H_a\), at least one group mean differs, so \(S_{tr}^2\) becomes larger than \(\sigma^2/n\), making \(F > 1\).
\end{itemize}

Intuitively, \(F\) measures how much the group means are separated relative to the noise inside groups.

\textbf{Sums of squares formulation}

In practice, the \(F\)-statistic is reported as a ratio of \textbf{mean sums of squares}:

\[
F = \frac{MST}{MSE},
\]

where

\begin{itemize}
\item
  \textbf{Mean Square for Treatments (MST):}\\
  \[
  MST = \frac{1}{k-1}\sum_{j=1}^k n(\bar{Y}_j-\bar{Y})^2
  = \frac{SS_{Group}}{k-1},
  \]
\item
  \textbf{Mean Square Error (MSE):}\\
  \[
  MSE = \frac{1}{k(n-1)}\sum_{j=1}^k\sum_{i=1}^n (Y_{ji}-\bar{Y}_j)^2
  = \frac{SS_{Residual}}{k(n-1)}.
  \]
\end{itemize}

Since sums of squares are quadratic forms of normally distributed variables, each scaled variance follows a chi-square distribution with its respective degrees of freedom. Their ratio follows an \textbf{F distribution}:

\[
F \sim F(k-1,\,k(n-1)).
\]

Here, \(k\) is the number of groups, and \(n\) the number of observations per group (balanced design). In unbalanced designs, \(n\) is replaced by the actual group sizes, and the degrees of freedom are adjusted accordingly.

\hypertarget{analysis-of-variance-anova}{%
\section{Analysis of variance (ANOVA)}\label{analysis-of-variance-anova}}

The hypothesis test for the difference in means across several groups is then

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\item
  \(H_0: \mu_1=\mu_2= ...=\mu_k\). There are no difference between group means. Then, the observed value of \(F\), \emph{e.g} \(f_{obs}\) will be near \(1\).
\item
  \(H_1:\) at least one \(\mu_i\) is different from the rest. Then, the observed value of \(F\), \emph{e.g} \(f_{obs}\) will be \textbf{greater} than \(1\).
\end{enumerate}

\textbf{Example (Mice knockouts)}

The value of the \(f_{obs}\) computed from the group (treatments) mean and residual (error) mean squares can be calculated in any statistical software that usually outputs the ANOVA table

\begin{verbatim}
Python:
import pandas as pd
import statsmodels.api as sm
from statsmodels.formula.api import ols
dat = pd.DataFrame({"group": ["Leptin+", "Leptin+",
"Leptin+", "Leptin+", "Leptin+","Leptin+", "Leptin+",
"Leptin+", "Leptin+", "Leptin+",
"Leptin+", "Leptin+", "Leptin+", "Leptin+", "Leptin+",
"Leptin+",  Leptin-", "Leptin-", "Leptin-", "Leptin-",
"Leptin-", "Leptin-", "Leptin-", "Leptin-(sup)",
"Leptin-(sup)", "Leptin-(sup)", "Leptin-(sup)",
"Leptin-(sup)", "Leptin-(sup)", "Leptin-(sup)",
"Leptin-(sup)", "Leptin-(sup)", "Leptin-(sup)"],
"weight": [27.67, 27.4, 25.77, 25.6, 25.03, 25.9, 26.67, 25.6, 
28.93, 31.83, 25.9, 26.3, 27.9, 26.77, 25.83, 20.87,
46.57, 40.43, 41.97, 41.17, 41.57, 46.17, 53.83, 
24.33, 22.37, 26.1, 17.5, 35.17, 25.97, 27.67,
23.37, 31.83, 22.37]})
model = ols('weight ~ group', data=dat).fit()
anova_table = sm.stats.anova_lm(model, typ=2)
print(anova_table)

R:
dat <- data.frame(
group=c("Leptin+", "Leptin+", "Leptin+", "Leptin+",
"Leptin+", "Leptin+", "Leptin+", "Leptin+", "Leptin+",
"Leptin+", "Leptin+", "Leptin+", "Leptin+", "Leptin+",
"Leptin+","Leptin+", "Leptin-", "Leptin-", "Leptin-",
"Leptin-", "Leptin-", "Leptin-", "Leptin-",
"Leptin-(sup)", "Leptin-(sup)", "Leptin-(sup)",
"Leptin-(sup)", "Leptin-(sup)", "Leptin-(sup)",
"Leptin-(sup)","Leptin-(sup)", "Leptin-(sup)",
"Leptin-(sup)"), weight=c(27.67, 27.4, 25.77, 25.6,
25.03, 25.9, 26.67, 25.6, 28.93, 31.83, 25.9, 26.3,
27.9, 26.77, 25.83, 20.87, 46.57, 40.43, 41.97, 41.17,
41.57, 46.17, 53.83, 24.33, 22.37, 26.1, 17.5, 35.17,
25.97, 27.67, 23.37, 31.83, 22.37))
summary(aov(lm(weight~group, data=dat)))
\end{verbatim}

\[
\begin{array}{cccccc}
&\mathbf{Df} & \mathbf{Sum Sq} & \mathbf{Mean Sq} & \mathbf{F value} & \mathbf{Pr(>F)} \\ 
\mathbf{group}    &    2 &1861.5&   930.8 & 63.37 &1.69\times10^{-11}\\
\mathbf{Residuals}&   30  &440.6&    14.7 &  & \\  
\end{array}
\]

In this table \(MST\) is the mean squares for the group (treatment), computed as the corresponding sum of squares divided by the degrees of freedom (\(k-1=2\)).

\[MST=\frac{1}{k-1}SSq_{group}=930.77\]
\(MSE\) is the mean squares for the residuals (error), computed as the corresponding sum of squares divided by the degrees on freedom (\(k(n-1)=30\)), where \(n=(16 + 7 + 10)/3=11\) is the average number of observations in each group

\[MSE=\frac{1}{k(n-1)}SSq_{Residual}=14.69\]

The observed value of the statistic is

\[f_{obs}=\frac{MST}{MSE}=63.373\]

To test the the null hypothesis, we then compute the probability that in a future experiment, if the null hypothesis is true, we observe a higher value than \(63.373\): \(P(F>63.373)\). This probability is the \textbf{upper tailed p-value}

\[pvalue=1-F_{Fisher(2,30)}(63.373)=1.694 \times 10^{-11}\]

which is much lower than \(\alpha=0.05\), suggesting significant differences in at least one group mean. Therefore, we reject the null hypothesis and accept that at least one of the groups has a different mean from the rest.

In this experiment, we observed a significant difference between groups (ANOVA test \(F(2,30)=63.373, pvalue= 1.69 \times 10^{-11}\)), indicating that at least one group has a different mean from the rest. We call it an omnibus test. ANOVA test is simultaneously testing all the groups and by itself it does not tell us which groups or group is different. Clearly, from the plots, the knockout mice has higher weights than the other two groups.

\hypertarget{anova-for-two-groups}{%
\section{ANOVA for Two Groups}\label{anova-for-two-groups}}

ANOVA is often complemented by pairwise comparisons. When there are only two groups, ANOVA is equivalent to the two-sample \(t\)-test. In fact, the \(F\)-test in ANOVA can be seen as a direct generalization of the \(t\)-test to more than two groups.

Let us explicitly compute the observed value of \(F\) when we have only two groups of equal size \(n\):

\[
f_{obs}=\frac{\tfrac{1}{2}(\bar{y}_{A}-\bar{y}_{B})^2}{\tfrac{s_p^2}{n}}
\]

Here, the numerator comes from simplifying \(S_{tr}^2\) for two groups, and \(s_p^2\) is the \textbf{pooled variance}:

\[
s_p^2 = \frac{s_A^2 + s_B^2}{2}
\]

which is obtained from \(S_e^2\) when both groups have the same number of observations.

Taking the square root of \(f_{obs}\) and rearranging, we recover the familiar \(t\)-statistic:

\[
t_{obs} = \sqrt{f_{obs}} = \frac{\bar{y}_{A} - \bar{y}_{B}}{\sqrt{\tfrac{2s_p^2}{n}}}
\]

Thus, the two-sample \(t\)-test for the difference between two balanced groups with equal variances is just a special case of the ANOVA \(F\)-test, with \(f_{obs}=t_{obs}^2\). In other words, ANOVA is a direct generalization of the \(t\)-test to multiple groups.

\textbf{Example (Wild type and knockout mice)}

We are now interested in comparing the weights between knockout mice with supplemented leptin (group B) and wild type mice (group C). The goal is to confirm that lean weight has been restored by exogenous supplementation of the hormone. We perform a \(t\)-test. The observed statistic is

\[t_{obs}=\frac{\bar{y}_A-\bar{y}_B }{\sqrt{\frac{s^2_p}{n_A}+\frac{s^2_p}{n_B}}}=\frac{26.49813-25.668}{\sqrt{\frac{12.66079}{10}+\frac{12.66079}{7}}}=0.5787438\]

using the pooled variance for unbalanced groups.

Therefore, the observed \(F\) is

\[f_{obs}=t_{obs}^2=(0.5787)^2=0.3349\]

The upper-tailed p-value for \(f_{obs}\) using the Fisher distribution is

\[pvalue=1-F_{Fisher,1,18}(0.3349)= 0.56\]

with \(k=2\) and \(n=18\) as the average number of observations per group.

We can confirm that the \(t\)-test and the ANOVA for two groups give the same result.

\begin{verbatim}
R:
t.test(weight~group, data=dat, subset = which(dat$group!="Leptin-"), 
equal.variance=FALSE)

summary(aov(lm(weight~group, data=dat, subset = which(dat$group!="Leptin-"))))
\end{verbatim}

Note that knockout mice with leptin supplementation recovered wild-type weight. When we perform a two-sample \(t\)-test between these two groups, we find that we cannot reject the hypothesis that their expected weights are the same (\(t_{obs}=0.5787\), \(pvalue=0.56\)). Demonstrating that a phenotype caused by knocking out a gene can be rescued by exogenous supplementation provides strong evidence that leptin deficiency can lead to significant weight gain.

\hypertarget{linear-model}{%
\section{Linear model}\label{linear-model}}

The following formulation is useful for integrating different types of analyses. Let us classify the observations of the random variable \(Y\) using the group \(j\) and the particular observation \(i\).

Let us consider first a group \(j\) and that the expected value of \(Y\) in group \(j\) is

\[E(Y_j)=E(Y_{ji}\mid j)=\mu_j.\]

We are making explicit that each condition has its own mean, which we estimate with the sample average \(\bar{Y}_j\). Now we can write

\[\mu_j=\mu + \alpha_j,\]

where \(\mu\) is the overall expected value of all outcomes across groups (\(E(Y)=\mu\)), and \(\alpha_j\) is the amount by which the mean in group \(j\) deviates from \(\mu\). For example, \(\mu_{\text{leptin+}}\) is the expected weight of a wild-type mouse, \(\mu\) is the overall average weight across all groups, and \(\alpha_{\text{leptin+}}\) is the deviation of the wild-type mean from the overall mean. Under the null hypothesis, \(\alpha_j=0\) for all \(j\).

Let us now take a random sample of size \(n_j\) from condition \(j\):

\[M=(Y_{j1}, Y_{j2}, \ldots, Y_{jn_j}).\]

For instance, \(Y_{\text{leptin+},3}\) is the random variable measuring the weight of the 3rd mouse under the wild-type condition, while \(y_{\text{leptin+},3}\) denotes its observed value.

We now assume that we can separate the systematic and random components of the variation and write the \textbf{linear model}:

\[Y_{ji} = \mu_j + \varepsilon_{ji},\]

or equivalently,

\[Y_{ji} = \mu + \alpha_j + \varepsilon_{ji},\]

where \(\varepsilon_{ji}\) is a \textbf{random variable} called the error, which has expected value \(E(\varepsilon_{ji})=0\) and variance \(V(\varepsilon_{ji})=\sigma^2\). It represents the deviation of the individual observation from its group mean. For instance, \(\varepsilon_{\text{leptin+},3}\) is the difference between the weight of the 3rd wild-type mouse and the group mean \(\mu_{\text{leptin+}}\).

In ANOVA, we assume that the variance of \(Y\) is the same across groups (homoscedasticity):

\[V(Y_j)=\sigma^2 \quad \text{for all } j.\]

This formulation makes explicit all the components that contribute to the variation of each observation in the experiment.

\textbf{Sources of variation}

When repeating a random experiment under condition \(j\) for the \(i\)-th time, the outcome \(Y_{ji}\) can be explained as the sum of:

\begin{itemize}
\tightlist
\item
  the overall mean \(\mu\)\\
\item
  the effect of condition \(j\) on the overall mean \(\alpha_j\)\\
\item
  the random error \(\varepsilon_{ji}\) around the group mean \(\mu_j\), with variance \(\sigma^2\)
\end{itemize}

\textbf{Hypothesis contrast}

In this formulation the hypothesis contrast can be written as:

\begin{itemize}
\tightlist
\item
  \(H_0:\ \alpha_j=0 \ \text{for all } j\). That is, there are no differences between the group means and the overall mean \(\mu\).\\
\item
  \(H_1: \alpha_j \neq 0\) for at least one \(j\). At least one group mean differs from the overall mean \(\mu\).
\end{itemize}

We again use the \(F\) statistic to decide between the two hypotheses.

\textbf{Example (Speed of light)}

In 1887, Michelson and Morley attempted to explain why stars appear to slightly change their positions in the sky depending on the Earth's motion around the Sun \citep{michelson1887ether}. If the Earth were moving away from a star, its light would arrive later, creating a small apparent shift in its position when compared to the stars toward which the Earth is moving. If light behaves as a wave, then the Earth would be moving relative to the medium that supports the wave --- the so-called ether. However, the angular deviations predicted by this theory were much larger than those actually observed.

To address this discrepancy, Michelson and Morley devised their famous experiment. They mounted a device on the edge of a rotating platform and sent a light beam into it. The beam was split at the center of the platform into two perpendicular rays. Each ray was reflected by mirrors placed at the edges of the apparatus and then recombined at the center, producing an interference pattern directed back toward the starting point. The interference generated a bright fringe whose position could be precisely measured. Crucially, the position of this fringe depended on the relative distances traveled by the two rays, which in turn should have been affected by the direction and speed of the Earth's motion through the ether.

They reasoned that when the apparatus was rotated to an angle \(r\) such that one ray aligned with the Earth's motion through the ether, the other ray would experience the maximum relative path difference. In that configuration, the displacement of the interference fringe would also reach its maximum at \(r\).

The apparatus, mounted on a mercury-supported rotating table, was turned through 16 different orientations (with the final position coinciding with the first) and the measurements repeated over three consecutive days. The outcomes, expressed in units of screw divisions (each corresponding to 50 times the wavelength of light), recorded the observed fringe deviations.

\[
\begin{array}{cccc}
\mathbf{Angle} & \mathbf{July 8} & \mathbf{July 9} & \mathbf{July 10} \\ 
1&   44.7&  57.4&   27.3\\
2&   44.0&  57.3&   23.5\\
3&   43.5&  58.2&   22.0\\
4&   39.7&  59.2&   19.3\\
5&   35.2&  58.7&   19.2\\
6&   34.7&  60.2&   19.3\\
7&   34.3&  60.8&   18.7\\
8&   32.5&  62.2&   18.8\\
9&   28.2&  61.5&   16.2\\
10&  26.2&  63.3&   14.3\\
11&  23.8&  65.8&   13.3\\
12&  23.2&  67.3&   12.8\\
13&  20.3&  69.7&   13.3\\
14&  18.7&  70.7&   12.3\\
15&  17.5&  73.0&   10.2\\
16&  16.8&  70.2&    7.3\\
17&  13.7&  72.2&    6.5\\
\end{array}
\]

If we treat the angles as different experimental conditions, each measured three times (once per day), then we can write

\[Y_{angle, day} = \mu + \alpha_{angle} +\varepsilon_{angle,day}\]

where \(E(Y_{\text{angle}, \text{day}})=\mu_{\text{angle}}=\mu + \alpha_{\text{angle}}\), and \(\text{day}\in\{\text{July 8}\), \(\text{July 9}\), \(\text{July 10}\}\). Our goal is to determine whether at least one \(\alpha_{\text{angle}}\) differs from the others, thereby rejecting the null hypothesis using ANOVA. When the board is set at a particular angle \(r\) such that the path difference between the two rays is maximal, the deviation of the fringe should also be maximal, and thus so should \(\alpha_r\).

However, as the figure shows, the experiment displayed large systematic errors that do not conform to an ANOVA model. First, within a single day, the measurements exhibited linear trends: the shift increased steadily with the angle. Second, across days, the results were systematically different: the measurements were consistently highest on July 9 and lowest on July 10.

\includegraphics{_main_files/figure-latex/unnamed-chunk-187-1.pdf}

To address the first error, the technique of the time was to compute the difference between the 17th and the 1st angle (which coincide because the board completes a full rotation) and divide it by 16. Each of these accumulated errors was then successively subtracted from the original measurements, producing a normalized outcome \(Y^*_{\text{angle}, i}\). The modern equivalent of this correction is to fit a regression line and use the residuals, as we will see in the next section.

For the second error, we average the measurements across days and explicitly include the day effect in the model of the normalized outcomes:

\[
Y^*_{\text{angle}, \,\text{day}} = \mu^* + \alpha^*_{\text{angle}} + \beta^*_{\text{day}} + \varepsilon^*_{\text{angle},\text{day}} .
\]

A further corrected outcome, obtained by subtracting the daily mean \(\beta^*_{\text{day}}\), satisfies the ANOVA model we want to test:

\[
Y^{**}_{\text{angle}, \,\text{day}} \;=\; Y^{*}_{\text{angle}, \,\text{day}} - \beta^*_{\text{day}} \;=\; \mu^* + \alpha^*_{\text{angle}} + \varepsilon^*_{\text{angle},\text{day}} .
\]

The bar plot shows that the corrected normalized outcomes at a given angle can now be considered as replications of the same experiment across three days. Michelson and Morley focused their analysis on half a rotation of the board (up to angle 8), resembling the pattern shown for the noon measurements, although they did not report in detail how they treated systematic errors \citep{handschy1982reexamination}.

In the plot, we can identify all the components of the model. The estimate of \(\mu^*\) is represented by the dotted line around zero, as the daily variation \(\beta^*_{\text{day}}\) has been removed. The effect of the angle \(\alpha^*_{\text{angle}}\) is shown by the bars, and the within-angle variability is reflected in the scatter of the individual points. The observed values of the model (denoted with lowercase) are

\[
y^{**}_{\text{angle}, \,\text{day}} \;=\; \hat{\mu}^* + \hat{\alpha}^*_{\text{angle}} + r^*_{\text{angle}, \,\text{day}} ,
\]

where \(r^*_{\text{angle}, \,\text{day}}\) are the residuals (the observed errors), \(\hat{\mu}^*\) is the overall mean of the corrected normalized observations, and \(\hat{\alpha}^*_{\text{angle}}\) are the estimated deviations at a fixed angle. Since the corrections for systematic errors were applied uniformly across all angles, they do not affect the hypothesis test, whose aim is to detect a privileged direction of Earth's movement relative to the ether, and thus a maximal fringe shift at a specific angle.

The ANOVA table for the normalized data is

\[
\begin{array}{cccccc}
&\mathbf{Df} & \mathbf{Sum Sq} & \mathbf{Mean Sq} & \mathbf{F value} & \mathbf{Pr(>F)} \\ 
\mathbf{Angle}     &     1  & 0.06  & 0.0633    & 0.04  & 0.842 \\
\mathbf{Residuals} &   22 & 34.44 & 1.5653 & &\\
\end{array}
\]

This ANOVA table shows that the data are consistent with the null hypothesis (\(pvalue = 0.842\)), meaning that all angles have the same average fringe shift after correcting for systematic errors. Thus, the experiment supports a model in which there is no privileged direction along which the speed of light could be reduced.

Based solely on the maximum observed fringe shift, Michelson and Morley's original analysis concluded that the Earth's velocity relative to the ether was undetectable with their apparatus, with an estimated measurement error of about one-sixth of the Earth's orbital speed. Later, Einstein argued that their results are consistent with the principle that the speed of light is invariant and constitutes the maximum possible speed, and he derived the theoretical consequences in the theory of special relativity.

\hypertarget{way-anova}{%
\section{2-way ANOVA}\label{way-anova}}

The ANOVA approach allows the analysis of the joint effects of \textbf{two} or more different types of conditions, each with potentially different numbers of groups. In the Michelson and Morley experiment, they had six unreported measurements for each angle on each day, and they reported only the averages. In the original (unavailable) data, not only the angle but also the day could be considered as two different types of conditions, each with its own levels or groups.

\textbf{Example (Leptin knockout)}

Let us consider again the experiment of Ramos-Lobo on leptin knockout mice \citep{Ramos-Lobo2019}. In this experiment, female mice were also included, introducing an additional condition: sex. Since weight depends on the sex of the animal, we may ask whether weight is still statistically dependent on leptin deficiency even when we further condition on sex.

Here we are asking two simultaneous questions:

\begin{itemize}
\tightlist
\item
  Is there an effect of \textbf{sex} on the weight of the mice?\\
\item
  Is there an effect of \textbf{leptin deficiency} on the weight of the mice?
\end{itemize}

\hypertarget{data-5}{%
\section{Data}\label{data-5}}

We introduce a conditional outcome. One random experiment --- that is, one mouse --- has three measurements: \((leptin, sex, weight)\).

Categorical variable for the first condition (leptin status) with two groups:

\begin{itemize}
\tightlist
\item
  \(leptin \in \{\text{KOplus: A}, \text{knockout: B}\}\)
\end{itemize}

Categorical variable for the second condition (sex) with two groups:

\begin{itemize}
\tightlist
\item
  \(sex \in \{\text{male: a}, \text{female: b}\}\)
\end{itemize}

Outcome of interest (continuous variable):

\begin{itemize}
\tightlist
\item
  \(weight \in (20, 60)\)
\end{itemize}

The data looks like

\[
\begin{array}{cccc}
\mathbf{Mouse} & \mathbf{Leptin} & \mathbf{Sex} & \mathbf{Weigth} \\ 
1&\text{Leptin-}&\text{M}&46.57 \\
2&\text{Leptin-}&\text{M}&40.43 \\
3&\text{Leptin-}&\text{M}&41.97 \\
4&\text{Leptin-}&\text{M}&41.17 \\
5&\text{Leptin-}&\text{M}&41.57 \\
6&\text{Leptin-}&\text{M}&46.17 \\
7&\text{Leptin-}&\text{M}&53.83 \\
8 & \text{Leptin-(sup)}&\text{M}&24.33 \\
9 & \text{Leptin-(sup)}&\text{M}&22.37 \\
10& \text{Leptin-(sup)}&\text{M}&26.10 \\
11& \text{Leptin-(sup)}&\text{M}&17.50 \\
12& \text{Leptin-(sup)}&\text{M}&35.17 \\
13& \text{Leptin-(sup)}&\text{M}&25.97 \\
14& \text{Leptin-(sup)}&\text{M}&27.67 \\
15& \text{Leptin-(sup)}&\text{M}&23.37 \\
16& \text{Leptin-(sup)}&\text{M}&31.83 \\
17& \text{Leptin-(sup)}&\text{M}&22.37 \\
18&\text{Leptin-}&\text{F}&  65.80 \\
19&\text{Leptin-}&\text{F}&  51.40 \\
20&\text{Leptin-}&\text{F}&  54.60 \\
21&\text{Leptin-}&\text{F}&  48.30 \\
22&\text{Leptin-}&\text{F}&  50.60 \\
23&\text{Leptin-}&\text{F}&  48.90 \\
24&\text{Leptin-}&\text{F}&  51.20 \\
25&\text{Leptin-}&\text{F}&  46.80 \\
26&\text{Leptin-}&\text{F}&  50.90 \\
27&\text{Leptin-}&\text{F}&  42.70 \\
28& \text{Leptin-(sup)}&\text{F}&  28.70 \\
29& \text{Leptin-(sup)}&\text{F}&  25.60 \\
30& \text{Leptin-(sup)}&\text{F}&  26.40 \\
31& \text{Leptin-(sup)}&\text{F}&  22.90 \\
32& \text{Leptin-(sup)}&\text{F}&  30.00 \\
33& \text{Leptin-(sup)}&\text{F}&  29.70 \\
34& \text{Leptin-(sup)}&\text{F}&  26.10 \\
35& \text{Leptin-(sup)}&\text{F}&  21.40 \\
36& \text{Leptin-(sup)}&\text{F}&  29.50 \\
37 &\text{Leptin-(sup)}&\text{F}&  21.90 \\
38 &\text{Leptin-(sup)}&\text{F}&  23.70 \\
39 &\text{Leptin-(sup)}&\text{F}&  21.00 \\
\end{array}
\]

Note that we will examine the association with sex only for the knockout mice that differ in leptin supplementation.

\hypertarget{modeling-residuals}{%
\section{Modeling residuals}\label{modeling-residuals}}

Using the following model, we want to test for significant differences in the expected weight between male and female mice, ignoring the leptin condition:

\[Y_{sex,i}= \mu + \alpha_{sex} + \varepsilon_{sex,i}\]
From the ANOVA table

\[
\begin{array}{cccccc}
&\mathbf{Df} & \mathbf{Sum Sq} & \mathbf{Mean Sq} & \mathbf{F value} & \mathbf{Pr(>F)} \\ 
\mathbf{sex}       & 1.0 &  134.9 & 134.9&  0.85  & 0.36 \\
\mathbf{Residual}  & 37.0  & 5844.8 & 157.9  &     &      \end{array}
\]

we see that there is no significant effect of sex on the weight of the mice when the leptin groups are not considered. From the bar plot, we also observe that the confidence intervals for each sex overlap, indicating that we cannot reject the null hypothesis that the mean weights are equal between sexes.

\includegraphics{_main_files/figure-latex/unnamed-chunk-190-1.pdf}

We see, however, that the points are clustered by leptin group, suggesting that we should adjust for differences in leptin status before testing for differences by sex. It is possible that, after removing the effect of leptin, the mean weight of males may be lower than that of females.

The way to remove the effect of leptin is to perform an ANOVA of mouse weight on the leptin group and then obtain the residuals for each observation. These residuals represent the \textbf{observed} values of the error term \(\varepsilon_{leptin,i}\) and are computed as:

\[
r_{leptin,i} = y_{leptin,i} - \bar{y}_{leptin}
\]

That is, we subtract the mean of the leptin group from each observation \(i\) within that group. These residuals can then be used to test the effect of sex independent of leptin status.

\begin{verbatim}
Python:
model1 = sm.OLS.from_formula('weight ~ leptin', data=filtered_data).fit()

model1.resid

R:
dat$residual.weight <- summary(lm(weight ~ leptin, data=dat))$residuals
\end{verbatim}

Then we can perform another ANOVA using the leptin-corrected residuals to test for differences by sex. Let the leptin-corrected outcome be:

\[
Y^*_{leptin,sex,i} = Y_{leptin,sex,i} - \bar{Y}_{leptin} = \mu^* + \alpha^*_{sex} + \varepsilon^*_{sex,i}
\]

where \(\bar{Y}_{leptin}\) is the mean weight of the leptin group for each observation, \(\alpha^*_{sex}\) is the effect of sex after correcting for leptin, and \(\varepsilon^*_{sex,i}\) is the random error variable within sex.

The ANOVA table for weight corrected for leptin differences is:

\[
\begin{array}{cccccc}
&\mathbf{Df} & \mathbf{Sum Sq} & \mathbf{Mean Sq} & \mathbf{F value} & \mathbf{Pr(>F)} \\ 
\mathbf{sex}       & 1 & 73.94 &  73.94  & 2.956 & 0.093 \\
\mathbf{Residual}  & 37 & 925.49 & 25.013  &     & \\      \end{array}
\]

The lower weight in males is almost significant for the leptin-corrected residuals. However, from this model we cannot determine the contribution of the leptin group to the overall variability of the data. Additionally, if the sample size \(n\) is large, it is possible that even if the null hypothesis is true, the means of each sex may still not be close to the grand mean due to a significant effect of leptin.

The bar plot of the residuals shows that we have effectively adjusted for the differences in leptin, as the dots for males and females are now intermixed, indicating that the leptin effect has been removed.

\includegraphics{_main_files/figure-latex/unnamed-chunk-193-1.pdf}

We can generalize ANOVA to explain the simultaneous contributions of sex and leptin groups.

\hypertarget{way-anova-linear-model}{%
\section{2-way ANOVA linear model}\label{way-anova-linear-model}}

The ANOVA approach allows for the simultaneous analysis of two factors, each with multiple groups or levels. For simplicity, we discuss a balanced design, where we have \(n\) repetitions of the random experiment in each condition (or the average number of repetitions in the general case).

Consider the \textbf{linear model}:

\[
Y_{jri} = \mu + \alpha_j + \beta_r + \varepsilon_{jri}
\]

for the \(i\)-th repetition of the random experiment (\(i = 1, \dots, n\)) in condition \(j\) of factor 1 and condition \(r\) of factor 2, with:

\begin{itemize}
\tightlist
\item
  \textbf{Grand mean}:
\end{itemize}

\[
E(Y_{jri}) = \mu
\]

which is the expected value of all observations, estimated by the overall average \(\bar{Y}\).

\begin{itemize}
\tightlist
\item
  \textbf{Random error}:
\end{itemize}

\[
\varepsilon_{jri}
\]

a \textbf{random variable} with mean \(E(\varepsilon_{jri}) = 0\) and variance \(V(\varepsilon_{jri}) = \sigma^2\).

\begin{itemize}
\tightlist
\item
  \textbf{Deviations of factor 1 (leptin) from the grand mean}:
\end{itemize}

\[
\alpha_j = \mu_{j\cdot} - \mu
\]

for each group \(j \in \{1, \dots, k\}\), with \(\sum_j \alpha_j = 0\).

\begin{itemize}
\tightlist
\item
  \textbf{Deviations of factor 2 (sex) from the grand mean}:
\end{itemize}

\[
\beta_r = \mu_{\cdot r} - \mu
\]

for each group \(r \in \{1, \dots, m\}\), with \(\sum_r \beta_r = 0\).

Here, the dot indicates that we ignore the other factor when computing the mean.

The model can also be expressed as:

\[
Y_{jri} = \mu_{jr} + \varepsilon_{jri}, \quad \text{where } \mu_{jr} = \mu_{j\cdot} + \mu_{\cdot r} - \mu
\]

is the mean in the \textbf{condition} defined by group \(j\) of factor 1 \textbf{and} group \(r\) of factor 2. For example, the condition ``male and leptin-'' is one of these groups. In our leptin example, there are four such groups corresponding to all combinations of sex and leptin status.

\hypertarget{hypothesis-tests}{%
\section{Hypothesis tests}\label{hypothesis-tests}}

ANOVA allows testing the effects of the two factors simultaneously:

\textbf{Factor 1 (leptin):}

\begin{itemize}
\tightlist
\item
  \(H_0: \alpha_1 = \alpha_2 = \dots = \alpha_k = 0\) --- no differences between group means for factor 1.
\item
  \(H_1\): at least one \(\alpha_j \neq 0\).
\end{itemize}

\textbf{Factor 2 (sex):}

\begin{itemize}
\tightlist
\item
  \(H_0: \beta_1 = \beta_2 = \dots = \beta_m = 0\) --- no differences between group means for factor 2.
\item
  \(H_1\): at least one \(\beta_r \neq 0\).
\end{itemize}

\hypertarget{variance-components}{%
\section{Variance components}\label{variance-components}}

To evaluate each hypothesis contrast, we need appropriate test statistics. We can estimate the dispersion of the outcomes from their means in each \textbf{condition}, determined by both factors.

For instance, the mean for knockout (KO) males (M) is

\[
\mu_{KO,M} = \mu_{\cdot M} + \mu_{KO\cdot} - \mu
\]

and can be estimated by the sample averages:

\[
\hat{\mu}_{KO,M} = \bar{Y}_{\cdot M} + \bar{Y}_{KO\cdot} - \bar{Y}
\]

These estimated means correspond to the dots in the following plot.

\includegraphics{_main_files/figure-latex/unnamed-chunk-194-1.pdf}
The dispersion of the knockout male observations around the estimated mean \(\hat{\mu}_{KO,M}\) is given by the sample variance:

\[
S^2_{KO,M} = \frac{1}{n-1} \sum_{i=1}^n \left(Y_{KO,M,i} - \hat{\mu}_{KO,M}\right)^2
\]

If we assume that the variance of weight is the same across all sex--leptin conditions, \(\sigma^2\), we can estimate it with the pooled sample variance:

\[
S_e^2 = \frac{1}{km} \sum_{j=1}^k \sum_{r=1}^m S^2_{jr}
\]

which sums the within-condition variances across all sex--leptin groups. \(S_e^2\) is an estimator of \(\sigma^2\).

We can also estimate the variance of the averages in each group of \textbf{factor 1} (e.g., leptin) relative to the overall mean:

\[
S^2_{tr1} = \frac{1}{k-1} \sum_{j=1}^k \left(\bar{Y}_{j\cdot} - \bar{Y}\right)^2
\]

This is an estimator of \(\sigma^2 / (nm)\) under the null hypothesis. The corresponding \(F\) statistic is

\[
F_1 = \frac{nm S^2_{tr1}}{S_e^2} \sim F(k-1, (kmn-1)-(k-1)-(m-1))
\]

If the null hypothesis is true (no differences between factor 1 group means), \(F_1\) will be close to 1. If the alternative is true, at least one group mean differs from the grand mean, and \(F_1\) will be greater than 1.

Similarly, for \textbf{factor 2} (e.g., sex), we define

\[
S^2_{tr2} = \frac{1}{m-1} \sum_{r=1}^m \left(\bar{Y}_{\cdot r} - \bar{Y}\right)^2
\]

and

\[
F_2 = \frac{nk S^2_{tr2}}{S_e^2} \sim F(m-1, (kmn-1)-(k-1)-(m-1))
\]

Observed values of \(F_2\) far from 1 indicate significant differences between the means of factor 2, leading to rejection of the null hypothesis.

All of this is summarized in a \textbf{two-way ANOVA table}:

\begin{verbatim}
import pandas as pd
import statsmodels.api as sm
from statsmodels.formula.api import ols
dat = pd.DataFrame({
    "Mouse": range(1, 40),
    "Leptin": [
        "Leptin-", "Leptin-", ...
    ],
    "Sex": [
        "M","M",...
    ],
    "Weight": [
        46.57, 40.43, 41.97, ...
    ]
})

# Two-way ANOVA
model = ols('Weight ~ Sex + Leptin', data=dat).fit()
anova_table = sm.stats.anova_lm(model, typ=2)
print(anova_table)


R: 
dat <- data.frame(
  Mouse = 1:39,
  Leptin = c(
    rep("Leptin-", 7),
    rep("Leptin-(sup)", 10),
    rep("Leptin-", 10),
    rep("Leptin-(sup)", 12)
  ),
  Sex = c(
    rep("M", 17),
    rep("F", 22)
  ),
  Weight = c(
    46.57, 40.43, 41.97, 41.17, 41.57, 46.17, 53.83,
    24.33, 22.37, 26.10, 17.50, 35.17, 25.97, 27.67, 23.37, 31.83, 22.37,
    65.80, 51.40, 54.60, 48.30, 50.60, 48.90, 51.20, 46.80, 50.90, 42.70,
    28.70, 25.60, 26.40, 22.90, 30.00, 29.70, 26.10, 21.40, 29.50, 21.90,
    23.70, 21.00
  )
)

# Two-way ANOVA 
summary(aov(Weight ~ Sex + Leptin, data = dat))
\end{verbatim}

\[ \begin{array}{lccccc} &\mathbf{Df} & \mathbf{Sum\ Sq} & \mathbf{Mean\ Sq} & \mathbf{F\ value} & \mathbf{Pr(>F)} \\ \mathbf{sex} & 1 & 134.98 & 134.98 & 5.251 & 0.0279 \\ \mathbf{group} & 1 & 4919.51 & 4919.51 & 191.389 & 5.59 \times 10^{-16} \\ \mathbf{Residual} & 36 & 925.35 & 25.704 & & \\ \end{array} \]

For example, the observed value of \(F_1\) is

\[
f_{1,obs} = \frac{MST_1}{MSE} 
= \frac{\frac{1}{k-1} SSq_{Sex}}{\frac{1}{(kmn-1)-(k-1)-(m-1)} SSq_{Residual}} 
= 5.251
\]

This value is sufficiently large, indicating rejection of the null hypothesis (\(pvalue = 0.028 < 0.05\)). Therefore, there is a \textbf{significant effect of sex} when correcting for leptin differences.

From the dispersion plot above, males appear to have lower weights than females, both in the leptin-supplemented (sup) and non-supplemented (KO) groups.

\hypertarget{way-anova-with-interaction}{%
\section{2-way ANOVA with interaction}\label{way-anova-with-interaction}}

In the previous ANOVA model, we computed the means at each sex-leptin condition \((j,r)\) as the sum of the contributions of group \(j\) from factor 1 (sex) and group \(r\) from factor 2 (leptin) to the overall mean:

\[
\mu_{jr} = \alpha_{j} + \beta_{r} + \mu
\]

However, the distribution of the data around these means is not symmetrical, because these are not the means computed \textbf{within each condition}.

When we take weights conditioned on each leptin-by-sex group, we observe:

\begin{itemize}
\tightlist
\item
  \(n_{sup,F} = 12\) supplemented leptin \textbf{female} mice had a weight average of \(\bar{y}_{F,plus} = 25.575\).\\
\item
  \(n_{sup,M} = 10\) supplemented leptin \textbf{male} mice had a weight average of \(\bar{y}_{M,plus} = 25.668\).\\
\item
  \(n_{KO,F} = 10\) control \textbf{female} mice had a weight average of \(\bar{y}_{F,KO} = 51.120\).\\
\item
  \(n_{KO,M} = 7\) leptin KO \textbf{male} mice had a weight average of \(\bar{y}_{M,KO} = 44.530\).
\end{itemize}

These condition-specific means can be visualized with a barplot including confidence intervals.

\includegraphics{_main_files/figure-latex/unnamed-chunk-195-1.pdf}

From this plot, we can see that most of the difference between male and female weights comes from the knock-out condition. In the leptin-supplemented groups, the sexes seem to have similar weights. However, the differences between sexes appear larger in the knock-out mice than in the supplemented-with-leptin mice.

How can we test the hypothesis that a particular combination of conditions has an effect on the weight of mice? For instance, how can we test whether male knock-outs are less sensitive to weight gain than female knock-outs?

The apparently lower-than-expected weight gain of leptin- (KO) males suggests a specific interaction between leptin KO and being male.

\hypertarget{linear-model-1}{%
\section{Linear model}\label{linear-model-1}}

We can then formulate the \textbf{linear model} with an \textbf{interaction term} that accounts for the specific contribution of each condition given by the groups \((j,r)\) of each factor:

\[
Y_{jri} = \mu + \alpha_j + \beta_r + (\alpha\beta)_{jr} + \varepsilon_{jri}
\]

Here, each observation in a condition \((j,r)\) has a specific expected value:

\[
E(Y \mid j,r) = \mu + \alpha_j + \beta_r + (\alpha\beta)_{jr}
\]

This is the mean \textbf{conditioned} on the groups \((j,r)\). The condition mean can be computed as the sum of the independent contributions of each group (\(\mu_{jr}\)) plus the specific contribution of the condition \((\alpha\beta)_{jr}\):

\[
E(Y \mid j,r) = \mu_{jr} + (\alpha\beta)_{jr}
\]

\hypertarget{hypothesis-tests-1}{%
\section{Hypothesis tests}\label{hypothesis-tests-1}}

This ANOVA model allows testing three null hypotheses:

\textbf{First:}

\[
H_0: \alpha_1 = \alpha_2 = \dots = \alpha_k = 0
\]

There is no difference between group means in the first factor.

\textbf{Second:}

\[
H_0: \beta_1 = \beta_2 = \dots = \beta_m = 0
\]

There is no difference between group means in the second factor.

\textbf{Third:}

\[
H_0: (\alpha\beta)_{jr} = 0
\]

There is no difference between specific condition means.

The alternatives are that at least one of the terms is different from 0.

\hypertarget{variance-components-1}{%
\section{Variance components}\label{variance-components-1}}

The first two hypothesis contrasts can be tested as shown in the previous section.

To test the first contrast, we define the \(F_1\) statistic that measures the dispersion of the group means of factor 1 relative to the dispersion of the residuals:

\[
F_1 = \frac{MS_1}{MSE} \sim F(k-1, (n-1) - (k-1) - (m-1))
\]

To test the second contrast, we define the corresponding \(F_2\) statistic for factor 2:

\[
F_2 = \frac{MS_2}{MSE} \sim F(m-1, (n-1) - (k-1) - (m-1))
\]

Finally, to test the hypothesis for the interaction term, we compute the \(F_I\) statistic, given by the dispersion of the group averages relative to the overall average with respect to the dispersion of the residuals:

\[
F_I = \frac{MS_I}{MSE} \sim F\Big((k-1)(m-1), (nkm-1) - (k-1) - (m-1) - (k-1)(m-1)\Big)
\]

These statistics are summarized in the ANOVA table with interaction, which can be obtained with the following code:

\begin{verbatim}
Python:

model = ols('weight ~ sex*group', data=filtered_data).fit()
anova_table = sm.stats.anova_lm(model, typ=1)
print(anova_table)

R:
summary(aov(Weight ~ Sex*Leptin, data = dat))
\end{verbatim}

\[ \begin{array}{lccccc} &\mathbf{Df} & \mathbf{Sum\ Sq} & \mathbf{Mean\ Sq} & \mathbf{F\ value} & \mathbf{Pr(>F)} \\ \mathbf{sex} & 1 & 134.98 & 134.98 & 5.757 & 0.02188 \\ \mathbf{group} & 1 & 4919.51 & 4919.51 & 209.836 & 2.37 \times 10^{-16} \\ \mathbf{sex:group} & 1 & 104.79 & 104.79 & 4.470 & 0.04169 \\ \mathbf{Residual} & 35 & 820.56 & 23.445 & & \\ \end{array} \]

As we observed from the bar plots, the statistical inference confirms that there are significant differences in weight between leptin groups, significant differences between sexes, and significant interactions between sex and leptin groups. In particular, being male reduces weight in knock-out mice, whereas it increases weight in mice with leptin supplementation. In other words, the effect of sex changes depending on the leptin context.

Interactions are better represented in plots displaying the means conditioned on both factors. Significant interactions are indicated by non-parallel lines in such plots.

\includegraphics{_main_files/figure-latex/unnamed-chunk-197-1.pdf}

\hypertarget{questions-13}{%
\section{Questions}\label{questions-13}}

\textbf{1)} We test ANOVA when we have

\textbf{\(\qquad\)a:} several continuous random variables;
\textbf{\(\qquad\)b:} several categorical variables;
\textbf{\(\qquad\)c:} several continuous random variables and several categorical variables;
\textbf{\(\qquad\)d:} a continuous random variable and several categorical outcomes;

\textbf{2)} The statistic \(F=\frac{MST}{MSE}\) is

\textbf{\(\qquad\)a:} the estimation of \(\sigma^2\) by the residuals;
\textbf{\(\qquad\)b:} the estimation of \(\sigma^2\) by the group means;
\textbf{\(\qquad\)c:} the estimation of \(\sigma^2\) by the group means minus by the estimation of \(\sigma^2\) by the residuals;
\textbf{\(\qquad\)d:} the estimation of \(\sigma^2\) by the group means divided by the estimation of \(\sigma^2\) by the residuals

\textbf{3)} We accept the null hypothesis that the groups have equal means when the value of \(F=\frac{MST}{MSE}\) is

\textbf{\(\qquad\)a:} close to 1;
\textbf{\(\qquad\)b:} different from 1;
\textbf{\(\qquad\)c:} small;
\textbf{\(\qquad\)d:} large;

\textbf{4)} ANOVA assumes that the observations in each condition have

\textbf{\(\qquad\)a:} the same mean \(\mu\);
\textbf{\(\qquad\)b:} the same variance \(\sigma^2\);
\textbf{\(\qquad\)c:} different variances;
\textbf{\(\qquad\)d:} different means;

\textbf{5)} The interaction term in a an ANOVA tests that

\textbf{\(\qquad\)a:} the group means of factor 1 are different from the group means of factor 2 ;
\textbf{\(\qquad\)b:} the mean of a group of factor 1 is different from the mean of the other groups of factor 1;
\textbf{\(\qquad\)c:} the means of the groups across both factors are not added contributions of each factor;
\textbf{\(\qquad\)d:} the means of the groups across both factors are different;

\hypertarget{practice-7}{%
\section{Practice}\label{practice-7}}

Load leptin data (
\url{https://alejandro-isglobal.github.io/SDA/data/dataleptin.txt})

\begin{itemize}
\item
  Use a t-test to test the hypothesis that the weight between sexes is different.
\item
  Use an ANOVA to test that the weight between sexes is different.
\item
  Extract residuals from the ANOVA on leptin group and do a second anova on sex.
\item
  Use an ANOVA on sex and group to test the whether sex and leptin groups are significantly associated with weight
\item
  Include an interaction term in the previous ANOVA
\item
  Make a bar plot of weight across all conditions given by both factors
\end{itemize}

\href{https://colab.research.google.com/drive/1K2SlNEd5hNn3lI4XnKsgj1tDeXVyprVE?usp=sharing}{Solutions}

\hypertarget{regression-and-correlation}{%
\chapter{Regression and Correlation}\label{regression-and-correlation}}

When analyzing a continuous variable, the conditions under which observations are made may also be continuous. Unlike categorical groups, clearly defined repeating conditions are not available, as a group is defined by a continuous variable whose values cannot be repeated exactly. For example, consider measuring the velocity of a galaxy and its distance from Earth. We might want to know whether the velocity depends on distance---that is, whether higher distances correspond to higher velocities, as evidence of the expansion of the Universe.

In this chapter, we will explore how to test the statistical dependence between two continuous random variables. We will introduce \textbf{correlation} as a parameter of a bivariate normal distribution and perform statistical tests on correlation, illustrating Hubble's law.

We will also discuss how to assess statistical dependence when one variable is considered the outcome (dependent) and the other the predictor (independent). We will introduce \textbf{regression analysis} to test linear relationships, using, for example, leptin levels in blood as explained by fat mass. Finally, we will discuss \textbf{multiple regression}, which allows adjusting associations for additional variables that may confound the relationship. For instance, we will examine how monthly variations in atmospheric CO2 can obscure the yearly trend first reported by Keeling in his groundbreaking 1960s work.

Sex differences are often important modulatory factors in biomedical research. We will show how the effect of fat mass on leptin levels differs between sexes. In models relating a continuous outcome to a continuous predictor, interactions with additional variables can explain important variation in the outcome. We will conclude by introducing regression analysis with interactions, highlighting how these models capture more complex relationships between variables.

\hypertarget{correlations}{%
\section{Correlations}\label{correlations}}

In 1929, Edwin Hubble reported the distances and velocities of 22 nebulae, providing the first evidence for the expansion of the Universe \citep{Hubble1929_distance_velocity}. He measured distances using different techniques depending on the scale. For relatively nearby galaxies, he relied on supergiant pulsating stars, whose pulsation periods are related to their intrinsic luminosity. By comparing the observed luminosity to the predicted luminosity, Hubble could estimate the distance to the galaxy containing the star. Velocities were determined from the redshift of spectral lines: the faster a galaxy moves away, the more its light is shifted toward the red.

From a statistical perspective, Hubble's data can be seen as a simultaneous measurement of two continuous variables---distance and velocity---for each galaxy. This dataset naturally leads to questions about the relationship between these variables, providing an early example of the kind of correlation and regression analysis that we discuss in this chapter.

Hubble's random experiment is the simultaneous measurement of two continuous outcomes on a galaxy:

\begin{itemize}
\tightlist
\item
  The velocity of a galaxy, which has a probability density
\end{itemize}

\[Y \sim N(\mu_y, \sigma_y^2)\]

\begin{itemize}
\tightlist
\item
  The distance of the galaxy, with density
\end{itemize}

\[X \sim N(\mu_x, \sigma_x^2)\]

\hypertarget{data-6}{%
\section{Data}\label{data-6}}

One random experiment in Hubble's study has two outcomes: \((velocity, distance)\).

Continuous variable (outcome of interest)

\begin{itemize}
\tightlist
\item
  \(velocity \in (-25.0, 1800.0)\) km/s
\end{itemize}

Continuous variable (explanatory variable)

\begin{itemize}
\tightlist
\item
  \(fatmass \in (0.62, 3.450)\) Mpc
\end{itemize}

Repeating the experiment \(n=21\) times, the data for look like

\[
  \begin{array}{ccc}
  \mathbf{Object} & \mathbf{Velocity} & \mathbf{Distance} \\ 
  278 & 650 & 1.52 \\
  584 & 1800 & 3.45 \\
  936 & 1300 & 2.37 \\
  1023 & 300 & 0.62 \\
  1700 & 800 & 1.16 \\
  2681 & 700 & 1.42 \\
  2683 & 400 & 0.67 \\
  2841 & 600 & 1.24 \\
  3034 & 290 & 0.79 \\
  3115 & 600 & 1.00 \\
  3368 & 940 & 1.74 \\
  3379 & 810 & 1.49 \\
  3489 & 600 & 1.10 \\
  3521 & 730 & 1.27 \\
  3623 & 800 & 1.53 \\
  4111 & 800 & 1.79 \\
  4526 & 580 & 1.20 \\
  4565 & 1100 & 2.35 \\
  4594 & 1140 & 2.23 \\
  5005 & 900 & 2.06 \\
  5866 & 650 & 1.73 \\
  \end{array}
\]

and the plot of velocity against distance is

\includegraphics{_main_files/figure-latex/unnamed-chunk-198-1.pdf}

We see a clear linear relationship between the outcomes. How can we test for it? We need to formulate a hypothesis test on a parameter of a join probability distribution that represents this linear relationship, which we will call the correlation between the variables.

\hypertarget{normal-bivariate}{%
\section{Normal bivariate}\label{normal-bivariate}}

Let us consider that one random experiment of the study consists on drawing a pair of measurements of both the distance \textbf{and} and the velocity of a a single galaxy. We, therefore, have a random variable that is a pair of measurements that follows a probability distribution. We will assume that the distribution is the \textbf{two-dimensional} version of a normal probability density

\[(Y, X) \sim N(\mu_y, \sigma^2_y, \mu_x, \sigma^2_x, \rho)\]

This is function has five parameters. More explicitly the function is of the form

\[f(y,x)=\frac{1}{2\pi \sigma_y\sigma_x \sqrt{1-\rho^2}}e^{\frac{(y-\mu_y)^2}{\sigma_y^2}-\frac{2\rho(y-\mu_y)(x-\mu_x)}{\sigma_y\sigma_x}+\frac{(x-\mu_x)^2}{\sigma_x^2}}\]

and the parameters are \(\mu_y, \mu_x, \sigma^2_y, \sigma_x^2, \rho\). These are

The \textbf{marginal mean} and variance of \(Y\)

\begin{itemize}
\tightlist
\item
  \(\mu_y=E(Y)\), \(\sigma^2_y=V(Y)\)
\end{itemize}

The \textbf{marginal mean} and variance of \(X\)

\begin{itemize}
\tightlist
\item
  \(\mu_x=E(X)\), \(\sigma^2_x=V(X)\)
\end{itemize}

The \textbf{correlation} between \(X\) and \(Y\)

\begin{itemize}
\tightlist
\item
  \(\rho=\frac{E[(Y-\mu_y)(X-\mu_x)]}{\sigma_y\sigma_x}\)
\end{itemize}

\(\rho\) is called the \textbf{correlation coefficient}.

When we draw the probability density in two dimensions, we see that the marginal densities are the projections of the densities on each axis. Let us define the densities for each variable

\[X \sim N(\mu_x, \sigma_x^2)\]

and

\[Y \sim N(\mu_y, \sigma_y^2)\]

with its parameters. We can plot the 2D density as contour lines and the marginal densities on each axis

\includegraphics{_main_files/figure-latex/unnamed-chunk-199-1.pdf}

We can also plot the density in a 3D plot, where the \(Z\) axis is the value of the probability density.

\includegraphics{_main_files/figure-latex/unnamed-chunk-200-1.pdf}

looking at the contour plots of the 2D distribution, we see that the fifth parameter \(\rho\) defines the direction on which the ellipses are elongated (mayor axis).

\includegraphics{_main_files/figure-latex/unnamed-chunk-201-1.pdf}

The density is entirely determined by 5 parameters.

\hypertarget{estimators}{%
\section{Estimators}\label{estimators}}

We can derive the estimator of all 5 parameters, if we formulate the likelihood function

\[L=\Pi_{i=1}^n f(y_i,x_i; \mu_x, \mu_y, \sigma^2_x, \sigma_y^2, \rho)\]
and maximize it for each parameter. As a result, we obtain the following estimators of the parameters.

The estimators for the means are the usual averages

\begin{itemize}
\tightlist
\item
  \(\bar{Y}=\frac{1}{n}\sum_{i=1}^n y_i\) estimates \(\mu_y\)
\item
  \(\bar{X}=\frac{1}{n}\sum_{i=1}^n x_i\) estimates \(\mu_x\)
\end{itemize}

The maximum likelihood of the estimators for the variances are the uncorrected sample variances (dividing by \(n\))

\begin{itemize}
\tightlist
\item
  \(S^2_y=\frac{1}{n}\sum_{i=1}^n (y_i-\bar{y})^2\) estimates \(\sigma^2_y\)
\item
  \(S^2_x=\frac{1}{n}\sum_{i=1}^n (x_i-\bar{x})^2\) estimates \(\sigma^2_x\)
\end{itemize}

The estimator for the correlation is

\begin{itemize}
\tightlist
\item
  \(R=\frac{\sum_{i=0}^n(X_i-\bar{X})(Y_i-\bar{Y})}{\sqrt{\sum_{i=1}^n(X_i-\bar{X})^2}\sqrt{\sum_{i=1}^n(Y_i-\bar{Y})^2}}\) estimates \(\rho\).
\end{itemize}

\hypertarget{correlation-coefficient}{%
\section{Correlation coefficient}\label{correlation-coefficient}}

\(R\) is then a \textbf{statistics} that can be computed from the data and we can take one value as an estimate of \(\rho\). The sampling distribution of \(R\) can be obtained if we transform it into a new variable (Fisher's z transformation)

\[Z=\frac{1}{2}\ln (\frac{1+R}{1-R})\]
The new variable \(Z\) is a normal variable

\[Z \sim_{aprox} N(\frac{1}{2}\ln (\frac{1+\rho}{1-\rho}), \frac{1}{n-3})\]

with mean \(\frac{1}{2}\ln (\frac{1+\rho}{1-\rho})\) and variance \(\frac{1}{n-3}\).

As \(R\) estimates \(\rho\) we have that

\begin{itemize}
\item
  If \(R\) is near \(0\) then there is no linear relationship between \(y\) and \(x\) (the mayor and minor axes of the 2D plot are the x and y axes).
\item
  If \(R\) is near \(1\) there is strong evidence of a linear relationship between \(y\) and \(x\) (the mayor axis does not align with neither the x or the y axis).
\end{itemize}

\(R\) is then a measure of the statistical dependence between \(Y\) and \(X\). We can use it to test that hypothesis.

\hypertarget{hypothesis-contrast}{%
\section{Hypothesis contrast}\label{hypothesis-contrast}}

The hypothesis contrast for the independence on the relationship between \(Y\) and \(X\) that follow a \textbf{bivariate normal distribution} can be formulated as

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\item
  \(H_0: \rho=0\) (null hypothesis). Therefore, \(Y\) and \(X\) are consider statistically \textbf{independent} and consequently \(f(y,x)=f(x)f(y)\) (the joint probability is the product of the marginals)
\item
  \(H_1: \rho \neq 0\) (alternative hypothesis). Therefore, \(Y\) and \(X\) are statistically \textbf{dependent}.
\end{enumerate}

We then use the statistic \(R\) to test whether there is a dependency between \(Y\) and \(X\). For testing the hypothesis contrast, we take the observed value of \(R\)

\[r_{obs}=\hat{\rho}=\frac{\sum_{i=0}^n(x_i-\bar{x})(y_i-\bar{y})}{\sqrt{\sum_{i=1}^n(x_i-\bar{x})^2}\sqrt{\sum_{i=1}^n(y_i-\bar{y})^2}}\]

and compute its \(pvalue\). That is the probability that we obtain a larger value if we repeat the sample when the \textbf{null hypothesis} is true. Under the null hypothesis \(\rho=0\), we then compute the two-tailed \(pvalue\)

\[pvalue=2(1- F(|r_{obs}|)= 2(1- F_{N(0, 1/(n-3
)})(|z_{obs}|)  \]

where

\[Z \sim_{aprox} N(0, \frac{1}{n-3})\]
\textbf{Example (Hubble's law)}

For Hubble's data, we have that the observed correlation coefficient is \(r_{obs}=0.956\) and its transformed value

\[z_{obs}=ln((1+0.956)/(1-0.956))/2=1.89\]

The \(pvalue\) is therefore

\[pvalue=2(1- F_{N(0,1/18)}(1.89))=1.4\times 10^{-11}\]

Therefore, since it is lower than \(\alpha=0.05\), we reject the null hypothesis that galaxy velocity and distance are independent. The correlation is strongly positive, showing that as more distance galaxies are observed, we expect that they are moving away from the Earth and between each other at faster speeds. This result clearly implies that at some time in the past all the mass of the Universe was close together, perhaps at a point before it exploded in a Big Bang.

In Python and R:

\begin{verbatim}
from scipy import stats
r, p_value = stats.pearsonr(dat["velocity"], dat["distance"])

R:
cor.test(dat$velocity, dat$distance)
\end{verbatim}

\textbf{Example (leptin and fat mass)}

Leptin is a hormone produced by adipose tissue. We want to study the serum leptin levels in the adult population. Zhang et al.~studied the gene expression of patients with metabolic syndrome, for which one of its characteristics is high weight \citep{Zhang2013_adiponectinQTLs}. They recorded relevant data of 188 individuals that included leptin levels in blood, lean fatmass(kg), age and sex. Data is available at the public repository GEO and can be accessed from R

We can use this publicly available data to inquiry on the relationship between fatmas and leptin and contrast it with expected results from previous studies, although this was not the main objective of the study.

From this data, we assume that the levels of fatmass (\(X\)) and (log) leptin in blood \(Y\) are random variables that have a join bivarite normal distribution

\[(X, Y) \sim N(\mu_x, \mu_y, \sigma_x^2, \sigma_y^2, \rho)\]
We are interested in finding whether letpin levels depend on the levels of fatmass. Leptin is a hormone released from adipose tissue. Therefore, as more fatmas we expect more leptin in ciculating blood that regulate regulates satiety. The breaking of satiety regulation by leptin may be a cause for obesity.

One random experiment in our study has two outcomes: \((leptin, fatmass)\).

Continuous variable (outcome of interest)

\begin{itemize}
\tightlist
\item
  \(lepting \in (0, 5)\)
\end{itemize}

Continuous variable (explanatory variable)

\begin{itemize}
\tightlist
\item
  \(fatmass \in (20,80)\)
\end{itemize}

Repeating the experiment \(n=188\) times, the data for the first five repetitions look like

\[
  \begin{array}{ccc}
  \mathbf{Subject} & \mathbf{Leptin} & \mathbf{Fatmass} \\ 
1 & 3.355677  & 45.721 \\ 
2 & 2.272126  & 43.895 \\ 
3 & 1.071584  & 47.871 \\ 
4 & 3.921082  & 65.801 \\ 
5 & 1.536867  & 56.644 \\ 
... & ...  & ... \\ 
 \end{array}
\]

For this data, we have that the observed correlation coefficient is \(r_{obs}=-0.276\) and its transformed value

\[z_{obs}=\frac{1}{2}\ln(\frac{1-0.276}{1+0.276})=-0.284\]

The \(pvalue\) is therefore

\[pvalue=2(1- F_{N(0,1/185)}(0.284))=0.0001\]

Therefore, since it is lower than \(\alpha=0.05\), we reject the null hypothesis that leptin and fat mass are independent. Note that leptin and fat mass are \textbf{weakly correlated} and \textbf{negatively} \(r=-0.28\), although the correlation is \textbf{highly significant} because \(pvalue=0.0001\).

We should take this result with caution:

\begin{itemize}
\item
  Looking at the data, we see the fitted 2D probability function with negative correlation, but the \textbf{marginal} histograms are not quite normally distributed.
\item
  From literature, we know that as we increase fat mass, we should obtain \textbf{more} leptin, not the contrary, as it is released from adipose tissue.
\end{itemize}

\includegraphics{_main_files/figure-latex/unnamed-chunk-205-1.pdf}

How can we improve our analysis, so that our results are as expected?

\hypertarget{regression-analysis}{%
\section{Regression analysis}\label{regression-analysis}}

To determine if \(leptin\) and \(fatmass\) are statistically independent, we can rather ask: What is the probability density of leptin at a \textbf{given value} of fat mass. Does the density changes when we change the value of fat mass? We will then consider the that the continuous fat mass value is a condition for leptin, or that changing fat mass will produce a change in leptin levels, as it is the tissue that releases it. We call the condition variable the explanatory or independent variable, and the outcome of interest the explained or dependent variable.

Let us explore first the independence of two bivariate normal variables in terms of conditinal probabilities. Remember, two variables \(Y\) and \(X\) are independent if

\[P(Y|X)=P(Y)\]

Therefore, we can calculate the \textbf{conditional probability density} of \(Y\) (leptin) given \(X\) (fat mass) from the definition

\[f(y|x)=\frac{f(y,x)}{f(y)}
=N(\mu_{y|x}, \sigma^2_{y|x})\]

This turns up to be a normal distribution. The conditional probability density is the profile, or a slice, of the 2D distribution at a given value of \(x\).

\includegraphics{_main_files/figure-latex/unnamed-chunk-206-1.pdf}

The conditional density at a given point is the dotted line in red. Its mean is the red dot and, in terms of the parameter of the binomial normal density, is given by

\[\mu_{y|x}=\mu_y+\frac{\sigma_y\rho}{\sigma_x}(x-\mu_x)\]

If we manipulate this equation, we see that the \textbf{conditional mean} is a linear function of \(x\)

\[\mu_{y|x}=(\mu_y-\mu_x \frac{\sigma_y\rho}{\sigma_x})+\frac{\sigma_y \rho}{\sigma_x} x=\alpha + \beta x\]

where \(\alpha=\mu_y-\mu_x \beta\) and \(\beta=\frac{\sigma_y \rho}{\sigma_x}\). This line is called a \textbf{regression} line (red dotted line in the plot). That is, when we move along the \(X\) axis, the conditional mean of \(Y\) moves linearly.

The variance of the conditional density is

\[\sigma^2_{y|x}= \sigma_y^2(1-\rho^2)\]
Solving for \(\rho^2\), we have

\[\rho^2=\frac{\sigma_y^2-\sigma^2_{y|x}}{\sigma_y^2} \]
\(\rho^2\) is then the proportion of the total variance that is explained by the regression line. For instance, \(\rho^2=1\) when the conditional variability \(\sigma^2_{y|x}\) is zero ans the conditional distribution shinks to its mean; that is, all the observations fall on the regression line.

\hypertarget{linear-model-2}{%
\section{Linear model}\label{linear-model-2}}

Consider the \textbf{linear model} for the values of \(Y\) \textbf{conditioned} to the value of \(x_i\)

\[Y_i = \alpha + \beta x_i +\varepsilon_{i}\]

\(i\) is the index of the observation from \((1,...n)\), typically one for every \(x_i\) as \(x_i\) is continuous, and \(\varepsilon_{i}\) is the random error, a \textbf{random variable} with expected value \(E(\varepsilon_{i})=0\) and variance \(V(\varepsilon_{i})=\sigma_{y|x}^2\). The expected value of \(Y_{x_i}\) is the regression line

\[\mu_{y|x_i}=\alpha + \beta x_i\]
with

\[\alpha=\mu_y-\beta\mu_x\]
and

\[\beta=\rho\frac{\sigma_y}{\sigma_x}\]

\hypertarget{hypothesis-contrast-1}{%
\section{Hypothesis contrast}\label{hypothesis-contrast-1}}

Null hypothesis:

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\tightlist
\item
  \(H_0: \beta=0\). \(Y\) and \(X\) are statistically \textbf{independent}, and therefore \(f(y|x)=f(y)\). If this is the case then the regression line is flat, as the conditional density is the same no matter the point of the slice. The null hypothesis model is shown in the plot
\end{enumerate}

\includegraphics{_main_files/figure-latex/unnamed-chunk-207-1.pdf}

Note that the marginal probability (projection of the probability on the \(y\) axis) would be identical to all the conditional probabilities, which is the condition for independence.

Alternative hypothesis:

\begin{enumerate}
\def\labelenumi{\alph{enumi}.}
\setcounter{enumi}{1}
\tightlist
\item
  \(H_1: \beta\neq 0\). \(Y\) and \(X\) are statistically \textbf{dependent}. If this is the case then we have the following model
\end{enumerate}

\includegraphics{_main_files/figure-latex/unnamed-chunk-208-1.pdf}

Note that the marginal probability would be the combination of the conditional probabilities.

\hypertarget{estimators-1}{%
\section{Estimators}\label{estimators-1}}

\(\beta\) is therefore our parameter of interest and we need a probability distribution to test its hypothesis contrast

\begin{itemize}
\tightlist
\item
  \(\beta=\rho\frac{\sigma_y}{\sigma_x}\) suggest the estimator for \(\beta\)
\end{itemize}

\[B=\frac{\sum_{i=1}^m(x_i-\bar{x})(Y_i-\bar{Y})}{\sum_{i=1}^n(x_i-\bar{x})^2}\]

\begin{itemize}
\tightlist
\item
  \(\alpha=\mu_y-\beta\mu_x\) suggests the estimator for \(\alpha\)
\end{itemize}

\[A=\bar{Y}- \hat{\beta}\bar{x}\]

The estimators \(A\) and \(B\) for \(\alpha\) and \(\beta\) can formally be derived from \textbf{minimizing the sum of squares} for the error given \(x_i\)

\[SSE=\sum_{i=1}^n(Y_i-\bar{Y_i})^2=\sum_{i=1}^n(Y_i-\alpha + \beta x_i)^2\]
with respect to \(\alpha\) and \(\beta\). If \(\bar{Y_i}\) is normal then

\[B \sim N(\beta, \frac{n\sigma^2_y}{{(n-2)s^2_x}})\]

with mean \(E(B)=\beta\) and, therefore, it is an unbiased estimator.

\hypertarget{hypothesis-testing-2}{%
\section{Hypothesis testing}\label{hypothesis-testing-2}}

The standardized error, we make when we estimate \(\beta\) with \(B\)

\[\frac{B -\beta}{\sqrt{\frac{ns^2_y}{{(n-2)s^2_x}}}} \sim t(n-2)\]
that follows a t-distribution with \(n-2\) degrees of freedom. If the null hypothesis is true then \(\beta=0\) and the observed error is

\[t_{obs}= \frac{\hat{\beta}}{\sqrt{\frac{ns^2_y}{{(n-2)s^2_x}}}} \sim t(n-2)\]
Where \(\hat{\beta}\) is the observed value of the random variable \(B\).

\textbf{Example (leptin and fatmass)}

We can perform the statistical test in Pyhton and R:

\begin{verbatim}
Python:
import pandas as pd
from scipy.stats import pearsonr

##loading dat
model = ols('leptin ~ fatmass', data=data).fit()
print(model.summary())

R:
##loading dat
summary(lm(leptin ~ fatmass, data=dat))
\end{verbatim}

The outut is a table like

\[
\begin{array}{lcccc}
 & \mathbf{Estimate} & \mathbf{Std.\ Error} & \mathbf{t\ value} & \mathbf{Pr(>|t|)} \\ 
\mathbf{(Intercept)} & 3.720 & 0.295 & 12.604 & <2 \times 10^{-16} \ ^{***} \\
\mathbf{fatmass}     & -0.0226 & 0.00576 & -3.926 & 0.000121 \ ^{***} \\
\end{array}
\]

The observed value of \(B\) is

\[\hat{\beta}= \frac{\sum_{i=1}^m(x_i-\bar{x})(y_i-\bar{y})}{\sum_{i=1}^n(x_i-\bar{x})^2}= -0.02262\]
and the observed value of \(A\) is

\[\hat{\alpha}=\bar{y}-\hat{\beta}\bar{x}= 3.72012\]
Only \(7\%\) of the variability is explained by the regression line, since (R-squared: 0.07653)

\[r_{obs}^2=0.07653\]

We can draw the regression line

\includegraphics{_main_files/figure-latex/unnamed-chunk-210-1.pdf}

We see that the regression line is negative and there is dispersion about it. Remember that the correct way to interpret the line is to think that these are the mean values of \(Y\) \textbf{conditioned} on \(X\). The bands, are the contious confidence intervals at a fix value of \(X\).

\hypertarget{stratified-analysis}{%
\section{Stratified analysis}\label{stratified-analysis}}

We can include other conditions in the regression analysis. In general, it is important \textbf{to adjust} for other factors that we believe are correlated with the outcome \(Y\) and the condition \(x\), as they may explain part of the association. These are called \textbf{confounders}.

\textbf{Example (Fat mass and leptin)}
In the datudy of Zhang and colleageues \citep{Zhang2013_adiponectinQTLs} they measure othre determinats of leptin levels, such as sex and age. The leptin level may have multiple covariates or explanatory variables that can affect the association between fat mass and letpin. The first observations of the complete data sex looks like

\[
  \begin{array}{ccccc}
  \mathbf{Subject} & \mathbf{Leptin} & \mathbf{Fatmass} & \mathbf{Sex} & \mathbf{Age} \\ 
1 & 3.356 & 45.721 & F & 45 \\ 
2 & 2.272 & 43.895 & F & 77 \\ 
3 & 1.072 & 47.871 & M & 79 \\ 
4 & 3.921 & 65.801 & F & 58 \\ 
5 & 1.537 & 56.644 & M & 42 \\ 
\vdots & \vdots & \vdots & \vdots & \vdots \\ 
 \end{array}
\]

Consider the previous regression and color the points according to their sex

\includegraphics{_main_files/figure-latex/unnamed-chunk-211-1.pdf}

We clearly see that the negative association between leptin and fat mass is given by the \textbf{effect of sex}. As males have lower leptin levels and higher fat mass then the negative correlation between leptin and fat mass is due to sex.

\hypertarget{multiple-regression}{%
\section{Multiple Regression}\label{multiple-regression}}

The linear regression can be extended to include several number of factors. Consider the \textbf{linear model} with one additional factor

\[Y_{jr} = \alpha + \beta x_j +\gamma z_{r}+\varepsilon_{jr}\]

Here, our effect of interest is the coefficient \(\beta\) of \(x_i\) (leptin) and we want to adjust for the effect of \(z_j\) (sex). The model is similar to the linear model we used for ANOVA, only that now the treatment effects are continuous and depend on the variable \(X\).

Running the model in Python or R we obtain

\begin{verbatim}
Python:
import pandas as pd
from scipy.stats import pearsonr
from statsmodels.formula.api import ols

model = ols('leptin ~ fatmass + sex', data=dat).fit()

print(model.summary())

R:
summary(lm(leptin ~ fatmass + sex, data=dat))
\end{verbatim}

\[
\begin{array}{lcccc}
 & \mathbf{Estimate} & \mathbf{Std.\ Error} & \mathbf{t\ value} & \mathbf{Pr(>|t|)} \\ 
\mathbf{(Intercept)} & 1.716 & 0.277 & 6.187 & 3.84 \times 10^{-9} \ ^{***} \\
\mathbf{fatmass}     & 0.0279 & 0.00604 & 4.630 & 6.88 \times 10^{-6} \ ^{***} \\
\mathbf{sexM}        & -1.636 & 0.136 & -12.023 & <2 \times 10^{-16} \ ^{***} \\
\end{array}
\]

Therefore, we observe that there is a positive increase in leptin when we adjust for sex \(\hat{\beta}=4.63\), \(pvalue=6.88 \times 10^{-6}\). In addition males have a negative association with leptin, compared with women \(\gamma=-12.02\), \(pvalue<2 \times 10^{-16}\). But within each sex the association is positive as expected.
The regression lines for each sex will be parallel but displaced by the average effect of changing from female (reference) to male.

\includegraphics{_main_files/figure-latex/unnamed-chunk-213-1.pdf}

\textbf{Example(C02 concentration in atmosphere)}

Perhaps one of the most consequential observations of the second half of the 20 century was reported by Keeling in 1960 \citep{Keeling1960}. Using continuous recording within less than three years (1958-1960), and over different geographic regions and methods, he showed that the levels of CO2 were significantly increasing in the atmosphere. He started CO2 survey in Hawaii in Mauna Loa, which have continue for 66 years. The data from on this station from his 1960 paper is

\[
  \begin{array}{cccc}
  \mathbf{Observation} & \mathbf{CO2 (ppm)} & \mathbf{Month} & \mathbf{Year} \\ 
1  & 313.4 &  3  & 1958 \\ 
2  & 314.4 &  4  & 1958 \\ 
3  & 315.1 &  5  & 1958 \\ 
5  & 312.9 &  7  & 1958 \\ 
6  & 312.3 &  8  & 1958 \\ 
7  & 311.6 &  9  & 1958 \\ 
9  & 310.6 & 11  & 1958 \\ 
10 & 311.6 & 12  & 1958 \\ 
11 & 312.5 &  1  & 1959 \\ 
12 & 313.5 &  2  & 1959 \\ 
13 & 314.0 &  3  & 1959 \\ 
14 & 314.7 &  4  & 1959 \\ 
15 & 315.3 &  5  & 1959 \\ 
16 & 315.2 &  6  & 1959 \\ 
17 & 313.5 &  7  & 1959 \\ 
18 & 311.9 &  8  & 1959 \\ 
19 & 311.1 &  9  & 1959 \\ 
20 & 310.5 & 10  & 1959 \\ 
21 & 311.8 & 11  & 1959 \\ 
22 & 312.5 & 12  & 1959 \\ 
23 & 313.4 &  1  & 1960 \\ 
24 & 313.7 &  2  & 1960 \\ 
25 & 314.4 &  3  & 1960 \\ 
  \end{array}
\]
The plot of the observations per month though out years (now known as Kelling plot is)

\includegraphics{_main_files/figure-latex/unnamed-chunk-214-1.pdf}

Can we say from this data that C02 concentrations are getting increasing in Mauna Loa thoughout the years?

Finding a year trend from the inspection of the data is difficult because the variations per month are large, in comparison with the year variation. Maximum CO2 coincides with summer in the northern hemisphere when vegetation is the greenest, and vegetation respiration maximum. Therefore, the monthly variation is a confounding factor for the regressing CO2 concentration on years. The model we are interested in fitting is

\[Y_{year, month} = \alpha + \beta x_{year} + z_{month}+\varepsilon_{year, month}\]
where \(Y\) is the CO2 concentrations (outcome), \(\beta\) is the effect of years (explanatory variable), and \(z_{month}\) is the effect of the month (covariate). The covariate month is better encoded as categorical as we expect each month to have its own specific effect (with respect to January). Encoded as numeric the model will fit pair-wise comarisons between January and each month, as 11 covariates for the effect of year.

The code in if Python and R

\begin{verbatim}
Pyhton:
import pandas as pd
import statsmodels.api as sm
import statsmodels.formula.api as smf

df = pd.DataFrame({
    'y': [313.4, 314.4, 315.1, 312.9, 312.3, 311.6, 310.6, 311.6,
          312.5, 313.5, 314.0, 314.7, 315.3, 315.2, 313.5, 311.9, 
          311.1, 310.5, 311.8, 312.5, 313.4, 313.7, 314.4],
    'month': [3,4,5,7,8,9,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3],
    'year': [1958,1958,1958,1958,1958,1958,1958,1958,
             1959,1959,1959,1959,1959,1959,1959,1959,
             1959,1959,1959,1959,1960,1960,1960]
})
model = smf.ols('y ~ year + C(month)', data=df).fit()
print(model.summary())


R:
df <- data.frame(
  y = c(313.4, 314.4, 315.1, 312.9, 312.3, 311.6, 310.6, 311.6,
        312.5, 313.5, 314.0, 314.7, 315.3, 315.2, 313.5, 311.9, 
        311.1, 310.5, 311.8, 312.5, 313.4, 313.7, 314.4),
  month = c(3,4,5,7,8,9,11,12,1,2,3,4,5,6,7,8,9,10,11,12,1,2,3),
  year = c(1958,1958,1958,1958,1958,1958,1958,1958,
           1959,1959,1959,1959,1959,1959,1959,1959,
           1959,1959,1959,1959,1960,1960,1960)
)

summary(lm(y~year+factor(month), data=df))
\end{verbatim}

with results

\[
\begin{array}{lcccc}
 & \mathbf{Estimate} & \mathbf{Std.\ Error} & \mathbf{t\ value} & \mathbf{Pr(>|t|)} \\ 
\mathbf{(Intercept)}    & -500.9962 & 286.0446 & -1.751 & 0.110420 \\
\mathbf{year}           & 0.4154 & 0.1460 & 2.846 & 0.017383 \ ^{*} \\
\mathbf{factor(month)2} & 0.6500 & 0.3722 & 1.746 & 0.111309 \\
\mathbf{factor(month)3} & 1.1910 & 0.3475 & 3.427 & 0.006466 \ ^{**} \\
\mathbf{factor(month)4} & 2.0154 & 0.3998 & 5.041 & 0.000506 \ ^{***} \\
\mathbf{factor(month)5} & 2.6654 & 0.3998 & 6.667 & 5.59 \times 10^{-5} \ ^{***} \\
\mathbf{factor(month)6} & 2.4577 & 0.4616 & 5.324 & 0.000336 \ ^{***} \\
\mathbf{factor(month)7} & 0.6654 & 0.3998 & 1.664 & 0.127009 \\
\mathbf{factor(month)8} & -0.4346 & 0.3998 & -1.087 & 0.302483 \\
\mathbf{factor(month)9} & -1.1846 & 0.3998 & -2.963 & 0.014211 \ ^{*} \\
\mathbf{factor(month)10}& -2.2423 & 0.4616 & -4.857 & 0.000664 \ ^{***} \\
\mathbf{factor(month)11}& -1.3346 & 0.3998 & -3.338 & 0.007511 \ ^{**} \\
\mathbf{factor(month)12}& -0.4846 & 0.3998 & -1.212 & 0.253294 \\
\end{array}
\]

We see an significant increase in \(\hat{\beta}=0.4154\) ppm per year \(pvalue=0.01\), adjusting for month variation. While this was preliminary data on global CO2 emissions, a year plot of the residuals from fitting the CO2 concentrations on moths clearly shows the trend. Additionally, continuous monitoring at Mauna Loa for over 66 year reveals how the underlying yearly trend has largely superseded the monthly variations.

\includegraphics{_main_files/figure-latex/unnamed-chunk-215-1.pdf}

\hypertarget{multiple-regression-interaction}{%
\section{Multiple Regression interaction}\label{multiple-regression-interaction}}

Sometimes the explanatory variable has an effect that is modulated by a covariate. That is, the strngth of the effect depends on another condition.

\textbf{Example (Fat mass, letptin and sex)}

For the association between fat mass and leptin we can fit the regression model \textbf{separating}, or stratifying, by sex. We find that the strength of the association between leptin and fat mass depends on sex. That is, the slope of the regression line changes between sexes.

\includegraphics{_main_files/figure-latex/unnamed-chunk-216-1.pdf}

We can furthermore include interactions between conditions in the regression.

Consider the \textbf{linear model}

\[Y_{jr} = \alpha + \beta x_{j} +\gamma z_{r} + \delta (xz)_{jr} +\varepsilon_{jr}\]
The parameter \(\delta\) will add a contribution to the slope \(\beta\) that is \textbf{specific} to the condition \(r\). For instance, if \(z_i \in (0,1)\) then when \(z=0\) the coefficient of \(x_i\) is \(\beta\). But when \(z=1\) the coefficient of \(x_i\) is \(\beta+\gamma\).

\textbf{Example (Fat mass and leptin)}

In the fat mass and leptin relationship, if \(z\) is sex, then \(\gamma\) will test the differences in \(\beta\)s between males and females. We can further adjust by age.

\[Y_{jrs} = \alpha + \beta\times fatmass_{j} +\gamma\times sex_{r} + \delta\times (fatmass\times sex)_{jr} + \epsilon_s age+\varepsilon_{jrs}\]
We can re-fit the model with the interaction term

\begin{verbatim}
Python:
import pandas as pd
from scipy.stats import pearsonr

model = ols('leptin ~ fatmass*sex + age', data=dat).fit()

print(model.summary())

R:

summary(lm(leptin ~ fatmass*sex + age, data=dat))
\end{verbatim}

with th efollowing results

\[
\begin{array}{lcccc}
 & \mathbf{Estimate} & \mathbf{Std.\ Error} & \mathbf{t\ value} & \mathbf{Pr(>|t|)} \\ 
\mathbf{(Intercept)} & 1.800 & 0.337 & 5.337 & 2.77 \times 10^{-7} \ ^{***} \\
\mathbf{fatmass}     & 0.0200 & 0.00695 & 2.881 & 0.00443 \ ^{**} \\
\mathbf{sexM}        & -3.218 & 0.775 & -4.151 & 5.07 \times 10^{-5} \ ^{***} \\
\mathbf{age}         & 0.00585 & 0.00294 & 1.989 & 0.0482 \ ^{*} \\
\mathbf{fatmass{:}sexM} & 0.0284 & 0.0135 & 2.104 & 0.0368 \ ^{*} \\
\end{array}
\]

The data suggest a steeper increase of leptin with body fat in males than in females (interaction: \(0.028427\), \(pvalue=0.03\)), as we saw in the figures of the stratified analysis. A positive and significant interaction means that the slope for males is higher than the slope for females. The effect of fatmass on letpin levels is stronger in males.

We also observe a signification effect of age on leptin levels. As the coefficient is a slope, we can say that for a year increase in the age of the patient, we see a significant increase in \(0.005\) units of leptin levels (\(pvalue=0.048\)). Therefore age is also a contributing factor of leptin levels. The model now explains \(50\%\) of the variance as \(R^2=0.5\).

\hypertarget{model-diagnostics}{%
\section{Model diagnostics}\label{model-diagnostics}}

All linear models have been made on the supposition that

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  Errors are distributed normally
\item
  Errors have the same variance
\end{enumerate}

There are a number of plots to check that at least the data is consistent with these suppositions. In particular, we are interested to see that the residuals distribute symmetrically against the fitted values, revealing equal variance (homoscedasticity). We also want to check that the points in a quantile-quantile (QQ) plot fall in the line, revealing how close the residuals distribute normally.

\begin{verbatim}
R:
mod <- lm(leptin ~ fatmass * sex + age, data = dat)

plot(mod, which = 1, main = "Residuals vs Fitted")
plot(mod, which = 2, main = "Normal Q-Q")
\end{verbatim}

\includegraphics{_main_files/figure-latex/unnamed-chunk-218-1.pdf}

The residuals versus fitted values plot shows that the residuals are fairly randomly scattered around zero, with no strong pattern, suggesting that the linear model fits the data reasonably well and that the assumption of linearity is broadly satisfied. While there may be a slight concentration of residuals at certain fitted values, it is not pronounced. The Normal Q-Q plot indicates that the residuals are approximately normally distributed, as most points follow the reference line given by the nornal quantiles, with only minor deviations at the extremes, which are typical in real datasets. Overall, the model age appears to provide a reasonable fit, with assumptions of linearity and normality largely met.

\hypertarget{questions-14}{%
\section{Questions}\label{questions-14}}

\textbf{1)} We perform a regression analysis when we have

\textbf{\(\qquad\)a:} one categorical variable;
\textbf{\(\qquad\)b:} two categorical variables;
\textbf{\(\qquad\)c:} one continuous random variable;
\textbf{\(\qquad\)d:} two continuous random variables;

\textbf{2)} The correlation coefficient \(\rho\) is

\textbf{\(\qquad\)a:} a parameter of a 2D probability density;
\textbf{\(\qquad\)b:} a statistic of the relationship between two continuous variables;
\textbf{\(\qquad\)c:} the linear coefficient btween of two variables;
\textbf{\(\qquad\)d:} the variance explained by one continuous variable on other;

\textbf{3)} \(\beta\) in the linear model

\[Y_{x_i} = \alpha + \beta x_i +E_{i}\]

\textbf{\(\qquad\)a:} is a statistic that measure the linear relationship between \(x\) and \(y\);
\textbf{\(\qquad\)b:} is zero for the null hypothesis;
\textbf{\(\qquad\)c:} is a parameter with expected value of zero;
\textbf{\(\qquad\)d:} is the correlation between \(x\) and \(y\)

\textbf{4)} Why do we adjust for a variable in a regression coefficient?

\textbf{\(\qquad\)a:} to test the interaction between the variable and \(x\);
\textbf{\(\qquad\)b:} to stratify the relationship between \(x\) and \(y\);
\textbf{\(\qquad\)c:} to remove confounding in the relationship between \(x\) and \(y\);
\textbf{\(\qquad\)d:} to improve the significance of the relationship between \(x\) and \(y\);

\textbf{5)} The regression analysis assumes that

\textbf{\(\qquad\)a:} the errors have mean 0 and equal variances;
\textbf{\(\qquad\)b:} the errors distribute normally with mean 0 and equal variances;
\textbf{\(\qquad\)c:} the errors distribute normally with different mean and equal variances;
\textbf{\(\qquad\)d:} the errors distribute normally with different mean and different variances;

\hypertarget{practice-8}{%
\section{Practice}\label{practice-8}}

\hypertarget{practice-9}{%
\subsubsection{Practice}\label{practice-9}}

Load misophonia data \url{https://alejandro-isglobal.github.io/SDA/data/data_0.txt}

We have four measures of anxiety:

\begin{itemize}
\tightlist
\item
  Trait: ansiedad.rasgo (are you an anxious person?) continuous:0-100
\item
  State: ansiedad.estado (are you currently feeling anxious?) continuous:0-100
\item
  Diagnosed: ansiedad.medicada (have you been diagnosed with an anxiety disorder?) binary (si, no)
\item
  Excess: ansiedad.dif (difference between State and Trait)
\end{itemize}

We formulate the following hypothesis:

Participants who enrolled in the study had an increased level of anxiety from their baseline (trait) that is related to their:

\begin{itemize}
\tightlist
\item
  age
\item
  sex
\item
  anxiety state.
\end{itemize}

We are interested in the variable anxiety.dif, that is the observed \textbf{excess} of anxiety from the trait

\(excess = state - trait\)

Answer the following questions:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  Are the state and trait of anxiety correlated?
\item
  Is excess in anxiety higher in older people?
\item
  Is excess in anxiety higher in older people after adjusting by sex?
\item
  Is the interaction between age and sex significant on excess anxiety?
\end{enumerate}

\href{https://colab.research.google.com/drive/14i2h9AYxHh4KIfxBB9Px7VwnYdSFVBq5?usp=sharing}{Solutions}

\hypertarget{apendix}{%
\chapter{Apendix}\label{apendix}}

\hypertarget{solutions-to-questions}{%
\section{Solutions to Questions}\label{solutions-to-questions}}

\[
\begin{array}{c|ccccc}
\textbf{Chapter} & Q1 & Q2 & Q3 & Q4 & Q5 \\ \hline
2  & c & a & d & d & b \\
3  & a & b & b & b & a \\
4  & c & b & d & b & b \\
5  & d & c & b & c & b \\
7  & d & a & d & a & d \\
8  & d & b & a &   &   \\
9  & c & a & c & d & b \\
10 & a & c & d & d & c \\
11 & c & b & c &   &   \\
12 & d & d & b & a &   \\
13 & b & a & d & b & b \\
15 & b & d & c & d & a \\
16 & c & b & b & b & a \\
17 & d & d & a & b & c \\
18 & d & a & b & c & b \\
\end{array}
\]

\hypertarget{summary-tables-python-and-r-code}{%
\section{Summary tables, Python and R code}\label{summary-tables-python-and-r-code}}

\hypertarget{creating-data-frames-and-loading-text-files}{%
\subsection{Creating Data Frames and Loading Text Files}\label{creating-data-frames-and-loading-text-files}}

\scriptsize

\begin{longtable}[]{@{}
  >{\raggedright\arraybackslash}p{(\columnwidth - 4\tabcolsep) * \real{0.2612}}
  >{\centering\arraybackslash}p{(\columnwidth - 4\tabcolsep) * \real{0.3955}}
  >{\centering\arraybackslash}p{(\columnwidth - 4\tabcolsep) * \real{0.3433}}@{}}
\toprule\noalign{}
\begin{minipage}[b]{\linewidth}\raggedright
Task
\end{minipage} & \begin{minipage}[b]{\linewidth}\centering
Python (pandas)
\end{minipage} & \begin{minipage}[b]{\linewidth}\centering
R
\end{minipage} \\
\midrule\noalign{}
\endhead
\bottomrule\noalign{}
\endlastfoot
\textbf{Define a simple data frame} & \texttt{import\ pandas\ as\ pd} & \texttt{df\ \textless{}-\ data.frame()} \\
& \texttt{df\ =\ pd.DataFrame(\{\textquotesingle{}col1\textquotesingle{}:\ {[}1,2,3{]},\ \textquotesingle{}col2\textquotesingle{}:{[}4,5,6{]}\})} & \texttt{df\ \textless{}-\ data.frame(col1\ =\ c(1,2,3),\ col2\ =\ c(4,5,6))} \\
\textbf{Load a CSV file} & \texttt{df\ =\ pd.read\_csv(\textquotesingle{}file.csv\textquotesingle{})} & \texttt{df\ \textless{}-\ read.csv(\textquotesingle{}file.csv\textquotesingle{})} \\
\textbf{Load a tab-delimited text file} & \texttt{df\ =\ pd.read\_csv(\textquotesingle{}file.txt\textquotesingle{},\ sep=\textquotesingle{}\textbackslash{}t\textquotesingle{})} & \texttt{df\ \textless{}-\ read.delim(\textquotesingle{}file.txt\textquotesingle{})} \\
\textbf{Preview first rows} & \texttt{df.head()} & \texttt{head(df)} \\
\textbf{Preview last rows} & \texttt{df.tail()} & \texttt{tail(df)} \\
\end{longtable}

\normalsize

\textbf{Note:} In Python, you need to import the following libraries:

\begin{verbatim}
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
\end{verbatim}

\hypertarget{python-and-r-functions-for-data-description}{%
\subsection{Python and R Functions for Data Description}\label{python-and-r-functions-for-data-description}}

\scriptsize

\begin{longtable}[]{@{}
  >{\raggedright\arraybackslash}p{(\columnwidth - 4\tabcolsep) * \real{0.1931}}
  >{\raggedleft\arraybackslash}p{(\columnwidth - 4\tabcolsep) * \real{0.3862}}
  >{\centering\arraybackslash}p{(\columnwidth - 4\tabcolsep) * \real{0.4207}}@{}}
\toprule\noalign{}
\begin{minipage}[b]{\linewidth}\raggedright
Function / Task
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedleft
Python
\end{minipage} & \begin{minipage}[b]{\linewidth}\centering
R
\end{minipage} \\
\midrule\noalign{}
\endhead
\bottomrule\noalign{}
\endlastfoot
\textbf{Mean (average)} & \texttt{np.mean(data)} & \texttt{mean(data)} \\
\textbf{Standard deviation (sd)} & \texttt{np.std(data,\ ddof=1)} & \texttt{sd(data)} \\
\textbf{Median} & \texttt{np.median(data)} & \texttt{median(data)} \\
\textbf{Quantiles / IQR} & \texttt{np.quantile(data,\ {[}0.25,\ 0.5,\ 0.75{]})} & \texttt{quantile(data,\ probs\ =\ c(0.25,\ 0.5,\ 0.75))} \\
\textbf{Relative frequencies} & \texttt{pd.value\_counts(data,\ normalize=True)} & \texttt{table(data)\ /\ length(data)} \\
\textbf{Cumulative frequencies} & \texttt{np.cumsum(np.bincount(data))\ /\ len(data)} & \texttt{cumsum(table(data))\ /\ length(data)} \\
\textbf{Histogram} & \texttt{plt.hist(data,\ bins=10,\ density=True)} & \texttt{hist(data,\ breaks=10,\ freq=FALSE)} \\
\textbf{Boxplot} & \texttt{plt.boxplot(data)} & \texttt{boxplot(data)} \\
\textbf{ECDF (empirical CDF)} & \texttt{np.sort(data);\ y\ =\ np.arange(1,len(data)+1)/len(data)} & \texttt{plot(sort(data),\ (1:length(data))/length(data),\ type=\textquotesingle{}s\textquotesingle{})} \\
\textbf{Bar plot} & \texttt{pd.value\_counts(data).plot.bar()} & \texttt{barplot(table(data))} \\
\textbf{Pie chart} & \texttt{pd.value\_counts(data).plot.pie()} & \texttt{pie(table(data))} \\
\end{longtable}

\normalsize

\hypertarget{summary-of-common-probability-models}{%
\section{Summary of common probability models}\label{summary-of-common-probability-models}}

\scriptsize

\begin{longtable}[]{@{}
  >{\raggedright\arraybackslash}p{(\columnwidth - 10\tabcolsep) * \real{0.1463}}
  >{\centering\arraybackslash}p{(\columnwidth - 10\tabcolsep) * \real{0.1951}}
  >{\centering\arraybackslash}p{(\columnwidth - 10\tabcolsep) * \real{0.1951}}
  >{\centering\arraybackslash}p{(\columnwidth - 10\tabcolsep) * \real{0.1707}}
  >{\centering\arraybackslash}p{(\columnwidth - 10\tabcolsep) * \real{0.1463}}
  >{\centering\arraybackslash}p{(\columnwidth - 10\tabcolsep) * \real{0.1463}}@{}}
\toprule\noalign{}
\begin{minipage}[b]{\linewidth}\raggedright
Model
\end{minipage} & \begin{minipage}[b]{\linewidth}\centering
Experiment
\end{minipage} & \begin{minipage}[b]{\linewidth}\centering
range of X
\end{minipage} & \begin{minipage}[b]{\linewidth}\centering
f(x)
\end{minipage} & \begin{minipage}[b]{\linewidth}\centering
E(X)
\end{minipage} & \begin{minipage}[b]{\linewidth}\centering
V(X)
\end{minipage} \\
\midrule\noalign{}
\endhead
\bottomrule\noalign{}
\endlastfoot
Uniform & Measuring an integer or real number & \([a, b]\) & \(\frac{1}{n}\) & \(\frac{b+a}{2}\) & \(\frac{(b-a+1)^2-1}{12}\) \\
Bernoulli & Observing \(A\) & \((0,1)\) & \((1-p)^{1-x}p^x\) & \(p\) & \(p(1-p)\) \\
Binomial & Counting \# of \(A\) events in \(n\) repetitions of Bernoulli trials & \((0,1,.., n)\) & \(\binom n x (1-p)^{n-x}p^x\) & \(np\) & \(np(1-p)\) \\
Negative Binomial for events & Counting \# of \(A'\) events in repetitions of Bernoulli trials until \(r\) events \(A\) are observed & \((0,1,..)\) & \(\binom {x+r-1} x (1-p)^xp^r\) & \(\frac{r(1-p)}{p}\) & \(\frac{r(1-p)}{p^2}\) \\
Hypergeom & Counting \# of \(A\) events in a sample \(n\) from population \(N\) with \(K\) \# of \(A\) events & \(\max(0, n+K-N)\), \ldots{} \(\min(K, n)\) & \(\frac{1}{\binom N n}\binom K x \binom {N-K} {n-x}\) & \(n*\frac{N}{K}\) & \(n \frac{N}{K} (1-\frac{N}{K})\frac{N-n}{N-1}\) \\
Poisson & Counting \# \(A\) events in an interval & 0,1, .. & \(\frac{e^{-\lambda}\lambda^x}{x!}\) & \(\lambda\) & \(\lambda\) \\
Exponential & Measuring an interval between two events \(A\) & \([0,\infty)\) & \(\lambda e^{-\lambda x}\) & \(\frac{1}{\lambda}\) & \(\frac{1}{\lambda^2}\) \\
Normal & Measuring values with symmetric errors whose most likely value is the average & \((-\infty, \infty)\) & \(\frac{1}{\sqrt{2\pi}\sigma}e^{-\frac{(x-\mu)^2}{2\sigma^2}}\) & \(\mu\) & \(\sigma^2\) \\
\end{longtable}

\normalsize

\hypertarget{python-and-r-functions-for-probability-models}{%
\subsection{Python and R Functions for Probability Models}\label{python-and-r-functions-for-probability-models}}

\scriptsize

\begin{longtable}[]{@{}
  >{\raggedright\arraybackslash}p{(\columnwidth - 6\tabcolsep) * \real{0.2095}}
  >{\centering\arraybackslash}p{(\columnwidth - 6\tabcolsep) * \real{0.3176}}
  >{\raggedright\arraybackslash}p{(\columnwidth - 6\tabcolsep) * \real{0.2432}}
  >{\raggedleft\arraybackslash}p{(\columnwidth - 6\tabcolsep) * \real{0.2297}}@{}}
\toprule\noalign{}
\begin{minipage}[b]{\linewidth}\raggedright
Model
\end{minipage} & \begin{minipage}[b]{\linewidth}\centering
Python (PDF/PMF)
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
R \texttt{f(x)}
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedleft
R \texttt{F(x)}
\end{minipage} \\
\midrule\noalign{}
\endhead
\bottomrule\noalign{}
\endlastfoot
\textbf{Uniform (continuous)} & \texttt{uniform.pdf(x,\ a,\ b)} & \texttt{dunif(x,\ a,\ b)} & \texttt{punif(x,\ a,\ b)} \\
\textbf{Binomial} & \texttt{binom.pmf(x,\ n,\ p)} & \texttt{dbinom(x,\ n,\ p)} & \texttt{pbinom(x,\ n,\ p)} \\
\textbf{Negative Binomial (events)} & \texttt{nbinom.pmf(x,\ r,\ p)} & \texttt{dnbinom(x,\ r,\ p)} & \texttt{pnbinom(x,\ r,\ p)} \\
\textbf{Hypergeometric} & \texttt{hypergeom.pmf(x,\ N,\ K,\ n)} & \texttt{dhyper(x,\ K,\ N-K,\ n)} & \texttt{phyper(x,\ K,\ N-K,\ n)} \\
\textbf{Poisson} & \texttt{poisson.pmf(x,\ lambda)} & \texttt{dpois(x,\ lambda)} & \texttt{ppois(x,\ lambda)} \\
\textbf{Exponential} & \texttt{expon.pdf(x,\ 0,\ 1/lambda)} & \texttt{dexp(x,\ lambda)} & \texttt{pexp(x,\ lambda)} \\
\textbf{Normal} & \texttt{norm.pdf(x,\ mu,\ sigma)} & \texttt{dnorm(x,\ mu,\ sigma)} & \texttt{pnorm(x,\ mu,\ sigma)} \\
\end{longtable}

\normalsize

\textbf{Notes:}

\begin{itemize}
\item
  All Python functions assume you have imported the relevant \texttt{scipy.stats} distribution.
\item
  For the exponential in Python, note the shape is defined via scale \(1/\lambda\) in scipy.stats.expon.
\end{itemize}

\hypertarget{summary-of-hypothesis}{%
\section{Summary of hypothesis}\label{summary-of-hypothesis}}

\scriptsize

\begin{longtable}[]{@{}
  >{\raggedright\arraybackslash}p{(\columnwidth - 8\tabcolsep) * \real{0.1161}}
  >{\centering\arraybackslash}p{(\columnwidth - 8\tabcolsep) * \real{0.1518}}
  >{\centering\arraybackslash}p{(\columnwidth - 8\tabcolsep) * \real{0.1518}}
  >{\centering\arraybackslash}p{(\columnwidth - 8\tabcolsep) * \real{0.3036}}
  >{\centering\arraybackslash}p{(\columnwidth - 8\tabcolsep) * \real{0.2768}}@{}}
\toprule\noalign{}
\begin{minipage}[b]{\linewidth}\raggedright
Outcome (Y)
\end{minipage} & \begin{minipage}[b]{\linewidth}\centering
Condition1 (X)
\end{minipage} & \begin{minipage}[b]{\linewidth}\centering
Condition2 (Z)
\end{minipage} & \begin{minipage}[b]{\linewidth}\centering
Null Hypothesis
\end{minipage} & \begin{minipage}[b]{\linewidth}\centering
Test
\end{minipage} \\
\midrule\noalign{}
\endhead
\bottomrule\noalign{}
\endlastfoot
Normal & - & - & \(\mu=\mu_0\) & T-test \\
Normal & - & - & \(\sigma^2=\sigma_0^2\) & \(\chi^2\)-test \\
Normal & - & - & \(\mu_1 - \mu_2=0\) & Paired T-test \\
Binary & - & - & \(p=p_0\) & Z-test \\
Binary (exact) & - & - & \(p=p_0\) & Binomial-test \\
Categorical & Categorical (A,B,\ldots) & - & \(p_A = p_B = ...\) & \(\chi^2\)-test \\
Categorical & Categorical (A,B) & - & \(p_A = p_B\) & Fisher-exact test \\
Normal & Binary & - & \(\mu_A = \mu_B\) & T-test \\
Normal & Categorical (A,B,\ldots) & - & \(\mu_A = \mu_B = ...\) & ANOVA (F-test) \\
Normal & Categorical (A,B,\ldots) & Categorical (L,M,\ldots) & \(\alpha_A = \alpha_B = ...; \beta_L = \beta_M = ...\) & 2-way ANOVA (F-test) \\
Normal & Categorical (A,B,\ldots) & Categorical (L,M,\ldots) & \(\alpha_A \times \beta_L = 0\) & 2-way interaction (F-test) \\
Normal & Normal & - & \(\rho = \rho_0\) & Correlation-test \\
Normal & any type & - & \(\beta = 0\) & Regression \\
Normal & any type & any type & \(\beta = 0\) & Adjusted regression \\
Normal & any type & any type & \(\delta_{x,z} = 0\) & Regression with interaction \\
\end{longtable}

\normalsize

\hypertarget{python-and-r-functions-for-hypothesis-tesing}{%
\subsection{Python and R Functions for Hypothesis tesing}\label{python-and-r-functions-for-hypothesis-tesing}}

\scriptsize

\begin{longtable}[]{@{}
  >{\raggedright\arraybackslash}p{(\columnwidth - 4\tabcolsep) * \real{0.2171}}
  >{\centering\arraybackslash}p{(\columnwidth - 4\tabcolsep) * \real{0.4806}}
  >{\centering\arraybackslash}p{(\columnwidth - 4\tabcolsep) * \real{0.3023}}@{}}
\toprule\noalign{}
\begin{minipage}[b]{\linewidth}\raggedright
Test
\end{minipage} & \begin{minipage}[b]{\linewidth}\centering
Python code
\end{minipage} & \begin{minipage}[b]{\linewidth}\centering
R code
\end{minipage} \\
\midrule\noalign{}
\endhead
\bottomrule\noalign{}
\endlastfoot
T-test & \texttt{stats.ttest\_1samp(data,\ popmean)} & \texttt{t.test(x,\ mu=mu0)} \\
\(\chi^2\)-test & \texttt{p\_value\ =\ 1\ -\ stats.chi2.cdf(chi\_square,\ df)} & \texttt{var.test(x,\ sigma.squared\ =\ sigma0\^{}2)} \\
Paired T-test & \texttt{stats.ttest\_rel(x1,\ x2)} & \texttt{t.test(x1,\ x2,\ paired=TRUE)} \\
Z-test & \texttt{proportions\_ztest(count,\ nobs,\ value)} & \texttt{prop.test(x,\ n,\ p=p0)} \\
Binomial-test & \texttt{binomtest(successes,\ trials,\ p)} & \texttt{binom.test(x,\ n,\ p=p0)} \\
\(\chi^2\)-test (categorical) & \texttt{chi2\_contingency(observed)} & \texttt{chisq.test(table(X))} \\
Fisher-exact test & \texttt{fisher\_exact(table(X,Y))} & \texttt{fisher.test(table(X,Y))} \\
T-test (binary groups) & \texttt{stats.ttest\_ind(groupA,\ groupB,\ equal\_var)} & \texttt{t.test(A,\ B)} \\
ANOVA & \texttt{model\ =\ ols(\textquotesingle{}Y\ \textasciitilde{}\ X\textquotesingle{},\ data).fit();\ sm.stats.anova\_lm(model)} & \texttt{aov(Y\ \textasciitilde{}\ X,\ data=df)} \\
2-way ANOVA & \texttt{model\ =\ ols(\textquotesingle{}Y\ \textasciitilde{}\ X+Z\textquotesingle{},\ data).fit();\ sm.stats.anova\_lm(model)} & \texttt{aov(Y\ \textasciitilde{}\ X\ +\ Z,\ data=df)} \\
2-way interaction & \texttt{model\ =\ ols(\textquotesingle{}Y\ \textasciitilde{}\ X*Z\textquotesingle{},\ data).fit();\ sm.stats.anova\_lm(model)} & \texttt{aov(Y\ \textasciitilde{}\ X*Z,\ data=df)} \\
Correlation & \texttt{pearsonr(X,\ Y)} & \texttt{cor.test(X,\ Y,\ method="pearson")} \\
Regression & \texttt{ols(\textquotesingle{}Y\ \textasciitilde{}\ X\textquotesingle{},\ data).fit()} & \texttt{lm(Y\ \textasciitilde{}\ X,\ data=df)} \\
Adjusted regression & \texttt{ols(\textquotesingle{}Y\ \textasciitilde{}\ X+Z\textquotesingle{},\ data).fit()} & \texttt{lm(Y\ \textasciitilde{}\ X\ +\ Z,\ data=df)} \\
Regression with interaction & \texttt{ols(\textquotesingle{}Y\ \textasciitilde{}\ X*Z\textquotesingle{},\ data).fit()} & \texttt{lm(Y\ \textasciitilde{}\ X*Z,\ data=df)} \\
\end{longtable}

\normalsize

\hypertarget{python-libraries-required}{%
\subsection{Python Libraries Required}\label{python-libraries-required}}

\scriptsize

\begin{longtable}[]{@{}
  >{\raggedright\arraybackslash}p{(\columnwidth - 4\tabcolsep) * \real{0.3559}}
  >{\raggedright\arraybackslash}p{(\columnwidth - 4\tabcolsep) * \real{0.1864}}
  >{\raggedleft\arraybackslash}p{(\columnwidth - 4\tabcolsep) * \real{0.4576}}@{}}
\toprule\noalign{}
\begin{minipage}[b]{\linewidth}\raggedright
Purpose
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
Library
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedleft
Functions Used
\end{minipage} \\
\midrule\noalign{}
\endhead
\bottomrule\noalign{}
\endlastfoot
\textbf{Basic statistics (t-test, chi-square, paired t-test, etc.)} & \texttt{scipy.stats} & \texttt{ttest\_1samp}, \texttt{ttest\_rel}, \texttt{ttest\_ind}, \texttt{chi2.cdf}, \texttt{pearsonr}, \texttt{fisher\_exact} \\
\textbf{Proportions \& binomial tests} & \texttt{statsmodels.stats.proportion} & \texttt{proportions\_ztest}, \texttt{binomtest} \\
\textbf{ANOVA \& regression} & \texttt{statsmodels.api} & \texttt{ols}, \texttt{anova\_lm} \\
\textbf{Data manipulation} & \texttt{pandas} & \texttt{DataFrame}, data selection and grouping \\
\textbf{Contingency tables} & \texttt{scipy.stats} or \texttt{statsmodels} & \texttt{chi2\_contingency}, \texttt{fisher\_exact} \\
\end{longtable}

\normalsize

  \bibliography{book.bib,packages.bib}

\end{document}
